{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-11T14:39:54.665432Z",
     "start_time": "2020-11-11T14:39:52.189098Z"
    }
   },
   "outputs": [],
   "source": [
    "from vectorai import ViClient\n",
    "import json\n",
    "import xmltodict as xtd\n",
    "import numpy as np\n",
    "import os\n",
    "from nltk.corpus import stopwords\n",
    "from nltk.tokenize import wordpunct_tokenize\n",
    "from nltk.stem import PorterStemmer\n",
    "from joblib import Parallel, delayed\n",
    "from gensim import corpora\n",
    "from gensim import models\n",
    "import pandas as pd\n",
    "from gensim import similarities\n",
    "from operator import itemgetter\n",
    "import glob\n",
    "import dask.bag as db\n",
    "from dask.distributed import Client, progress\n",
    "from vectorai.models.deployed import ViText2Vec"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-11T14:49:12.167422Z",
     "start_time": "2020-11-11T14:49:11.071071Z"
    }
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "B:\\Anaconda3\\lib\\site-packages\\distributed\\dashboard\\core.py:79: UserWarning: \n",
      "Failed to start diagnostics server on port 8787. [WinError 10048] Solo se permite un uso de cada dirección de socket (protocolo/dirección de red/puerto)\n",
      "  warnings.warn(\"\\n\" + msg)\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table style=\"border: 2px solid white;\">\n",
       "<tr>\n",
       "<td style=\"vertical-align: top; border: 0px solid white\">\n",
       "<h3 style=\"text-align: left;\">Client</h3>\n",
       "<ul style=\"text-align: left; list-style: none; margin: 0; padding: 0;\">\n",
       "  <li><b>Scheduler: </b>tcp://127.0.0.1:52184</li>\n",
       "  <li><b>Dashboard: </b><a href='http://127.0.0.1:52187/status' target='_blank'>http://127.0.0.1:52187/status</a>\n",
       "</ul>\n",
       "</td>\n",
       "<td style=\"vertical-align: top; border: 0px solid white\">\n",
       "<h3 style=\"text-align: left;\">Cluster</h3>\n",
       "<ul style=\"text-align: left; list-style:none; margin: 0; padding: 0;\">\n",
       "  <li><b>Workers: </b>6</li>\n",
       "  <li><b>Cores: </b>12</li>\n",
       "  <li><b>Memory: </b>17.13 GB</li>\n",
       "</ul>\n",
       "</td>\n",
       "</tr>\n",
       "</table>"
      ],
      "text/plain": [
       "<Client: 'tcp://127.0.0.1:52184' processes=6 threads=12, memory=17.13 GB>"
      ]
     },
     "execution_count": 15,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "client = Client(n_workers=6, threads_per_worker=2)\n",
    "\n",
    "client"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-11T14:49:22.928221Z",
     "start_time": "2020-11-11T14:49:21.987445Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Logged in. Welcome cesar-cabeza. To view list of available collections, call list_collections() method.\n"
     ]
    }
   ],
   "source": [
    "username = 'cesar-cabeza'\n",
    "api_key = 'amdXU0VYVUJXN1RRYmVKWGUwcG46Ni1zRW15R01SNFNmLUZ4Ry00MHFmZw'\n",
    "\n",
    "vi_client = ViClient(username=username, api_key = api_key)\n",
    "text_encoder = ViText2Vec(username, api_key)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-11T14:49:23.820455Z",
     "start_time": "2020-11-11T14:49:23.003435Z"
    }
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'status': 'complete', 'message': 'covid-19 deleted'}"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "vi_client.delete_collection(\"covid-19\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-11T14:49:24.477448Z",
     "start_time": "2020-11-11T14:49:24.475447Z"
    }
   },
   "outputs": [],
   "source": [
    "path_test = \"B:/document_parser/document_parses/test\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-11T14:50:02.462401Z",
     "start_time": "2020-11-11T14:50:02.455400Z"
    }
   },
   "outputs": [],
   "source": [
    "def load_corpus(files_path, client, text_encoder):\n",
    "    files = glob.glob(files_path + \"/*.json\")\n",
    "    stopset = set(stopwords.words(\"english\"))\n",
    "    stemmer = PorterStemmer()\n",
    "    b = db.from_sequence(files).map(preprocess_json, stopset, stemmer)\n",
    "    client.insert_documents(\"covid-19\", b.compute(), models={\n",
    "                            \"abstract\": text_encoder.encode, \"title\": text_encoder.encode}, workers = 12, chunksize = 20)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-11T14:50:03.283058Z",
     "start_time": "2020-11-11T14:50:03.280055Z"
    }
   },
   "outputs": [],
   "source": [
    "def preprocess_json(file_path, stopset, stemmer):\n",
    "    with open(file_path) as file:\n",
    "        file_json = json.load(file)\n",
    "    returned = preprocess_document(file_json, stopset, stemmer)\n",
    "    return returned"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-11T14:50:03.474841Z",
     "start_time": "2020-11-11T14:50:03.471840Z"
    }
   },
   "outputs": [],
   "source": [
    "def preprocess_document(file, stopset, stemmer):\n",
    "    returned = {}\n",
    "    returned[\"_id\"] = file[\"paper_id\"]\n",
    "    title = file[\"metadata\"][\"title\"]\n",
    "    abstract = \" \".join([paragraph[\"text\"] for paragraph in file[\"abstract\"]])\n",
    "    body = \" \".join([paragraph[\"text\"] for paragraph in file[\"body_text\"]])\n",
    "    returned[\"title\"] = title\n",
    "    returned[\"abstract\"] = abstract#preprocess_document_part(abstract, stopset, stemmer)\n",
    "    returned[\"body\"] = body#preprocess_document_part(body, stopset, stemmer)\n",
    "    return returned"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-11T14:50:03.642388Z",
     "start_time": "2020-11-11T14:50:03.639388Z"
    }
   },
   "outputs": [],
   "source": [
    "def preprocess_document_part(part, stopset, stemmer):\n",
    "    if part != \"\":  # actually has content\n",
    "        tokens = [word.lower() for word in wordpunct_tokenize(\n",
    "            part) if word.lower() not in stopset and len(word) > 2]\n",
    "        stems = [stemmer.stem(token) for token in tokens]\n",
    "        return \" \".join(stems)\n",
    "    return \"no content\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "ExecuteTime": {
     "start_time": "2020-11-11T14:50:03.838Z"
    }
   },
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "4e6bb3b241a945e69138dea0e796bfac",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "HBox(children=(FloatProgress(value=0.0, max=27.0), HTML(value='')))"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "load_corpus(path_test, vi_client, text_encoder)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-11T14:16:29.701686Z",
     "start_time": "2020-11-11T14:16:27.632Z"
    }
   },
   "outputs": [],
   "source": [
    "vi_client.collection_schema(\"covid-19\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-11T14:16:29.705687Z",
     "start_time": "2020-11-11T14:16:28.478Z"
    }
   },
   "outputs": [],
   "source": [
    "stopset = set(stopwords.words(\"english\"))\n",
    "stemmer = PorterStemmer()\n",
    "tokens = [word.lower() for word in wordpunct_tokenize(\"what is the origin of coronavirus\") if word.lower() not in stopset and len(word) > 2]\n",
    "stems = [stemmer.stem(token) for token in tokens]\n",
    "query = \" \".join(stems)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-11T14:16:29.705687Z",
     "start_time": "2020-11-11T14:16:28.478Z"
    }
   },
   "outputs": [],
   "source": [
    "advanced_search_query = {\n",
    "    'text' : {'vector': text_encoder.encode(\"what is the origin of coronavirus\"), 'fields' : ['abstract_vector_', \"title_vector_\"]}\n",
    "}\n",
    "results = vi_client.advanced_search('covid-19', advanced_search_query, page_size=10)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-11T14:16:29.708685Z",
     "start_time": "2020-11-11T14:16:29.088Z"
    }
   },
   "outputs": [],
   "source": [
    "results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-11T14:16:29.821685Z",
     "start_time": "2020-11-11T14:16:29.809686Z"
    }
   },
   "outputs": [],
   "source": [
    "[(result[\"title\"],result[\"_search_score\"]) for result in results[\"results\"]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-11T14:18:24.997349Z",
     "start_time": "2020-11-11T14:18:23.929833Z"
    }
   },
   "outputs": [],
   "source": [
    "vi_client.random_documents(\"covid-19\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-11T14:45:34.918853Z",
     "start_time": "2020-11-11T14:45:34.906853Z"
    }
   },
   "outputs": [],
   "source": [
    "files = glob.glob(path_test + \"/*.json\")\n",
    "stopset = set(stopwords.words(\"english\"))\n",
    "stemmer = PorterStemmer()\n",
    "b = db.from_sequence(files).map(preprocess_json, stopset, stemmer)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-11T14:46:09.160203Z",
     "start_time": "2020-11-11T14:46:09.134203Z"
    }
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "({'_id': '000a0fc8bbef80410199e690191dc3076a290117',\n",
       "  'title': 'PfSWIB, a potential chromatin regulator for var gene regulation and parasite development in Plasmodium falciparum',\n",
       "  'abstract': 'Background: Various transcription factors are involved in the process of mutually exclusive expression and clonal variation of the Plasmodium multigene (var) family. Recent studies revealed that a P. falciparum SWI/SNF-related matrix-associated actin-dependent regulator of chromatin (PfSWIB) might trigger stage-specific programmed cell death (PCD), and was not only crucial for the survival and development of parasite, but also had profound effects on the parasite by interacting with other unknown proteins. However, it remains unclear whether PfSIWB is involved in transcriptional regulation of this virulence gene and its functional properties. Methods: A conditional knockdown system \"PfSWIB-FKBP-LID\" was introduced to the parasite clone 3D7, and an integrated parasite line \"PfSWIB-HA-FKBP-LID\" was obtained by drug cycling and clone screening. Growth curve analysis (GCA) was performed to investigate the growth and development of different parasite lines during 96 h in vitro culturing, by assessing parasitemia. Finally, we performed qPCR assays to detect var gene expression profiling in various comparison groups, as well as the mutually exclusive expression pattern of the var genes within a single 48 h life-cycle of P. falciparum in different parasite lines. In addition, RNA-seq was applied to analyze the var gene expression in different lines. Results: GCA revealed that conditional knockdown of PfSWIB could interfere with the growth and development of P. falciparum. The parasitemia of PfSWIB∆ showed a significant decline at 96 h during in vitro culture compared with the PfSWIB and 3D7 lines (P < 0.0001). qPCR and RNA-seq analysis confirmed that depletion of PfSWIB not only silences upsA, upsC and partial upsB var genes, as well as removes the silencing of partial upsB var genes at the ring stage in PfSWIB∆ line, but also leads to aberrant expression of upsA and partial upsB/upsC var genes at the mature stage of P. falciparum, during a single 48-h life-cycle. We demonstrated that PfSWIB was involved in the process of clonal variation in var gene expression, and crucial for the survival and development of Plasmodium parasite. These findings could provide better understanding of the mechanism and function of PfSWIB contributing to the pathogenesis in malaria parasites.',\n",
       "  'body': 'Plasmodium falciparum, which causes malignant malaria such as cerebral malaria (CM) or pregnancy-associated malaria (PAM) [1, 2] , is the cause of death to 435,000 individuals annually [3] . Plasmodium falciparum erythrocyte membrane protein 1 (PfEMP1), which is encoded by the var gene family comprising approximately 60 members, is the major virulence factor involved in the antigenic variation and clinical pathogenicity of falciparum malaria [4, 5] . It is remarkable that the expression of var family members is mutually exclusive. For each parasite at a time, only one or a few var genes are expressed, while the remaining members are silenced [6, 7] . In each generation, P. falciparum is able to express different var genes, and the switches in var gene expression lead to antigenic variation of PfEMP1, which results in immune evasion and chronic infection [8] [9] [10] [11] . This process has proven to be mediated by epigenetic mechanisms, including chromatin modification, nuclear architecture and gene relocation [12, 13] . It has been shown that the single active var gene is enriched in euchromatic modifications, such as histone 3 lysine 9 acetylation (H3K9ac) and histone 3 lysine 4 trimethylation (H3K4me3), particularly near the transcriptional start site (TSS) [14] , while the silent var genes are usually enriched in the heterochromatin marker (histone 3 lysine9 trimethylation, H3K9me3) in the 5\\'UTR or the coding sequence [14] [15] [16] [17] . In this process, histone modifying enzymes such as PfMYST, PfGCN5, PfSIR2 and PfSET, have been considered important for epigenetic control of reversible histone modifications, according to previous studies [12, 13] . PfMYST, a putative histone acetyltransferases (HATs) in P. falciparum, is capable of acetylating multiple lysines on H4 [18, 19] , while an orthologue of the yeast HAT PfGCN5 influences acetylation of H3K9 and K14, both are essential marks of gene activation [18, 20] . Two paralogues of the class III HDAC (histone deacetylases) Sir2, play key roles in maintaining heterochromatin and mutually exclusive var gene expression [15, 21, 22] . PfSir2A deacetylates H3K9ac, H3K14ac and H4K16ac [23] , and is more important in silencing subtelomeric var genes such as upsA-, upsEand upsC-subtype vars. In contrast, PfSir2B silences upsB vars [22] . Furthermore, a total of ten P. falciparum histone lysine methyltransferases (HKMTs) belonging to the SET domain family have been found in P. falciparum [18, 24, 25] . PfSET10 localizes to a specialized region at the nuclear periphery with the active vars such as H3K4me4 transferase [26] . Plasmodium falciparum variant-silencing SET gene (PfSETvs) controls the H3K36me3 on var genes (silencing var genes) [27] . Moreover, subnuclear architecture contributes to the regulation of var gene expression [13] . var genes are by default silent since they are located at the nuclear periphery [28] . However, active var promoters seem to occupy a privileged expression site of this region, making it permissive to transcribe [21] . PF11_0091, an ApiAP2 member, binds a conserved motif in var introns, mediates episomal anchoring towards the nuclear periphery, and recruiting an actin protein complex and polymerized nuclear actin de-represses silent var genes [29] . On the other hand, various trans factors are intimately involved in regulating chromatin structure and gene activity. PfAlbas contribute to heterochromatin structure and regulation of var gene expression [30] . PfSIP2, another ApiAP2 member, is involved in the recruitment and organization of subtelomeric heterochromatin through binding to SPE2 [31] , and induces a comprehensive silencing of a var promoter. Particularly, four major classes of ATP-dependent chromatin remodelers had been found in P. falciparum, facilitating the spatial and temporal remodeling of chromatin [13] . All these remodeler families consist of a SWI2/SNF2family ATPase subunit featuring an ATPase domain [32, 33] , which includes an HSA (helicase-SANT), a post-HSA, and a C-terminal bromodomain. Some other conserved subunits carry extra conserved domains, such as hBAF155/170 (SANT, SWIRM), hBAF60 (SWIB) and human polybromo (multiple bromodomains). BAF60b is generally known as SWI-B or SWIB, an alternative form of the SWI/SNF complex (Complex B) [34] , while human BAF60a protein of SWI/SNF complex is responsible for facilitating cell cycle halting and has the ability to tune the balance between repair and apoptosis induction [35] [36] [37] . Recently, a SWI/ SNF-related matrix-associated actin-dependent regulator of chromatin, PfSWIB (GenBank: PF3D7_0611400), which is characterized by its SWIB/MDM2 domain (SWI complex, BAF60b domain), has been identified in P. falciparum. PfSWIB might trigger stage-specific PCD by which transient nuclear localization causes removal of these parasites [38] . Further studies revealed that PfSWIB was not only crucial for the survival and growth of the parasite, but also had profound effects on the parasite by interacting with other unknown proteins [38] [39] [40] . However, it remains unclear whether PfSWIB is involved in transcriptional regulation of virulence gene, especially vars, and its functional properties. To address the molecular mechanism of PfSWIB, we propose a new conditional knockdown system to study the function and mechanism of chromatin remodeler PfSWIB in regulating var gene expression. We found that the dysfunction of PfSWIB might lead to a significant downregulation of upsA, upsC and partial upsB var genes at the ring stage in a PfSWIB-knockdown line, as well as the aberrant expression of certain var genes at the mature stage in a single 48-h life-cycle of P. falciparum, which was characterized by the reversion of transcriptional downregulation of upsA and partial upsB/ upsC var genes, suggesting that PfSWIB might play a key role on var gene regulating. In addition, PfSWIB knockdown could affect the growth and development of parasite. This study provides a better understanding of the regulatory function of chromatin remodeling complexes (CRCs) in regulating the clonal variation of var genes, and an essential insight into the regulation of the major virulence gene family contributing to the pathogenesis in malaria parasites. Plasmodium falciparum lines 3D7, PfSWIB and PfSWIB∆ were first thawed and established for continuous cultivation in vitro according to standard procedures [41] . The cultivation was as follows: thawed isolates were cultured in RPMI 1640 medium (Invitrogen/Thermo Fisher Scientific, Carlsbad, CA, USA) containing 25 mM Hepes, 2 mM l-glutamine, 0.1 mM hypoxanthine (Sigma-Aldrich, Shanghai, China), 20 mg/ml gentamicin (Sigma-Aldrich, Shanghai, China), 0.5% Albumax II (Invitrogen/Thermo Fisher Scientific) and 2% human serum (type AB+; Changhai Hospital, The Second Military Medical University, Shanghai, China). Cultures were grown in media with type O+ erythrocytes at a 3% hematocrit under 2% O 2 , 5.5% CO 2 , 92.5% N 2 at 37 °C. To obtain tightly synchronized ring stage parasites, a standard percoll/sorbitol synchronization method was performed [42, 43] . In brief, schizonts were isolated over a 40 and 60% percoll cushion and allowed to invade fresh RBCs for 0 to 6 h. The resulting rings were subjected to 5% sorbitol treatment for 5 min at 37 °C. This process was repeated twice. The culture was washed and re-suspended in complete medium containing 2% RBC after the final sorbitol treatment. All transfected parasite lines originated from the 3D7 clone. DNA extraction was performed using the QIAamp DNA Mini Kit (Qiagen, Hilden, Germany). The high fidelity KOD-plus-Neo enzyme (Toyobo, Osaka, Japan) was used for PCR amplification to detect the endogenous PfSWIB locus or integration of the PfSWIB locus (PfSWIB-HA-FKBP-LID). PCR amplification reaction conditions are as follows: 95 °C for 3 min, followed by 32 cycles of 95 °C for 30 s, 50 °C for 30 s, 62 °C for 1 min, and a final extension cycle of 65 °C for 10 min. The specific primers used for PCR are listed in Additional file 1: Table S1 . pLN-ENR-GFP [44] (GenBank: DQ813653.3) was modified by deleting the whole pfenr-gfp cassette, which was then used to clone various DNA fragments in ApaI/BamHI multicloning site. For the pLN-HA-FKBP-LID construct (6900 bp), the calmodulin promoter (5\\' CaM fragment) was initially substituted by a 1.5 kb C terminus of PfSWIB in the ApaI/ASCI multicloning site. Three duplicates of HA tag (YPYDVPDYA) and FKBP-LID tag were ligated towards the C terminus of PfSWIB via a linker sequence (AAAAVDAAAA). The resulting vector pLN-PfSWIB-HA-FKBP-LID was used to transfect the 3D7 clone. The HA tag consists of three tandem repeat sequences of influenza hemagglutinin, whereas the FKBP-LID tag is comprised of the FK506-and rapamycin-binding protein and a 19-amino-acid degron fused towards the C terminus of FKBP ( Fig. 1a ) [45] . Transfections were performed on synchronized ringstage parasites of 3D7 in a 2 mm BioRad Gene Pulser ® cuvettes (Bio-Rad Laboratories, Hercules, CA 94547, USA). A minimum of 100 μg recombinated plasmid DNA was initially purified and used for electroporation using the Bio-Rad GenePulse Xcell ™ electroporator (BioRad Laboratories, Hercules, CA 94547, USA), at 310 V, with a resistance of 950 μF and a transfection time of less than 15 ms. Positive BSD drug selection at the final concentration of 2 nM, was subsequently applied and maintained after the first growth cycle. Those parasites that adopted the pLN-PfSWIB-HA-FKBP-LID construct carrying the bsd gene would offer a level of resistance to the drug. The transfected parasites were subjected to in vitro culturing for 25 to 40 days and subsequent BSD drug cycling to select stable single-crossover parasites. The initial obtained transfected parasite line was a mixture of the episomal and integrated form. Therefore, a \"3 + 2\" mode of drug screening process was subsequently introduced to enrich the integrated lines. In this process, the original transfected parasites were first cultured in vitro in the absence of BSD for 3 weeks, followed by 2 weeks of culture with BSD (2.5 µg/ml), therefore integrated lines of PfSWIB-HA-FKBP-LID could possibly be further sorted after 2-3 rounds of drug cycling (Fig. 1b) . After two cycles of BSD \"off \" (3 weeks) and BSD \"on\" (2 weeks), PCR was applied to detect the integration events of PfSWIB-HA-FKBP-LID in transfected parasites lines by using specific primers (Additional file 1: Table S1 ). Primers P2, P4 and P5 are specific to 3\\'UTR of endogenous PfSWIB, the upstream sequence of PfSWIB C-terminus and downstream plasmid sequence of HA-FKBP-LID fusion expression system, respectively, while Fn is the forward primer used for PCR amplification of PfSWIB C-terminus. In order to obtain purified integrated parasites clones, we utilized a clone screening assay by serial dilution of parasites in 96-well plates, as detailed previously [46] . Two stable transfections of PfSWIB-FKBP-LID integrated parasites clones, PfSWIB-C2 and PfSWIB-B10 were obtained. The integration events were detected on screened clones after extended the culture to 24-well plates. The primers used for testing the clones are described above (see also Additional file 1: Table S1 ). PCR amplification reaction conditions were as follows: 94 °C for 5 min, followed by 32 cycles of 94 °C for 30 s, 54 °C for 30 s, 65 °C for 2 min, and a final extension cycle of 65 °C for 10 min. To analyze the expression of the PfSWIB fusion system in integrated parasites lines, total parasite extracts were prepared by treatment with 0.15% saponin and re-suspended in SDS-loading buffer (Bio-Rad), then separated on a 12% SDS-PAGE gel (Bio-Rad) and subjected to western blot analysis. Total proteins from the 3D7 clone were used as controls. An antibody to aldolase (1:1000 dilution; Roche, Indianapolis, USA) was used as a positive control. Rabbit anti-HA (1:1000 dilution; Abcam, Cambridge, UK) was used to identify PfSWIB fusion proteins in different parasite lines. An enhanced chemiluminescence (ECL) western blotting kit (GE Healthcare, Uppsala, Sweden) was used to develop western blots. The theoretical molecular weight of endogenous PfSWIB, HA tag and HA-FKBP-LID tag is 92kD, 3.5kD and 22kD, respectively. The graylevel image analysis procedure was implemented with ImageJ software. Statistical significance was determined (*P < 0.05, **P < 0.01). Synchronized ring parasites (0-6 h after re-invasion) of the PfSWIB integrated clone (PfSWIB-HA-FKBP-LID) and the 3D7 clone were collected and diluted to 0.2% parasitemia. Each diluted parasite clone was divided into two wells of parasite cultures in the 24-well plate (50 µl RBC pellets/1 ml medium). Shield1 (0.5 mM (diluted in ethanol); Clontech, Mountain View, USA) was added into one of the two wells (final concentration: 0.5 μM). Consequently, four parasite lines including PfSWIB, PfSWIBΔ, 3D7 (shield1 induced) and 3D7 sh− (shield1 not induced) were subject to in vitro culture for 4-5 days. The parasitemia of each parasite line was counted every 12 h up to 96 h and repeated twice. The rate of parasitemia (%) of different lines were compared statistically by a one-way ANOVA followed by Bonferroni\\'s multiple comparison test (*P < 0.05, **P < 0.01). Growth curve analysis was performed in triplicate. For the transcription level assay of individual var genes, we used the var primer pairs published elsewhere [47] and listed in Additional file 2: Table S2 . qPCR assays were performed as described previously [10] . In brief, total RNA of synchronous parasite culture was extracted using TRIzol reagent (Invitrogen/Thermo Fisher Scientific, Carlsbad, CA, USA) [48] . cDNA was prepared in accordance with the manufacturer\\'s recommendations (Thermofisher Scientific, Carlsbad, CA, USA). The fold change between the experimental and reference groups was calculated as 2 −ΔΔCq . qPCR was performed on a QuantStudio 6 Flex Real-Time PCR System (Applied Biosystems, Singapore) using a program of 1 cycle of 15 s at 95 °C and 40 cycles of 30 s at 95 °C, 30 s at 54 °C, 30 s at 60 °C. Seryl-tRNA synthetase was applied as an endogenous control gene and to normalize the transcriptional level of each var gene. For the qPCR data, statistical significance was determined with the two-tailed Student\\'s t-test (*P < 0.05, **P < 0.01). RNA-seq was used to screen differentially expressed genes in three parasite lines and was carried out in duplicate. The reference genome and transcript information (P. falciparum 3D7) were obtained from PlasmoDB (http://plasm odb.org/). The RNA-seq aligner STAR v2.5.3a [49] was used to create a genome catalog, based on two data files including a genome sequence fasta file and GFF3 annotation file (PlasmoDB-34_Pfalcipa-rum3D7_Genome.fasta, PlasmoDB-34_Pfalciparum3D7. gff ). FastQC v0.11.5 was subsequently used to analyze the quality of the RNA-seq reads. The qualified reads were subsequently mapped and aligned to the reference genome. Reads with a minimum mapping quality of 20 (− q 20) were further screened out by using SAM tools. Heatmaps of gene expression profiles in different lines were created by using the R package edgeR [50] . Hierarchical clustering was performed by using one minus Pearson\\'s correlation metric and average linkage method. Line plots were generated with ggplot2 [51] . In order to carry out an analysis of variance (ANOVA)-like differential gene expression analysis, a statistical analysis of summarized read counts per genomic feature obtained from feature Counts function [52] was performed by using the R package ALDEx2 [53, 54] . A fold change greater than 1.5 or lower than − 1.5, FDR < 0.05, and P < 0.05 were selected as filters for defining the sets of differentially expressed genes. [44] , which harbored an antibiotic selection marker (bsd for Blasticidin S resistance, pink). HA tags (red) and FKBP-LID tag (blue and green) were ligated towards the 1.5 kb C terminus of PfSWIB. b Drug screening of the PfSWIB fusion expression system. The integration events and expression of PfSWIB modified locus were identified by PCR. \\'BSD+\\' and \\'BSD-\\' indicate that the transfected parasites were cultured in the presence or absence of BSD. Day 24: in vitro culturing in the absence of BSD for 24 days. Day 5, 10 and 15: in vitro culturing in the presence of BSD for 5, 10 and 15 days, respectively. The details of specific primers used for detecting the endogenous, episomal and integrated PfSWIB locus are provided in \"Methods\" Due to the low transfection proficiency after inserting the full-length PfSWIB gene, we subsequently selected a 1.5 kb C terminal sequence of PfSWIB and cloned this into the modified recombinant plasmids pLN-ENR-GFP forming fusion gene expression system pLN-PfSWIB (C-terminus)-HA-FKBP-LID. Through homologous recombination caused by a single crossover event, a transfected parasite clone known as PfSWIB-HA-FKBP-LID was obtained by transfecting the recombinant plasmid pLN-PfSWIB (C-terminus)-HA-FKBP-LID into the 3D7 strain, which was selected by Blasticidin S deaminase (BSD). The HA tag contains three tandem repeat sequences of influenza hemagglutinin, whereas FKBP-LID tag is comprised of the FK506and rapamycin-binding protein and a 19-amino-acid degron (ligand-induced degradation domain) fused to the C terminus of FKBP (Fig. 1a) [45] . A drug cycling process was subsequently introduced to enrich the integrated lines. The integration events and expression of PfSWIB-HA-FKBP-LID fusion gene were first identified by PCR (Fig. 1b) . The specific primers designed for detecting the transfection proficiency of the PfSWIB modified locus are listed in Additional file 1: Table S1 . Additional file 3: Figure S1 shows the screening of integrated clones which were performed by serial dilution and identification of integration events by PCR. Stable transfections of PfSWIB-FKBP-LID integrated parasites clones PfSWIB-C2 and PfSWIB-B10 were finally obtained from the cloning and screening process in 96-well plates. Since the PfSWIB-C2 clone exhibited stability and reproducibility in the subsequent assays, such as PCR detection and qPCR analysis, we selected clone PfSWIB-C2 rather than clone PfSWIB-B10 for the present study. Western blotting was subsequently performed to detect the expression of PfSWIB-HA-FKBP-LID fusion protein in the presence and absence of shield1 (1:2000, 0.5 μM). According to the previous study, PfSWIB is theoretically stable in the absence of the small molecule shield1 [45] . Alternatively, shield1 binds tightly to FKBP thereby substituting the degron and triggering rapid, continuous degradation of the LID domain and PfSWIB [45] . As shield1 may take time to exert its inhibition function, we selected the in vitro culturing ring and trophozoite parasite stage within their third life-cycle (> 96 h) as testing samples, according to the GCA results (Fig. 2a) . The results indicated that although the PfSWIB-HA-FKBP-LID (114 kDa) fusion gene could be successfully expressed in both stages in the presence or absence of shield1 (Fig. 2b , Additional file 4: Figure S2a) , there was still a significant difference of PfSWIB fusion protein expression between PfSWIB∆ and PfSWIB lines in the trophozoite stage, the expression of PfSWIB fusion protein decreased in PfSWIB∆ line (t (2) = 4.890, P = 0.0394). In contrast, no significant difference had been found in the ring stage parasite (t (2) = 1.006, P = 0.4205) (Fig. 2b, Additional file 4: Figure S2b) . To study the specific functions of PfSWIB, we subsequently analyzed the growth curve by counting parasitemia of different groups within 96 h (Fig. 2 , Additional file 5: Table S3 ). Both PfSWIB-C2 and 3D7 parasite clones were initially synchronized into 0-6 h ring stage parasites and equally divided into two groups. One group contained shield1 in the culture medium, whereas the other group was lacking shield1. Thus, four groups of lines including PfSWIB∆ (PfSWIB shield1 induced), PfSWIB (PfSWIB sh− shield1 not induced), 3D7 sh+ and 3D7 sh− were subject into the in vitro culture, with exactly the same preliminary parasitemia of 0.2%. The results of GCA revealed no significant difference in parasitemia between these lines during the first 48 h life-cycle (F (3, 6)  Based on the GCA result, we subsequently performed RNA-seq analysis in duplicate with synchronized 0-6 h ring stage parasites of three P. falciparum lines within their third life-cycle, to study the change of expression profiling when knocking down PfSWIB. The scatter plot illustrates the comparison of screening of differentially expressed genes in three parasite lines, including 3D7, PfSWIB and PfSWIB∆ (Additional file 6: Figure S3a ). Among all differentially expressed genes, 112 and 450 were upregulated in the PfSWIB∆ line when compared with the PfSWIB and 3D7 line, respectively, whereas 1 and 616 genes were downregulated (fold change, FC > 1.5, P < 0.05, FDR < 0.05). We further performed hierarchical clustering of the individual gene expression profiles by using the R package edgeR. RNA-seq data were normalized for sequencing depth and gene length using log 2 -transformed FPKM values (Additional file 6: Figure  S3b ). According to the special characteristics of differentially expressed proteins in each line, we further screened out transcription factors such as PfApiAP2 family and PfSir2B, as well as certain histone modification enzymes, which proved to be involved in regulating on var gene transcription. These genes were significantly upregulated in the PfSWIB∆ line, suggesting the existence of their different transcription profiling when knocking down PfSWIB. In addition, a number of vars also exhibited differential expression among different lines (Additional file 2: Table S2 ). To further study the transcription patterns of upsA-, upsB-and upsC-subtype var genes in different parasite lines, we first selected 44 differentially expressed vars among different lines based on the RNA-seq results (fold change, FC > 1.5, P < 0.05, FDR < 0.05), and tested the expression patterns of these preferred vars in each comparison group at the ring stage (0-10 h after re-invasion in 3rd life-cycle, > 96 h) by real-time PCR (Additional file 2: Table S2 ). upsB or upsA corresponds to subtelomeric and upsC corresponds to chromosome internal var gene members [55, 56] . Figure 3 shows the fold change of var gene expression profiling between shield1 pre-and post-induced lines PfSWIB∆ vs PfSWIB. Most upsA-and upsC-type vars were significantly downregulated in the PfSWIB∆ line, especially the upsA vars, when compared to the PfSWIB line (Fig. 3 , Additional file 7: Figure S4 ). However, the expression pattern of partial upsB vars presented a distinct change in the PfSWIB∆ line. A group of 9 upsB vars exhibited a high level of expression (Fig. 3 , red asterisk) (fold change ≥ 2.0, P < 0.05), while 5 upsB vars present an extremely low expression level (Fig. 3 , black asterisk) (fold change ≤ − 2.0, P < 0.05), when compared the PfSWIB∆ line to other lines. We subsequently re-analyzed the RNA-seq results of vars in these lines by using the R package edgeR, to identify similarities and differences in expression patterns for different var variants. The heatmap representing the hierarchical clustering of the var genes is shown in Fig. 4 and Additional file 8: Figure S5 . Figure 4 indicates two major clusters of 44 selected vars in relation to the expression pattern: one major cluster consisting of 37 var variants covered most of upsA, upsB and all upsC vars, while the other cluster covered only 5 upsB and 2 upsA vars. In addition, there were 24 and 20 vars variants showed an expression maximum in the 3D7 line and PfSWIB∆ line, respectively (red box). Additional linear regression analysis of the correlation between qPCR and RNA-seq data was performed to investigate the transcriptome data accuracy on 44 var genes (Additional file 9: Figure S6 , Additional file 10: Table S4 ). In the PfSWIB vs PfSWIB∆ comparison, the qPCR data showed a significant linear correlation with the RNA-seq data (r (44) = 0.6281, P < 0.0001). The RNA-seq results of 9 upregulated and 5 downregulated upsB vars (described above by qPCR analysis in the PfSWIB∆ line), were also re-analyzed by computing the log 2 fold change of FPKM values in different lines. Of these, 5 out of 9 upregulated upsB vars (except for PF3D7_1300100, PF3D7_1200100, PF3D7_0900100 and PF3D7_0712300) (Fig. 4, red arrow) , and 4 out of 5 downregulated (except for PF3D7_1240300) (Fig. 4 , black arrow) showed consistency with the qPCR results. Additional file 8: Figure S5 shows the expression pattern and hierarchical clustering of all the 60 vars in the three lines. According to previous studies, var genes were highly expressed in the ring stage, while drastically downregulated in the trophozoite stage [6, 10, 11] . To confirm this, we initially collected RNA samples from the ring and trophozoite stages of the 3D7 clone within the third . Approximately all of the var genes, were downregulated at the mature stage of 3D7 sh+ and 3D7 sh− parasite lines, except for PF3D7-0632500 (upsA), and PF3D7_1219300 and PF3D7_0712900 (upsB), suggesting that shield1 may have no influence on the expression of most var genes in a single 48 hour-life cycle of P. falciparum (Additional file 11: Figure S7) . We subsequently analyzed the log 2 ratio of T/R in invitro cultured PfSWIB∆ and PfSWIB lines within their third life-cycle after collecting RNA samples from their synchronized ring and trophozoite parasites. Among these parasite lines, PfSWIB exhibited a quite similar log 2 ratio of T/R to 3D7, which was characterized by the downregulation of most var genes at the mature stage (Fig. 5) . In contrast, approximately all of the upsA vars, as well as partial upsB (PF3D7_1200400, PF3D7_0632500, PF3D7_1219300, PF3D7_1240300 and PF3D7_1240400) and upsC (PF3D7_0412900, PF3D7_0420900 and PF3D7_0617400) vars were activated at the mature stage of P. falciparum in the PfSWIB∆ line (Fig. 5 ). The interplay between altered histone modification, chromatin modifying machinery, and general or sequence-specific transcription factors results in var gene regulation by modulating chromatin structure [57] . In this process, ATP-dependent chromatin remodeling complexes (CRCs) play a key role in chromatin remodeling through removal and movements of nucleosomes and deposition of alternative histones [13] . There are at least 11 proteins containing putative SWI2/SNF2 ATPase catalytic domains in P. falciparum [58] , seven of which are predicted to be chromatin-modifying proteins, including ISWI, Swr1, CHD1, SNF2L, MAL8P1.65, PFB0730w, PFF0225w and PF13_0308 [18] . However, it remains unclear how these PfCRCs regulate var gene expression by remodeling certain chromatin and therefore resulting in a high level of antigenic variation, immunity evasion and pathopoiesis. The SWI/SNF complex, originally revealed in yeasts, allows for transcriptional activation by remodeling chromatin [59] . It has also been demonstrated previously that mammalian SWI/SNF homologs complexes, also known as BRG1-associated factors (BAFs), can be found in multiple forms which consist of 9-12 proteins. Among these BAFs, BAF60b exists in an alternative form of the SWI/ SNF complex (Complex B), which is generally known as SWI-B or SWIB [34] . In humans, the BAF60a protein of the SWI/SNF complex is responsible for p53 binding and regulates its activities via an N-terminal region of the SWIB/MDM2 domain [35] , as well as facilitating cell cycle halting and tune the balance between repair and apoptosis induction [35] [36] [37] . SWIB/MDM2 domains have in fact been shown to participate in protein-protein [60] and chromatin-related interactions [61] , but their functional roles in other species are poorly understood. Similarly, the SWI/SNF complex is composed of constant units and includes the Swp73p/SNF12 protein comprising a SWIB/MDM2 domain, proposed to facilitate specificity and/or functionality to some extent [38, 62] . According to recent studies, the P. falciparum genome encodes two putative SWIB/MDM2 domain-containing proteins: PfMDM2 (PF3D70518200, SWIB/MDM2 domaincontaining protein) and PfSWIB (PF3D70611400, SWI/ SNF-related matrix-associated actin-dependent regulator of chromatin) [38] , and PfSWIB was not only crucial for the survival and growth of the parasite, but additionally had profound effects on the parasite by interacting with other unknown proteins [38] [39] [40] . Bioinformatics analysis demonstrated that PfSWIB is characterized by its SWIB domain (BAF60b domain, PlasmoDB, http://plasm odb. org/). Bonger et al. [45] have recently developed a new technique by which the degradation of a specific protein is definitely induced by way of a small molecule, shield1. Hence, an in-depth functional analysis of PfSWIB based on the conditional knock-down assay was implemented in the present study. By using a transgenic assay, drug screening and subsequent screening of the clones, we obtained two integrated parasite clones PfSWIB-C2 and B10, with a stable transfection of the modified locus PfSWIB-FKBP-LID into the Plasmodium parasites (Fig. 1 , Additional file 3: Figure S1 ). Western blot analysis confirmed that the fusion expression protein was successfully expressed in the transfected parasite lines, and there was a significant decrease of PfSWIB fusion protein expression in the PfSWIB∆ line (t (2) = 4.890, P = 0.0394), at its trophozoite stage within third life-cycle of P. falciparum. However, it should be noted that no significant change was detected in the protein expression level of PfSWIB fusion system during the ring stage, either in the presence or absence of drug pressure (t (2) = 1.006, P = 0.4205) (Fig. 2b , Additional file 4: Figure S2 ). This could be due to at least two reasons. First, it takes time for shield1 to work at full capacity through binding to LID, therefore, a portion or slight loss of fused PfSWIB protein could not be identified. Secondly, this protein might be expressed predominantly at the mature stage of P. falciparum. It is worth noting that PfSWIB may lead to stage-specific PCD, in the same way as BAF60a of the mammalian SWI/SNF complex induces p53-directed apoptosis, by which transient nuclear localization results in removal of these parasites, as previously described [35] . Vieira et al. [38] further demonstrated that PfSWIB could have a stage-specific, pro-survival function and takes part in the heat stress response of the Plasmodium parasite. They discovered that PfSWIB and PfARK3 participate in preventing p53-mediated apoptosis within the nucleus in reaction to heat stress [38] . Alternatively, human Aurora kinase B and its yeast homologue play key roles during gene transcription through histone H3 phosphorylation [39, 40] . Likewise, PfSWIB could also have an effect on parasite survival through transcriptional regulation by guiding the PfARK3\\'s phosphorylation activities [39, 40] . In our study, growth curve analysis of P. falciparum revealed that a relatively low parasitemia was observed in the PfSWIB knockdown line (PfSWIB∆) when compared to other parasite lines during their second life-cycle. Significant change was observed after 96 hours, when the parasites developed into the third life-cycle. The parasitemia of PfSWIB∆ showed a sharp decline (Fig. 2a) , suggesting that PfSWIB might play a key role in the proliferation and survival of P. falciparum. On the other hand, Jiang et al. [27] proved that knocking out the P. falciparum variant-silencing SET gene, which encodes an ortholog of Drosophila melanogaster ASH1 and controls histone H3 lysine 36 trimethylation (H3K36me3) on var genes, leads to expression of all vars. In another study focused on a novel chromatin-associated exoribonuclease, termed PfRNase II, Zhang et al. [63] revealed that this PfRNase II controls the silencing of upsA var genes, by marking their transcription start site and intron promoter regions leading to short-lived cryptic RNA. Depletion of PfRNase II specifically removes the silencing of upsA vars. Similarly, in the present study, knocking down of PfSWIB results in the silencing of upsA, upsC and partial upsB var genes, as well as removes the silencing of partial upsB vars (Fig. 3 , Additional file 7: Figure S4 ). Therefore, PfSWIB might be one of the trans factors activating var genes, especially the upsA vars, and silencing partial upsB vars. In addition, a few var genes exhibited inconsistent RNA-seq results with qPCR, especially the upsA vars. They are more likely to be conditionally activated in the PfSWIB knockdown line, according to the RNA-seq analysis (Fig. 4 , Additional file 8: Figure  S5 ). The minor inconsistency between the RNA-seq and the qPCR results for partial vars could be due to false positives/negatives resulting from either of these two methods, or experimental variation caused by different parasite lines or other unidentified factors. Nevertheless, by comparing the log 2 fold change data from qPCR and RNA-seq on 44 var genes, we confirmed that the linear regression and linear correlation analysis showed a significant linear correlation between the two methods (r (44) =v0.6281, P < 0.0001) (Additional file 9: Figure S6) . Furthermore, previous studies have demonstrated that var genes were highly expressed in the ring stage of P. falciparum, while drastically downregulated in the trophozoite stage. The var transcriptional levels in trophozoites are usually 10-100 times less than those in rings [6, 7, 10, 11] . A single P. falciparum parasite simultaneously transcribes multiple var genes but selects only one PfEMP-1 to attain the surface of the host cell by way of a developmentally regulated process [6, 7] . In order to test whether the transcription pattern had been disrupted when knocking down PfSWIB, we subsequently compared the transcription level of selected var genes between ring and trophozoite stage parasites, by testing the log 2 fold change of T/R in each group during a single life-cycle of P. falciparum (Fig. 5 , Additional file 11: Figure S7 ). Strikingly, we discovered that most of the upsA, upsB and upsC vars were downregulated in PfSWIB and 3D7 lines, while virtually all the upsA vars and partial upsB/upsC vars were significantly upregulated at the trophozoite-stage of the PfSWIB∆ line within the third life-cycle of P. falciparum (Fig. 5) , suggesting that PfSWIB might be involved in temporal regulation of var genes, particularly group A vars. The dysfunction of PfSWIB could disrupt the interactions between PfSWIB and other transcription factors which are essential for gene silencing or inhibit PfSWIB activity through unknown mechanisms. As we mentioned previously, P. falciparum varies its surface protein expression, particularly the var gene family-encoded clonally variant surface protein expression, to evade host immune responses [6] . In the process of monoallelic expression, specific silent var genes are usually retained at unique perinuclear sites and relocated to transcriptionally active zones [21, 28] . Nuclear actin has been discovered complexed with PfSET10, which has been suggested to play a role in the movement of the locus during activation and heterochromatic silencing [26, 29] . Actin and actin-related proteins (Arps) are common components of chromatin remodeling complexes and are involved in nuclear positioning. According to biological characteristics, PfSWIB may be involved in the process of relocating episomes, as an actin-dependent regulator of chromatin. Further studies regarding the possible role of PfSWIB and actin interaction would be worthwhile. In the present study, we revealed that actin-related chromatin remodeling factor PfSWIB is involved in the process of clonal variation in var gene expression, and is crucial for the survival and growth of the Plasmodium parasite. Depletion of PfSWIB not only silences upsA, upsC and partial upsB var genes and removes the silencing of partial upsB var genes, but also leads to the aberrant expression of upsA and partial upsB/upsC var genes at the mature stage of P. falciparum, during a single 48-hour life-cycle. These findings could provide a better understanding of the regulatory function of PfCRCs in regulating the clonal variation of the var gene family contributing to the pathogenesis in malaria parasites.'},\n",
       " {'_id': '000affa746a03f1fe4e3b3ef1a62fdfa9b9ac52a',\n",
       "  'title': 'Correlation between antimicrobial consumption and incidence of health-care- associated infections due to methicillin- resistant Staphylococcus aureus and vancomycin-resistant enterococci at a university hospital in Taiwan from 2000 to 2010',\n",
       "  'abstract': 'Objectives: This study was conducted to investigate the correlation between antibiotic consumption and the incidence of health-care-associated infections (HCAIs) caused by methicillin-resistant Staphylococcus aureus (MRSA) (HCAI-MRSA) and vancomycin-resistant enterococci (VREs) (HCAI-VREs) at a university hospital in Taiwan during the period from 2000 to 2010. Methods: Data on annual patient-days and annual consumption (defined daily dose/1000 patient-days) of glycopeptides (vancomycin and teicoplanin), linezolid, fusidic acid, tigecycline, and daptomycin were analyzed. Yearly aggregated data on the number of nonduplicate clinical MRSA and VRE isolates causing HCAI were collected. Journal of Microbiology, Immunology and Infection (2015) 48, 431e436 aureus; Results: Overall, the consumption of teicoplanin and linezolid significantly increased during the study period. A significant decrease in the incidence of HCAI-MRSA and a significant increase in the incidence of HCAI-VRE were found during the study period. A significant correlation was found between the increased use of teicoplanin and linezolid and the decreased incidence of HCAI-MRSA. By contrast, positive correlations were found between the consumption of teicoplanin and tigecycline and the incidence of HCAI-VRE. Conclusion: This study identified various correlations between the consumption of antibiotics and the incidence of HCAI-MRSA and HCAI-VRE. Strict implementation of infection-control guidelines and reinforcement of administering appropriate antibiotic agents would be helpful in decreasing the incidence of MRSA and VRE in hospitals.',\n",
       "  'body': 'The incidence of health-care-associated infection (HCAI) caused by multidrug-resistant bacteria has gradually risen during the last decade, especially in immunocompromised patients. 1, 2 The most common causative agents of HCAI in the United States are the Gram-positive bacteria Staphylococcus aureus and Enterococcus. 3 Methicillin (oxacillin)resistant S. aureus (MRSA) is of particular concern because patients with MRSA infection tend to have higher mortality rates, longer hospital stays, and higher health-careassociated costs than patients with methicillin-susceptible S. aureus infections. 1, 4 In addition, DiazGranados et al 5 found that the mortality rate among patients with vancomycin-resistant enterococci (VREs) infection was significantly higher than that among patients with vancomycin-susceptible Enterococcus infections. Taiwan is no exception, and VRE and MRSA infections have become emerging infectious diseases. 6e10 Antibiotic use is one of the risk factors for antibiotic resistance among bacterial species; however, the nature of this relationship is complicated. Although several studies have examined the relationship between antimicrobial consumption and antibiotic resistance, the findings were inconsistent, possibly due to differences in resistance profiles as well as due to differences in antibiotic-prescribing practices in different countries. 11e18 In those studies, the use of glycopeptides, extended-spectrum cephalosporins, and fluoroquinolones was demonstrated to be associated with the prevalence of MRSA and VRE. 12e18 Few studies, however, have investigated the relationship between the use of linezolid or fusidic acid and the prevalence of MRSA and VRE. In addition, the association between the consumption of tigecycline, a novel anti-Gram-positive agent derived from minocycline that has been shown to be effective against many Gram-negative rods as well as Gram-positive cocci, and the prevalence of MRSA and VRE has never been studied. 19 Similarly, no studies have so far investigated the association between the prevalence of MRSA and VRE and the consumption of daptomycin, an anti-MRSA antibiotic that has been approved by the U.S. Food and Drug Administration for the treatment of complicated skin and skin-structure infections and bacteremia due to MRSA. 20, 21 In this study, we investigated the correlation between consumption of antibiotics, including vancomycin, teicoplanin, linezolid, tigecycline, fusidic acid, and daptomycin, and the incidence of HCAI-MRSA and HCAI-VRE during the period from 2000 to 2010 at a medical center in Taiwan. The National Taiwan University Hospital (NTUH) is a 2500bed, academically affiliated medical center that provides both primary and tertiary care in northern Taiwan. The number of annual inpatient-days at the hospital increased from 624,675 in 2000 to 763,772 in 2010. Linezolid and fusidic acid were introduced into the hospital formulary in 2002. Tigecycline and daptomycin have been prescribed at the NTUH since 2007 and 2009, respectively. Some of the data analyzed in this study were included in a previous study. 16, 17 Bacterial isolates Data on the susceptibilities of S. aureus to oxacillin were collected during the period from 2000 to 2010. These isolates were nonduplicate and isolates of each species from each patient recovered within 7 days were considered as a single isolate. Susceptibility testing for S. aureus and Enterococcus species followed the Clinical and Laboratory Standards Institute guidelines. 22 S. aureus ATCC 25923 was used as the control strain for routine disk-susceptibility testing. 22 Methicillin resistance among S. aureus isolates was routinely screened by measuring their growth on oxacillin (6 mg/L) in 2% NaCl-containing trypticase soy agar plate that had been incubated in ambient air at 35 C for 24 hours. 22, 23 Vancomycin resistance among Enterococcus species was confirmed by growth of the isolate on a brain heart infusion agar plate containing vancomycin (6 mg/L) that had been incubated in ambient air at 35 C for 24 hours. 22  Yearly aggregated data on the number of nonduplicate clinical MRSA and VRE isolates causing HCAI were collected. HCAI was defined according to the National Nosocomial Infection Surveillance guidelines. 18 The incidence rates of HCAI-MRSA and HCAI-VRE were defined as the number of patients with HCAI-MRSA and HCAI-VRE, respectively, per 1000 inpatient-days. Data on annual consumption [defined daily dose (DDD)/1000 inpatient-days] of glycopeptides (vancomycin and teicoplanin), linezolid, fusidic acid, tigecycline, and daptomycin from 2000 to 2010 were obtained from the pharmacy department of the hospital. Linear regression analysis was used to analyze the trends in annual consumption of antimicrobial agents and the trends in incidence of HCAI-MRSA and HCAI-VRE over time. The Pearson product moment correlation coefficient was used to determine the relationship between annual antibiotic consumption and trends in resistance. A p value < 0.05 was considered statistically significant. In general, the use of each antimicrobial agent varied over time (Table 1) . Overall, the consumption of teicoplanin and linezolid significantly increased, whereas the consumption of vancomycin, glycopeptides (including vancomycin and teicoplanin), tigecycline, and fusidic acid remained stable during the study period. Trend in HCAI due to MRSA and VRE and the incidence of HCAI-MRSA and HCAI-VRE during the period 2000e2010 During the study period, a total of 4657 nonduplicate S. aureus isolates and 4219 enterococcal isolates causing HCAI were identified. Fig. 1A shows the trends in MRSA isolates and incidence of HCAI-MRSA. There was a significant decrease in the incidence of HCAI-MRSA over time. Fig. 1B shows the trends in VRE isolates and the incidence of HCAI-VRE. A significant rise in the incidence of HCAI-VRE over time was noted. Data on the correlation between the incidence of HCAI-MRSA, the incidence of HCAI-VRE, and the annual consumption of vancomycin, teicoplanin, linezolid, fusidic acid, tigecycline, and daptomycin are shown in Tables 2  and 3 . A significant correlation was found between the increased use of linezolid and teicoplanin and the decreased prevalence of MRSA. By contrast, no significant correlation was found between the increased use of vancomycin, glycopeptides (vancomycin and teicoplanin), tigecycline, and fusidic acid and the incidence of HCAI-MRSA. There was, however, a positive correlation between the incidence of HCAI-VRE and the use of teicoplanin and tigecycline. This study evaluated the association between antibiotic consumption and the incidence of HCAI-MRSA and HCAI-VRE in a medical center in Taiwan during an 11-year period. . analyzed the disease density (per 1000 patient-days) due to resistant bacteria rather than resistance rate. This parameter (density) is now considered more appropriate to present the real situation. In addition, we evaluated two newly available antibiotics (tigecycline and daptomycin) and the relationship with disease density. Using this new analysis and adding some new data, we had several significant novel findings. We found that the consumption of teicoplanin and linezolid significantly increased during the study period; however, the consumption of vancomycin, glycopeptides (both vancomycin and teicoplanin), and fusidic acid remained stable. It is speculated that physicians in the hospital are more likely to prescribe teicoplanin and linezolid than vancomycin for the treatment of MRSA infections because teicoplanin and linezolid are associated with fewer side effects, such as renal toxicity. In addition, the consumption of tigecycline increased from 1.8 DDD/1000 inpatient-days in 2007 to 6.0 DDD/1000 inpatient-days in 2010. However, although the consumption of both tigecycline and daptomycin increased over time, there was no significant association between the consumption of those antimicrobial agents and the incidence of nosocomial MRSA infections during the study period. Long-term studies are needed to evaluate the trend in HCAI-MRSA infections associated with the consumption of tigecycline and daptomycin. Although the relationship between the incidence of MRSA and the use of b-lactam antibiotics and fluoroquinolones has been investigated, 11e14,24 few studies have focused on the association between consumption of glycopeptides, linezolid, and fusidic acid and the incidence of infection due to Gram-positive bacteria. 16, 17 In this study, we found that the incidence of infections due to MRSA significantly decreased from 0.6023/1000 inpatientdays in 2000 to 0.2108/1000 inpatient-days in 2010. In addition, we noted a negative correlation between the use of teicoplanin and linezolid and the incidence of HCAI-MRSA. Although further research is needed to clarify this association, our findings suggest that teicoplanin and linezolid might exert a protective effect against the emergence of MRSA. We also found that there was no significant correlation between the consumption of vancomycin, glycopeptides including vancomycin and teicoplanin, and fusidic acid and incidence of HCAI-MRSA, a finding that is consistent with that reported in our previous study. 17 In addition, the consumption of tigecycline and daptomycin was not associated with the incidence of HCAI-MRSA. The reason for that finding is most likely due to the short duration of tigecycline and daptomycin use in our hospital. A longer study period is needed to clarify the correlation between those two antimicrobial agents and the incidence of MRSA infections. Our data show that the incidence of HCAI-VRE significantly increased during the study period. We found that the increase in use of teicoplanin positively correlated with the increase in incidence of HCAI-VRE. This finding is consistent with that in our previous study. 17 By contrast, we found that there was a negative correlation between the use of teicoplanin and the incidence of HCAI-MRSA. Our data also show a positive correlation between the consumption of tigecycline and the incidence of HCAI-VRE. Although tigecycline was used for only 4 years in this study, the strong positive correlation between its use and the incidence of VRE infections implies that tigecycline should be administered with caution. This study has several limitations. Although we found a significant correlation between antibiotic use and the incidence of HCAI-MRSA, the etiology behind the emergence of drug-resistant bacteria in our hospital is complicated and selective pressure from widespread use of antimicrobial agents might be only one of the causes. The prevalence of MRSA could have been affected by multiple factors such as infection-control measures and hand hygiene, or operational changes in the hospital. After the emergence of severe acute respiratory syndrome in 2003, the infection prevention and control program at the NTUH was upgraded to include hand hygiene, antibiotic-control policies, and an annual, intensive, project-based control program. In fact, it has been demonstrated that those policies are directly associated with the decrease in rates of HCAIs and bloodstream infections at the NTUH. 25 However, those effects were not measured in this study. Whether those policies had an impact on MRSA trends remains unknown. In addition, because this was an epidemiological surveillance study, we did not analyze the impact that duration of exposure to antibiotics, or clonal spread of resistant bacteria had on the trend in incidence of nosocomial MRSA infection. Furthermore, the overall DDD of a given antibiotic is a notoriously problematic denominator as there will be changes in low-dose usage in specific groups of patients (i.e., children and patients with renal insufficiency). Therefore, an analysis using number of prescriptions for those patient groups would be more appropriate. However, the impact should be limited in this study because those specific patient groups comprised only a small fraction of the study patients and the effect would be diluted and equal in each year. In conclusion, in this 11-year study in a single medical institution, we found that the use of teicoplanin and linezolid significantly increased. Furthermore, we found a negative correlation between the usage of individual antibiotics, such as teicoplanin and linezolid, and the incidence of HCAI-MRSA. However, we also found a positive correlation between consumption of teicoplanin and tigecycline and the incidence of HCAI-VRE. Therefore, strict implementation of infection-control policies including administration of appropriate antimicrobial agents may be positively correlated with a decrease in the presence of MRSA in hospitals.'},\n",
       " {'_id': '000b0174f992cb326a891f756d4ae5531f2845f7',\n",
       "  'title': 'Full Title: A systematic review of MERS-CoV (Middle East Respiratory Syndrome Coronavirus) 2 seroprevalence and viral RNA prevalence in dromedary camels: implications for animal vaccination',\n",
       "  'abstract': '',\n",
       "  'body': 'Since the first human case of Middle East Respiratory Syndrome Coronavirus (MERS-CoV) infection 53 was detected in 2012 (1), a substantial evidence base has built up showing dromedary camels to be the 54 zoonotic source of this virus (2) . MERS-CoV circulates extensively in dromedary populations causing 55 no impactful disease. Human infection, however, is associated with a measured case fatality ratio of 56 around 35% (3). Following spillover events, human-to-human transmission of MERS-CoV is 57 relatively inefficient and limited to close, unprotected contact environments such as hospitals (4, 5) . Phylogenetic analysis of viral sequences isolated from dromedaries and humans indicates that 59 hundreds of camel-to-human spillover events are likely to have occurred since 2012 (6). Taken 60 together, recurring dromedary-to-human transmission is driving ongoing human infection. The key role of dromedaries in human MERS-CoV infection has led decision-makers to consider 62 dromedary vaccination as part of MERS-CoV prevention interventions (2) . Dromedary-targeted  Additionally, although whole-virus isolation and culture is necessary to confirm whether the shedding 78 is infectious, detection of viral RNA through RT-PCR can be used as a proxy for the prevalence and 79 distribution of infectious dromedaries (8-10). By conducting a qualitative synthesis of the study findings, considering reported heterogeneities, and 81 summarising the results of longitudinal studies of infection and immunity, we aim to assess the extent 82 of current understanding of MERS-CoV epidemiology in dromedaries, implications for control, and 83 gaps to be addressed going forward. We note that a similar systematic review of the literature up until May 2018 was unknowingly carried out in parallel to our own (11), with no discussion between the 85 two groups. Here, we confirm and update the results of the parallel review, discussing our results in 86 the context of potential animal vaccination and mathematical modelling of MERS-CoV in dromedary 87 camels. f. neutralisation at dilution >1:80 gave 15% seropositivity but regional range only available for ELISA results -reported accordingly 138 g. study also tested different site-types but measured RNA in serum sample rather than nasal swab so this was not included 139 h. range is across weeks rather than regions 140 i. both (66) and (65) Fig 3) . (67) UAE 2012 - - - - 4% 1113 farm (68) UAE 2014 - - - - 100% j 6 j - - (69) UAE 2015 100% 8 - 1:40 100% 8 - -(70 Although seroprevalence was 80-100% in adult dromedaries in most populations across the Middle Three studies have found dromedaries to be shedding MERS-CoV RNA despite having high 266 antibody titres months or weeks prior to detectable infection. Both older animals whose 267 antibodies reflect past exposure (29, 76, 79) , and young calves whose high antibody titres 268 were maternally-acquired immediately post-partum, became infected (79). One study directly observed recurring infection amongst a herd of dromedaries in Egypt. Four animals were shedding 1-3 months prior to a herd-wide epidemic in which they were Age-dependent seroprevalence patterns suggest that the higher prevalence of viral shedding 311 in juveniles compared with adults is likely due to immunological naivety. The age-312 distribution of reported infections synthesised here, suggests that contact with juveniles may 313 pose greater risks of human transmission than adults, making them potential targets for 314 vaccination. However, frequency of human contact with dromedaries may also be animal- respiratory syndrome coronavirus specific antibodies in naturally exposed Israeli llamas, 543 alpacas and camels. One Health. 2018;5:65-8. 544'},\n",
       " {'_id': '000b7d1517ceebb34e1e3e817695b6de03e2fa78',\n",
       "  'title': 'Supplementary Information An eco-epidemiological study of Morbilli-related paramyxovirus infection in Madagascar bats reveals host-switching as the dominant macro-evolutionary mechanism',\n",
       "  'abstract': '',\n",
       "  'body': '- Figure S1 : Phylogeny of all sequences belonging to the UMRV phylogroup. - Table S4 : Bats cytochrome b sequences data set. -Table S5 : Test of host-parasite co-evolution using global fit methods ParaFit. Figure S1 . Phylogeny of all sequences belonging to the UMRV phylogroup. A global phylogeny of 308 partial L-gene sequences calculated in 10,000,000 iterations in MrBayes with the GTR + G + I evolutionary model and a 10% burn-in rooted with an Aquaparamyxovirus sequence (GenBank number EF646380). All Malagasy bat paramyxoviruses sequences obtained within this study fell within group of Unclassified Morbillivirus-Related viruses. Genbank accession numbers used for each virus genera are indicated in Table S6 . '},\n",
       " {'_id': '000bc3d4637530496ac4cb798e10d95709eff92d',\n",
       "  'title': 'FDA Perspectives on Diagnostic Device Clinical Studies for Respiratory Infections',\n",
       "  'abstract': '',\n",
       "  'body': \"Two pathways are described for submission to FDA for clearance of a diagnostic device: a Premarket Application (PMA), which can lead to approval of a diagnostic device, and a Premarket Notification, which can lead to clearance. The latter is often called a 510(k), named for the statute providing for this path. Recent FDA clearance of molecular-based multiplex panels represents the beginning of a new era for the diagnosis of respiratory infections. The ability to test for multiple pathogens simultaneously, accompanied by the increasing availability of molecular-based assays for newly recognized respiratory pathogens will likely have a major impact on patient care, drug development, and public health epidemiology. We provide a general overview of how FDA evaluates new diagnostics for respiratory tract infections and the agency's expectations for sponsors developing new tests in this area. The Food and Drug Administration (FDA) has regulatory responsibility in the United States for review and oversight of the products that form the backbone of modern medical practice. FDA premarket and postmarket activities are administratively organized around centers, with the Center for Drug Evaluation and Research (CDER), Center for Biologics, Evaluation, and Research (CBER), and Center for Devices and Radiological Health (CDRH) bearing most of the responsibility for patient diagnosis and treatment. CDRH, through the Division of Microbiology Devices, in the Office of In Vitro Diagnostic Device Evaluation and Safety has regulatory responsibility for in vitro diagnostic devices (IVDs) intended for use in the diagnosis of almost all infectious diseases, including respiratory tract infections. The specific regulations that guide FDA are defined under Title 21 of the Code of Federal Regulations (CFR); in this overview, we highlight both the elements of the CFR relevant to in vitro microbiology diagnostic devices as well as how FDA approaches the review of microbiology device applications for respiratory infections. (Specific regulations cited are included in brackets when referenced.) IVDs are defined as ''reagents, instruments, and systems intended for use in the diagnosis of disease or other conditions, including a determination of the state of health, in order to cure, mitigate, treat, or prevent disease or its sequelae.for use in the collection, preparation, and examination of specimens from the human body'' [21 CFR 809.3] . Similar to drug and biological products, new in vitro devices must undergo FDA review prior to marketing. For a new IVD, both ''safety'' and ''effectiveness'' of a device must be demonstrated. The broad criteria for evaluating the safety and effectiveness of new IVD are also defined by regulation: Requirements for marketing also include the need for labeling that includes, among other requirements, product performance characteristics ''as appropriate . . . describing such things as accuracy, precision, specificity, and sensitivity'' [21 CFR 809.10]. The following discussion provides an overview of how FDA approaches these requirements during the development and review of a new microbiology diagnostic device. FDA regularly issues guidances in an effort to make the FDA submission and review process more consistent. Guidances are not rules but provide FDA's current thinking on a topic. Guidances may be applicable to all of FDA (eg, see [1] ), to a center within FDA, to a class of products, or tied to a novel product (often called Special Controls Guidance; eg, see [2, 3] ). Some are initially issued as draft guidances where the public has an opportunity to comment for a prespecified period. In this article, we provide references to several FDA guidances. The FDA drug approval process is more familiar to physiciansadequate and well-controlled trials to establish safety and efficacy [21 CFR 314.50] are required as the standard for regulatory approval, usually conducted as randomized, comparative trials. In contrast, study designs used to support device applications are typically not randomized, controlled trials; most often, clinical performance is documented by a single prospective multicenter study in the intended use population, supported by extensive nonclinical analytical studies. For diseases of low prevalence, prospective studies may not be feasible. One major difference between drugs and devices is the existence of two distinct pathways for the marketing of devices: devices may be ''cleared'' through the 510(k) process when a new device is determined to be substantially equivalent to a previously marketed product. Devices for entirely new indications or higher-risk intended use are ''approved'' by the Premarket Application (PMA) process. It is important that clinical and analytical requirements for submissions are more aligned with the technology and the clinical settings for use of the device rather than the specific pathway used to go to market. Some devices are novel, but may be considered to be moderate risk in the context of how they are used. These devices may still may be cleared via the 510(k) process using what is called a de novo 510(k). However, the potential for a de novo pathway should be discussed with FDA well in advance of a formal submission. Essential elements considered in the review of a new in vitro diagnostic device for a respiratory pathogen are outlined in Table 1 and discussed in more detail below. How diagnostic devices are evaluated is strongly influenced by the intended use (IU) and the risks associated with that use. The proposed IU for a device is an explicit statement of the analyte that the device is measuring or detecting (eg, a specific organism or a biomarker for that organism, and the clinical disease resulting from infection). The IU also describes how the results are reported: qualitative assays most often report whether a specific pathogen is detected or not (eg, culture or direct antigen tests); less commonly (for respiratory pathogens), the results may be quantitative (eg, viral load assays or antibody titers). The specimen type (also referred to as the sample matrix) that can be tested is also an important component of the IU of the device. Typical specimens for an upper respiratory tract infection diagnostic device are nasal swabs, nasopharyngeal swabs, nasal washes; or for lower respiratory tract specimens, bronchoalveolar lavage or sputum. Less obvious matrices for respiratory pathogens include urine or gastric lavage specimens. Evaluation of an analyte in a specific matrix may be critical for a proposed IU as test performance may significantly vary across different matrices. The setting (and/or timing) where test specimens should be collected may also be part of the IU; for example, some devices (including several rapid influenza tests) are cleared for point-ofcare settings (ie, used near the patients while under the auspices of a laboratory with professional laboratorians). Timing may be equally critical to test performance; for example, sensitivity of certain rapid antigen tests may drop dramatically as an illness evolves. New devices must be compared with a benchmark for establishing whether or not a specific target condition is present. FDA recognizes two major situations for assessing the diagnostic performance of new qualitative diagnostic tests: when a clinical reference method is available or when a comparator other than a reference method is used. In the former circumstance, a clinical reference method is considered to be the best available method for establishing the presence or absence of the target organism [1] but the reference method cannot incorporate results reported by the new device; for example, a new enzyme- linked immunosorbent assay (ELISA) could not be used as one of the diagnostic criteria for the reference method in a study used to evaluate the performance of the assay. Confusion may sometimes arise when distinguishing between analytical and clinical reference method since these may be identical or separate in different contexts; for example, the performance characteristics for a new assay for group A streptococcus against an analytical reference method such as culture on appropriate media may only be valid in the clinical setting of pharyngitis, since group A streptococcus may be a normal colonizer and performance may be different in the setting of colonization versus infection. In contrast, the analytical and clinical reference methods for influenza A are identical since the presence of influenza A is always considered abnormal. Determining the appropriate clinical reference methods may be particularly complex in the setting of multiplex assays for diseases with multiple etiologies such as community-acquired pneumonia, and where two or more possible pathogens may be isolated. Newly identified organisms introduce different challenges in selecting the appropriate reference method; for example, more recently identified pathogens such as metapneumovirus may be difficult to isolate or confirm by traditional methods such as culture. Because a clinical reference method cannot include the new device in its ascertainment, methods such as polymerase chain reaction (PCR) followed by bidirectional DNA sequencing may be necessary to confirm test performance. Similar concerns exist for assays that differentiate subtypes of novel influenza, a disease where diagnostics are essential but where relatively few clinical specimens are available for confirming performance. There are recent FDA guidances that are useful in aiding companies in developing valid comparators. One is a draft guidance on establishing the performance of an influenza IVD [4] , whereas the other two guidances are special controls guidances: one for multiplex devices for respiratory viral pathogens [2] and the other for multiplex devices for influenza A subtyping [3] . Another common issue that arises in clinical studies for new devices is result discordance; that is, when discrepancies occur between the result with the new device and the reference method. The objective of a pivotal study is to ascertain performance of the new device. Retesting discrepant samples tends to result in an overly optimistic view of performance of the new device. If the new device is truly better than the current clinical reference method, companies should consider contacting FDA to discuss an alternative comparator prior to beginning their pivotal trial. Clinical studies for evaluating new devices should support the intended use of the device and be conducted in a manner that will yield results consistent with how the device would be integrated into US clinical practice. New devices under study may include foreign sites depending on the specific analyte and the specific intended use for the device; however, clinical studies with foreign sites must include a justification for why similar performance is expected in the United States. Study protocols should include clear inclusion and exclusion criteria, case report forms, and testing procedures. Sample sizes should be sufficient to ensure that the clinical performance is acceptable in its IU setting. See section on Assay performance characteristics: Statistics 101 for Diagnostic Devices for further discussion. The interpretation of results from a new investigational device should be determined prior to conducting 'pivotal' clinical studies. For a qualitative assay, this includes the criteria for scoring device results (eg, as either analyte detected, analyte not detected, equivocal, or invalid). Invalid results may occur when one or more test parameters fail to meet the expected criteria. For equivocal results, rules for retesting specimens should be specified. Assay Performance Characteristics: Statistics 101 for Diagnostic Devices Sensitivity/specificity. The simplest approach to describing diagnostic performance characteristics is when every test result can be described qualitatively (eg, analyte detected or not detected), and where a reference method exists. For a prospectively enrolled study where patients meet the inclusion/exclusion criteria for the IU of the device, agreement of the test results with the reference method establishes sensitivity and specificity of the test device. Sensitivity is the probability that the new device will have a positive test result given that the clinical reference method is positive; it is estimated in a clinical study as the fraction of patients that are positive by the new test among those that are positive by the clinical reference method. For example, in a study of novel H1N1 influenza, it is the percent of patients that both test positive by the new device and are positive via the reference assay divided by the number of patients positive by the reference assay. Low test sensitivity reflects less security with the results of a negative test; that is, patients with the disease may be falsely labeled as negative by the new assay. (This fundamental consideration, i.e., -equating ''analyte not detected'' with the absence of disease [or the analyte]-is a common misinterpretation that bedevils the clinical use of devices with imperfect sensitivity.). Specificity is the converse; that is, the probability that the new device will have a negative test result given that the clinical reference method is negative. Specificity is estimated in the pivotal study by the fraction of subjects that are negative by the new test among those that are negative by the clinical reference method. Higher specificity yields greater confidence in a positive result. It is important to recognize that although patients included in specificity calculations do not have the disease of interest, in most cases they are still symptomatic (depending on the specific IU for the test); asymptomatic subjects would not be considered part of the IU population for such a device and therefore would not be appropriate for use in a specificity calculation. There are examples where a new device is not compared with a reference but to a previously cleared/approved device. In this setting, sensitivity and specificity cannot be estimated due to the lack of known infected status. (For example, a benchmark reference standard for aspergillosis infection may include galactomannan as part of the definition of aspergillosis infection in certain populations, even in the absence of invasive procedures to document infection.). Identical statistical calculations can be used in this setting but are interpreted as being positive and negative percent agreement rather than sensitivity and specificity; these considerations are also described in FDA guidance [1] . Percent agreement estimates may be misleading as similar devices can both be in error; for example, 100% agreement between two similar devices would be unlikely to indicate the new device has perfect clinical sensitivity/specificity were the clinical truth available. There may be significant challenges with estimating the sensitivity of a new assay if the prevalence of the disease being tested for is low. As Table 2 illustrates, estimates of sensitivity are dependent both on the true underlying sensitivity and sample size. In a clinical study with only 5 specimens positive for a specific pathogen (via the reference method) and an observed sensitivity of 5/5 (100%) results in a lower confidence bound of only 55.6%, meaning diagnostic performance is very uncertain. Even for a test with perfect sensitivity, at least 35 patients with the condition of interest would be needed to yield a 95% lower bound greater than 90%. In a pivotal study for devices to detect more common conditions, it is not uncommon to see a sample size of 50 or more patients that are positive via the reference method. FDA considers the observed sensitivity and specificity and their respective 95% lower confidence bounds as part of device performance. When sensitivity, specificity, and prevalence are known, one can calculate positive and negative predictive values (PPV and NPV, respectively). PPV and NPV ask an alternative question: given a positive or negative test result, what is the probability that the patient has (or does not have) the condition being tested for? In a prospective study, this would be estimated as the percentage of patients with the condition of interest from all patients that test positive; similarly, the negative predictive value is estimated as the percentage of patients without the condition of interest among those that test negative. The relationship between sensitivity, specificity, prevalence, PPV, and NPV is illustrated in Table 3 . As can be seen, PPV and NPV are expressed as percentages; ideally, both will be close to 100%. The importance of test prevalence is illustrated in Table 4 where, despite high sensitivity and specificity, test utility is extremely limited in the context of ruling in a diagnosis as the PPV of the test result is still only 8%, underscoring the value of tests as an adjunct to diagnosis rather than the sole means for diagnosis. However, tests may still have substantial clinical value even when only the NPV is very high. A high NPV indicates that if the patient tests negative, it is very unlikely that the disease is present and the physician may want to pursue further work up to obtain a diagnosis. However, the formulas for NPV and PPV in Table 3 are inappropriate when prevalence is unknown (e.g., when representative cases and controls are selected for study). Instead, a more complex set of formulas found in Pepe [5] provides a means of estimating NPV and PPV under various scenarios for prevalence. Assuming one believes reasonable estimates of sensitivity and specificity are available, these equations can be used to illustrate what impact prevalence has on PPV and NPV: a Lower limit of a two-sided 95% score confidence interval (see [1] ). The lower confidence bound is influenced by the number of specimens with pathogen of interest and how good the device is. FN 1 TN 1 FP) 5 100% x (N 1 /N) , positive predictive value (PPV) is 100% x TP/(TP 1 FP), negative predictive value (NPV) is 100% x TN/(TN 1 FN) . In these equations, prevalence is assumed known (ie, measured without error). So if we had a device with a sensitivity of 90% and a specificity of 90% as in Table 4 , but the prevalence of the target condition of interest was now 10% rather than the 1% in Table 4 , the PPV would be 0.50 or 50% rather than 8.33% as in Table 3 , and NPV would be 0.99 or 99%. NPV is negligibly lower than in Table 3 . When a clinical study only assesses agreement to another previously cleared device rather than to a reference method, reporting NPV and PPV would be inappropriate. Prospective studies in the IU population provide the best estimate of real-world performance for a diagnostic device. However, clinical performance of devices for detecting rare conditions, such as potential bioterrorism agents (eg, anthrax) may be impossible to ascertain. Low prevalence often mandates the use of retrospective or banked specimens as clinical specimens for new devices. With banked specimens and/or enriched studies, prevalence usually cannot be estimated. (Retrospective samples that were prospectively archived [e.g., all patients meeting a prespecified inclusion/exclusion criteria are included] can still be representative provided storage conditions do not impact the assay results. However, banked [repository] specimens may not be representative and should be considered separately.). As an alternative, the PPV and NPV can be calculated for a set of plausible prevalence estimates so that the clinical impact of a different disease prevalence can be understood. It is also important to appreciate that including a nontrivial proportion of banked specimens or substantial enrichment of specimens can change the disease spectrum in the study, which in turn changes the estimates of sensitivity and specificity and not just prevalence; in this situation, sensitivity, specificity, PPV, and NPV could all be biased and misleading. Given the critical role of study design in determining performance characteristics for a new test, companies are strongly encouraged to discuss their study proposals with FDA prior to initiation of studies essential for device clearance or approval. To evaluate the performance of the assay in multiple settings, precision studies must be conducted to confirm the reproducibility of assay results. These studies usually include multiple days or multiple runs across clinical sites. Studies should be designed to capture all major sources of variability, including site-to-site variation, the effect of different operators, and instrument-specific variability. Detailed guidance regarding precision studies for in vitro diagnostic devices is available in the Clinical and Laboratory Standards Institute (CLSI) guideline EP5-2A [6] . For qualitative assays, test panels used for precision/reproducibility studies should not only include both positive and negative samples, but particularly samples close to assay cutoffs; that is, thresholds for distinguishing positive from negative or equivocal. For quantitative assays, panels should include samples that are at the extremes of the analytical measurement range as well as values near key clinical decision points. Analytical sensitivity (limit of detection) for each detected pathogen (analyte) and each specimen type (ie, matrix) need to be assessed [2-4, 7, 8] . Analytical specificity, including crossreactivity with other possible respiratory pathogens, should be assessed, as well as interference by common substances such as blood, medications, and so forth. For some quantitative assays, it may be necessary to rule out a possible hook effect with high positive samples. For studies utilizing banked specimens to establish clinical or analytical performance, it would be necessary to provide evidence that storage conditions do not impact assay results. Other analytical studies may be requested depending on the type of technology used. A recent major advance for respiratory in vitro diagnostic testing has been FDA clearance of multiplex devices that can test for multiple pathogens simultaneously. The technical definition of a multiplex assay is ''two or more targets simultaneously detected via a common process of sample preparation, target or signal amplification, allele discrimination, and collective interpretation'' [7] . Essentially, a multiplex assay takes a single sample and then provides more than one output (eg, adenovirus present or absent and influenza A present or absent). There are several known hurdles to developing a multiplex assay with many pathogens, as illustrated by the multiplex respiratory viral infection panel example [2] . Adequate performance must be demonstrated for each viral pathogen included Adding a high-risk pathogen such as severe acute respiratory syndrome (SARS) coronavirus to a multiplex assay would mean submitting a PMA for approval. Thus, the regulatory path may be a consideration when designing a multiplex device. An FDA special controls guidance for respiratory viral panel multiplex devices has been recently published [2] and addresses some of these issues. FDA also publishes special control guidance on devices detecting new pathogens, which can be either standalone tests or part of multiplex assays. The Clinical Laboratory Improvement Amendments of 1988 (CLIA) defines categorization for a subset of devices sufficiently simple to use in a CLIA-waived settings, thus in essence permitting the use of these diagnostic tests outside of a clinical laboratory [9] . (A CLIA-waived test is different from ''home use devices'' [eg, pregnancy tests] which can be used with no medical oversight, or ''point-of-care'' tests [eg, tests performed in an emergency room, which remains under the auspices of the hospital laboratory]. ''Point-of-care'' tests may or may not be CLIA-waived depending on the specific device.) Upon application by a sponsor, FDA has the authority to designate a specific in vitro diagnostic test as a waived test. Waived tests such as urine dipsticks or group A streptococcal throat swab tests have widespread clinical use due to their availability at sites such as physician offices or commercial pharmacies. Devices being considered for a CLIA waiver are first subject to the same regulatory process as other diagnostic devices; that is, the device must demonstrate it has adequate performance characteristics in the hands of a trained laboratorian. However, beyond the demonstration that the device has adequate performance characteristics, the device must perform well in a CLIA-waived setting. This can be done either after a device is cleared or approved, or the initial study of the device can be designed to support a CLIA waiver application, which can be submitted after clearance/approval is granted. As part of an application for a CLIA waiver, sponsors are required to include studies to show the outcomes of not following the ''instructions for use'' provided with the device (e.g., prolonging the incubation of reagents, adding reagents in the wrong order, etc). These are studies designed to ensure that the device is robust and simple enough to operate without a trained laboratorian. By this definition, a sample that required centrifugation for processing, for example, would not be eligible for a waiver. The study participants in a CLIA-waiver study represent end users typical of those who would perform the test in a waived setting (eg, nurses or aides in a doctor's office or nursing home). Three or more sites should be included in the study, with 1-3 users per site for a minimum of 9 intended users. Testing must also include specimens with results close to the test cutoff values (ie, specimens that may be the most challenging to nonlaboratorians). Detailed information regarding the design of studies for a CLIA waiver application, including the minimal performance criteria acceptable for a CLIA-waived device, is described in ''FDA guidance for industry and FDA staff: recommendations for Clinical Laboratory Improvement Amendments of 1988 (CLIA) waiver applications for manufacturers of in vitro diagnostic devices'' [9] . It is highly recommended that sponsors seeking to market a device for use in a waived setting consult with FDA early in the regulatory process, especially for new products under development. It is important to note that CLIA-waiver requirements are very stringent; sponsors should consider the value of device use in CLIA-waived settings before pursuing the waiver process. Most pivotal studies for IVDs do not involve managing the patient based on the result of the investigational device; thus, IVD companies are rarely required to have an investigational device exemption (IDE) to start their pivotal study. However, FDA Center for Devices and Radiological Health, including the Division of Microbiology Devices, provides advice to sponsors on whether an IDE is required and other issues regarding the pre-IDE submission consultative process. Given the extensive number of diagnostic indications, the evolving technologies, and potential regulatory pathways, FDA strongly recommends that companies consider participating in the pre-IDE submission process early in the device development cycle and especially prior to conducting pivotal studies. The pre-IDE process allows FDA to evaluate if the proposed study designs are appropriate to support the IU of a device. For new instrumentation that have not been reviewed by the FDA, documentation of hardware, software, and manufacturing processes will need to be assessed, and could be topics for discussion in a pre-IDE submission [10, 11] . As an aid to sponsors developing new diagnostic devices, FDA posts summaries of the review decisions for cleared devices on the FDA Web site (http://www.accessdata.fda.gov/scripts/cdrh/ cfdocs/cfpmn/pmn.cfm). Companies may want to examine these summaries for recently cleared devices that are similar to the ones they are proposing to submit. In addition, substantial advice concerning the device development process is available through published FDA guidance [12] . The previous discussion has only briefly reviewed some of the concerns manufacturers of new in vitro diagnostic tests should consider during their development. The extensive number of diagnostic indications, evolving technologies, increasing use of multiplex instruments in microbiology, and potential regulatory pathways mandate careful planning by sponsors when developing new diagnostic assays. It is essential that sponsors plan for clinical and analytical studies that consider the device performance needed to support the proposed intended use. The use of appropriate statistical methods in both planning and evaluating these studies is also highly recommended. Interactive dialogue with FDA during the device development process is strongly encouraged.\"})"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "b.take(100)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.6"
  },
  "latex_envs": {
   "LaTeX_envs_menu_present": true,
   "autoclose": true,
   "autocomplete": true,
   "bibliofile": "biblio.bib",
   "cite_by": "apalike",
   "current_citInitial": 1,
   "eqLabelWithNumbers": true,
   "eqNumInitial": 1,
   "hotkeys": {
    "equation": "Ctrl-E",
    "itemize": "Ctrl-I"
   },
   "labels_anchors": false,
   "latex_user_defs": false,
   "report_style_numbering": false,
   "user_envs_cfg": false
  },
  "varInspector": {
   "cols": {
    "lenName": 16,
    "lenType": 16,
    "lenVar": 40
   },
   "kernels_config": {
    "python": {
     "delete_cmd_postfix": "",
     "delete_cmd_prefix": "del ",
     "library": "var_list.py",
     "varRefreshCmd": "print(var_dic_list())"
    },
    "r": {
     "delete_cmd_postfix": ") ",
     "delete_cmd_prefix": "rm(",
     "library": "var_list.r",
     "varRefreshCmd": "cat(var_dic_list()) "
    }
   },
   "types_to_exclude": [
    "module",
    "function",
    "builtin_function_or_method",
    "instance",
    "_Feature"
   ],
   "window_display": false
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
