{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-05T08:25:37.035557Z",
     "start_time": "2020-11-05T08:25:30.596629Z"
    }
   },
   "outputs": [],
   "source": [
    "from vectorai import ViClient\n",
    "import json\n",
    "import xmltodict as xtd\n",
    "import numpy as np\n",
    "import os\n",
    "from nltk.corpus import stopwords\n",
    "from nltk.tokenize import wordpunct_tokenize\n",
    "from nltk.stem import PorterStemmer\n",
    "from joblib import Parallel, delayed\n",
    "from gensim import corpora\n",
    "from gensim import models\n",
    "import pandas as pd\n",
    "from gensim import similarities\n",
    "from operator import itemgetter\n",
    "import glob\n",
    "import dask.bag as db\n",
    "from dask.distributed import Client, progress\n",
    "from vectorai.models.deployed import ViText2Vec"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-05T08:25:38.841449Z",
     "start_time": "2020-11-05T08:25:37.050558Z"
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<table style=\"border: 2px solid white;\">\n",
       "<tr>\n",
       "<td style=\"vertical-align: top; border: 0px solid white\">\n",
       "<h3 style=\"text-align: left;\">Client</h3>\n",
       "<ul style=\"text-align: left; list-style: none; margin: 0; padding: 0;\">\n",
       "  <li><b>Scheduler: </b>tcp://127.0.0.1:51227</li>\n",
       "  <li><b>Dashboard: </b><a href='http://127.0.0.1:8787/status' target='_blank'>http://127.0.0.1:8787/status</a>\n",
       "</ul>\n",
       "</td>\n",
       "<td style=\"vertical-align: top; border: 0px solid white\">\n",
       "<h3 style=\"text-align: left;\">Cluster</h3>\n",
       "<ul style=\"text-align: left; list-style:none; margin: 0; padding: 0;\">\n",
       "  <li><b>Workers: </b>6</li>\n",
       "  <li><b>Cores: </b>12</li>\n",
       "  <li><b>Memory: </b>17.13 GB</li>\n",
       "</ul>\n",
       "</td>\n",
       "</tr>\n",
       "</table>"
      ],
      "text/plain": [
       "<Client: 'tcp://127.0.0.1:51227' processes=6 threads=12, memory=17.13 GB>"
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "client = Client(n_workers=6, threads_per_worker=2)\n",
    "\n",
    "client"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-05T08:25:39.666511Z",
     "start_time": "2020-11-05T08:25:38.849450Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Logged in. Welcome cesar-cabeza. To view list of available collections, call list_collections() method.\n"
     ]
    }
   ],
   "source": [
    "username = 'cesar-cabeza'\n",
    "api_key = 'amdXU0VYVUJXN1RRYmVKWGUwcG46Ni1zRW15R01SNFNmLUZ4Ry00MHFmZw'\n",
    "\n",
    "vi_client = ViClient(username=username, api_key = api_key)\n",
    "text_encoder = ViText2Vec(username, api_key)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-05T08:25:58.091091Z",
     "start_time": "2020-11-05T08:25:57.671088Z"
    }
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'status': 'error', 'message': 'covid-19 does not exist'}"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "vi_client.delete_collection(\"covid-19\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-05T08:25:59.169201Z",
     "start_time": "2020-11-05T08:25:59.166201Z"
    }
   },
   "outputs": [],
   "source": [
    "path_test = \"B:/document_parser/document_parses/test\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-05T08:30:44.290959Z",
     "start_time": "2020-11-05T08:30:44.287961Z"
    }
   },
   "outputs": [],
   "source": [
    "def load_corpus(files_path, client, text_encoder):\n",
    "    files = glob.glob(files_path + \"/*.json\")\n",
    "    stopset = set(stopwords.words(\"english\"))\n",
    "    stemmer = PorterStemmer()\n",
    "    b = db.from_sequence(files).map(preprocess_json, stopset, stemmer)\n",
    "    client.insert_documents(\"covid-19\", b.compute(), models={\n",
    "                            \"title\": text_encoder.encode, \"abstract\": text_encoder.encode}, workers = 12, chunksize = 20)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-05T08:30:44.304959Z",
     "start_time": "2020-11-05T08:30:44.300960Z"
    }
   },
   "outputs": [],
   "source": [
    "def preprocess_json(file_path, stopset, stemmer):\n",
    "    with open(file_path) as file:\n",
    "        file_json = json.load(file)\n",
    "    returned = preprocess_document(file_json, stopset, stemmer)\n",
    "    return returned"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-05T08:30:44.323961Z",
     "start_time": "2020-11-05T08:30:44.315959Z"
    }
   },
   "outputs": [],
   "source": [
    "def preprocess_document(file, stopset, stemmer):\n",
    "    returned = {}\n",
    "    returned[\"_id\"] = file[\"paper_id\"]\n",
    "    title = file[\"metadata\"][\"title\"]\n",
    "    abstract = \" \".join([paragraph[\"text\"] for paragraph in file[\"abstract\"]])\n",
    "    body = \" \".join([paragraph[\"text\"] for paragraph in file[\"body_text\"]])\n",
    "    returned[\"title\"] = title\n",
    "    returned[\"abstract\"] = preprocess_document_part(abstract, stopset, stemmer)\n",
    "    returned[\"body\"] = preprocess_document_part(body, stopset, stemmer)\n",
    "    return returned"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-05T08:30:44.340959Z",
     "start_time": "2020-11-05T08:30:44.335960Z"
    }
   },
   "outputs": [],
   "source": [
    "def preprocess_document_part(part, stopset, stemmer):\n",
    "    if part != \"\":  # actually has content\n",
    "        tokens = [word.lower() for word in wordpunct_tokenize(\n",
    "            part) if word.lower() not in stopset and len(word) > 2]\n",
    "        stems = [stemmer.stem(token) for token in tokens]\n",
    "        return \" \".join(stems)\n",
    "    return \"no content\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-05T08:32:58.197915Z",
     "start_time": "2020-11-05T08:30:46.080323Z"
    }
   },
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "dede34a15aab4ebdb0c0ff792a54d496",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "HBox(children=(FloatProgress(value=0.0, max=27.0), HTML(value='')))"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n"
     ]
    }
   ],
   "source": [
    "load_corpus(path_test, vi_client, text_encoder)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-05T08:33:25.480814Z",
     "start_time": "2020-11-05T08:33:24.983721Z"
    }
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'abstract': 'text',\n",
       " 'abstract_vector_': 'vector',\n",
       " 'body': 'text',\n",
       " 'insert_date_': 'date',\n",
       " 'title': 'text',\n",
       " 'title_vector_': 'vector'}"
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "vi_client.collection_schema(\"covid-19\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-05T08:34:36.559402Z",
     "start_time": "2020-11-05T08:34:35.416287Z"
    }
   },
   "outputs": [],
   "source": [
    "advanced_search_query = {\n",
    "    'text' : {'vector': text_encoder.encode('what is the origin coronavirus'), 'fields' : ['abstract_vector_', \"title_vector_\"]}\n",
    "}\n",
    "results = vi_client.advanced_search('covid-19', advanced_search_query, page_size=10)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-05T08:34:37.471680Z",
     "start_time": "2020-11-05T08:34:37.455684Z"
    }
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'results': [{'_id': 'a3116a1ea2e157c8855023d968cc0f0bd6c9846f',\n",
       "   'insert_date_': '2020-11-05T08:31:41.743675',\n",
       "   'abstract': 'concern new coronaviru 2019 ncov global public health threat articl provid preliminari evolutionari molecular epidemiolog analysi new viru phylogenet tree built use avail whole genom sequenc 2019 ncov whole genom sequenc highli similar sequenc avail gene bank sar mer bat sar like coronaviru fubar analysi show nucleocapsid spike glycoprotein site posit pressur homolog model help explain molecular structur differ virus phylogenet tree show 2019 ncov significantli cluster bat sar like coronaviru sequenc isol 2015 wherea structur analysi reveal mutat nucleocapsid protein result 2019ncov could consid coronaviru distinct sar viru probabl transmit bat anoth host mutat confer upon abil infect human author funder right reserv reus allow without permiss',\n",
       "   'title': 'The 2019-new coronavirus epidemic: evidence for virus evolution',\n",
       "   'body': 'famili coronavirida compris larg singl plu strand rna virus isol sever speci previous known caus common cold diarrheal ill human previous unknown sever acut respiratori syndrom sar manual edit use bioedit program maximum likelihood method employ analys allow test author funder right reserv reus allow without permiss copyright holder preprint peer review http :// doi org 1101 2020 915157 doi biorxiv preprint differ phylogenet hypothes calcul probabl given model evolut gener observ data compar probabl nest model likelihood ratio test best fit nucleotid substitut model chosen jmodeltest softwar tree reconstruct use generalis time revers gtr plu gamma distribut invari site evolutionari model use mega adapt evolut server http :// www datamonkey org use find eventu site posit neg select purpos follow test use fast unconstrain bayesian approxim fubar test allow infer site specif pervas select episod diversifi select across region interest identifi episod select individu site statist signific posit neg select base valu homolog model built reli websit swissmodel structur templat search valid use softwar avail within swissmodel environ hhpred homolog model valid use qmean tool threedimension structur analyz display use pymol select pressur structur analysi mutat surfac protein spike protein nucleocapsid protein confer stabil viral particl shown viral spike protein respons viru entri cell bind cell receptor membran fusion two key role viral infect pathogenesi protein structur protein involv virion assembl play pivot role viru transcript assembl effici mutat protein could determin two import characterist coronaviru isol 2019ncov epidem higher stabil bat like sar coronaviru lower pathogen sar coronaviru featur explain author funder right reserv reus allow without permiss copyright holder preprint peer review http :// doi org 1101 2020 915157 doi biorxiv preprint 2019 ncov zoonot transmiss initi lower sever sar epidem result exclud mutat due posit select pressur lead epidem evolut could favor ehanc pathogen transmiss novel viru recent describ homolog recombin within spike glycoprotein 2019ncov favor cross speci transmiss suggest snake probabl viru reservoir human infect resampl similar codon usag rscu bia similar bungaru multicinctu snake compar anim human previou articl proven composit properti mutat pressur natur select gene express dinucleotid affect codon usag bia bungaru speci data along data studi enforc bat origin infect could clarifi transmiss dynam 2019 ncov support infect control polici ongo epidem author funder right reserv reus allow without permiss copyright holder preprint peer review http :// doi org 1101 2020 915157 doi biorxiv preprint author funder right reserv reus allow without permiss copyright holder preprint peer review http :// doi org 1101 2020 915157 doi biorxiv preprint',\n",
       "   '_search_score': 0.8915696},\n",
       "  {'_id': 'a05179c57d5e1cdddc839ff3c0b70a614efbb6cb',\n",
       "   'insert_date_': '2020-11-05T08:31:44.405486',\n",
       "   'abstract': 'recent outbreak pneumonia wuhan china caus 2019 novel coronaviru 2019 ncov emphas import detect novel virus predict risk infect peopl report introduc vhp viru host predict predict potenti host virus use deep learn algorithm predict suggest 2019 ncov close infect human coronavirus especi sever acut respiratori syndrom coronaviru sar cov bat sar like coronavirus middl east respiratori syndrom coronaviru mer cov base predict compar coronavirus infect vertebr bat coronavirus assign similar infect pattern 2019 ncov furthermor compar infect pattern virus host vertebr found mink virus show closer infect pattern 2019 ncov consequ infect pattern analysi illustr bat mink may two candid reservoir 2019 ncov result warn bewar 2019 ncov guid explor properti reservoir one sentenc summari great valu identifi whether newli discov viru risk infect human guo propos viru host predict method base deep learn detect kind host viru infect dna sequenc input',\n",
       "   'title': 'Host and infectivity prediction of Wuhan 2019 novel coronavirus using deep learning algorithm',\n",
       "   'body': 'coronavirus strong potenti infect human method help futur viral analysi earli prevent control viral pathogen recent outbreak pneumonia wuhan china brought 2019 novel coronaviru current urgent need identifi host potenti infect 2019 ncov virus larg amount infecti virus obtain domest anim bird vector anim environ virus may pose great threat human sinc lot virus safe anim may potenti infect human addit owe high mutat rate virus also non human infecti virus origin spread anim mutat infect human later besid unexpect host novel virus also concern detect potenti host pathogen new virus convent way base sequenc similar known virus either build phylogenet tree blast howev method seem rudimentari sever limit fact owe intrins heterogen within genom differ sequenc fragment novel viru may assign quit differ known virus ident similar local align consequ lead ambigu predict host novel virus addit routin method phylogenet tree analysi alway util whole genom assembl shotgun read intern licens author funder made avail copyright holder preprint peer review http :// doi org 1101 2020 914044 doi biorxiv preprint frequent influenc sequenc assembl mistak moreov potenti host novel virus predict phylogenet perspect peopl usual accept assumpt virus closer phylogenet relationship common host pathogen infect behavior howev assumpt prove unsustain abil infect human vari even within genu virus harmless human exampl betacoronaviru genu coronavirina contain virus infect human sar virus report infect human rousettu bat coronaviru hku9 tabl tabl besid report human infect coronavirus assign lowest valu predict vhp method tabl similar probabl 2019 ncov human coronavirus illustr high risk 2019 ncov part valu result shown tabl rank decreas order valu viru infect human predict result warn 2019 ncov strong potenti infect human similar sar bat sar like coronavirus mer sar assign slightli lower valu human bat sar like coronavirus given closer valu 2019 ncov indic 2019 ncov probabl less infecti within human sar infecti bat sar like coronavirus furthermor explor reservoir 2019 ncov infect pattern shown tabl compar valu host type intern licens author funder made avail copyright holder preprint peer review http :// doi org 1101 2020 914044 doi biorxiv preprint explor differ 2019 ncov sar sar cov like coronaviru analyz core gene aa ident nonsynonym snp singl nucleotid polymorph tabl copyright holder preprint peer review http :// doi org 1101 2020 914044 doi biorxiv preprint one origin besid compar two bat sar like coronavirus 2019 ncov 130 220 nonsynonym snp three core gene respect show differ among 2019 ncov sar cov like coronavirus except 2019 ncov predict coronavirus indic night heron coronaviru hku19 report infect vertebr night heron also strong potenti infect human coronavirus larg famili virus usual circul among anim caus fatal diseas mutat evolv abl infect human copyright holder preprint peer review http :// doi org 1101 2020 914044 doi biorxiv preprint genbank order develop method expert predict potenti host type novel virus viral sequenc data releas 2018 use build train set releas 2018 util test integr host virus five type includ plant germ invertebr vertebr human host subtyp contain five type list tabl medic colleg genom sequenc share also acknowledg gisaid intern licens author funder made avail copyright holder preprint peer review http :// doi org 1101 2020 914044 doi biorxiv preprint',\n",
       "   '_search_score': 0.8064966},\n",
       "  {'_id': 'a09090caa2498f6ff886f0fd7b10a7e45818751c',\n",
       "   'insert_date_': '2020-11-05T08:31:42.005113',\n",
       "   'abstract': 'controversi remain whether coronaviru diseas 2019 covid viru may infect incub period onset symptom author opportun examin infect covid incub period conduct epidemiolog survey confirm patient visit jeju island incub period epidemiolog find support claim covid viru infect incub period open access articl distribut term creativ common attribut non commerci licens http :// creativecommon org licens bync permit unrestrict non commerci use distribut reproduct medium provid origin work properli cite',\n",
       "   'title': 'A Chinese Case of Coronavirus Disease 2019 (COVID-19) Did Not Show Infectivity During the Incubation Period: Based on an Epidemiological Survey',\n",
       "   'body': 'world health organ declar outbreak novel coronaviru 2019 2019 ncov januari 2020 arriv chines tourist domest travel conjunct chines new year day korean new year day holiday case confirm korea februari 2020 jeju island popular tourist destin risk influx coronaviru diseas 2019 covid commun spread higher region korea furthermor still controversi whether viru infecti incub period onset symptom chines author report januari 2020 covid infecti incub period addit news stori januari 2020 research paper publish januari 2020 report con pissn 1975 8375 eissn 2233 4521 firm patient germani contact asymptomat woman china two day onset symptom seem evid support infect incub covid infect incub period seriou concern effect neuter current communic diseas control system start time infect confirm case yet defin fortun author opportun observ infect covid incub period conduct epidemiolog survey confirm patient visit jeju island incub period year old chines woman live wuhan china visit jeju island daughter yangzhou china januari 2020 tour jeju island 22nd 24th return yangzhou 25th woman fever start 26th day return home diagnos 30th confirm infect inform commun jeju provinci offic februari 2020 afternoon via fax airlin travel due confirm diagnosi china onset symptom return home subject epidemiolog investig base guidelin center diseas control prevent korea kcdc howev basi german case jeju provinci govern decid seek subject face face convers patient 24th 25th two day day symptom onset take measur quarantin day maxim incub period use social network servic patient daughter china check close circuit televis cctv imag instal hotel throughout jeju island patient travel rout person came close contact confirm base author identifi eleven person talk directli patient longer second recommend quarantin none show symptom cov infect midnight februari 2020 last contact releas quarantin quarantin period suggest februari 2020 chines woman contact first german patient alreadi develop symptom take antipyret analges stay germani inform contrari claim cov infect incub period addit accord testimoni doctor korea clinic progress mild patient condit stabl possibl onset express sudden start slowli manifest date point patient may later actual light possibl explain confus claim made chines author accordingli would reason conduct epidemiolog investig one day onset confirm patient 5th edit kcdc effect begin februari 2020 convers symptom mild actual onset chines case may earlier januari 2020 howev author conclud symptom manifest stay jeju island follow fact first third parti hotel front desk staff tour guid pharmacist face face contact patient state exhibit ill symptom dri cough second cctv video walk long time appear healthi tourist without diseas epidemiolog find support claim infect incub period howev uncertainti possibl transmiss latent infect asymptomat patient remain epidemiolog spectrum covid yet accur identifi paper perspect need ethic consider',\n",
       "   '_search_score': 0.7831983999999999},\n",
       "  {'_id': 'a18236e9c75536c022502b9a9d0efdc6b0a3f0c2',\n",
       "   'insert_date_': '2020-11-05T08:31:43.655076',\n",
       "   'abstract': 'ongo global pandem infect diseas covid caus 2019 novel coronaviru sar cov formerli 2019 ncov present critic threat public health economi sinc identifi china decemb 2019 genom sar cov sequenc structur annot yet littl known intrins organ evolut genom end present mathemat method genom spectrum kind barcod sar cov common human coronavirus genom spectrum construct accord period distribut nucleotid therefor reflect uniqu characterist genom result demonstr coronaviru sar cov exhibit dinucleotid island non structur protein analysi dinucleotid region suggest dinucleotid repeat increas evolut may confer evolutionari fit viru special dinucleotid region sar cov genom identifi studi may becom diagnost pharmaceut target monitor cure covid diseas',\n",
       "   'title': 'Dinucleotide repeats in coronavirus SARS-CoV-2 genome: evolutionary implications',\n",
       "   'body': 'keyword covid sar cov genom spectrum dinucleotid repeat evolut fit virul introduct current global pandem covid caus novel coronaviru sar cov formerli 2019 ncov sever threat public health economi sinc emerg wuhan china decemb 2019 may 2020 million covid case globe confirm 357 688 death occur covid diseas 2020 sar cov aetiolog agent respons larg scale outbreak fatal diseas understand notabl featur sar cov genom zoonot origin evolutionari trend import reveal intervent target diseas control prevent coronavirus cov largest group envelop posit sens singl strand rna virus taxonom coronaviru sar cov belong nidoviral order coronavirida famili coronavirina subfamili beta cov genu fehr perlman 2015 sar cov highli pathogen similar lower pathogen sever acut respiratori syndrom sar coronaviru sar cov 2002 2003 lower pathogen middl east respiratori syndrom coronaviru mer cov 2012 nevertheless sar cov highli transmiss human date seven human cov hcov identifi among alpha cov hcov 229e hcov nl63 remain five beta cov includ hcov oc43 hcov hku1 sar cov mer cov sar cov four common human cov hcov 229e hcov oc43 hcov nl63 hcov hku1 usual caus mild symptom like common cold diarrhea 2016 contrast sar cov mer cov current sar cov particularli pathogen caus sar covid respect resembl sar cov sar like coronavirus bat like act natur reservoir host progenitor sar cov andersen 2020 establish sar cov transmit palm civet human wang 2006 mer cov dromedari camel human yet zoonot origin sar cov still unclear genom sequenc analysi show sar cov closest rel sar like slcov ratg13 found horsesho bat rhinolophu affini yunnan china zhou 2020 sar cov overal genom sequenc ident sar like coronaviru slcov ratg13 zhou 2020 therefor natur reservoir sar cov could horsesho bat recent evid suggest pangolin host candid lam 2020 howev host pangolin firmli determin bat sar like cov infect human exampl bat coronaviru swine acut diarrhoea syndrom coronaviru sad cov found 2017 zhou 2018 caus million piglet death human case may postul sar cov evolv progenitor mutat evolutionari fit host shift adapt sar cov fast evolv pathogen continu undertak mutat gener infect host yin 2020 wang 2020 korber 2020 fact suggest sar cov evolv mutat critic protein genom structur prior establish human infect understand sar cov jump anim adapt infect human import surveil viru evolut divers therefor ultim control covid pandem prevent futur sar like outbreak sar cov envelop posit strand rna viru except long 9kb genom zhou 2020 genom consist leader cap sequenc along poli tail gene encod non structur protein nsp structur protein well sever accessori protein approxim two third genom compris two larg overlap open read frame orf1a orf1ab encod polyprotein subsequ cleav viral proteas gener non structur protein nsp1 nsp16 non structur protein essenti rna replic transcript immun evas nsp3 larg multi domain multi function protein play essenti role viru replic papain like proteas plpro activ nsp3 respons initi process ofr1a protein addit nsp3 togeth nsp4 nsp6 recruit intracellular membran form doubl membran vesicl dvm support viral rna replic nsp5 second viral proteas like proteas 3clpro split orf1a orf1ab protein downstream region genom encod structur protein spike protein nucleocapsid protein envelop protein membran protein four structur protein requir produc structur complet viral particl protein mediat viral attach ace2 receptor host subsequ fusion viral host cell membran enabl viru enter host cell nucleocapsid protein one abund viral protein bind rna genom particip process replic assembl host cell respons viral infect mcbride 2014 sar cov genom sequenc annot littl known complex structur genom light evolutionari fit host infect convent rna virus use encod protein interact compon cellular respons howev numer discoveri show viru rna structur determin genom composit may also play central role maxim viru replic evolutionari fit jensen thomsen 2012 exampl recent studi show increas dinucleotid code noncod region echoviru inhibit replic initi post entri sever cell line fro 2017 therefor rna virus simul host mrna composit exampl dinucleotid frequenc fro 2017 anim genom bia dinucleotid composit heavi represent dinucleotid especi well known anim rna small dna virus suppress genom dinucleotid frequenc appar mimick host mrna composit giallonardo 2017 viru rna composit structur differ host mrna rig retino acid induc gene like receptor may detect rna molecul absent uninfect host goubau 2013 detect evolutionari microbi structur known pathogen associ molecular pattern pamp import featur innat immun system host cell possess intrins defens pathway prevent replic virus increas frequenc mechan independ codon usag belalov lukashev 2013 genom spectrum demonstr dinucleotid trinucleotid multi nucleotid distribut dinucleotid distribut often consid signatur genom kariin burg 1995 period pattern distinguish characterist protein code region tsoni 1991 strength period period genom determin perfect level copi number dinucleotid trinucleotid repeat respect period period essenti characterist genom studi examin two period genom spectrum analyz genom studi present genom spectrum sar cov identifi dinucleotid repeat orf1a nsp instrument interact host immun system pathogen host infect dinucleotid repeat essenti surviv infect microb consid one pamp sar cov genom element evid evolutionari fit sar cov host shift adapt track evolut element may provid insight zoonot origin sar cov control covid diseas inspect insight trait sar cov genom util period analysi method survey nucleotid distribut render period genom previous propos period analysi method quantit detect nucleotid repeat period genom yin 2017 method employ nucleotid distribut period posit genom identifi approxim repeat structur signatur genom includ function smooth period profil origin method describ method detail though technic algorithm delin previous yin 2017 comput program period analysi genom avail public github repositori http :// github com cyinbox dnadu nucleotid distribut period posit dna sequenc repres congruenc deriv vector yin wang 2016 yin 2017 vector nucleotid specif period construct cumul frequenc nucleotid period posit definit definit dna sequenc length let nucleotid appear posit otherwis congruenc deriv vector nucleotid dna sequenc period defin mod modulo oper return remaind divis )). four congruenc deriv vector period nucleotid form congruenc deriv matrix size column matrix indic nucleotid frequenc period posit posit index dna sequenc exampl consid matrix period dna sequenc first column matrix show nucleotid frequenc period posit second column matrix show nucleotid frequenc period posit third column matrix show nucleotid distribut period posit matrix dna sequenc describ nucleotid frequenc period posit use effici comput fourier power spectrum determin period dna sequenc yin wang 2016 therefor vector reflect arrang repetit sequenc element inner period dna sequenc sinc matrix contain nucleotid frequenc period posit varianc matrix element measur nucleotid distribut matrix period summat element matrix equal length dna sequenc mean element matrix therefor quantifi nucleotid distribut defin normal distribut uniform ndu dna sequenc use matrix definit definit dna sequenc length let element matrix period normal distribut uniform period dna sequenc defin definit notic normal distribut uniform period intuit descript level unbal nucleotid frequenc period posit depend quadrat function nucleotid frequenc sequenc period length ndu use indic exist period dna sequenc method offer elabor repetit element repeat consensu copi number perfect level yin 2017 use slide window along genom period rang calcul window segment therefor two dimens period spectrum form genom two dimens spectrum consid genom signatur barcod signatur genom locat posit repeat region genom smooth filter correspond slide window period move averag convolut jong 1989 peak period profil detect use score algorithm brakel 2020 peak posit use demarc repeat genom nutshel comput distribut uniform differ period dna sequenc first scan sequenc differ period size construct congruenc deriv matrix period comput ndu period period maximum distribut uniform reflect predomin pattern repetit element ndu valu period indic perfect level copi number correspond repeat region studi depend complet genom coronavirus includ sar cov 2020 sever acut respiratori syndrom sar relat coronaviru middl east respiratori syndrom coronaviru mer cov human infect coronavirus human cov genom data retriev nation center biotechnolog inform ncbi gene bank bat slcov ratg13 complet sequenc report zhou 2020 download gisaid repositori http :// www gisaid org shu mccauley 2017 genom data studi list supplementari materi result analysi genom spectrum coronaviru sar cov reveal rich dinucleotid pattern nsp identifi signatur featur coronaviru sar cov genom employ period spectrum analysi identifi characterist period genom creat genom spectrum barcod sar cov fig use slide window ndu method compar counterpart slcov ratg13 fig sar cov tor2 fig mer cov fig spectrum comparison observ sar cov slcov ratg13 pronounc period four region sar cov tor2 mer cov extrem low level period correspond region strong dinucleotid signal sar cov encourag investig caus detail locat region rich dinucleotid repeat verifi period period strong signal among genom period fig detect peak slidingwindow period profil fig peak posit use demarc dinucleotid repeat region genom dinucleotid repeat region dinucleotid island orf1a correspond gene list tabl genom spectrum analysi rel abund dinucleotid repeat particularli dinucleotid map gene orf1a nsp3 nsp4 nsp6 fig tabl .). howev dinucleotid repeat signal weak impercept correspond region sar cov mer cov genom coronaviru sar cov rna genom gene replicas encod two overlap polyprotein orf1a replicas orf1ab replicas 1ab genom structur correspond dinucleotid region identifi illustr fig two polyprotein respons viral replic transcript express proxim portion pp1ab requir ribosom frame shift first dinucleotid repeat code region papain like proteinas proteinas non structur protein nps3 nsp3 largest essenti compon replic transcript complex proteinas nsp3 cleav nsp block host innat immun respons promot cytokin express lei 2018 serrano 2009 second dinucleotid repeat code region non structur protein nsp4 nsp4 respons form doubl membran vesicl dmv third dinucleotid repeat code region termin 3clpro proteas chymotrypsin like proteinas 3clpro nsp6 3clpro proteas essenti rna replic 3clpro proteinas respons process terminu nsp4 nsp16 coronavirus anand 2003 therefor conserv structur catalyt site 3clpro may serv attract target antivir drug kim 2012 togeth nsp3 nsp4 nsp6 induc dmv angelini 2013 summari dinucleotid repeat island found studi locat host interact region genom sar cov special dinucleotid repeat region orf1a like contribut adapt immun respons therefor impli evolut fit coincident previou work mer cov use evolut analysi reveal nsp3 repres preferenti select target adapt evolut zoonot mer cov new host forni 2016 find nsp3 involv evolut fit consist discoveri mer cov investig correl dinucleotid content sar cov genom virul examin dinucleotid genom region increas period genom region sar cov sar like cov result unbalanc distribut dinucleotid understand evolutionari tendenc coronaviru genom examin genom spectra four major bat sar like coronavirus slcov pangolin slcov slcov zxc21 slcov wiv1 slcov shaanxi2011 natur live bat rhinolophida horsesho pangolin slcov found similar sar cov pangolin exploratorili postul intermedi anim host sar cov (). slcov zxc21 second similar strain sar cov similar 2018 slcov wiv1 close relat sar cov tor2 term genom ident ace2 bind human cell 2013 slcov shaanxi2011 found 2011 result show pangolin slcov display three major dinucleotid repeat nsp3 nsp4 lack correspond dinucleotid repeat 3clpro nsp6 found sar cov fig sar cov mostli close slcov ratg13 fig slcov zxc21 fig pangolin slcov pangolin intermediari host sar cov sar cov inde evolv pangolin slcov may infer pangolin slcov would need evolv dinucleotid repeat 3clpro nsp6 evolut fit infect human host also observ evolut trend slcov wiv1 fig slcov shaanxi2011 fig genom spectrum slcov shaanxi2011 similar slcov wiv1 addit increas dinucleotid repeat 3clpro nsp6 new dinucleotid repeat region 3clpro nsp6 slcov shaanxi2011 consist region found sar cov slcov ratg13 slcov zxc21 therefor dinucleotid repeat region 3clpro nsp6 probabl play import role evolutionari fit sar cov human host result show sar cov sar like cov low dinucleotid repeat low dinucleotid content consid earli evolut fit interact human immun system low dinucleotid may render high viru virul viru adapt host immun system host immun system act intens investig correl dinucleotid repeat pathogen coronavirus produc compar genom spectra four common human coronavirus fig spectrum analysi observ higher period hcov 229e hcov oc43 fig dinucleotid repeat region correspond three sub region sar cov high 2period valu may attenu viru replic therefor reduc sever virul agreement correl dinucleotid repeat pathogen previous spectra hcov nl63 hcov hku1 demonstr extrem high period well period correspond region fig hcov nl63 hcov hku1 common human cov caus mild cold symptom symptom pyrc 2007 high dinucleotid repeat may contribut light pathogen two virus wish know whether sar cov origin evolv sar cov sar cov would evolv sar cov answer question may help predict evolut sar cov viru better diseas predict control genom sar relat coronavirus long time period rare avail infer evolut coronavirus track trend hcov 229e coronavirus last six decad first human infect hcov 229e identifi 1962 thiel 2001 hcov 229e viru caus common cold occasion associ sever respiratori infect children elderli person underli ill use measur dinucleotid genom region trend hcov 229 coronavirus may infer evolutionari stage bat coronavirus similar method may determin origin sar cov trend sar cov compar sar like coronavirus human coronaviru hcov 229e strain use dinucleotid trend analysi differ histor period refer genom hcov 229 evolutionari analysi obtain infecti hcov 229e 1973 deposit laboratori adapt prototyp strain hcov 229e 740 hcov 229e prototyp strain origin isol 1962 patient chicago first clinic hcov 229e isol patient 2012 includ studi farsani 2012 result period trend hcov 229e coronavirus demonstr 2period fig period fig coronavirus increas evolutionari time sar cov rel high period period result suggest trend period period increas time evolutionari origin coronavirus infer trend period period therefor may compar period period sar cov sar like coronavirus understand evolutionari origin sar cov investig correl dinucleotid repeat viru virul sar relat coronavirus compar spectra genom region coordin 12k five sar relat coronavirus genom region contain abund dinucleotid repeat region 12k genom contain three dinucleotid repeat approxim locat 10k 10k 12k sub region five sar relat coronavirus differ level virul virul viru mer cov follow sar cov period magnitud reflect distribut dinucleotid compar shown fig may see mer cov genom region lowest dinucleotid level three dinucleotid sub region sar cov also low dinucleotid level higher mer cov therefor may infer low dinucleotid level correl high virul lower dinucleotid level higher viru virul postul support observ dinucleotid three sar relat coronavirus compar sar cov sar like bat slov rp3 similar dinucleotid distribut sub region 10k 12k higher dinucleotid distribut sub region 10k slcov rp3 lower viru virul sar sar like slcov zxc21 share highest sequenc ident sar cov show slightli lower dinucleotid distribut two sub region slightli higher dinucleotid distribut sub region notic trend period period mer cov sar cov sarslik cov sar cov increas fig base previou analysi period trend coronavirus differ time may infer sar cov origin sar like cov well studi dinucleotid composit bia rna virus may impact viru replic specif attenu strengthen viru virul evolut fro 2017 2019 clinic evid suggest sar cov lower virul sar cov investig increas dinucleotid repeat correl viru virul compar dinucleotid frequenc three dinucleotid rich island sar cov sar cov tor2 result show sar cov sar cov abund dinucleotid whole genom three promin dinucleotid repeat region fig extrem defici sub region fig dinucleotid increas sar cov compar sar cov fig role increas dinucleotid sar cov genom possibl attenu viru replic one mechan attenu viru dinucleotid bia dinucleotid region fold special structur target cell rna cleavag fundament host respons control viral infect zhou 1993 oligoadenyl synthetas rnase system innat immun pathway respond pathogen associ molecular pattern pamp induc degrad viral cellular rna therebi block viral infect higher vertebr process often regul interferon ifn ribonucleas rnase latent interferon ifn induc antivir ribonucleas upon activ destroy rna within tabl cellular viral silverman 2007 rnase cleav hepat viru hcv rna singl strand dinucleotid throughout open read frame orf interest discoveri bacterium mycoplasma pneumonia respiratori infect agent genom also rel abund extrem dinucleotid karlin 1998 therefor may postul dinucleotid region sar cov possibl cleav rnase infect studi identifi uniqu dinucleotid repeat sar cov genom dinucleotid repeat genom spectra reveal period analysi discov strength repeat correl evolutionari fit viru human host maxim surviv epidem instead destroy host therefor rna virus simul host mrna composit dinucleotid composit fro 2017 vertebr rna small dna virus suppress genom dinucleotid frequenc appar mimick host mrna composit abund dinucleotid like common pathogen island microbi genom studi sar cov provid addit evid increas dinucleotid sar cov result interact host viru evolut consid three dinucleotid abund region pathogen island sar cov genom addit special region may contribut rna replic recogn cell rnaas rna degrad immun respons howev studi theoret analysi genom actual function consequ impact transmiss pathogenesi dinucleotid repeat determin biochem experi anim model human mammal apobec apolipoprotein mrna edit enzym catalyt polypeptid like3 system help protect organ viral infect origin dinucleotid repeat sar cov genom specul molecular mechan increas dinucleotid repeat evolut fit possibl apobec3 mediat edit viral rna cytosin often mutat uracil deamin bishop 2004 therefor mani dinucleotid repeat could gener period island rna genom apobec defens system cross speci transmiss coronavirus wildlif reservoir may lead diseas outbreak human pose sever threat human health date studi zoonot origin sar cov primarili focu spike protein essenti entri viru particl cell mutat acquisit potenti cleavag site furin proteas spike protein may confer zoonot transmiss sar cov andersen 2020 hoffmann 2020 howev spike protein requir suffici zoonot coronaviru transmiss instanc spike protein sar like shc014 cov enabl chimer viru shc014 ma15 infect human cell spike protein integr wild type sar cov backbon menacheri 2015 sar cov backbon contain orf1ab replic compon studi provid evid import orf1ab replic region evolutionari fit consequ orf1ab replic compon critic zoonot transmiss studi genom spectrum sar cov reveal high dinucleotid region orf1a might contribut evolut fit host immun evas accordingli monitor sar cov develop antivir drug envisag molecular characterist chang nsp vital compon interact human host special dinucleotid repeat region investig detail function phenotyp chang sar cov importantli dinucleotid repeat region possibl prophylact therapeut target control covid',\n",
       "   '_search_score': 0.7324622000000001},\n",
       "  {'_id': 'a03517f26664be79239bcdf3dbb0966913206a86',\n",
       "   'insert_date_': '2020-11-05T08:31:44.398003',\n",
       "   'abstract': 'background bat born viru surveil necessari determin inter speci transmiss risk import due wide rang bat speci may harbour potenti pathogen studi aim monitor coronavirus cov paramyxovirus pmv bat roost northwest italian region investig focus cov pmv due proven abil switch host zoonot potenti provid phylogenet character highli conserv polymeras gene fragment result famili wide pcr screen use test 302 bat belong differ bat speci thirti eight anim locat confirm pcr posit overal detect rate cov rna found bat belong eight speci pmv rna three pipistrellu spp phylogenet character obtain alpha cov beta cov three pmv moreov one pipistrellu result infect cov pmv diverg alpha cov clade myoti nattereri spa also describ compact cluster beta cov ferrumequinum roost expand current viral sequenc databas specif speci europ knowledg first report cov plecotu auritu oxygnathu pmv kuhlii conclus studi identifi alpha beta cov new bat speci previous unsurvey italian region knowledg repres first uniqu report pmv itali new bat genet sequenc present expand current molecular bat born viru databas consid amount novel bat born pmv associ emerg zoonot infect anim human last year definit viral divers within european bat speci need perform surveil studi within specif geograph area provid awar viral burden bat roost close proxim spillov host form basi appropri control measur potenti threat public health optim manag bat habitat',\n",
       "   'title': 'Coronavirus and paramyxovirus in bats from Northwest Italy',\n",
       "   'body': 'bat order chiroptera repres least one fifth exist mammal consist 1300 known speci least present europ itali speci divers expect increas taxa myoti nattereri complex process defin cryptic speci use molecular approach rather use morpholog characterist bat group two subord fruit eat megabat megachiroptera fli fox consist singl famili pteropodida echoloc insectivor microbat microchiroptera compris bat famili bat born virus arous increas interest sinc viral infect bat associ zoonot diseas outbreak human domest anim includ livestock rabi viru hendra nipah virus sever acut respiratori syndrom sar middl east respiratori syndrom mer coronavirus well filovirus exemplifi role bat spread virus last fifteen year least two widespread outbreak caus novel coronavirus jump speci barrier sar 2002 2003 mer start arabian peninsula sinc 2012 genet similar viral sequenc detect outbreak cov sequenc bat suggest virus origin fli mammal presum pass human previou adapt intermedi host civet cat dromedari coronavirus famili coronavirida subfamili coronavirina divid four main genera alphacoronaviru alpha cov betacoronaviru beta cov found mainli mammal gammacoronaviru detect bird marin mammal deltacoronaviru found mainli bird sever alpha beta cov describ worldwid differ bat speci first report china rhinolophu speci specif associ sar like cov belong lineag beta cov genu investig need clarifi origin mammalian coronavirus assum viral ancestor resid bat until recent discoveri new highli diverg cov wesv hous shrew china 2010 circul cov italian bat popul notifi publish studi sar like beta cov identifi rhinolophu speci cov sequenc avail italian pipistrellu kuhlii hypsugo savii nyctalu noctula epseticu serotinu myoti blythii hipposidero speci fecal sampl despit rapid accumul bat cov sequenc last decad viral isol trial differ mammalian bat cell line fail till 2013 first isol sarslik cov bat fecal sampl succeed china list emerg zoonos broad divers bat born paramyxovirus pmv belong wide paramyxovirida famili emerg nipah viru hendra viru henipavirus rubulavirus menangl viru tioman viru tuhoko viru refer therein detect isol paramyxovirus tissu urin obtain mainli fli fox genu pteropu africa asia south america australia also microbat speci previous indic pmv reservoir test posit pmv rna africa europ moreov ever increas attent paid bat associ pathogen led discoveri numer novel yet unclassifi pmv reveal unexpect genet divers paramyxovirina subfamili pmv identif report studi insectivor bat europ germani bulgaria romania luxembourg none novel virus close relat highli human pathogen paramyxovirus follow increas need surveil bat born virus wide rang bat speci potenti repres reservoir known unknown pathogen studi aim estim viral divers distribut bat popul resid northwest itali investig focus coronavirus paramyxovirus due prove abil switch host zoonot potenti provid phylogenet character viral polymeras gene fragment highli conserv within viral famili investig sinc bat speci europ protect habitat direct european union agreement conserv popul european bat sampl collect bat speci identif perform expert chiropterologist author italian ministri environ author number dpn 2010 0011879 000882 pnm 08052014 bat captur three year surveil 2013 2016 northwestern italian region piedmont liguria follow ethic safeti recommend sampl conduct mid june octob period approxim correspond pregnanc lactat dispers mate activ european bat minim anim disturb bat caught soon parturit nylon mist net mesh size posit reproduct temporari roost along flight path toward forag drink area autumn catch focus particularli swarm site cave individu differ coloni meet mate net check everi min captur bat remov care net soon possibl minim injuri drown strangul stress individu place dispos cloth bag await speci identif collect biometr data biolog sampl speci identif carri accord dietz kiefer individu detail age class sex reproduct statu forearm length bodi mass record saliva urin drop present collect directli anim swab fece recov present cotton bag bat releas place captur minim manipul tag base result first two year surveil increas fece collect perform 2016 set random non invas fece sampl underneath singl speci reproduct roost briefli plastic film left ground differ area reproduct coloni min later singl fresh drop collect clean dispos fork place buffer pepton water kept till analys dead anim good post mortem condit also collect store analys swab fece maintain utm viral transport medium catalog number 360c copan diagnost corona california store analys took place presenc rabi viru antigen investig dead anim direct immunofluoresc stain bsl3 laboratori necropsi rabi infect exclud sampl underw pre treatment submit automat nucleic acid purif magnet bead pre treatment tissu involv prepar tissu pool compos heart lung spleen intestin individu anim pool homogen ratio depc treat pb tissuelys qiagen hilden germani tissu homogen clarifi 000 min 200 tissu pool supernat incub min 180 atl buffer qiagen proteas provid ez1 viru mini kit qiagen hilden germani avoid biosafeti risk pre treatment swab saliva urin fece suspens involv direct inactiv 200 suspens 200 atl buffer bsl3 hood nucleic acid purif rna dna final accomplish ez1 advanc instrument use amount 400 sampl input final elut volum rnase dnase free water follow manufactur guidelin rna store amplif protocol perform cdna synthet rna dna sampl transcriptor first strand cdna synthesi kit roch diagnost mannheim germani accord manifactur instruct coronaviru detect cdna amplifi end point pcr assay target conserv rna depend rna polymeras rdrp gene fragment 537 describ poon amplif set reaction mixtur contain deoxynucleosid triphosph mgcl2 primer platinum taq polymeras invitrogen carlsbad cycl condit min cycl min min min final elong step min anneal temperatur primer modifi upon amplif pcr product run agaros gel electrophoresi visual gelgreen nucleic acid gel stain biotium stain band expect size excis gel sequenc paramyxoviru detect broadli reactiv seminest pcr assay specif rna polymeras gene 538 paramyxovirina subfamili appli cdna amplifi use par primer design tong protocol optim taguchi method kurth briefli first round final concentr reaction mix deoxynucleosid triphosph mgcl2 par par primer platinum taq polymeras invitrogen carlsbad cycl condit min cycl final elong step min first round pcr product use second round concentr except mgcl set use par par primer cycl paramet ident first round pcr product run recov agaros gel describ amplicon purifi gel extract qiaquick gel extract kit qiagen hilden germani accord manufactur instruct elut nucleic acid quantif recov dna done use thermo scientif nanodrop spectrofotomet submit direct sequenc bmr genom padua itali obtain chromatogram manual check unclear base call edit use geneiou softwar geneiou auckland new zealand sequenc align use muscl implement geneiou softwar align use evalu best evolutionari model modeltest ver draw bayesian phylogenet tree mrbay ver consensu tree creat least million heurist search gener elimin first evalu tree topolog burnin %). total genom dna extract perform pcr posit individu start origin swab suspens use qiamp dna mini kit qiagen hilden germani follow manufactur protocol confirm speci identif genet determin complet mitochondri cytochrom gene cytb amplifi puechmail pcr product submit direct sequenc bmr genom padua itali obtain chromatogram manual check unclear base call edit use geneiou softwar geneiou auckland new zealand speci identif conduct compar obtain sequenc line avail refer sequenc blast align ncbi web site tabl anim captur activ surveil show sign diseas necropsi macroscop lesion refer infecti diseas observ examin bat neg rabi viru antigen test cov pmv posit sampl type includ fece 158 urin swab %). none tissu pool dead bat oral swab pcr posit significantli greater percentag femal bat pcr posit male 117 coronaviru paramyxoviru rna found anim belong eight bat speci tabl specif cov rna detect bat sampl site piedmont one liguria pmv rna three anim three sampl site piedmont map show posit site present fig sampl set detect rate cov 302 confid interv rang kuhlii despit repres abund speci sampl ferrumequinum phylogenet analysi perform uniqu sequenc obtain sampl yield pcr product expect size cov pcr screen posit sampl collect nattereri myoti oxygnathu kuhlii pipistrellu auritu ferrumequinum new sequenc identifi submit genbank access number assign given tabl pmv strain detect three differ provinc two kuhlii cn2 al10 site one pipistrellu vc8 site moreov phylogenet analysi base gene fragment possibl three strain retriev studi interestingli one pipistrellu vc8 site coinfect cov pmv pcr posit result obtain urin sampl detail posit sequenc sampl display tabl rdrp phylogeni present fig show cov strain studi cluster alphacoronaviru genu beta coronaviru genu shown fig pmv phylogeni base repres gene sequenc avail genbank present fig new pmv strain detect three differ locat one pipistrellu vc8 site two kuhlii bat cn2 al10 site distanc 1015 pipistrellu strain reveal nucleotid ident e95 pmv strain jn086951 detect bat speci germani 2009 diverg known pmv sequenc two pmv sequenc kuhlii 600 site cn2 621 site al10 similar cluster separ previous known pmv sequenc diverg gene fragment phylogenet tree knowledg kuhlii speci never previous implic paramyxoviru host recent emerg diseas surveil program intensifi investig role bat evolut spillov zoonot pathogen wildlif studi involv three year activ passiv surveil character viral divers northwestern italian bat popul use viral famili wide pcr identifi phylogenet character new cov pmv strain date studi bat cov phylogeni mainli base dataset short sequenc 440 due difficulti obtain isol good qualiti viral rna bat ideal long sequenc fragment would benefici infer reliabl phylogeni high preval posit fecal sampl studi concord studi identifi fece best sampl type cov detect bat rather collect sampl individu caught bat time consum labor intens collect singl fecal drop mono speci roost turn reliabl non invas method virolog surveil bat roost reproduct period moreov urin confirm suitabl appropri sampl type detect paramyxovirus bat popul consid pmv posit sampl studi urin swab 2013 2014 coronaviru circul identifi least four speci specif reproduct roost piedmont to7 site nattereri vc8 site pipistrellu cn2 al10 kuhlii unfortun attempt test roost 2016 fail sinc vc8 coloni move due effect human disturb robberi copper roof cover use refug pipistrellu bat three coloni locat privat build inaccess due logist reason likelihood roost disturb taken account put place bat surveil plan enabl steadi follow coloni time bat social behavior could explain significantli higher infect rate detect studi femal bat sampl august near matern roost previou studi document higher viru detect rate femal juvenil captur near matern roost summer support hypothesi viru amplif occur mainli reproduct roost identif cov strain 100 ident differ roost bat speci ferrumequinum pipistrellu locat also 100 distanc seem confirm bat cov appear speci specif thu close associ host speci sampl locat interestingli identifi diverg alpha cov lineag nattereri spa repres cryptic lineag within myoti nattereri speci complex mediterranean region lineag known present itali howev inform avail germani hungari follow host viru coevolut theori base close phylogenet concord small diverg nattereri spa cov strain german hungarian nattereri one could indic resid nattereri spa host consid molecular speci identif specimen lack detect ident alpha cov sequenc two differ speci belong myoti genu oxygnathu myoti two distinct roost site sv12 cn4 apart could due expans overlap habitat forag area myoti spp maritim alpin chain valley knowledg first report cov oxygnathu speci compact cluster almost ident beta cov lineag strain two separ ferrumequinum roost give indic rhinolophu genu may repres specif host sar like cov give import contribut term avail beta cov sequenc speci europ knowledg first report cov auritu speci sequenc cluster separ within beta cov lineag show diverg mer strain identifi camel egypt detect highli diverg alpha cov strain within one pipistrellu reproduct roost circul alpha beta cov within one ferrumequinum roost infect pipistrellu cov pmv provid evid bat abl carri one viru infect multipl cov speci bat coloni well known previous report china europ apart metagenom studi notabl bias toward identif sequenc dsdna virus knowledg coinfect differ ssrna viral famili anim far report one studi europ pygmaeu hungari specif coinfect two ssrna viral famili within host may explic light ifn inhibit use paramyxovirus circumv host innat immun respons mechan known ifn antagon may exploit virus abl escap adapt immun cov introduc prolifer host observ mallard increas viral surveil consider number novel paramyxovirus discov pteropoid non pteropoid speci date number bat pmv sequenc europ scarc bat speci three new pmv strain two kuhlii one pipistrellu speci classifi within current seven known pmv genera cluster crowd unassign pmv clade compris sever bat deriv strain report repres first identif pmv kuhlii speci worldwid two sequenc retriev two roost locat apart diverg previous known pmv cluster may indic stronger associ host speci rather geograph area also paramyxovirus viral tropism also strongli support high similar pipistrellu sequenc one e95 pmv sequenc retriev germani bat speci support hypothesi studi renal tissu african bat underlin paramyxoviru diverg pteroid non pteroid bat correl bat taxonomi suggest strong associ bat genera gene fragment use genet marker aforement studi overlap sequenc use phylogenet compar nevertheless given high similar pipistrellu sequenc show e95 pmv strain find support associ moreov extens collect urin sampl coloni would necessari facilit pmv isol remain critic requir full genom pathogen character strain detect compar previou studi publish itali detect alpha beta cov previous survey italian region new bat speci moreov report repres first novel identif pmv itali new bat genet sequenc fill gap expand current molecular bat born viru databas consid amount novel bat born pmv associ emerg zoonot infect anim human last year defin viru divers within european bat speci need perform surveil studi within specif geograph area provid awar viral burden bat roost close proxim spillov host form basi appropri control measur curb potenti threat public health optim manag bat habitat financi support studi public provid italian ministri health context ricerca sanitaria corrent 2013 code iz plv data gener analyz studi includ paper made avail correspond author upon reason request sequenc data obtain studi deposit genbank databas full sequenc name genbank access indic tabl author contribut mlm sr1 particip concept coordin studi perform captur bat speci identif field perform experi wrote paper analyz genet data perform necropsi sr2 contribut reagent materi analysi tool author read approv final manuscript ethic approv consent particip sampl collect bat speci identif perform expert chiropterologist author italian ministri environ author number dpn 2010 0011879 000882 pnm 08052014 field oper perform complianc habitat direct european union agreement conserv popul european bat respect ethic safeti recommend applic',\n",
       "   '_search_score': 0.7292778000000002},\n",
       "  {'_id': '00f244245f02c27974ee0dd3bc21fe47abcc03cb',\n",
       "   'insert_date_': '2020-11-05T08:31:38.934173',\n",
       "   'abstract': 'shed tissu distribut quasispeci composit felin coronavirus studi natur experiment infect cat infect remain subclin major anim shed viru via faec throughout experi sequenc correspond viral nucleocapsid region amplifi revers transcript polymeras chain reaction cortex dura mater pancrea lung third eyelid heart muscl four case interestingli orf7b viral region suppos virul factor detect fewer organ rais possibl region affect delet viru replic vivo demonstr composit viral quasispeci differ organ genom region differ function undergo distinct process select consid evolut felin coronavirus',\n",
       "   'title': 'Preliminary studies on feline coronavirus distribution in naturally and experimentally infected cats',\n",
       "   'body': 'felin coronavirus fcov member famili coronavirida genom consist singlestrand posit sens rna groot 1987 luytj 1995 regard pathogen fcov compris two biotyp felin infecti periton fip virus fipv felin enter coronavirus fecv pedersen 1987 base vitro neutralis use monoclon antibodi exist least two serotyp fcov type shown serotyp contain virus fipv fecv biotyp hohdatsu 1991a type virus differ vitro growth characterist pedersen 1987 importantli composit spike protein herrewegh 1998 field preval fcov type seem higher sinc per cent fip case due infect type virus hohdatsu 1992 fip highli fatal diseas felida foley 1997a pathogenesi diseas still fulli understood diagnosi diseas rather complic therefor repres one major pet anim health problem still remain solv thu serolog data alway inform olsen 1993 diagnosi vaccin may provid protect sinc may even induc acceler pathogenesi earli death associ antibodi depend enhanc infect vennema 1990 harvey 1996 nevertheless commerci avail vaccin shown promis result fehr 1997 fecv consid direct caus morbid cat may produc mild enter experiment natur infect remain subclin howev present consider health risk cat reveal parent fipv foley 1997b vennema 1998 recent find suggest genet chang frequent occur felin coronavirus herrewegh 1995 vennema 1998 instanc demonstr type felin coronavirus arisen rna recombin event canin coronaviru type felin coronaviru herrewegh 1998 observ indic delet differ part fecv genom may respons format fip virus vennema 1998 howev mechan event total understood known type fecv establish persist infect cat viru replic enterocyt crypt lieberkhn larg intestin viru replic specif chang viral genom may lead develop highli pathogen fip virus consequ manifest diseas possibl malfunct immun system contribut process poland 1996 laboratori diagnosi control fip serious hamper failur fcov replic cell cultur consid fact aim studi develop appli effici approach detect differenti fcov subpopul use techniqu investig persist distribut quasispeci composit fcov natur experiment infect cat order mimic natur condit randomli select mix breed domest cat gender use anim origin vendor clinic healthi month age cat immunis vaccin faec cat no posit fcov use revers transcript polymeras chain reaction pcr therefor anim regard carrier remain eight cat repeatedli neg fcov start experi six fcov free anim no experiment infect fcov two cat no use neg control anim kept room separ cage physic contact fed commerci avail cat food milk fresh water faecal sampl collect cat no begin experi fresh sampl pool homogenis pb centrifug 3100 minut supernat fluid given oral six fcov free cat no dose cat neg control fcov carrier mock infect pb cat monitor daili clinic sign rectal swab sampl collect day start experi day experiment inocul post infect day pid swab process describ foley 1997b briefli swab vortex doubledistil water suspens centrifug 5000 minut pid cat kill gross histopatholog examin carri follow tissu sampl collect pcr brain cortex medulla oblongata dura mater liver kidney spleen lymph node lung small larg intestin pancrea third eyelid heart muscl organ sampl per cent suspens prepar doubl distil water centrifug 7000 minut supernat use immedi rna extract store supernat faec tissu homogen use rna prepar procedur boom 1990 cheung 1994 follow 100 suspens ad size fraction silica 900 lysi buffer 120 guanidin thiocyan dissolv 100 tri hcl edta triton 100 sampl vortex left minut room temperatur supernat remov quick spin 12000 minut silica rna pellet wash twice 450 wash buffer 120 guanidin thiocyan dissolv 100 tri hcl twice ethanol final wash aceton pellet dri minut rna elut resuspend pellet diethyl pyrocarbon treat depc water minut pellet silica particl 12000 minut supernat fluid contain rna use revers transcript cdna synthesi perform reaction mixtur preliminari mixtur contain rna depc treat water random hexam pharmacia biotech uppsala sweden incub minut rna denatur tube place ice mix contain deoxynucleotid triphosph pharmacia reaction buffer tri hcl 375 kcl mgcl rnaguard pharmacia moloney murin leukaemia viru revers transcriptas 200 gibco brl bethesda usa ad reaction mixtur incub minut follow incub minut inactiv enzym primer select highli conserv nucleocapsid orf7b region fip viru strain 1146 genom vennema 1991 use oligo program nation bioscienc inc plymouth usa sequenc primer shown tabl note pcr appli year routin diagnost laboratori diagnosi fip sever hundr sampl clinic case found posit unpublish data two pcr assay carri volum reaction mixtur contain geneamp pcr buffer perkin elmer cetu norwalk usa pmol ofip1a ofip2a primer first nucleocapsid pcr pmol p399 p997 primer first orf7b pcr deoxynucleotid triphosph pharmacia mgcl amplitaq dna polymeras perkin elmer cetu cdna two droplet miner oil sigma chemic loui usa ad prevent evapor second nest pcr assay perform amount first pcr product pmol primer ofip4 ofip3 nucleocapsid pmol primer p503 p970 orf7b follow temperatur profil use second second second first five cycl minut second round amplif second minut minut appli cycl profil repeat time follow final extens step minut amplifi dna product detect standard per cent agaros gel electrophoresi avoid fals posit neg result precaut routin diagnost pcr laboratori use throughout experi belk ballagi pordni 1993 sensit two pcr assay evalu compar test fold dilut tissu cultur supernat refer viru strain 1683 black ucd ucd clinic sampl rectal swab heart muscl homogen origin cat concentr nucleic acid solut determin spectrophotomet genequ rna dna calcul pharmacia uppsala sweden serial fold dilut test two pcr assay show sensit calcul total rna minim target visibl pcr product obtain shown order gener singl strand pcr product sscp analysi asymmetr pcr perform amount complet first pcr mixtur use nest primer ratio pmol cycl profil appli second round amplif five microlitr amount singl strand pcr product ad sscp load buffer per cent formamid per cent xylen cyanol per cent bromophenol blue kept minut put ice minut final load onto per cent polyacrylamid gel bio rad hercul usa electrophoresi perform mini protean cell bio rad 140 constant power hour room temperatur subsequ gel submerg ethidium bromid solut minut visualis light experiment infect clinic statu cat sever affect mild diarrhoea observ among experiment infect anim pid appetit behaviour bodi temperatur remain normal bodi weight cat increas per cent end experi test swab sampl pcr orf7b pcr assay gave consist result cat no fcov detect occas test anim viru detect variou occas shown test larg intestin pcr assay viru detect eight anim cat no neg cat no viru also demonstr follow organ cat cortex dura mater lung pancrea third eyelid heart muscl cat third eyelid heart muscl cat no heart muscl orf7b region detect larg intestin eight anim pcr assay also demonstr pancrea cat heart muscl cat no sscp assay quasispeci natur detect virus reveal illustr fig band lane repres viral subpopul slight nucleotid sequenc differ lzaro estivil 1992 gener region show higher variabl orf7b region consist pattern confirm repeat pcr sscp analys data shown chang viral quasispeci larg intestin cat fig could characteris follow day infect appear one major one minor variant viru subpopul pid addit minor variant latter remain termin experi pid first minor variant appear replac third minor variant pid also remain present end major variant alway present throughout experi never replac third minor variant sequenc gene appear pid might correl variant gene sequenc fig cat fig major minor variant present day like variant present throughout experi day pid appear disappear addit minor subpopul chang region show similar characterist moder extent sscp pattern organ homogen swab sampl demonstr fig appar organ consist contain three predomin variant orf7b sscp pattern organ substanti pattern swab shown recent studi suggest fecv parent fipv alter genom harmless parent lead develop fatal diseas infecti periton vennema 1992 herrewegh 1995 poland 1996 vennema 1998 consid fecv extrem widespread cat popul foley 1997b detail studi requir investig biolog two viru biotyp particular regard genet stabil fecv genom region fig sscp pattern product amplifi nucleocapsid region specif pcr organ sampl cat lane repres follow organ cortex dura mater lung larg intestin pancrea heart muscl third eyelid respons develop virus high pathogen investig reli abil detect characteris differ virus variou stage infect present studi investig invas tissu distribut variabl fcov aim establish compar two pcr assay respect applic monitor purpos appli assay amplifi two conserv region fcov genom found eight cat shed viru faec throughout day cours experi find support observ herrewegh 1997 report fcov infect maintain chronic infect carrier short last mild diarrhoea observ among experiment infect anim earli stage infect could clinic symptom fcev infect may consequ massiv exposur anim fcov suggest foley 1997a two cat no becam remain pcr neg rel earli stage experi might strong effect immun respons fcov crucial step pathogenesi fip certain viru variant leav intestin reach differ organ pedersen 1981 known long process take condit requir event present experi show despit rel short observ period day one cat fcov appar invad differ organ consid suggest pedersen 1981 may import step develop fip interest orf7b pcr assay gave ident result swab larg intestin sampl test howev case addit organ pcr score consider posit sampl obviou explan would differ sensit two system like howev consid two assay yield ident result swab larg intestin sampl also serial dilut fcov clinic specimen test therefor hypothesis virus detect cat could presum orf7b region affect delet phenomenon report avirul tissu cultur adapt felin coronavirus conclud loss virul associ delet orf7b region groot horzinek 1995 accord vennema 1998 well virul factor presenc region advantag viru replic therefor phenomenon investig vitro vivo studi also involv orf3c region consid exert regulatori effect orf7b vennema 1998 quasispeci natur member coronavirida famili well document mainli murin coronavirus makino 1986 baric 1990 holland 1992 lai 1996 present studi reveal fcov also compris viru subpopul howev differ preval viral subpopul bodi compart variant present swab sampl present organ sampl variant preval organ sampl larg intestin addit vivo replic sscp pattern region show higher variabl orf7b region find suggest presum select viru variant becam abl leav larg intestin invad blood circul differ organ appear howev select pressur gene sinc predomin subpopul master sequenc could demonstr pcr posit organ specimen nevertheless presum new select process take place everi infect organ may result viru quasispeci differ composit origin replic larg intestin process conjunct rel impair reaction respect immun respons particular cat could eventu lead develop fip present studi demonstr fcov abl establish chronic enter infect cat data indic virus undergo continu subtl chang chronic replic extent variat composit viral quasispeci differ organ addit even neighbour genom region show distinct variabl region vari orf7b region possibl role orf7b viru replic indic highli conserv sscp pattern region exhibit throughout experi exact inform orf7b product gp7b reveal role region develop fip conclus studi provid inform concern altern genom fcov vivo replic shown composit viral quasispeci differ organ genom region differ function undergo distinct process select consid evolut felin coronavirus futur investig aim determin signific chang particular viral genom region beyond immun respons fcov infect probabl impair function eventu particip pathogenesi fip elabor',\n",
       "   '_search_score': 0.7255031999999999},\n",
       "  {'_id': 'a1949f80e726d045fe1cccb24a0e911cc0b912b9',\n",
       "   'insert_date_': '2020-11-05T08:31:43.659076',\n",
       "   'abstract': 'porcin epidem diarrhea viru pedv porcin deltacoronaviru pdcov emerg reemerg virus pig togeth transmiss gastroenter viru tgev pose signific econom concern swine industri virus infect epitheli cell small intestin caus wateri diarrhea dehydr high mortal neonat piglet type interferon ifn -/) major antivir cytokin form host innat immun turn enter coronavirus evolv modul host innat immun signal infect accumul evid howev suggest ifn induct signal intestin epitheli cell differ epitheli cell larg due distinct featur gut epitheli mucos surfac commens microflora appear type iii interferon ifn play key role maintain antivir state gut review describ recent understand immun evas strategi porcin enter coronavirus role differ type ifn intestin antivir innat immun',\n",
       "   'title': 'Immune evasion of porcine enteric coronaviruses and viral modulation of antiviral innate signaling',\n",
       "   'body': 'coronaviru enter highli contagi viral diseas pig character sever diarrhea vomit dehydr high mortal especi neonat piglet less week age transmiss gastroenter viru tgev frequent caus endem epidem viral enter neonat older pig kim 2000 laud 1993 porcin epidem diarrhea viru pedv becom major concern swine industri sinc emerg april 2013 tgev pedv caus sever enter newborn piglet mortal much higher may reach 100 epidem debouck pensaert 1980 song park 2012 ped first describ earli 1970 sinc becom endem part europ highli virul pedv emerg asia subsequ chen 2014a marthal 2013 mole 2013 stevenson 2013 pedv sinc quickli spread state april 2016 state becom endem pedv rais signific econom concern www aphi usda gov anim health secd pdcov newli emerg swine enter coronaviru caus enter sever diarrhea piglet first describ surveil studi 2012 hong kong china woo 2012 pdcov addit enter viru emerg februari 2014 pdcov identifi fece intestin sampl pig experienc sever diarrhea without pedv tgev retrospect studi show pdcov present least sinc august 2013 mccluskey 2016 sinha 2015 pdcov also spread least state www aphi usda gov anim health secd korea china lee lee 2014 pdcov infect seem clinic milder lower death rate neonat typic pedv infect infect pedv tgev porcin rotaviru common result sever form diseas marthal 2014 wang 2014 clinic symptom pdcov reproduc gnotobiot pig convent pig present diarrhea accompani sever histopatholog lesion villi atrophi absenc pathogen jung 2015c 2015 although clinic sign patholog lesion indistinguish among pedv tgev pdcov jung 2014 coronavirus antigen distinct thu cross protect absent lin 2015 pathogenesi coronavirus may includ destruct enterocyt villou atrophi intestin mucosa jejunum ileum debouck pensaert 1980 jung 2014 coronavirus envelop virus contain genom singl strand posit sens rna famili coronavirida coronaviru genom rang length current largest known viral rna genom coronavirus group coronavirina subfamili compris genera alpha beta gamma delta coronavirus http :// www ictvonlin org virustaxonoym asp ictv 2014 evolutionari studi suggest alphacoronaviru betacoronaviru origin bat gammacoronaviru deltacoronaviru bird woo 2012 human coronavirus infect respiratori tract caus symptom rang common cold sever pneumonia acut respiratori distress sever acut respiratori syndrom coronaviru sar cov middl east respiratori syndrom coronaviru mer cov belong betacoronaviru caus lower respiratori tract infect signific public concern gralinski baric 2015 among swine enter coronavirus tgev pedv belong alphacoronaviru share similar genom organ wherea pdcov belong deltacoronaviru http :// www ictvonlin org virustaxonomi asp pedv genom approxim length cap polyadenyl tail genom arrang orf1a orf1b orf3 order termini flank untransl region utr duart 1993 orf1a code larg polyprotein pp1a orf1b express pp1ab fusion protein via ribosom frameshift polyprotein proteolyt cleav nonstructur protein nsp1 nsp16 proteinas activ nsp3 nsp5 tgev genom similar pedv arrang orf1a orf1b orf3a orf3b orf7 order alonso 2002 pedv orf3 encod sole accessori protein wherea tgev three accessori protein encod orf3a orf3b orf7 brian baric 2005 pdcov genom smallest three coronavirus length includ two accessori gene ns6 ns7 gene order orf1a orf1b ns6 ns7 woo 2012 notabl pdcov lack nsp1 gene thu code nsp total genom similar dissimilar -, -, -, coronavirus illustr fig upon viru infect host react quickli invad viru produc type interferon ifn -/) elicit antivir state infect cell uninfect neighbor cell three differ type ifn known type ifn -/), type ifn -), type iii ifn ifn tabl human type ifn contain subtyp ifn singl subtyp ifn -, ifn ifn ifn pig ifn produc mani function gene type ifn singl subtyp ifn report pig produc larg immun cell type iii ifn includ ifn interleukin 28a 28b ifn kotenko 2003 prokunina olsson 2013 sheppard 2003 differ type ifn signal differ receptor ifn -/ signal heterodimer complex compos singl chain ifn receptor ifnar1 ifnar2 believ express cell surfac nucleat cell gibbert 2013 pestka 2004 ifn form homodim signal ifn receptor ifngr complex compos dimer two transmembran span receptor ifn receptor ifngr1 ifngr2 broad tissu distribut ifn signal heterodim interleukin receptor il10r2 ifn receptor ifnlr1 10r2 wide distribut differ cell type ifnlr1 larg restrict epitheli cell sommereyn 2008 thu subtyp ifn limit antivir function epitheli cell ifn activ janu kinas jak signal transduc activ transcript stat signal pathway establish antivir state sinc ifngr1 wide distribut cell type capabl respond ifn valent 1992 ifn pleiotrop effect immun cell direct antivir activ rather limit contrast ifn -/ ifn induc potent antivir state regul express hundr interferon stimul gene isg even though ifn -/ ifn util differ receptor downstream signal induct antivir mechan similar vivo studi ifn show high degre redund two ifn system lung epitheli cell ank 2008 mordstein 2008 mordstein 2010 rais question two seemingli redund antivir system exist ifn signal cascad summar fig swine enter virus mostli transmit via fecal oral rout possibl aerosol transmiss highli infecti tgev infect replic primarili small intestin enterocyt lesser extent respiratori tract kim 2000 pedv pdcov also infect small intestin enterocyt jung 2015c jung saif 2015 thu intestin epitheli surfac first line host defens prevent enter viral infect due distinct featur intestin epitheli mucos surfac induct signal ifn uniqu coronavirus larg genom evolv carri redund mechan counteract host innat immun respons sar cov least viral protein identifi type ifn antagonist kindler thiel 2014 shi 2014a totura baric 2012 recent studi pedv show least viral protein type ifn antagonist contrast acut infect tgev induc high level ifn newborn pig bonnardier laud 1981 despit variat genom organ swine enter coronavirus seem util similar distinct mechan activ evas host innat immun intestin epitheli surfac wealth inform avail ifn mechan induct signal cascad function mani fundament question still remain answer especi role intestin antivir innat immun current review discuss activ modul innat immun respons mediat swine enter coronavirus specif mechan viral protein modul host innat antivir defens role differ type ifn intestin innat immun describ host innat immun system util prr patternrecognit receptor sens respond pamp pathogenassoci molecular pattern invad virus kawai akira 2011 recognit membran bound prr trigger activ ifn induct pathway toll like receptor tlr among best character group prr kawai akira 2011 tlr constitut famili singl transmembran protein ectodomain contain leucin rich repeat recognit pamp cytosol tir toll receptor domain key domain transduc signal downstream adaptor includ trif tir contain adaptor protein induc ifn -) myd88 myeloid differenti primari respons gene kawai akira 2010 among tlr tlr involv endosom nucleic acid sens tlr3 dsrna sensor respond poli synthet rna analog kariko 2004 okahira 2005 tlr7 tlr8 recogn ssrna deriv rna virus diebold 2004 heil 2004 wherea tlr9 sens cpg dna motif genom dna virus vollmer 2004 tlr13 recent identifi sensor bacteri ribosom rna oldenburg 2012 rlr rig like receptor famili respons sens cytosol rna rlr member includ rig activ retino acid induc gene mda5 melanoma differenti gene lpg2 laboratori genet physiolog rig mda5 respond poli specif base length dsrna rig sens shorter dsrna 300 contain triphosph panhandl structur wherea mda5 recogn long dsrna fragment kato 2008 schlee 2009 schmidt 2009 cytoplasm dna sensor address includ sting stimul ifn gene cga cyclic gmp amp synthas cytosol dna sens ishikawa barber 2008 sun 2013 2013 type ifn product activ rig mda5 bind mitochondri adaptor protein mav ip recruit traf3 tnf receptor associ factor traf6 traf3 activ downstream ikk kinas relat kinas tbk1 tank bind kinas ikk induct irf3 irf7 depend type ifn product fitzgerald 2003 sharma 2003 traf6 lead tak1 activ follow activ type ifn cytokin product rajsbaum garcia sastr 2013 within ifn promot four regulatori ci element name posit regulatori domain prd iii prd iii prd prd bind site irf atf jun respect honda 2006 express ifn requir assembl regulatori factor prd form enhanceosom complex recruit cbp ifn product randal goodbourn 2008 follow product secret ifn bind cell surfac receptor trigger activ jak stat pathway ifn signal cascad three differ ifn system signal distinct ifn receptor type ifn type iii ifn trigger format isgf3 complex product hundr isg contrast type ifn trigger phosphoryl stat1 subsequ format ifn activ factor gaf antivir gene express schneider 2014 express membran bound prr cell typedepend specif molecular featur prr cell type recogn two main factor specif ifn subtyp product hoffmann 2015 nearli nucleat cell capabl produc ifn activ irf3 ifn subtyp primarili produc leukocyt cantel 1981 plasmacytoid dendrit cell pdc potent type ifn produc hundr thousand time ifn cell type siegal 1999 activ pdc migrat lymph node potenti influenc variou cell type secret ifn ifn respons vari viru infect still unclear recent studi show differ ifn respons depend activ pdc interact infect cell viru alon frenz 2014 level total ifn product pdc vari greatli invad viru time amount inoculum thoma 2014 anoth studi show pdc produc highli uniform ifn profil regardless pamp szubin 2008 ratio differ ifn subtyp may optim evolut specif cell type similar ifn -/, ifn induc differ cell type variou virus ifn express varieti cell type hematopoiet lineag also produc ifn -/ abund among nonhematopoiet cell epitheli cell potent produc ifn ifn respons larg restrict epithelium lung epitheli cell respond ifn -/ ifn intestin epitheli cell respond exclus ifn mordstein 2010 pott 2011 sommereyn 2008 small intestin epitheli cell iec form physic barrier segreg mucos immun system commens microbi commun lumen paneth goblet cell secret mucu antimicrobi protein exclud invad bacteria virus epitheli surfac renew epitheli cell layer control intestin stem cell crypt secret iga siga also contribut barrier function lamina propria made stromal cell cell cell dendrit cell macrophag intraepitheli lymphocyt iel local iec cell constitut larg highli conserv cell compart serv cytolysi dysregul iec renew iec microfold cell found gut associ lymphoid tissu galt peyer patch mucosa associ lymphoid tissu malt part gastrointestin tract special antigen sampl initi mucos immun respons peterson arti 2014 unlik lung epithelium intestin epithelium maintain toler symbiot gastrointestin microflora mount effect immun respons infect intestin epitheli barrier depict fig tabl modul innat immun respons signal pedv suppress ifn induct upon poli stimul inhibit irf3 nuclear transloc vero cell xing 2013 suppress type ifn induct degrad cbp nucleu marc 145 cell zhang 2016 lack poli mediat ifn induct block activ ip rig mediat pathway intestin epitheli cell iec cao 2015a ativ pathway tlr2 tlr3 tlr9 iec cao 2015b activ extracellular signal regul kinas erk pathway earli infect 2015 pedv infect caus undetect natur killer cell activ suckl piglet compar wean pig pig annamalai 2015 mark induct serum ifn suckl pig day post infect declin significantli base level day post infect pig annamalai 2015 type ifn ifn -/) major cytokin restrict viral infect recent studi howev shed new light import ifn restrict viral infect intestin epithelium rotaviru model ifn -/ rather play minor role restrict viral infect instead ifn play major role inhibit viral replic intestin epithelium angel 1999 pott 2011 iec strongli respond ifn margin respond ifn -/ vivo effect ifn -/ ifn compartment major effect ifn -/ intestin lymphocyt ifn epitheli cell pott 2011 distinct role ifn -/ ifn control system local dissemin rotaviru respect suggest ifn -/ produc mucos pdc enhanc cell respons promot viral clearanc protect infect deal 2013 rotaviru signific ifn -/ seem vari depend viral strain host synergi type ifn ifn determin local intestin epitheli antivir defens recent find show activ iel upregul ifn -/ ifn express mount rapid antivir state protect iec swami 2015 activ iel vivo rapidli provok isg express ifn ifnreceptor thu activ iel offer overt mean promot innat antivir potenti intestin epithelium resist invad virus gut epithelium reli mainli ifn ifn -/, rais question ubiquit express receptor ifn -/. recent studi show mous iec minim express function type ifn receptor thu respond type ifn explain ifn -/ system unabl compens ifn defici enter epitheliotrop viral infect studi show viru infect iec potent produc ifn suggest gut mucosa possess compartment ifn system epitheli cell respond predominantli ifn wherea cell gut reli ifn -/ antivir defens time product ifn iec drive rapid clearanc intestin viral infect limit viral shed fece intestin epithelium respond exclus ifn non redund fashion control epitheliotrop virus pott 2011 reoviru ifn restrict initi viral replic reduc viral shed fece wherea ifn -/ critic prevent system infect similarli noroviru infect also requir ifn control persist enter infect although ifn -/ restrict system spread gut protect effect innat immun particular ifn counteract intestin commens bacteria pedv infect caus decreas commens bacteria normal present healthi gastrointestin tract induc unbalanc shift gut microbiota koh 2015 liu 2015b suggest possibl modul ifnrespons pedv border surfac intestin epithelium major entri site enter virus specif mechan need fail safe protect multi layer surfac although ifn product iec requir suffici protect epithelium rotaviru induc cell damag product releas iec follow rotaviru infect enhanc product recent studi identifi cooper ifn synergist curtail mucos viral infect receptor ifn restrict epitheli cell simultan engag receptor iec requir restrict rotaviru infect suggest second layer control necessari downregul ifnlr1 interfer stat1 signal may import strategi viral evas sen 2014 stimul poli infect reoviru iec hematopoiet cell intestin epithelium express ifn strongli quickli express ifn -/ gene thu mechan seem evolv produc ifn specif potent intestin epithelium sinc common pathway believ util induct ifn -/ ifn onoguchi 2007 ifn -/ pathway specif block iec unknown mechan anoth studi show peroxisom bound mav may preferenti activ ifn express odendal 2014 nlrp6 nucleotid bind oligomer domain like receptor intracellular protein specif tissu cell type highest express iec elinav 2011 nlrp6 bind viral rna via rna helicas dhx15 rna sensor trigger mav depend express ifn -/ ifn plu isg thu play import role intestin antivir respons wang 2015b use gene knockout mice specif ifn receptor ifn identifi play import role defens sar cov respiratori viral infect mordstein 2010 fecal shed sar cov detect mice lack receptor ifn -/ ifn mice lack singl receptor suggest ifn control viral infect epitheli cell respiratori gastrointestin tract rel contribut variou ifn subtyp restrict enter coronaviru infect remain elus tgev pedv infect replic pam respiratori tract vivo park shin 2014 pedv also shown infect macrophag infiltr lamina propria modul immun respons infect monocyt deriv dc modc intestin dc lee 2000 summari ifn seem critic effici restrict enter coronaviru infect fig coronaviru nsp1 major ifn antagonist degrad creb bind protein cbp pedv nsp1 cell seed coverslip transfect pedv nsp1 gene porcin reproduct respiratori syndrom viru prrsv nsp1 known degrad cbp nucleu thu includ control cell stain post transfect indirect immunofluoresc assay ifa use rabbit anti flag polyclon antibodi red mous anti cbp monoclon antibodi green nuclei stain dapi blue white arrow indic nsp1 express cell yellow arrow indic control cell pedv nsp1 alter express p300 cell grown confluenc tag p300 gene transfect pedv nsp1 gene empti vector pxj41 cell stain post transfect detect nsp1 red use rabbit anti flag polyclon antibodi p300 green use mous anti monoclon antibodi nuclei stain dapi blue suppress intestin epitheli surfac includ pedv tgev possibl pdcov infect turn virus may modul host ifn respons great interest investig enter coronavirus regul ifn respons pig fulllength infecti cdna clone avail pedv use studi individu gene function use replic mutant vivo abl better understand pathogen mechan porcin enter coronavirus host respons infect beall 2016 jengarn 2015 coronavirus protein interact receptor suscept cell determin host rang viral tropism bosch 2003 gallagh buchmeier 2001 unlik mani coronavirus cell cultur adapt pedv trigger mutat protein sequenc lawrenc 2014 requir supplement exogen proteas typic trypsin activ protein induc membran fusion bind cellular receptor hofmann wyler 1988 park 2011 wicht 2014 trypsin depend pedv vitro consist tissu tropism trypsin rich small intestin pig cell adapt pedv strain lose trypsindepend nam lee 2010 wicht 2014 pedv infect cell produc distinct cytopath effect includ cell fusion multinucl syncytia format vacuol detach due apoptot cell death kim lee 2014 recent studi show heparan sulfat cell surfac may function attach factor pedv huan 2015 tgev pedv also util aminopeptidas apn cd13 function receptor apn heavili glycosyl transmembran protein express cell myeloid origin includ monocyt macrophag granulocyt hematopoiet precursor chen 1996 mina osorio 2008 riemann 1999 also abundantli express brush border epitheli cell small intestin renal proxim tubul cell membran small intestin play role final digest peptid gener hydrolysi protein gastric pancreat proteas kruse 1988 tgev util apn receptor entri swine testi cell howev cannot render effici propag pedv delma 1992 differenti may due rel low level express apn cell express apn enhanc pedv infect increas apn densiti enhanc pedv suscept cultur cell nam lee 2010 2003 porcin intestin tissu sensit high titer viru product pedv natur infect may due abund express apn porcin intestin brush border membran collect high level express apn intestin barrier correl effici replic tgev pedv infect pedv depend apn densiti addit pedv also infect cultur cell pig human monkey bat suggest pedv may util apn speci origin unidentifi receptor receptor effici entri liu 2015a zhang 2016 pedv tgev infect alveolar macrophag besid small intestin epitheli cell laud 1984 park shin 2014 pedv also infect infiltr macrophag lamina propria vivo lee 2000 even though clinic sign epidem pedv tgev infect indistinguish great interest investig whether cell tissu tropism contribut differenti pathogenesi vivo porcin enter coronavirus replic iec transmit via fecal oral rout viral entri releas tgev restrict apic surfac polar epitheli cell rossen 1994 experiment infect newborn piglet tgev induc strong earli ifn product serum intestin secret bonnardier laud 1981 block viral receptor still induc ifn respons indic entri need ifn induct leukocyt riffault 1997 earli strong ifn product also induc nonimmun cell exposur tgev infect epitheli cell charley laud 1988 rapid massiv releas ifn serum mediat protein tgev charley laud 1988 laud 1992 riffault 1997 specif point mutat protein result lower ifn induct compar wildtyp viru splichal 1997 viru like particl made protein induc ifn product porcin leukocyt almost effici fulli infecti tgev baudoux 1998 howev reconstitut virosom lose abil induc ifn product indic nativ envelop structur essenti riffault 1997 immunohistochemistri analysi tissu tgev infect piglet show ifn -- produc cell ipc almost exclus detect intestin tissu mesenter lymph node mln earli time infect disappear riffault 2001 major ipc local enterocyt epitheli layer lamina propria around peyer patch accumul mln riffault 2001 number ipc spleen peripher lymph node limit thu circul ifn like origin gut mln trigger viru riffault 2001 ipc pig identifi pdc summerfield 2003 interestingli gut associ ipc express swine leucocyt antigen sla distinct tgev posit cell suggest cell may mucos counterpart intestin pdc riffault 2001 contrari pdc produc high level ifn tnf infect porcin monocytederiv dc respond tgev guzylack piriou 2006 frequenc intestin pdc howev low compar dc thu limit popul intestin pdc may major subset antigen present dc product ifn intestin pdc mln may flood cell area influenc immun outcom charley 2006 addit product ifn -/, tgev induc phosphoryl nuclear transloc stat1 viru infect cell enhanc express isg tgev infect also promot p53 signal regul apoptosi viral nucleocapsid protein respons action cell cycl arrest ding 2014a huang 2013 ifn shown inhibit tgev protein express without reduc viral rna tgev infect cell jordan derbyshir 1995 infect tgev upregul express type ifn induct nsp1 bovin coronaviru bcov transmiss gastroenter coronaviru tgev hela cell seed well plate cotransfect pifn -- luc along individu nsp1 gene prl ratio prrsv nsp1 known ifn suppressor includ control post transfect cell stimul poli luciferas activ measur report experi repeat three time triplic asterisk indic statist signific statist analysi perform student test **, ***, 001 oligoadenyl synthetas inhibit viral replic bosworth 1989 jordan derbyshir 1995 swine ifn shown enhanc protect tgev allevi clinic sign piglet oral administ salmonella enterica serovar typhimurium express swine ifn without cytokin kim 2010 lee 2011 viral load significantli reduc intestin segment piglet treat ifn induc challeng reduct viral yield seen simultan inocul tgev ifn -, suggest ifn may use prophylact jordan derbyshir 1994 pedv cytolyt acut necrosi iec lead villou atrophi small intestin jung 2014 high mortal pedv piglet associ extens dehydr result sever villou atrophi sever dehydr may due extens structur destruct disorgan tight junction adheren junction infect nurs piglet jung 2015b slower turnov enterocyt mediat regener crypt stem cell nurs piglet compar wean piglet probabl respons age depend resist ped jung 2015a pedv antigen mainli observ villou enterocyt small larg intestin pedv also infect macrophag infiltr lamina propria vivo lee 2000 experiment infect tgev show lung tract sustain viral replic ifn synthesi bonnardier laud 1983 tgev replic porcin alveolar macrophag pam vitro induc mark synthesi type ifn laud 1984 similarli pedv also infect replic pam vivo park shin 2014 suggest extra intestin replic pedv addit pedv shown induc apoptot cell death vitro vivo caspas independ mitochondri apoptosisinduc factor aif pathway kim lee 2014 anoth studi claim pedv may induc death enterocyt small intestin apoptosi pathway infect pig bind apn pedv intern target cell direct membran fusion genom rna releas cytosol replic sens viral nucleic acid prr endosom cytosol initi innat immun signal like coronavirus pedv like replic doubl membran vesicl hide viral rna sens prr attenu innat immun respons angelini 2013 snijder 2006 recent studi show tlr2 tlr3 tlr9 may contribut activ respons pedv infect iec vitro caus rig activ cao 2015b tlr2 plasma membran respond viral glycoprotein envelop induc product ifn inflammatori cytokin tlr3 tlr9 local larg endosom sens dsrna cpg dna respect boehm compton 2004 suggest pedv may initi innat immun respons surfac glycoprotein viral nucleic acid endosom pedv activ erk extracellular signal regul kinas pathway earli infect independ viral rna replic knockdown inhibit erk suppress viral transcript protein express progeni product suggest activ erk pathway benefici requir pedv replic proteom analysi pedv infect cell show differenti express protein involv apoptosi signal transduct stress respons erk1 significantli downregul pedv infect thu erk pathway modul pedv inter est studi one studi six differ signal pathway shown differenti regul pedv includ rlr rap1 pi3k akt mapk jak stat tlr signal pathway suggest modul host innat immun pedv pdcov littl known viral modul host immun pdcov protein form non coval link oligom local nucleolu protein alter express cellular protein relev cell divis metabol stress respons protein biosynthesi transport cytoskeleton network cell commun notabl pdcov protein upregul two member heat shock protein hsp70 famili two line swine cell llc pk1 use isol propag pdcov jung 2016 apoptosi induc vitro apoptosi observ vivo pdcov infect intestin enterocyt jung 2016 suggest better cell system need mimic vivo condit pdcov infect purpos iec intestin enteroid organoid may use finkbein 2012 jung 2016 pedv infect suckl pig alter natur killer cell activ compar wean pig show significantli higher cell activ blood ileum annamalai 2015 interestingli pedv infect seem stimul mark induct serum ifn suckl pig day post infect declin basal level day post infect annamalai 2015 wean pig howev pedv infect caus slight induct serum ifn day post infect decreas basal level day post infect peak product serum ifn coincid peak viral rna shed fece serum suckl pig serum wean pig onset diarrhea pedv infect differenti modul inflammatori cytokin suckl wean pig induc similar level tnf ifn annamalai 2015 accord studi pedv capabl modul ifn product especi wean pig sinc pedv infect occur small intestin tgev interest examin ifn intestin secret pedv modul type ifn induct intestin pdc intestin dc highli effect antigen present cell distribut beneath epitheli line unlik tgev pedv infect modc produc high level inf without alter express interact pedv modc affect stimul cell respons vitro pedv may also enhanc abil porcin intestin dc sampl antigen activ cell prolifer vivo intestin pdc respond pedv infect modul innat immun signal remain determin ifn -/ play import role restrict viral infect enter coronavirus evolv redund mechan counteract host innat immun optim viral adapt replic even though tgev activ ifn -/ product accessori protein modul ifn -/ express interact pp1 prolong viral dissemin increas surviv accumul evid show pedv also abil evad host ifn respons pedv activ ifn express even poli stimul virusinfect cell xing 2013 zhang 2016 appear pedv infect inhibit irf3 nuclear transloc thu block irf3 mediat ifn induct use marc 145 cell ifncompet pedv permiss laboratori investig innat immun modulatori role pedv studi cultur supernat pedv infect cell suppress growth vsv gfp provid evid sup pression ifn -/ pedv studi show pedv abil degrad creb bind protein cbp nucleu thu inhibit enhanceosom assembl result suppress ifn transcript iec major target cell pedv pig cell pedv block activ mav ip irf3 nuclear transloc rig mediat pathway induc ifn express cao 2015a ipec cell epitheli cell deriv jejunum colostrum depriv neonat pig age zhao 2014 blikslag north carolina state univers cell howev appear non permiss pedv infect unpublish data thu less use initi anticip studi pedv innat immun evas individu clone code sequenc pedv express screen ifn antagonist luciferas report assay viral protein protein found suppress ifn induct includ structur protein nonstructur protein nsp1 nsp3 nsp7 nsp14 nsp15 nsp16 orf3 zhang 2016 protein antagon ifn irf3 activ among nsp1 potent ifn antagonist studi show nsp1 interfer irf3 phosphoryl nuclear transloc rather interrupt interact irf3 creb bind protein cbp thu inhibit ifn enhanceosom assembl pedv nsp1 trigger cbp degrad nucleu proteasomedepend manner seem key mechan nsp1 mediat ifn suppress pedv dramat decreas level ubiquitin conjug protein cytoplasm papain like proteinas plp2 nsp3 pedv contain deubiquitinas dub activ process cytosol k48 k63 link polyubiquitin chain xing 2013 plp2 nsp3 cov cov could process lysin 48and lysin link polyubiquitin chain cytosol protein chen 2014a clementz 2010 sar cov plp2 reduc level ubiquitin form rig sting traf3 tbk1 irf3 sting traf3 tbk1 complex inhibit type interferon signal pathway chen 2014a similarli pedv plp2 interact deubiquitin rig sting antagon ifn respons plp2 sar cov specif deubiquitin mdm2 mous doubl minut homolog degrad p53 suppress ifn signal yuan 2015 similarli pedv upregul mdm2 express degrad p53 significantli inhibit irf7 express suggest pedv plp2 may also target p53 pathway innat immun evas inhibit p53 depend apoptosi yuan 2015 collect pedv differenti modul ubiquitin level cytoplasm nucleu modul innat immun respons pedv infect lead activ pathway iec cao 2015b pedv stimul nuclear local exogen p65 post infect increas nuclear local endogen p65 post infect anoth studi pedv normal induc activ follow poli stimul post infect xing 2013 studi rel late stage pedv infect pedv may inhibit activ earli stage optim surviv activ late stage inflamm porcin reproduct respiratori syndrom viru prrsv porcin arteriviru nidoviral order activ min infect 2012 thu pedv inhibit earli time infect interest need explor sar cov protein activ liao 2005 wherea protein suppress probabl direct interact ikk fang 2007 pedv protein caus stress lead activ pathway regul bcl 2013a 2013b protein pedv report induc cell cycl arrest phase via cyclin pathway 2015 studi protein block ifn product isg express lead suppress irf3 activ ding 2014b protein rna bind protein shuttl cytoplasm nucleu shi 2014b furthermor interact tbk1 sequest associ irf3 block phosphoryl nuclear transloc irf3 lead inhibit irf3 mediat ifn product ding 2014b nsp5 protein pedv like proteinas shown ifn antagonist cystein proteinas activ nsp5 cleav nemo essenti modul disrupt rig mda5 signal wang 2015a modul innat immun respons signal pedv infect summar tabl sar cov nsp14 exoribonucleas nsp15 endoribonucleas function specif digest dsrna subsequ remov rna pamp lead suppress ifn respons kindler thiel 2014 sar cov nsp16 methlytransferas modifi cap structur viral rna function avoid detect host immun system totura baric 2012 pedv nsp14 nsp15 nsp16 correspond nsp14 nsp15 nsp16 sar cov presum share similar function innat immun evas need verifi experiment protein play import role viral assembl induct pedv neutral antibodi recent studi show addit ifn antagonist sinc protein tgev respons abund express ifn -, great interest determin pedv protein suppress ifn respons function differ tgev protein orf3 sole accessori protein pedv role pathogenesi remain enigmat strain cell adapt pedv display intern delet orf3 gene park 2008 suggest orf3 relat adapt pathogen orf3 also shown function ion channel protein thu suppress orf3 express inhibit viral product addit orf3 delay phase divid cell suppress cell cycl progress 2015 contradictori data exist orf3 caus inabl recov progeni viru full length infecti clone suggest orf3 neg modul pedv replic vitro jengarn 2015 howev delet entir orf3 target rna recombin possibl orf3 appear nonessenti pedv replic vitro orf3 inhibit type ifn product facilit vesicl format orf3 express cell 2015 implic import role orf3 innat immun evas viral antagonist may target differ signal pathway modul host innat immun synergist effici infect modul type ifn respons pedv ifn antagon pedv protein shown fig coronaviru nsp1 termin cleavag product polyprotein pp1a pp1a process papain like proteinas plp1 activ nsp3 ziebuhr 2005 nsp1 gene one diverg gene among four differ genera genom cov cov code nsp1 wherea genom cov cov lack nsp1 gene woo 2010 woo 2012 ziebuhr 2005 ziebuhr 2007 nsp1 gene cov much smaller cov sequenc homolog also limit pedv nsp1 gene encod 110 amino acid predict molecular migrat protein sequenc nsp1 harbor known cellular function motif connor roper 2007 thu genu specif genet marker snijder 2003 howev nsp1 cov cov share similar function coronaviru nsp1 especi cov nsp1 regul host cell viral gene express narayanan 2014 nsp1 protein cov cov potent ifn antagonist among coronavirus sar cov nsp1 best character sar cov nsp1 inhibit report gene express constitut promot well ifn promot kamitani 2006 sar cov nsp1 murin hepat coronaviru mhv nsp1 hcov 229e nsp1 affect cellular gene express suppress ifn -, ifn stimul respons element isr sv40 promot zust 2007 mhv nsp1 facilit viral replic mice suppress ifn -/ unknown mechan mhv nsp1 sar cov nsp1 distribut cytoplasm viru infect cell brockway 2004 kamitani 2006 sar cov nsp1 suppress ifn mrna accumul without alter irf3 dimer kamitani 2006 sar cov nsp1 block irf3 irf7 activ sar cov nsp1 also inhibit ifn depend signal interf stat1 phosphoryl wathelet 2007 bat cov nsp1 inhibit type ifn isg product tohya 2009 hcov 229e nsp1 hcov nl63 nsp1 also inhibit ifn -, isr isg15 activ well sv40 hsv cmv promot zust 2007 pedv nsp1 shown degrad cbp nucleu antagon type ifn induct fig p300 protein transcript activ protein cbp function histon acetyltransferas play central role coordin integr multipl signal depend event appropri level gene transcript pedv nsp1 alter express level p300 fig confirm specif degrad cbp examin whether nsp1 protein coronavirus also possess activ type ifn suppress nsp1 gene clone tgev bovin coronaviru bcov examin ifn suppress activ use ifn luciferas assay similar pedv nsp1 tgev nsp1 bcov nsp1 also significantli suppress ifn promot activ fig tgev nsp1 shown inhibit host protein synthesi without bind ribosom subunit huang 2011 bcov infect activ host innat defens via tlr7 depend pathway inhibit type ifn pro inflammatori cytokin aich 2007 interest investig detail mechan tgev nsp1 bcov nsp1 mediat ifn suppress two emerg enter coronavirus pedv pdcov caus epidem endem infect pig popul mani countri becom major econom threat pork industri intestin epitheli surfac first line defens restrict enter viral infect ifn -/ ifnar import antivir cytokin virus evolv counteract pedv least viral ifn -/ antagonist identifi laboratori sinc enter coronavirus infect intestin epitheli surfac sever question still remain answer fig firstli whether pedv pdcov select activ modul ifn product whether ifn play protect role intestin surfac secondli differ cell type iec pdc iel intestin barrier induc differ type ifn function mucos defens thirdli intestin epithelium constantli interact commens bacteria opportunist viral infect thu ifn system must abl deal organ without sever inflamm tissu damag thu better research model cell anim need truli reflect viral evas innat immun defens lastli specif cytokin mediat produc iec respond porcin enter viral infect addit interest',\n",
       "   '_search_score': 0.7171661999999999},\n",
       "  {'_id': 'a6203f5bc0f1ac39f4c722a6d9aa6918858f5de1',\n",
       "   'insert_date_': '2020-11-05T08:32:26.069963',\n",
       "   'abstract': 'outbreak new coronaviru sar cov diseas becom pandem effect control diseas critic set optim quarantin period case develop symptom retain isol moment establish quarantin period day base previou report studi small size hospit case 100 respect howev adult child case necessari stay hospit therefor hospit therefor question current infer median incub time repres whole covid popul current quarantin period optim compil analyz patient level inform 2015 laboratori confirm covid case includ children chines provinc cohort repres wide rang spectrum covid diseas hospit non hospit case full rang incub period covid case rang day among 2015 case symptom free case includ femal median age year male median age year median incub period male femal adult similar day significantli shorter day child case cohort contain transmiss gener incub period case gener significantli differ suggest viru rapidli adapt human be interestingli incub period 233 case longer establish quarantin period day data model suggest adult take extra day day isol quarantin period day respect peopl develop symptom effect quarantin patient transmit via lunch dinner parti gastrointestin tract infect intern licens made avail author funder grant medrxiv licens display preprint perpetu certifi peer review',\n",
       "   'title': 'Is a 14-day quarantine period optimal for effectively controlling coronavirus disease 2019 (COVID-19)?',\n",
       "   'body': 'outbreak new coronaviru sar cov diseas covid becom worldwid pandem understand characterist incub period crucial design public health effort effect isol case contact control outbreak covid sar cov transmit either respiratori droplet contamin surfac object oral fecal transmiss rout lead gastrointestin tract infect outbreak occur mani countri critic analyz real time data covid case gain insight inform public health intervent strategi manag prevent diseas one key issu control covid set suitabl quarantin period could captur case develop symptom practic isol effect save public health resourc thu far incub time studi focus hospit patient highli select base medic condit sampl size infer incub time small exampl 100 respect howev major adult children covid case necessari hospit therefor hospit due mild condit ask median incub time infer hospit patient repres whole popul covid current quarantin period determin base hospit patient optim answer question manual curat patient level data case report offici websit china provinci public health agenc cohort repres gener popul covid case without select base medic condit cohort contain 2015 case includ children symptom onset januari 2020 februari 2020 curat case contain exposur onset diagnosi date analyz incub time covid popul contain hospit nonhospit case intern licens made avail author funder grant medrxiv licens display preprint perpetu certifi peer review copyright holder preprint version post march 2020 read case report chines curat age sex demograph inform name provinc citi date exposur symptom onset diagnosi confirm sar cov infect laboratori test avail patient data curat wxc ycc nativ chines speaker translat summari patient english make sure data accuraci curat clean check time four research ywn mainli focus case allow infer incub time among 4567 case remov case exposur date whose exposur date sure obtain 2015 case whose incub time could infer inform covid case hubei provinc includ wuhan citi center epidem publicli avail cohort includ non hubei case studi design data curat patient level data describ character patient baselin inform incub period data studi deidentifi publicli avail local chines health agenc therefor patient consent ethic approv requir incub period covid case time infecti exposur symptom onset 000 case suffici inform infer incub period clear record date exposur symptom onset import case travel wuhan citi exposur date defin date leav epicentr citi wuhan case quarantin monitor diseas diagnos date set date symptom onset test test use statist analysi use free softwar environ statist comput consid valu less signific intern licens made avail author funder grant medrxiv licens display preprint perpetu copyright holder preprint version post march 2020 collect patient level data 2015 covid case report januari februari 2020 tabl supplementari tabl 887 case femal adult median age year iqr 1029 male adult median age year iqr case children median age year iqr boy girl ratio 774 import case report travel histori wuhan 133 resid wuhan 641 %), 240 import case either resid return travel non wuhan region incub period essenti control infecti diseas halt pandem suitabl quarantin period set investig incub period thu understand characterist incub period crucial design public health effort cohort full rang incub period covid case rang day among 2015 case male femal adult median incub period day day longer report see discuss howev similar result releas recent news chines medic associ median incub period day median incub period day children significantli longer adult test adult incub period major case day male femal respect day incub importantli found incub period 233 123 105 male femal respect case longer day current set day quarantin period result agre recent studi also report case whose incub period longer day data suggest offici day quarantin period captur popul develop covid howev among 233 case longer incub period 198 day incub period suggest implement day quarantin period everybodi captur case howev knew fewer children infect intern licens made avail author funder grant medrxiv licens display preprint perpetu copyright holder preprint version post march 2020 http :// doi org 1101 2020 20036533 doi medrxiv preprint plot age distribut case longer quarantin period fig base fig conclud adult take extra day quarantin captur case data model also show day day quarantin period captur respect case incub period male femal adult respect total case day case transmit contact covid patient develop symptom onset yet suggest sar cov might contagi shed viru incub period patient could suscept viru symptom free case includ femal median age year iqr male median age year iqr symptom free case much younger major patient fecal oral transmiss found covid case food share lunch dinner parti could facilit fecal oral transmiss interestingli incub time day iqr case transmit lunch dinner parti contact significantli longer other suggest oral transmiss via share food longer incub period transmit via respiratori droplet gener transmit case base contact trace inform wuhanimport case defin gener case gener case includ non wuhan import case new case transmit directli 1gener case unknown caus gener case one transmit directli gener case incub time significantli differ patient transmiss gener suggest viru rapidli adapt human incub period essenti make intervent strategi control infecti diseas thu understand incub time wide rang spectrum covid establish optim quarantin period crucial design public health intervent two recent studi report incub period covid case hospit patient intern licens made avail author funder grant medrxiv licens display preprint perpetu copyright holder preprint version post march 2020 http :// doi org 1101 2020 20036533 doi medrxiv preprint highli select repres adult child case gener covid popul sampl size 100 case respect infer incub period small studi manual curat covid cohort contain baselin incub inform nearli 000 patient includ children cohort includ hospit non hospit case analyz cohort character incub time covid adult children well examin effect day quarantin period median incub period adult cohort day longer day report previous recent chines medic associ also found coronaviru median incub period day kid incub period day significantli longer adult base previou studi set day quarantin period covid howev found incub period 233 case longer day mean current quarantin polici stop case viru spread find similar anoth report show case incub time longer day gener quarantin period retain case develop symptom effect thu analyz subset 233 found respect men women case develop diseas quarantin everybodi adult alon respect take extra day isol quarantin period day take quarantin period day day respect case develop symptom success quarantin outbreak fast move world base analysi recommend extens adult quarantin period day could effect note outbreak occur chines new year holiday holiday period extend anoth day moreov march 2020 school univers still shut thu due implement aggress social distanc measur mani peopl china isol day intern licens made avail author funder grant medrxiv licens display preprint perpetu copyright holder preprint version post march 2020 http :// doi org 1101 2020 full rang incub period covid case rang zero day studi includ case whose incub period rang day symptom free case suggest wide rang heterogen respons sar cov viru popul challeng prevent viru spread furthermor case start transmit viru symptom onset differ transmiss pattern sar sever acut respiratori syndrom transmiss rare occur day symptom onset viru infecti symptom onset control outbreak use contact trace isol challeng interestingli case transmit lunch dinner parti contact day longer incub period case transmit via respiratori droplet chines style dish foodshar lunch dinner parti easili contamin share dish viru carrier thu could facilit oral transmiss lead gastrointestin tract infect report transmiss mean covid report patient present diarrhea nausea day develop fever respiratori symptom observ explain case transmit lunch dinner parti contact significantli longer incub period therefor earli sign case transmit lunch dinner parti could diarrhea nausea often develop fever respiratori symptom patient sever limit studi follow cohort includ patient level inform patient hubei provinc includ wuhan citi exhaust deal outbreak could make inform offici websit case chines patient acknowledg design project wxc ycc manual curat data translat summari patient english ywn check curat clean data ywn conduct data analysi wrote manuscript work support aish translat chair program cancer genom intern licens made avail author funder grant medrxiv licens display preprint perpetu certifi peer review copyright holder preprint version post march 2020 distribut case longer incub period figur show age case incub day day day longer day red blue green repres men women children respect intern licens made avail author funder grant medrxiv licens display preprint perpetu certifi peer review copyright holder preprint version post march 2020 http :// doi org 1101 2020 20036533 doi medrxiv preprint',\n",
       "   '_search_score': 0.6793499999999999},\n",
       "  {'_id': 'a911adbaf5828c2f84558d48e2e72e00810069aa',\n",
       "   'insert_date_': '2020-11-05T08:32:55.838074',\n",
       "   'abstract': 'object sever coronaviru diseas 2019 covid assess term absolut mortal sar cov posit cohort assess mortal rel mortal gener popul present design retrospect popul base studi set individu inform symptomat confirm sar cov patient subsequ death caus compar caus mortal swiss popul 2018 start februari 2020 mortal covid patient monitor day compar popul mortal observ time year start februari 2018 particip 160 595 inhabit switzerland age without covid gener popul spring 2018 769 person test posit covid spring 2020 measur sex age specif mortal rate estim use cox proport hazard model absolut probabl death predict risk assess term rel mortal take ratio sex age specif absolut mortal covid patient correspond mortal 2018 gener popul result confirm sar cov infect substanti increas probabl death across patient group rang nine time popul mortal year old infect femal fold increas year old infect male highest rel risk observ among male older patient magnitud effect smaller compar increas observ absolut mortal risk male covid patient exceed popul hazard male hazard ratio addit year age increas popul hazard covid patient hazard ratio limit inform distribut relev comorbid avail popul level associ risk quantifi conclus health care profession decis maker societi provid addit popul adjust assess covid mortal risk combin absolut measur risk rel risk present help develop comprehens understand actual impact covid',\n",
       "   'title': 'Relative Coronavirus Disease 2019 Mortality: A Swiss Population-based Study',\n",
       "   'body': 'earli report china itali diseas sever covid caus unpreced public health intervent around world rang social distanc measur school univers close complet lockdown societi absolut mortal patient diagnos covid case fatal rate main entiti use commun risk associ diseas risk factor mortal risk promin higher age male preexist medic condit becom public known current occas liberalis stringent public health intervent mani countri assess actual impact covid pandem call health care profession politician societi larg current engag discuss appropri mitig measur taken first scientif contribut matter arisen probabl death estim hundr thousand covid patient constantli report mani countri number howev hard compar due differ test regim vari ascertain mortal differ age structur societi differ health care public health system altern risk differ diseas impact excess number death report popul number observ caus death time covid pandem end februari mid may 2020 european countri minu expect number death given popul report number excess death observ sinc onset pandem avail portug spain northern itali variou european countri unit state howev cross countri comparison difficult success failur public health intervent possibl overrun hospit capac reflect presenc magnitud excess mortal emerg matur data cours covid pandem mani countri allow actual impact societi health care system discuss light shortterm rel mortal rel risk compar absolut caus mortal observ patient diagnos covid spring 2020 outbreak absolut caus mortal uninfect popul earlier year calendar time year popul mortal vari femal male attain age male older individu experienc higher short term risk die much increas mortal report male older covid patient attribut increas popul mortal risk gener import question await answer furthermor sex age specif rel covid mortal allow stratifi assess risk increas caus sar cov infect report age adjust rel covid mortal femal male base analysi individu popul level covid death record switzerland cover time intern licens made avail author funder grant medrxiv licens display preprint perpetu copyright holder preprint version post june 2020 http :// doi org 1101 2020 20127670 doi medrxiv preprint 2020 2020 popul base studi swiss gener popul data 2014 2018 includ offici sar cov test switzerland perform polymeras chain reaction pcr base lower upper respiratori tract sampl symptomat person individu experienc follow symptom elig test cough sore throat muscl pain dyspnea without fever acut anosmia ageusia test asymptomat person recommend control local outbreak hospit nurs home posit neg sar cov test result directli report feder offic public health patient follow subsequ studi popul consist two cohort first cohort consist person sar cov posit pcr test year age time test individu test post mortem hospitalis includ aim studi rel mortal cohort newli infect peopl short term increas mortal risk refer cohort swiss covid cohort sequel second cohort defin inhabit switzerland aliv februari 2018 age year sex age specif hazard ratio hr confid interv ci estim use stratifi cox proport hazard model allow separ baselin rate swiss covid swiss 2018 popul cohort year old femal refer sex age model main effect swiss 2018 cohort wherea swiss covid cohort interact effect sar cov statu sex age includ captur addit mortal effect among patient valu confid interv hazard ratio adjust multipl quantifi impact mortal associ sar cov diagnosi day probabl death predict fit model estim captur death occur within day februari posit test swiss 2018 popul cohort swiss covid cohort respect use probabl sex specif rel mortal associ confid band calcul take ratio two along age gradient year rel mortal incorpor uncertainti swiss covid cohort assumpt proport hazard assess fit model allow time vari effect potenti deviat linear age effect assess cox model allow nonlinear effect age main effect model compar model includ sex age interact sensit analysi model fit use swiss 2014 2017 gener popul refer detail statist analys perform found onlin supplement analys perform system statist comput daili number death observ day studi period 2020 2020 swiss covid cohort increas rapidli mid march peak first day april male femal figur number reduc less ten report death per day last week studi period figur intern licens made avail author funder grant medrxiv licens display preprint perpetu certifi peer review copyright holder preprint version post june 2020 http :// doi org 1101 2020 20127670 doi medrxiv preprint characterist swiss covid cohort swiss 2018 popul cohort earlier swiss popul cohort 2014 2015 2016 2017 present tabl ratio femal male indic patient swiss covid cohort like femal mean age similar two cohort sar cov posit patient one half femal three year male older individu swiss popul cohort similar observ compar whole age distribut among femal male cohort figur howev slight represent older peopl year age sar cov posit individu tabl caus mortal rate among year old male swiss 2018 popul cohort time among femal age mortal rate increas factor addit year age tabl 2018 male experienc mortal femal age year older addit increment mortal rate individu swiss covid cohort smaller sex age effect observ swiss 2018 popul cohort differ log hr popul patient sex effect age effect male covid patient associ rel male gener popul tabl hazard ratio compar hazard male femal swiss covid cohort addit year age increas covid mortal rate factor tabl day sex age specif probabl death consider larger covid cohort figur year old infect male associ probabl death around equival year old infect male gener popul probabl death infect individu femal older male older year rel mortal year old covid patient approxim three time higher rel mortal year old covid patient intern licens made avail author funder grant medrxiv licens display preprint perpetu certifi peer review probabl death impli nearli log linear function age femal male cohort studi popul henc follow rel risk compar mortal young patient mortal patient age year older exampl would constant regardless age younger group patient slope probabl death function age larger swiss covid cohort due model age effect attribut infect tabl covid age figur sex age specif rel mortal figur bottom panel tabl rang year old femal year old male interpret year old infect femal time like die within day posit test year old uninfect femal like die within day follow februari 2018 fatal time like year old infect versu non infect male howev confid band demonstr substanti uncertainti estim rel risk lower confid band year old person larg femal male rel mortal largest year old femal year old male tabl nine fold increas probabl death found femal male young old symptomat covid patient switzerland comparison swiss popul 2018 provid novel sex age specif inform sever pandem covid mortal never previous report rel risk direct relat mortal gener popul comparison absolut mortal case fatal rate risk group covid patient without popul adjust like estim increas mortal male older peopl present half hazard increas observ male older patient could attribut gener increas popul mortal risk group intern licens made avail author funder grant medrxiv licens display preprint perpetu copyright holder preprint version post june 2020 http :// doi org 1101 2020 20127670 doi medrxiv preprint popul base set match calendar time allow estim covid relat mortal unselect cohort symptomat diagnos patient without need differenti death caus caus covid term death certif inform season effect implicitli account compar cohort time year unlik popul studi estim excess mortal probabl death cohort could confound ongo public health intervent test coverag use data switzerland especi use type analysi border northern itali austria caus covid outbreak earli pandem broad uniform symptom base test overseen feder author implement quickli test result report symptom base test regim carri test asymptomat person recommend control local outbreak hospit nurs home howev number hospit start test patient admit hospit differ time point regardless symptom contrast report northern itali number covid patient need hospitalis never exceed health care capac everi patient receiv best possibl treatment circumst absolut mortal swiss covid cohort smaller report italian chines covid patient year old number expect higher germani overal case fatal rate test perform younger patient mild symptom due differ test protocol substanti higher lower rel mortal may therefor expect countri swiss covid cohort exclud person known increas mortal risk test posthum hospitalis well old thu priori frail person figur present nevertheless inform model develop comput prognos number expect death real hypothet popul rel mortal affect public health intervent lead reduc even nonexist excess mortal mani european countri prognos assum year rel mortal risk larger two uniformli femal male age short term day rel mortal larger nine found suggest actual risk larger assum base prognost model howev gener question true covid rel mortal lower report studi inclus criteria covid cohort defin symptomat test posit sar cov true proport asymptomat case actual preval ongo question sex age effect caus mortal attribut covid howev still relev male older person associ higher risk male higher mortal rate femal howev uncertainti around hazard ratio larg effect absent popul earlier year model sex age interact see onlin supplement rel mortal ratio probabl death approxim three time higher among year old year old covid patient popul adjust comparison risk young old patient suggest less drastic rel impact age intern licens made avail author funder grant medrxiv licens display preprint perpetu copyright holder preprint version post june 2020 http :// doi org 1101 2020 20127670 doi medrxiv preprint justifi base continu increas age specif rel mortal report result suggest covid risk assess term case fatal rate excess mortal complement popul adjust rel mortal complet pictur emerg potenti lead improv age base risk stratif contribut mjc design studi preprocess analys data prepar initi draft author review improv earli version manuscript approv final version fund swiss nation scienc foundat funder studi role studi design data collect data analysi data interpret write final report compet interest paid consult lectur present novarti roch pricewaterhousecoop hfg receiv unrestrict research grant gilead scienc roch fee data safeti monitor board membership merck consult advisori board membership fee gilead scienc merck viiv healthcar grant systemsx nation institut health mjc paid consult lectur present roch reinsur group america servic limit data share result reproduc studi cohort stata comput code public avail http :// gitlab switch torsten hothorn relative_covid 19_ mortal intern licens made avail author funder grant medrxiv licens display preprint perpetu copyright holder preprint version post june 2020 intern licens made avail author funder grant medrxiv licens display preprint perpetu copyright holder preprint version post june 2020 http :// doi org 1101 2020 20127670 doi medrxiv preprint intern licens made avail author funder grant medrxiv licens display preprint perpetu copyright holder preprint version post june 2020 intern licens made avail author funder grant medrxiv licens display preprint perpetu copyright holder preprint version post june 2020 intern licens made avail author funder grant medrxiv licens display preprint perpetu copyright holder preprint version post june 2020 intern licens made avail author funder grant medrxiv licens display preprint perpetu copyright holder preprint version post june 2020 intern licens made avail author funder grant medrxiv licens display preprint perpetu copyright holder preprint version post june 2020 intern licens made avail author funder grant medrxiv licens display preprint perpetu copyright holder preprint version post june 2020 intern licens made avail author funder grant medrxiv licens display preprint perpetu copyright holder preprint version post june 2020 intern licens made avail author funder grant medrxiv licens display preprint perpetu copyright holder preprint version post june 2020 intern licens made avail author funder grant medrxiv licens display preprint perpetu copyright holder preprint version post june 2020 intern licens made avail author funder grant medrxiv licens display preprint perpetu copyright holder preprint version post june 2020 popul cohort given year inhabit switzerland aliv age year old februari enter risk set inhabit surviv day censor day die within day februari record event abl evalu mortal rate ratio timescal allow season effect covid cohort delay entri approach taken patient becom risk death date posit test death cohort treat event individu without death record administr right censor day februari therefor model time death caus within day februari first posit sar cov test record februari 2020 databas close 2020 mortal rate swiss covid cohort compar mortal rate swiss 2018 popul cohort time year hazard ratio estim cox proport hazard model express distribut time death februari day fit cox model allow separ baselin hazard function assum 2018 baselin mortal rate time year old femal swiss 2018 popul cohort covid describ mortal rate swiss covid cohort calendar timescal account season effect take valu individu covid cohort individu 2018 cohort linear predictor parameteris way quantifi effect age male gener popul hazard ratio exp exp respect addit increas decreas hazard attribut age male covid cohort describ exp exp respect proport hazard assumpt assess compar sampl log likelihood model describ sampl log likelihood cox model time vari sex age effect simpler model assum proport hazard maximis log likelihood 5386 model assum proport hazard 5379 suggest mortal appropri model proport hazard estim comparison absolut risk probabl die time model mortal rate covid cohort time sinc diagnosi timescal therefor model 2018 cohort remain interest probabl death day februari 2018 covid cohort interest probabl death day diagnosi two separ cox model fit cohort appropri timescal rel mortal time defin sex age sex age sex age intern licens made avail author funder grant medrxiv licens display preprint perpetu copyright holder preprint version post june 2020 http :// doi org 1101 2020 20127670 doi medrxiv preprint supplementari tabl swiss 2018 popul covid cohort log hazard ratio log hazard ratio express risk male male year age age compar baselin hazard year old femal base cox proport hazard model sex age interact estim given standard error se log hr confid interv hr latter valu adjust multipl log valu femal ratio probabl sex age die within day posit sar cov test probabl sex age die within day februari 2018 without infect femal male person specif age use time horizon day rel mortal femal mean probabl die within sar cov infect time larger popul probabl die within day februari 2018 year old femal proport hazard model absolut risk estim log cumul baselin hazard function parameteris term bernstein polynomi use parametr equival semiparametr cox model model assumpt linear age effect assess compar linear cox proport hazard model cox proport model featur nonlinear addit effect age log hazard figur 2018 popul cohort differ margin justifi use linear age term swiss covid cohort nonlinear model suggest stronger age effect year level older peopl discrep nonlinear linear cox model term estim probabl death day infer figur 2018 popul cohort probabl death model similar variat peopl older year explain sharp increas correspond loghazard ratio figur age rang year two model rel similar %), howev nonlinear model suggest probabl death reduc compar probabl death linear model mean rel mortal assum linear effect age log hazard scale figur tabl could two time smaller peopl younger two time larger person around year old complex model allow nonlinear effect age also consid interact sex age found differ tabl intern licens made avail author funder grant medrxiv licens display preprint perpetu copyright holder preprint version post june 2020 comput perform version cox proport hazard model fit semiparametr estim log hazard ratio use surviv packag absolut mortal obtain cox model parametr parameteris flexibl logcumul baselin hazard function use packag mlt confid band comput use multcomp packag primari analys conduct howev ensur valid robust analys replic independ stata use flexibl parametr surviv model use stpm2 merlin command consid parametr equival cox model use restrict cubic spline directli model baselin log cumul hazard function found complet agreement across softwar platform model approach furthermor sensit main find report assess conduct analys use popul data 2014 2017 independ compar popul instead 2018 cohort age distribut baselin cumul hazard studi popul given figur absolut rel mortal obtain figur hazard ratio studi popul present tabl deviat main result margin intern licens made avail author funder grant medrxiv licens display preprint perpetu copyright holder preprint version post june 2020 intern licens made avail author funder grant medrxiv licens display preprint perpetu copyright holder preprint version post june 2020 http :// doi org 1101 2020 20127670 doi medrxiv preprint intern licens made avail author funder grant medrxiv licens display preprint perpetu copyright holder preprint version post june 2020 intern licens made avail author funder grant medrxiv licens display preprint perpetu copyright holder preprint version post june 2020 intern licens made avail author funder grant medrxiv licens display preprint perpetu copyright holder preprint version post june 2020 intern licens made avail author funder grant medrxiv licens display preprint perpetu certifi peer review copyright holder preprint version post june 2020 intern licens made avail author funder grant medrxiv licens display preprint perpetu certifi peer review copyright holder preprint version post june 2020 intern licens made avail author funder grant medrxiv licens display preprint perpetu certifi peer review copyright holder preprint version post june 2020 intern licens made avail author funder grant medrxiv licens display preprint perpetu certifi peer review copyright holder preprint version post june 2020 intern licens made avail author funder grant medrxiv licens display preprint perpetu certifi peer review copyright holder preprint version post june 2020 http :// doi org 1101 2020 20127670 doi medrxiv preprint',\n",
       "   '_search_score': 0.6775235999999998},\n",
       "  {'_id': 'a17525ba0365d376469cb2e8faeb0bad6d6ed682',\n",
       "   'insert_date_': '2020-11-05T08:31:43.650242',\n",
       "   'abstract': 'background novel sever acut respiratori syndrom coronaviru sar cov caus rapidli spread ill coronaviru diseas 2019 covid affect thousand peopl around world urgent guidanc clinician care sickest patient need form panel expert countri panel member complet world health organ conflict interest disclosur form panel propos question relev manag covid icu search literatur direct indirect evid manag covid critic ill patient icu identifi relev recent systemat review question relat support care assess certainti evid use grade recommend assess develop evalu grade approach gener recommend base balanc benefit harm resourc cost implic equiti feasibl recommend either strong weak form best practic recommend surviv sepsi campaign covid panel issu statement best practic statement strong recommend weak recommend recommend provid question topic infect control laboratori diagnosi specimen hemodynam support ventilatori support covid therapi',\n",
       "   'title': 'Surviving Sepsis Campaign: guidelines on the management of critically ill adults with Coronavirus Disease 2019 (COVID-19)',\n",
       "   'body': 'end 2019 novel coronaviru name sever acut respiratori syndrom coronaviru sar cov result acut respiratori ill epidem wuhan china world health organ term ill coronaviru diseas 2019 time guidelin panel assembl covid becom pandem affect 120 000 individu countri result 5000 death worldwid unit state center diseas control prevent cdc issu preliminari guidanc infect control screen diagnosi gener popul limit guidanc acut manag critic ill patient sever ill due covid guidelin provid recommend support hospit clinician manag critic ill adult covid intens care unit icu target user guidelin frontlin clinician alli health profession policymak involv care patient covid icu guidelin appli high low middl incom set surviv sepsi campaign ssc covid subcommitte select panel member way obtain balanc topic expertis geograph locat far possibl gender ssc covid panel assembl work within tight timelin order issu recommend time manner panel includ expert guidelin develop infect control infecti diseas microbiolog critic care emerg medicin nurs public health panel divid four group infect control test hemodynam support ventilatori support therapi guidelin intens care develop evalu guid group provid methodolog support throughout guidelin develop process panel member complet conflict interest coi form prior join guidelin panel use gradepro guidelin develop tool gdt onlin softwar http :// gdt guid line evelo pment org administ coi disclosur form particip panel member direct financi industri relat coi permit consid disqualifi develop guidelin includ industri input fund financi non financi contribut member guidelin panel receiv honoraria remuner role guidelin develop process guidelin develop process summar fig action guidelin question structur popul intervent control outcom pico format explicit definit wherea descript question content method expert group particip develop guidelin question pico format provid basi defin inclus exclus criteria literatur search perform identif relev studi facilit rapid develop recommend perform novel systemat priorit outcom use outcom priorit inform ongo ssc guidelin 2020 work expert input accordingli focus hospit mortal seriou advers event outcom question includ outcom deem critic decis make question help profession medic librarian electron search major databas cochran central medlin identifi relev systemat review random control trial rct observ studi case seri electron search perform look studi publish english incept march 2020 inform recommend hemodynam ventilatori support use recent publish systemat review ask expert identifi new relev studi select pico question pair review screen titl abstract retriev bibliograph databas pico question potenti elig studi assess elig accord prespecifi criteria content expert ask indic addit studi identifi search subsequ pair review independ abstract relev data correspond pico question item relev risk bia obtain intent treat data whenev avail otherwis use complet case data ignor miss data use grade recommend assess develop evalu grade approach assess qualiti evid confid estim effect support recommend qualiti evid rate high moder low low use gdt onlin softwar http :// gdt guid line evelo pment org gener evid profil evid summari given recent emerg covid anticip would scarciti direct evid therefor use predefin algorithm decid whether indirect evid could inform specif question figur ssc covid panel decid popul extrapol evid base context recommend likelihood presenc effect modifi figur accordingli use sourc indirect evid data middl east respiratori syndrom coronaviru mer cov sever acut respiratori syndrom sar coronavirus way consid indirect evid publish data support care icu studi influenza respiratori viral infect acut respiratori distress syndrom ard sepsi use principl outlin evid decis framework etd formul recommend tight timelin complet onlin etd tabl etd framework cover follow domain prioriti set magnitud benefit harm certainti evid patient valu balanc desir undesir effect resourc cost equiti accept feasibl four subgroup draft preliminari recommend use word recommend strong recommend suggest suggest weak recommend implic recommend strength present tabl final list recommend develop panel discuss consensu vote recommend requir present guidelin statement recommend tabl period autom electron search sent assign panel member everi week identifi relev new evid emerg accordingli issu guidelin releas order updat recommend need formul new one recent report chines center diseas control prevent describ 314 case covid china 672 laboratori confirm among laboratori confirm case 1716 healthcar worker 1080 1716 acquir infect wuhan report describ 247 1668 infect healthcar worker sever critic ill die itali march 2020 2026 document covid case among healthcar worker although incid data avail data point consider burden infect among healthcar worker risk patient patient transmiss icu current unknown therefor adher infect control precaut paramount healthcar worker follow infect control polici procedur alreadi place healthcar institut provid follow recommend suggest consider rather requir chang institut infect control polici healthcar worker perform aerosol gener procedur patient covid icu recommend use fit respir mask n95 respir ffp2 equival oppos surgic medic mask addit person protect equip glove gown eye protect face shield safeti goggl best practic statement healthcar worker perform aerosol gener procedur patient covid icu recommend use fit respir mask n95 respir ffp2 equival oppos surgic medic mask addit person protect equip glove gown eye protect face shield safeti goggl best practic statement neg pressur room engin control intend prevent spread contagi airborn pathogen room room measl tuberculosi main goal avoid accident releas pathogen larger space open facil therebi protect healthcar worker patient hospit set neg air pressur creat patient room keep pathogen insid avoid diffus adopt precaut aerosol gener procedur like tracheal intub bronchoscopi non invas posit pressur ventil nippv perform within room lower risk crosscontamin among room infect staff patient outsid room neg pressur creat maintain ventil system allow extra air enter isol room differenti pressur exhaust directli outsid filter high effici particul air hepa filter directli recircul moreov presenc unnecessari staff room avoid neg pressur room proven effect measur help avoid cross contamin sar epidem accordingli aerosolgener procedur guidanc covid recommend use neg pressur room minimum air chang per hour least 160 second patient facil natur ventil bronchoscopi among procedur highest risk aerosol use minim non invas ventil also high risk aerosol strategi describ contain risk viru spread also accord previou report sar infect feasibl portabl hepa filter use room wherev possibl hepa filter mechan air filter use isol maximum reduct remov submicron particul matter air requir hepa filter demonstr reduc viru transmiss simul set healthcar worker provid usual care non ventil covid patient suggest use surgic medic mask oppos respir mask addit person protect equip glove gown eye protect face shield safeti goggl weak recommend low qualiti evid healthcar worker perform non aerosol gener procedur mechan ventil close circuit patient covid suggest use surgic medic mask oppos respir mask addit person protect equip glove gown eye protect face shield safeti goggl weak recommend low qualiti evid recommend line guidanc current evid suggest surgic medic mask probabl inferior n95 respir provid protect laboratori confirm season respiratori viral infect influenza measl updat recent systemat review meta analysi rct identifi one new rct overal rct 5549 individu random healthcar worker n95 respir medic mask use medic mask oppos n95 respir increas laboratori confirm respiratori infect although point estim suggest use medic mask associ increas risk influenza like ill clinic respiratori infect differ statist signific recommend insuffici evid issu recommend use recombin rifn alon combin antivir critic ill adult covid recommend insuffici evid issu recommend use chloroquin hydroxychloroquin critic ill adult covid recommend insuffici evid issu recommend use tocilizumab critic ill adult covid recommend recent systemat review meta analysi reach similar conclus one rct report coronaviru test season coronaviru oc43 hku1 229e nl63 mean pcr non cluster rct 212 nurs medic mask group pcr confirm coronaviru infect compar 210 n95 respir group make recommend panel consid lack convinc evid n95 respir improv clinic outcom cost resourc associ n95 mask use need preserv n95 respir suppli aerosol gener procedur therefor panel issu suggest use medic mask context howev sar cov appear easili transmiss lethal season influenza specif earli estim reproduct number sar cov averag number peopl infect person subsequ infect function biolog properti pathogen combin social environment factor comparison estim averag 1918 influenza pandem result estim million death global estim averag season influenza therefor minimum surgic medic mask recommend healthcar worker care non ventil covid patient healthcar worker perform non aerosol gener procedur mechan ventil close circuit patient covid scarciti issu use fit respir use fit respir mask reason option patient covid suggest use video guid laryngoscopi direct laryngoscopi avail weak recommend low qualiti evid direct evid compar use videolaryngoscopi direct laryngoscopi intub patient covid sar cov appear predominantli spread larg respiratori droplet intub like small particl less aerosol gener procedur increas risk transmiss healthcar worker intub particularli riski given close contact healthcar worker patient airway respiratori secret thu techniqu reduc number attempt endotrach intub durat procedur minim proxim oper patient priorit potenti reduc risk complic hypox covid patient systemat review includ studi 7044 patient video laryngoscopi reduc risk fail intub without signific impact upon proport success first pass attempt hypoxia time tracheal intub patient difficult airway first attempt success rate may improv video laryngoscopi thu set video laryngoscopi avail staff skill use suggest use prefer direct laryngoscopi maxim chanc success recogn center rapid access video laryngoscopi skill user recommend condit covid patient requir endotrach intub recommend endotrach intub perform healthcar worker experienc airway manag order minim number attempt risk transmiss best practic statement similar reason factor maxim chanc first pass success use intub patient suspect confirm covid thu recommend healthcar oper experi skill airway manag first attempt intub recent declar covid pandem accordingli everi critic ill patient arriv evid respiratori infect consid potenti infect sar cov real time polymeras chain reaction pcr gold standard similar viral infect includ sar notabl covid pose sever diagnost challeng due extend incub period approxim week includ prolong interv approxim day viral shed prior onset symptom moreov durat asymptomat shed variabl may also differ base anatom level upper versu lower infect respiratori system accordingli perform biomolecular assay may vari site sampl diagnost test suggest obtain lower respiratori tract sampl prefer upper respiratori tract nasopharyng oropharyng sampl weak recommend low qualiti evid regard lower respiratori sampl suggest obtain endotrach aspir prefer bronchial wash bronchoalveolar lavag sampl weak recommend low qualiti evid covid diagnosi base pcr test respiratori sampl nasopharyng oropharyng swab lower respiratori tract sampl whenev possibl bronchoalveolar lavag limit perform indic adequ precaut due risk aerosol consequ exposur healthcar profession similarli sputum induct avoid due increas risk aerosol tracheal aspir specimen appear carri lower risk aerosol sometim obtain without disconnect patient ventil procedur involv laboratori pcr test sar cov use number assay current use well describ despit gener high sensit specif pcr base assay may enough reli oropharyng swab specimen alon sar cov diagnosi due low neg predict valu recent studi oropharyng swab covid patient test posit pcr similar data report use pcr 2002 2003 sar epidem use seroconvers gold standard sar diagnosi pcr assay perform nasopharyng throat specimen posit time respect howev fals posit observ indic assay specif 100 similarli studi account scan find among suspect covid case neg oropharyng nasal swab consid highli like case consid probabl case consequ singl neg swab upper airway rule sar cov infect repeat sampl multipl site includ lower airway increas diagnost yield similarli given coinfect viral pathogen observ posit test anoth respiratori viru rule covid delay test high suspicion covid given high specif singl posit swab confirm diagnosi covid enough trigger infect control precaut appropri treatment patient lower respiratori tract specimen consid give higher diagnost yield upper respiratori specimen patient pneumonia consist observ sar therefor obtain whenev possibl report preval shock adult patient covid highli variabl %), depend patient popul studi sever ill definit shock recent report summar epidemiolog characterist 415 chines patient covid 2087 diagnos critic case defin sever hypoxemia presenc organ failur includ shock anoth chines studi 1099 patient covid similar sever ill develop shock hospit patient incid like higher tabl may reach among patient icu cardiac injuri elev cardiac injuri biomark 99th percentil upper refer limit report patient covid wuhan china preval cardiac injuri may correl preval shock lack systemat screen cardiac dysfunct hemodynam stabl patient mean associ cannot taken certain tabl prognosi patient covid shock systemat report studi 150 patient hospit wuhan china shock major reason death may least part due fulmin myocard studi risk factor associ shock patient covid lack major avail report unadjust estim despit methodolog limit studi suggest older age comorbid especi diabet cardiovascular diseas includ hypertens lower lymphocyt count higher dimer level possibl cardiac injuri risk factor consid recommend adult covid shock suggest use dynam paramet skin temperatur capillari refil time serum lactat measur static paramet order assess fluid respons weak recommend low qualiti evid direct evid address optim resuscit strategi patient covid shock therefor panel base recommend indirect evid drawn critic ill patient gener systemat review meta analysi rct 1652 examin effect dynam assess fluid therapi import patient outcom adult icu patient requir fluid resuscit use dynam assess guid fluid therapi found reduc mortal icu length stay day durat mechan ventil note one trial focus patient septic shock dynam paramet use trial includ stroke volum variat svv puls pressur variat ppv stroke volum chang passiv leg rais fluid challeng among examin dynam paramet passiv leg rais follow ppv svv appear predict fluid respons highest accuraci static paramet includ compon earli goal direct therapi central venou pressur cvp mean arteri pressur map use serum lactat level guid resuscit patient shock summar systemat review meta analysi rct 1301 compar central venou oxygen satur scvo2 guid therapi earli lactat clearanc direct therapi associ reduct mortal shorter icu length stay day shorter durat mechan ventil howev high lactat level alway indic hypovolemia may also caus mitochondri dysfunct liver failur beta agonist mesenter ischemia epinephrin andromeda shock trial capillari refil test crt everi min associ nonsignific reduct mortal compar serum lactat measur everi crt simpl easi test use almost set given possibl improv mortal length stay durat mechan ventil may produc well avail suggest use dynam paramet skin temperatur capillari refil time lactat measur static paramet assess fluid respons patient covid shock acut resuscit adult covid shock suggest use conserv liber fluid strategi weak recommend low qualiti evid direct evid exist patient covid shock therefor panel use indirect evid critic ill patient sepsi ard inform recommend recent systemat review rct 637 patient compar restrict versu liber fluid volum initi resuscit patient sepsi found statist signific differ mortal seriou advers event howev assess outcom favor conserv fluid therapi lower volum importantli quantiti qualiti evid judg low suggest research need correspondingli 2017 meta analysi rct 2051 patient adult children ard sepsi manag accord conserv fluid strategi post resuscit phase critic ill ventil free day shorter icu stay patient manag accord liber fluid strategi see section respiratori support detail 2011 larg rct 3141 febril african children feast found children random fluid bolus salin albumin increas mortal compar children receiv fluid bolus absenc data demonstr benefit use liber fluid strategi critic ill patient sepsi ard consid major covid patient icu develop ard suggest initi conserv approach fluid resuscit patient covid shock recommend use crystalloid colloid strong recommend moder qualiti evid sinc exist direct evid shock patient covid panel base recommend indirect evid critic ill patient gener systemat review rct 020 patient compar use crystalloid versu colloid critic ill patient outcom favor use colloid consid colloid harm see colloid costli crystalloid avail colloid limit set low middl incom countri recommend use crystalloid fluid resuscit patient covid shock suggest use buffer balanc crystalloid unbalanc crystalloid weak recommend moder qualiti evid direct evid address question patient covid shock panel therefor base recommend indirect evid critic ill patient gener systemat review meta analysi rct 213 patient compar intraven buffer balanc crystalloid solut versu salin resuscit critic ill adult children report signific differ hospit mortal acut kidney injuri treatment howev point estim outcom suggest potenti benefit buffer crystalloid solut absenc appar harm consid roughli equival cost suggest use buffer crystalloid solut unbalanc crystalloid solut resuscit patient covid shock set limit avail buffer solut salin remain reason altern acut resuscit adult covid shock recommend use hydroxyethyl starch strong recommend moder qualiti evid given absenc direct evid patient covid shock panel base recommend indirect evid critic ill patient gener systemat review rct 020 patient compar use crystalloid versu colloid critic ill patient rct 177 patient compar use crystalloid use starch data pool statist signific differ mortal observ end follow within day within day author howev report increas risk blood transfus renal replac therapi rrt starch given risk clinic signific harm appar absenc benefit use hydroxyethyl starch recommend use resuscit patient covid shock suggest use gelatin weak recommend low qualiti evid sinc studi evalu question patient covid shock panel base recommend indirect evid critic ill patient gener systemat review rct 020 patient compar crystalloid versu colloid use critic ill patient crystalloid compar gelatin rct 1698 statist signific differ caus mortal observ end follow within day within day although point estim favor use crystalloid consid absenc benefit gelatin higher cost suggest use gelatin resuscit patient covid shock acut resuscit adult covid shock suggest use dextran weak recommend low qualiti evid given absenc direct evid patient covid shock panel base recommend indirect evid critic ill patient gener systemat review meta analysi crystalloid versu colloid use critic ill patient identifi trial compar crystalloid dextran 4736 report similar mortal rate end follow within day possibl increas risk blood transfus dextran arm view possibl increas risk blood transfus bleed higher cost associ dextran suggest use resuscit patient covid shock suggest routin use albumin initi resuscit weak recommend moder qualiti evid sinc direct evid patient covid shock panel base recommend indirect evid critic ill patient gener systemat review meta analysi identifi rct 047 compar albumin crystalloid use demonstr signific differ caus mortal end follow absenc benefit albumin consid cost limit avail suggest routin use initi resuscit patient covid shock recommend adult covid shock suggest use norepinephrin first line vasoact agent agent weak recommend low qualiti evid direct evid patient covid shock therefor panel base recommend indirect evid critic ill patient gener systemat review rct 3497 patient clinic practic guidelin 2016 summar avail bodi evid best first line vasopressor patient shock norepinephrin wide studi vasoact agent low priori risk undesir effect suggest use norepinephrin first line vasoact agent patient covid shock norepinephrin avail suggest use either vasopressin epinephrin first line vasoact agent vasoact agent adult covid shock weak recommend low qualiti evid absenc direct evid patient covid shock panel base recommend indirect evid critic ill patient gener systemat review rct 3497 patient norepinephrin compar vasopressin epinephrin trial directli compar two option norepinephrin avail suggest use either vasopressin epinephrin agent assess rct without show clear evid harm factor determin choic vasopressin epinephrin may includ avail contraind two agent vasopressin digit ischemia may concern epinephrin tachycardia excess lactat product may consider use dopamin norepinephrin avail strong recommend high qualiti evid direct evid address question patient covid shock panel base recommend indirect evid critic ill patient 2016 cochran systemat review found rct 1400 compar norepinephrin dopamin patient shock pool result show signific differ caus mortal point estim favor norepinephrin increas risk arrhythmia found dopamin arm basi increas risk harm includ possibl increas risk mortal patient treat dopamin recommend use dopamin patient covid shock norepinephrin altern avail see recommend adult covid shock suggest ad vasopressin second line agent titrat norepinephrin dose target mean arteri pressur map cannot achiev norepinephrin alon weak recommend moder qualiti evid absenc data patient covid shock panel base recommend indirect evid critic ill patient gener recent clinic practic guidelin use vasopressin vasopressin analog critic ill adult distribut shock assess analyz rct 3737 patient author found low certainti reduct mortal high certainti reduct atrial fibril moder certainti increas risk digit ischemia addit vasopressin analog catecholamin anoth recent systemat review reach similar conclus view find suggest ad vasopressin second line agent titrat norepinephrin dose target map cannot achiev norepinephrin alon patient covid shock adult covid shock suggest titrat vasoact agent target map mmhg rather higher map target weak recommend moder qualiti evid direct evid inform recommend base indirect evid critic ill patient recent individu patient data meta analysi rct 894 patient compar higher versu lower blood pressur target vasopressor therapi adult patient shock report signific differ risk arrhythmia increas patient alloc higher target group correspondingli recent publish trial report absolut risk differ mortal favor map target mmhg lower target compar standard care map target higher target indic improv outcom lower map target firm indic harm suggest titrat vasoact agent target mmhg adult covid shock evid cardiac dysfunct persist hypoperfus despit fluid resuscit norepinephrin suggest ad dobutamin increas norepinephrin dose weak recommend low qualiti evid absenc direct evid patient covid shock panel use indirect evid critic ill patient inform recommend clinic practic guidelin 2018 assess optim inotrop agent patient acut circulatori failur shock rct compar dobutamin placebo treatment identifi base physiolog rational suggest ad dobutamin treatment patient covid shock evid cardiac dysfunct persist hypoperfus despit fluid resuscit high dose norepinephrin use dobutamin shock includ covid patient shock research prioriti adult covid refractori shock suggest use low dose corticosteroid therapi shock revers \"), corticosteroid therapi weak recommend low qualiti evid remark typic corticosteroid regimen septic shock intraven hydrocortison 200 per day administ either infus intermitt dose sinc data exist use steroid patient covid shock panel base recommend indirect evid critic ill patient gener time resolut shock length stay especi icu import cost consider suggest use low dose corticosteroid therapi patient covid refractori shock provid guidanc patient covid respiratori failur absenc refractori shock preval hypox respiratori failur patient covid recent report china show covid patient studi receiv non invas posit pressur ventil nippv requir invas mechan ventil tabl although true incid hypox respiratori failur patient covid clear appear develop sever diseas requir oxygen therapi requir icu admiss mechan ventil anoth studi report critic ill covid patient patient ard receiv high flow nasal cannula hfnc invas mechan ventil nippv risk factor associ respiratori failur requir mechan ventil clearli describ publish report although limit avail data risk factor associ critic ill icu admiss includ older age year male gender presenc underli comorbid diabet malign immunocompromis state cdc report overal case fatal rate cfr cfr patient age year older critic ill patient cfr higher receiv invas mechan ventil presenc pre exist comorbid condit cardiovascular diseas diabet chronic respiratori diseas hypertens cancer associ higher risk death adult covid suggest start supplement oxygen peripher oxygen satur spo weak recommend low qualiti evid recommend start supplement oxygen spo strong recommend moder qualiti evid adult covid acut hypoxem respiratori failur oxygen recommend spo maintain higher strong recommend moder qualiti evid recent studi describ diseas cours 1009 patient covid china show hospit patient sever diseas requir supplement oxygen critic ill patient hypoxia detriment associ poor outcom random non random studi use oxygen adult covid howev panel use indirect evid acut ill popul inform recommend systemat review meta analysi rct 037 patient show liber oxygen strategi associ increas risk hospit mortal rr1 acut ill patient furthermor meta regress show linear associ risk death higher spo target median spo liber oxygen group iqr across trial recent clinic practic guidelin recommend spo maintain higher subsequ trial provid guidanc oxygen target icu rox trial random 1000 critic ill patient receiv either conserv oxygen base protocol dial oxygen usual care trial show differ 180 day mortal two group icu rox trial compar hyperoxia conserv oxygen strategi instead compar usual care conserv oxygen strategi recent loco2 trial random patient ard conserv oxygen arm target spo liber oxygen arm target spo %). trial stop earli futil possibl harm death occur 205 includ patient day mortal risk differ day conserv oxygen arm higher risk death consid associ patient harm extrem spo target increas cost liber oxygen use well potenti reduc equiti oxygen resourc deplet panel issu strong recommend use oxygen target spo strong recommend avoid lower valu spo %). therefor reason spo rang patient receiv oxygen despit convent oxygen therapi suggest use hfnc convent oxygen therapi weak recommend low qualiti evid direct evid patient covid panel use indirect evid critic ill popul inform recommend rct compar hfnc convent oxygen therapi patient acut hypox respiratori failur hfnc result reduc day mortal reduc risk intub systemat review meta analysi rct 2093 patient show hfnc reduc intub compar convent oxygen affect risk death icu length stay even though evid mortal length stay strong reduct need intub import find particularli perspect pandem covid resourc critic care bed ventil may becom limit addit sar report increas transmiss diseas healthcar worker especi nurs endotrach intub although find base mostli retrospect observ studi hfnc seem confer increas risk transmiss diseas studi evalu bacteri environment contamin hfnc present contamin risk similar convent oxygen sar healthcar worker expos hfnc increas risk develop diseas final patient may find hfnc comfort least comfort convent oxygen therapi although author advis avoid use hfnc patient covid due fear diseas transmiss studi support advic lack although propos patient wear face mask hfnc therapi uncertain efficaci safeti approach question could address futur studi suggest use hfnc nippv weak recommend low qualiti evid adult covid acut respiratori failur suggest use hfnc nippv rct compar hfnc nippv patient acut hypox respiratori failur hfnc result reduc mortal day significantli affect need intub failur rate nippv convent oxygen hfnc group anoth meta analysi compar hfnc nippv show hfnc decreas need intub patient yet without significantli reduc mortal icu length stay addit patient may find hfnc comfort nippv given evid decreas risk intub hfnc compar nippv acut hypoxem respiratori failur studi suggest nippv may carri greater risk nosocomi infect healthcar provid suggest hfnc nippv howev patient receiv hfnc nippv monitor close care set intub facilit event decompens failur rate may high emerg intub uncontrol set may increas risk nosocomi infect healthcar provid hfnc avail urgent indic endotrach intub suggest trial nippv close monitor short interv assess worsen respiratori failur weak recommend low qualiti evid abl make recommend regard use helmet nippv compar mask nippv option certain safeti efficaci covid adult covid receiv nippv hfnc recommend close monitor worsen respiratori statu earli intub control set worsen occur best practic statement adult present hypox respiratori failur covid direct evid support use nippv furthermor prior studi suggest may associ increas risk infect transmiss healthcar worker metaanalys rct show reduct intub mortal risk nippv hypox respiratori failur howev meta analys includ studi focus immunocompromis acut cardiogen pulmonari edema post oper patient find may therefor less applic covid patient acut hypoxem respiratori failur ard common present acut hypoxem respiratori failur etiolog cardiogen pulmonari edema nippv high failur rate one rct failur report patient hypox respiratori failur ventil nippv patient therefor requir intub addit patient hypox respiratori failur random nippv higher mortal treat convent oxygen therapi hfnc cohort middl east respiratori syndrom mer patient nippv associ improv mortal length stay compar patient intub without tri nippv howev nippv associ high failur rate %), lead intub patient receiv nippv prior intub increas inhal nitric oxid requir increas mortal failur rate pandem influenza h1n1 sar rang demonstr efficaci mainli come case seri observ studi rather rct lead practic variat china use nippv pandem respiratori infect common wherea guidelin europ hong kong advis nippv first line therapi h1n1 addit concern use nippv respiratori pandem like covid nippv may aggrav sever form lung injuri result injuri transpulmonari pressur larg tidal volum may delay initi invas mechan ventil lead emerg unstabl intub increas risk transmiss healthcar team addit nippv aerosol gener procedur increas risk transmiss diseas healthcar worker sever studi meta analys sar also highlight risk nosocomi spread diseas nippv balanc benefit harm use nippv adult covid unclear certain covid patient form respiratori failur acut hypercapn respiratori failur acut cardiogen pulmonari edema known caus respiratori failur nippv may benefici howev limit experi nippv pandem suggest high failur rate recommend patient receiv nippv monitor close care set intub facilit event decompens howev resourc becom stretch may insuffici abil provid invas ventil even moder chanc success nippv may justifi use nippv use helmet nippv attract option avail singl center rct show decreas intub improv mortal nippv deliv helmet ard patient particular import set pandem covid nippv helmet also shown reduc exhal air dispers wherea face mask insuffici howev helmet nippv expens without direct evid benefit covid patient resourc util acquir equip alreadi avail figur summar recommend hfnc nippv patient covid recommend mechan ventil adult covid ard recommend use low tidal volum ventil predict bodi weight higher tidal volum strong recommend moder qualiti evid current studi address mechan ventil strategi covid patient howev panel expert believ mechan ventil patient covid manag similarli patient acut respiratori failur icu mechan ventil potenti life save intervent worsen lung injuri ventil induc lung injuri vili contribut multiorgan failur patient ard one main ventil strategi minim vili low ventil systemat review meta analysi rct found invers associ larger gradient mortal addit author found use protocol low strategi high peep rct 1629 patient reduc risk death analysi rct 1181 patient show reduct hospit mortal low ventil basi avail bodi evid sever guidelin recommend use low predict bodi weight patient ard 100 panel judg magnitud benefit moder cost low intervent accept feasibl implement therefor issu strong recommend use low predict bodi weight ventil patient ard fig ardsnet studi protocol set initi increas patient doubl trigger inspiratori airway pressur decreas peep strict adher target spontan breath patient ard challeng patient ventil dyssynchroni uncommon 101 recommend target plateau pressur pplat strong recommend moder qualiti evid clinic trial examin effect plateau pressur pplat limit covid induc ard howev larg bodi indirect evid patient ard along low ventil pplat limit lung protect strategi limit vili systemat review meta analysi rct found use lung protect strategi includ protocol low pplat cmh rct 1629 patient reduc risk death subsequ meta analysi rct compar ventilatori strategi low high pplat patient ard studi found short term mortal higher patient pplat cmh 100 panel judg magnitud benefit moder cost low patient valu consist intervent accept feasibl implement therefor issu strong recommend keep pplat cmh ventil patient ard ardsnet studi protocol set initi measur pplat inspiratori paus pplat cmh could reduc step pplat within rang sever ard suggest use higher peep strategi lower peep strategi weak recommend low qualiti evid remark use higher peep strategi peep clinician monitor patient barotrauma ard extrins peep use prevent repeat open close alveoli atelectotrauma therefor reduc vili addit peep increas sustain alveolar recruit improv oxygen reduc oxygen requir clinic trial examin effect peep coronaviru induc ard howev larg bodi indirect evid patient ard individu patient data meta analysi ipdma largest trial 2299 patient high peep 103 104 105 found differ hospit mortal patient 106 howev patient ard higher peep strategi result lower icu mortal lower hospit mortal reduct use rescu therapi expens possibl increas risk pneumothorax 106 recent systemat review meta analysi rct 3612 patient examin effect higher peep strategi patient import outcom 107 overal higher peep strategi reduc hospit mortal howev subgroup trial enrol patient oxygen respons peep rct 1888 patient use high peep significantli reduc hospit mortal compar lower peep strategi although bodi evid suggest benefici effect higher peep select patient result like confound fact low ventil use control arm trial 108 clear agre upon definit higher peep moreov optim peep level ard patient unknown like vari base extent diseas lung complianc factor aforement ipdma median peep level high peep arm day respect compar median valu day low peep arm 106 although arbitrari clinician could consid peep level constitut higher peep strategi peep level lower peep strategi ipdma combin differ strategi set higher peep reason start point would implement strategi use larg rct includ alveoli lov express 103 104 105 increas peep level clinician monitor patient evid barotrauma importantli higher peep may result higher pplat associ risk benefit pplat cmh clinician use ard network protocol strategi determin optim peep level avail strategi includ decrement peep strategi esophag balloon techniqu electr imped tomographi howev effect use techniqu clinic outcom unknown mechan ventil adult covid ard suggest use conserv fluid strategi liber fluid strategi weak recommend low qualiti evid optim fluid strategi covid known howev plausibl patient respond fluid similarli ard patient limit data avail covid show cardiac failur alon combin respiratori failur caus covid death anoth studi show covid patient arrhythmia data suggest presenc myocardi injuri patient covid rct publish compar conserv deresuscit liber fluid strategi ard recent systemat review includ rct enrol 1206 patient ard risk death similar group conserv fluid strategi group liber strategi group studi includ rct critic ill patient without ard author found conserv fluid strategi increas ventil free day day day reduc icu length stay day day compar liber fluid strategi differ harm includ renal failur two group landmark trial ard patient factt found signific reduct durat mechan ventil conserv fluid strategi 109 furthermor major patient covid icu elderli may develop myocardi dysfunct could limit abil handl larg fluid volum view moder benefit observ ard popul possibl reduc cost administ less fluid feasibl intervent panel issu weak recommend favor conserv fluid strategi patient covid ard sever ard suggest prone ventil prone ventil weak recommend low qualiti evid seri patient covid radiograph featur progress first week symptom onset predomin ground glass opac mix pattern predomin basilar consolid latter pattern may suggest role prone ventil 110 prone posit theoret make ventil homogen decreas ventral alveolar distent dorsal alveolar collaps 111 may reduc differ dorsal ventral transpulmonari pressur addit reduc lung compress 112 improv perfus 113 recent studi describ clinic cours covid icu show prone ventil use patient howev studi avail describ clinic cours patient covid ventil prone posit recent systemat review meta analysi rct 2129 patient show prone ventil least patient moder sever ard reduc mortal rct effect mortal studi use prone ventil rct hand prone ventil increas risk pressur sore endotrach tube obstruct 114 systemat review reach similar conclus 115 116 117 moder certainti prone ventil patient moder sever ard reduc mortal may increas risk pressur sore endotrach tube obstruct healthcar worker prone patient covid train proper techniqu prone take infect control precaut event accident endotrach tube disconnect ventil prone associ signific cost believ may provid signific benefit prone implement low middl incom set effort made provid necessari train educ healthcar worker facilit practic http :// www youtu com watch e_6jt9r 7wj protocol prone use institut base avail resourc level train prone ventil use healthcar worker awar complic pressur sore vascular line endotrach tube displac facial edema transient hemodynam instabl corneal abras brachial plexu injuri hemodialysi vascular access flow issu addit clinician familiar absolut contraind prone ventil unstabl spine open abdomen open chest surgeri trauma enter nutrit via nasogastr nasoduoden tube continu prone 118 119 sever profession societi issu recommend use nmba ard 100 120 121 122 123 issu recommend favor use nmba infus patient moder sever ard recommend mostli base pool estim rct 431 patient show reduct day mortal nmba infus compar nmba infus 124 howev result evalu system earli neuromuscular blockad rose trial challeng previou trial rose trial investig random 1006 patient moder sever ard receiv either infus nmba intermitt nmba bolus need basi 125 rose trial show continu infus cisatracurium improv patient import outcom due differ design rose trial earlier trial perform meta analysi mortal outcom although pool estim barotrauma favor continu nmba infus panel suggest continu nmba infus reserv patient indic ongo paralysi intermitt dose may suffic patient persist ventil dyssynchroni patient need ongo deep sedat prone ventil persist high plateau pressur effect nmba longterm outcom unclear mechan ventil adult covid ard recommend routin use inhal nitric oxid strong recommend low qualiti evid mechan ventil adult covid sever ard hypoxemia despit optim ventil rescu strategi suggest trial inhal pulmonari vasodil rescuetherapi rapid improv oxygen observ treatment taper weak recommend low qualiti evid studi describ use pulmonari vasodil covid patient cochran review identifi rct 1243 patient inhal nitric oxid ard treatment show signific effect mortal associ increas risk acut kidney injuri inhal nitric oxid result transient improv oxygen subgroup studi report pao fio valu intervent show statist signific differ favor inhal nitric oxid present beyond studi assess use inhal nitric oxid rescu therapi 126 possibl harm inhal nitric oxid absenc clear mortal benefit panel issu strong recommend routin use patient ard howev view find improv oxygen trial inhal nitric oxid rescu therapi tri option reason avail inhal nitric oxid use without good respons term oxygen taper avoid rebound pulmonari vasoconstrict occur prolong use abrupt discontinu adequ power rct evalu inhal prostacyclin ilioprost therefor could recommend use sever ard despit optim ventil suggest use recruit maneuv use recruit maneuv weak recommend low qualiti evid recruit maneuv use recommend use staircas increment peep recruit maneuv strong recommend low qualiti evid studi assess role recruit maneuv rm patient ard secondari covid rm aim improv oxygen increas transpulmonari pressur open atelectat alveoli 127 howev exposur high level posit pressur may lead barotrauma well caus transient hypotens alreadi critic ill unstabl patient assess indirect rct assess rm ard patient includ patient sepsi due bacteri viral pneumonia vari strategi use help recruit atelectat lung howev two strategi particular common rct includ meta analysi tradit rm describ prolong inspiratori hold set durat time higher level cpap commonli 104 128 129 increment peep titrat rm describ increment increas peep min 130 131 132 133 systemat review meta analysi rct 1423 patient rm reduc mortal use rescu intervent improv oxygen without increas risk barotrauma 134 similarli identifi rct 2544 patient report hospit mortal studi rm associ reduc mortal howev subgroup analys suggest tradit rm significantli reduc mortal wherea increment peep titrat rm increas mortal effect rm oxygen may transient studi show signific improv oxygen trial use differ peep strategi intervent control arm rm best combin higher peep strategi patient sever ard hypoxemia may benefit tradit recruit maneuv along higher level peep evid specif covid need patient receiv rm monitor close sever desatur hypotens barotrauma rm stop lead patient deterior hypoxemia despit optim ventil use rescu therapi prone suggest use venoven ecmo avail refer patient ecmo center weak recommend low qualiti evid remark due resourc intens natur ecmo need experienc center healthcar worker infrastructur ecmo consid care select patient covid sever ard clinic trial ecmo covid patient recent report china suggest covid case icu receiv ecmo clinic cours outcom patient report yet ministri health saudi arabia establish ecmo program mer cov epidem retrospect cohort studi patient mer cov refractori hypoxemia group patient receiv ecmo lower hospit mortal 100 135 howev cohort studi high risk select bia given retrospect design two rct evalu ecmo convent mechan ventil sever ard guidelin publish 2017 unabl provid specif guidanc use ecmo research recommend although recent rct eolia stop earli futil 136 analysi trial use bayesian approach provid favor interpret suggest lower mortal ecmo sever ard 137 recent systemat review includ two rct 429 patient found reduct day mortal ecmo risk major bleed higher ecmo 138 ecmo resourc intens techniqu restrict special center remain extrem limit resourc therefor use rescu therapi reserv care select patient 139 futur studi describ outcom covid patient ecmo mechan death advanc understand guid practic section discuss possibl treatment option sar cov complic includ antivir agent immunosuppress agent immunomodul therapi cytokin storm syndrom hyperinflammatori state character fulmin multi organ failur elev cytokin level recent studi china show covid associ cytokin elev profil reminisc secondari hemophagocyt lymphohistiocytosi hlh author even suggest screen critic ill covid patient secondari hlh use hscore 140 corticosteroid immunosuppress agent use patient high likelihood hlh 141 evid need make recommend treatment option cytokin storm mechan ventil adult covid respiratori failur without ard suggest routin use system corticosteroid weak recommend low qualiti evid mechan ventil adult covid ard suggest use system corticosteroid use corticosteroid weak recommend low qualiti evid remark major panel support weak recommend suggest use steroid sickest patient covid ard howev low qualiti evid expert panel prefer issu recommend higher qualiti direct evid avail control clinic trial use corticosteroid covid patient coronavirus publish peer review report patient sever covid report use methylprednisolon day day associ shorter durat supplement oxygen use day day 001 improv radiograph find 142 although interest judg preliminari report insuffici basi formul recommend due risk confound therefor use indirect evid commun acquir pneumonia ard viral infect inform recommend sever rct use system corticosteroid hospit patient commun acquir pneumonia mostli non icu patient sepsi septic shock systemat review meta analysi rct show use corticosteroid may reduc need mechan ventil rct 1060 patient ard rct 945 patient durat hospit rct 1499 patient day increas risk hyperglycemia requir treatment 143 howev trial includ differ popul effect mortal outcom unclear use differ drug dose regimen addit concern corticosteroid use viral pneumonia therefor result may generaliz covid popul mani publish observ studi use steroid viral pneumonia influenza viru coronavirus other prone confound sicker patient usual receiv corticosteroid updat recent cochran review use corticosteroid influenza 144 search studi coronavirus includ total cohort studi influenza coronavirus meta analysi adjust or show associ corticosteroid use increas mortal effect patient coronavirus unclear also studi limit signific heterogen found signific homogen observ studi use corticosteroid ard caus coronavirus gener viral ard respect furthermor case summari statist tend toward harm use steroid updat recent cochran review 145 identifi addit rct 146 deal ard overal includ rct enrol 851 patient ard use corticosteroid reduc mortal durat mechan ventil day howev trial focus viral ard limit generaliz result covid patient addit review observ studi corticosteroid use viral ard identifi cohort studi although point estim show increas mortal includ substanti harm benefit recent rct interest trial use recombin interferon 1b rifn 1ba reduc mortal ard patient subgroup patient receiv corticosteroid rifn 1ba use associ increas mortal 147 direct evid come retrospect cohort studi 201 patient covid pneumonia studi show associ corticosteroid use lower mortal patient covid ard howev estim adjust confound factor 148 effect corticosteroid covid patient sepsi septic shock may differ recent systemat review meta analys rct sepsi show small improv mortal faster resolut shock corticosteroid use compar use corticosteroid 149 150 see previou section hemodynam support wide recogn corticosteroid rang advers effect viral pneumonia icu sever studi show increas viral shed corticosteroid use 151 152 153 potenti indic viral replic clinic implic increas viral shed uncertain consid panel issu suggest routin use system corticosteroid respiratori failur covid suggest use corticosteroid sicker popul covid ard clinician use corticosteroid ard use lower dose shorter treatment cours mechan ventil patient covid respiratori failur suggest use empir antimicrobi antibacteri agent antimicrobi weak recommend low qualiti evid remark treat team initi empir antimicrobi assess escal daili evalu durat therapi spectrum coverag base microbiolog result patient clinic statu control clinic trial evalu use empir antimicrobi covid patient coronavirus recommend therefor base upon extrapol data viral pneumonia particularli influenza 154 identifi bacteri coinfect superinfect patient covid challeng symptom may similar underli viral infect diagnost difficulti reflect high rate intraven antibiot administ wuhan non sever diseas patient admit hospit icu data preval bacteri superinfect patient covid limit larger case studi clinician often overwhelm systemat obtain high qualiti sampl critic ill patient mer bacteri viral infect 155 infect staphylococcu aureu common influenza pneumonia especi virul 154 recent clinic practic guidelin recommend initi empir antibacteri therapi adult commun acquir pneumonia test posit influenza 154 data critic ill patient demonstr secondari infect case although number small isol organ includ gram neg organ pneumonia aeruganosa marcescen basi limit data difficult determin pattern superinfect includ risk aureu infect commonli seen influenza patient covid hypox respiratori failur requir mechan ventil panel suggest empir antimicrobi treatment basi superinfect reason common popul may lead substanti increas mortal pandem influenza 156 157 158 therefor critic ill patient suspect confirm covid treat empir antimicrobi therapi accord clinic syndrom commun acquir hospit acquir pneumonia secondari infect occur patient covid incid unknown given limit data 159 infect treat accord clinic microbiolog data critic ill adult covid develop fever suggest use acetaminophen paracetamol temperatur control treatment weak recommend low qualiti evid major patient covid develop fever hospit sever diseas largest report china median temperatur across 1099 patient iqr data critic ill patient gener avail review literatur identifi rct 1785 patient examin effect fever control critic ill popul exclud neurolog indic temperatur control 160 161 162 163 164 165 166 167 168 169 170 171 activ temperatur manag pharmacolog non pharmacolog reduc risk death icu length stay day effect reduc bodi temperatur lower given safeti acetaminophen lack harm bodi evid increas patient comfort fever manag mayb import therefor issu suggest clinician consid use pharmacolog agent control fever coivd patient use non steroid anti inflammatori drug treat fever patient covid continu debat evid avail suggest use acetaminophen paracetamol treat fever critic ill adult covid suggest routin use standard intraven immunoglobulin ivig weak recommend low qualiti evid use intraven immunoglobulin ivig report sever seri covid patient efficaci data avail 172 absenc adequ titer neutral antibodi standard intraven immunoglobulin unlik biolog effect covid ivig may immunomodulatori action use rare also associ increas risk seriou advers event includ anaphylact reaction asept mening renal failur thromboembol hemolyt reaction transfus relat lung injuri late reaction 173 prepar anti sar cov polyclon monoclon antibodi develop howev data recent trial use antibodi base therapi immun plasma hyperimmun globulin monoclon antibodi hemagglutinin stalk 173 hospit season influenza patient demonstr improv outcom 174 175 176 critic ill adult covid suggest routin use convalesc plasma weak recommend low qualiti evid convalesc plasma obtain patient recov covid suggest potenti therapi may provid passiv immun sar cov2 specif antibodi 177 convalesc plasma use treat sever viral infect includ caus sar coronaviru avian influenza h5n1 viru influenza h1n1 pdm09 viru 178 179 180 181 182 recent meta analysi observ studi use passiv immunotherapi treatment sever acut respiratori infect viral etiolog suggest convalesc plasma therapi associ reduct mortal 183 current outbreak china convalesc plasma use patient covid 184 howev data efficaci safeti convalesc plasma limit target suffici level neutral antibodi titer sar cov unknown studi mer conclud use convalesc plasma might feasibl challeng due small pool potenti donor suffici high antibodi titer 185 rct patient confirm ebola viru diseas show convalesc plasma unknown level neutral antibodi associ improv surviv 186 anoth rct patient season influenza treat high titer versu low titer anti influenza immun plasma termin futil lack effect primari outcom measur point ordin scale clinic statu day 187 given lack convinc evid rct uncertainti surround optim prepar convalesc plasma safeti suggest routin use treat patient covid evid avail critic ill adult covid suggest routin use lopinavir ritonavir weak recommend low qualiti evid insuffici evid issu recommend use antivir agent critic ill adult covid prolong detect sar cov rna respiratori tract sometim site serious ill covid patient provid rational administr antivir agent reduc replic effort improv clinic outcom present direct act antivir proven inhibit replic provid clinic benefit covid mer patient consider number agent approv indic propos use comment address promis one sever other undergo test arbidol umifenovir favipiravir ribavirin tradit chines medicin inhal interferon alon combin one countri lopinavir antiretrovir proteas inhibitor use combin ritonavir ensur adequ lopinavir exposur treatment human immunodefici viru hiv infect 188 found show vitro activ sar cov lopinavir ritonavir administ combin high dose oral ribavirin taper cours system corticosteroid cohort patient sar found associ significantli fewer advers clinic outcom ard death compar ribavirin alon use 111 histor control receiv ribavirin corticosteroid 189 high throughput screen antivir compound lopinavir inhibit replic mer cov vitro 190 anim model mer cov infect treatment lopinavir ritonavir ifn 1b associ virolog histolog clinic improv versu placebo 191 lopinavir ritonavir combin interferon beta test rct mer cov patient 192 combin consid second candid research priorit list therapeut agent 193 drug gener good safeti profil may interact mani drug commonli use critic ill patient http :// www covid drugi ntera ction org /). recent rct compar use lopinavir ritonavir usual care 199 hospit patient covid china 194 trial lopinavir ritonavir significantli reduc day mortal time clinic improv day addit lopinavir ritonavir associ advers event 194 trial avail direct evid use lopinavir ritonavir patient covid howev sever limit trial unblind enrol small number patient 199 small number event death total limit confid result nevertheless routin use lopinavir ritonavir critic ill patient probabl warrant weak recommend routin use lopinavir ritonavir critic ill covid patient reason lopinavir ritonavir one arm plan core treatment protocol hospit patient covid remap cap random embed multifactori adapt platform trial commun acquir pneumonia trial nct02735707 result ongo trial help increas precis estim certainti evid remdesivir prodrug adenosin analog incorpor nascent viral rna chain result prematur termin consid promis drug inform consult research priorit candid therapeut agent 195 current publish case report publish trial use remdesivir covid remdesivir demonstr effect inhibit sar cov mer cov sar cov vitro studi 196 furthermor studi anim model mer cov show effect control superior lopinavir ritonavir combin system ifn 197 198 although intraven remdesivir appear adequ toler recent rct show less effect sever antibodi therapi ebola viru diseas 199 sever ongo rct aim examin efficaci safeti intraven remdesivir sever covid clinicaltri gov nct04257656 mild moder covid clinicaltri gov nct04252664 anoth trial sponsor nation institut allergi infecti diseas recruit patient usa clinicaltri gov nct04280705 updat guidelin new evid emerg insuffici evid issu recommend use recombin rifn alon combin antivir critic ill adult covid recombin interferon often combin ribavirin therapi use patient mer sar 179 200 201 202 differ prepar recombin rifn rifn 2a rifn 2b rifn 1a rifn 1b shown activ mer cov vero llc mk2 cell rhesu macaqu model mer cov infect 200 201 203 largest cohort critic ill patient mer show rifn 2a rifn 2b rifn 1a ribavirin associ lower mortal reduc viral clearanc adjust time vari covari 204 rel effect differ interferon sar cov unknown point vitro data show rifn display strongest mer cov inhibit among differ rifn prepar rifn 2b rifn rifn univers rifn 2a rifn time lower previous report inhibitori concentr ic50 rifn 2b 203 205 rct examin effect combin lopinavir ritonavir rifn mortal hospit patient mer current recruit patient 206 unpublish data indic ifn inhibit sar c0v cell cultur ifn priorit studi covid insuffici evid issu recommend use chloroquin hydroxychloroquin critic ill adult covid chloroquin metabolit hydroxychloroquin antimalari agent demonstr antivir effect sar cov sar cov vitro 196 207 208 prior studi found inhibitori effect chloroquin multipl rna virus vitro rct treatment dengu chikungunya viru infect influenza prophylaxi fail demonstr antivir clinic benefit 209 one non human primat model chikungunya infect shown chloroquin immunomodulatori effect associ delay immun respons higher level viral replic wors ill 210 news brief suggest use 100 patient show superior control inhibit exacerb pneumonia improv lung imag find promot viru neg convers shorten diseas cours data publish yet 211 recent consensu document recommend chloroquin phosphat 500 twice daili minimum day dose modif sever gastrointestin side effect occur 212 sinc chloroquin avail countri hydroxychloroquin altern recent studi china explor variou dose regimen chloroquin hydroxychloroquin use physiologicallybas pharmacokinet model 208 studi found hydroxychloroquin potent chloroquin inhibit sar cov vitro base model hydroxychloroquin load dose 400 twice daili follow 200 twice daili day recommend 208 recent systemat review found publish studi covid patient 213 pend result ongo trial unabl issu recommend chloroquin insuffici evid issu recommend use tocilizumab critic ill adult covid tocilizumab human immunoglobulin function immun respons block receptor bind approv cr inflammatori condit relat relat inflamm rheumatoid arthriti juvenil idiopath arthriti 214 215 216 217 sever ill patient covid may extrem immun respons lead sever respiratori failur case inhibit may help attenu cytokin releas syndrom reduc cytokin concentr acut phase reactant product 218 ongo trial tocilizumab help address safeti efficaci therapi covid rheumatoid arthriti literatur systemat review meta analysi rct dose dose show increas risk advers event compar control treatment increas risk infect 219 anoth systemat review meta analysi rct tocilizumab rheumatoid arthriti found increas risk infecti respiratori advers event 220 sinc data safeti efficaci tocilizumab covid unabl issu recommend nafamostat synthet serin proteas inhibitor potent inhibitor mer cov nitazoxanid antiprotozo agent antivir potenti sever respiratori virus includ influenza parainfluenza respiratori syncyti viru rhinoviru vitro studi show nafamostat nitazoxanid inhibit sar cov 196 rct patient acut uncompl influenza demonstr use nitazoxanid reduc durat symptom 221 howev hospit patient sever acut respiratori infect mexico nitazoxanid found superior placebo 222',\n",
       "   '_search_score': 0.6727056999999999}],\n",
       " 'count': 556}"
      ]
     },
     "execution_count": 25,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-05T08:34:38.715694Z",
     "start_time": "2020-11-05T08:34:38.711694Z"
    }
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[('The 2019-new coronavirus epidemic: evidence for virus evolution',\n",
       "  0.8915696),\n",
       " ('Host and infectivity prediction of Wuhan 2019 novel coronavirus using deep learning algorithm',\n",
       "  0.8064966),\n",
       " ('A Chinese Case of Coronavirus Disease 2019 (COVID-19) Did Not Show Infectivity During the Incubation Period: Based on an Epidemiological Survey',\n",
       "  0.7831983999999999),\n",
       " ('Dinucleotide repeats in coronavirus SARS-CoV-2 genome: evolutionary implications',\n",
       "  0.7324622000000001),\n",
       " ('Coronavirus and paramyxovirus in bats from Northwest Italy',\n",
       "  0.7292778000000002),\n",
       " ('Preliminary studies on feline coronavirus distribution in naturally and experimentally infected cats',\n",
       "  0.7255031999999999),\n",
       " ('Immune evasion of porcine enteric coronaviruses and viral modulation of antiviral innate signaling',\n",
       "  0.7171661999999999),\n",
       " ('Is a 14-day quarantine period optimal for effectively controlling coronavirus disease 2019 (COVID-19)?',\n",
       "  0.6793499999999999),\n",
       " ('Relative Coronavirus Disease 2019 Mortality: A Swiss Population-based Study',\n",
       "  0.6775235999999998),\n",
       " ('Surviving Sepsis Campaign: guidelines on the management of critically ill adults with Coronavirus Disease 2019 (COVID-19)',\n",
       "  0.6727056999999999)]"
      ]
     },
     "execution_count": 26,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "[(result[\"title\"],result[\"_search_score\"]) for result in results[\"results\"]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-04T18:22:20.808426Z",
     "start_time": "2020-11-04T18:22:19.365038Z"
    }
   },
   "outputs": [],
   "source": [
    "import dask.bag as db\n",
    "from dask.distributed import Client, progress\n",
    "client = Client(n_workers=6, threads_per_worker=2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-04T18:22:20.901427Z",
     "start_time": "2020-11-04T18:22:20.898428Z"
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<table style=\"border: 2px solid white;\">\n",
       "<tr>\n",
       "<td style=\"vertical-align: top; border: 0px solid white\">\n",
       "<h3 style=\"text-align: left;\">Client</h3>\n",
       "<ul style=\"text-align: left; list-style: none; margin: 0; padding: 0;\">\n",
       "  <li><b>Scheduler: </b>tcp://127.0.0.1:60739</li>\n",
       "  <li><b>Dashboard: </b><a href='http://127.0.0.1:8787/status' target='_blank'>http://127.0.0.1:8787/status</a>\n",
       "</ul>\n",
       "</td>\n",
       "<td style=\"vertical-align: top; border: 0px solid white\">\n",
       "<h3 style=\"text-align: left;\">Cluster</h3>\n",
       "<ul style=\"text-align: left; list-style:none; margin: 0; padding: 0;\">\n",
       "  <li><b>Workers: </b>6</li>\n",
       "  <li><b>Cores: </b>12</li>\n",
       "  <li><b>Memory: </b>17.13 GB</li>\n",
       "</ul>\n",
       "</td>\n",
       "</tr>\n",
       "</table>"
      ],
      "text/plain": [
       "<Client: 'tcp://127.0.0.1:60739' processes=6 threads=12, memory=17.13 GB>"
      ]
     },
     "execution_count": 51,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "client"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-04T18:43:57.773137Z",
     "start_time": "2020-11-04T18:43:57.746620Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "B:/document_parser/document_parses/test\\000a0fc8bbef80410199e690191dc3076a290117.json\n"
     ]
    }
   ],
   "source": [
    "files = glob.glob(\"B:/document_parser/document_parses/test\"+\"/*.json\")\n",
    "print(files[0])\n",
    "b = db.from_sequence(files).map(preprocess_json)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 147,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-04T15:16:35.022280Z",
     "start_time": "2020-11-04T15:15:10.556781Z"
    }
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "84420"
      ]
     },
     "execution_count": 147,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "b.count().compute()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-04T19:00:47.456615Z",
     "start_time": "2020-11-04T18:44:01.977273Z"
    }
   },
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "f4c7512d174c4ceebec4b224bb74cc67",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "HBox(children=(FloatProgress(value=0.0, max=37.0), HTML(value='')))"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "{'inserted_successfully': 556, 'failed': 0, 'failed_document_ids': []}"
      ]
     },
     "execution_count": 56,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "vi_client.insert_documents(\"corpus-dask\", b.compute(), models ={\"title\": text_encoder.encode, \"abstract\":text_encoder.encode})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-05T08:35:04.216562Z",
     "start_time": "2020-11-05T08:34:58.481582Z"
    }
   },
   "outputs": [],
   "source": [
    "advanced_search_query = {\n",
    "    'text' : {'vector': text_encoder.encode('what is the origin coronavirus'), 'fields' : ['abstract_vector_', \"title_vector_\"]}\n",
    "}\n",
    "resultsb = vi_client.advanced_search('corpus-dask', advanced_search_query, page_size=20)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-05T00:41:10.128170Z",
     "start_time": "2020-11-05T00:41:10.074162Z"
    }
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'results': [{'_id': 'a3116a1ea2e157c8855023d968cc0f0bd6c9846f',\n",
       "   'insert_date_': '2020-11-04T18:51:15.374255',\n",
       "   'abstract': 'There is concern about a new coronavirus, the 2019-nCoV, as a global public health threat. In this article, we provide a preliminary evolutionary and molecular epidemiological analysis of this new virus. A phylogenetic tree has been built using the 15 available whole genome sequence of 2019-nCoV and 12 whole genome sequences highly similar sequences available in gene bank (5 from SARS, 2 from MERS and 5 from Bat SARS-like Coronavirus). FUBAR analysis shows that the Nucleocapsid and the Spike Glycoprotein has some sites under positive pressure while homology modelling helped to explain some molecular and structural differences between the viruses. The phylogenetic tree showed that 2019.nCoV significantly clustered with Bat SARS-like Coronavirus sequence isolated in 2015, whereas structural analysis revealed mutation in S and nucleocapsid proteins. From these results, 2019nCoV could be considered a coronavirus distinct from SARS virus, probably transmitted from bats or another host where mutations conferred upon it the ability to infect humans. author/funder. All rights reserved. No reuse allowed without permission.',\n",
       "   'title': 'The 2019-new coronavirus epidemic: evidence for virus evolution',\n",
       "   'body': \"The family Coronaviridae comprises large, single, plus-stranded RNA viruses isolated from several species and previously known to cause common colds and diarrheal illnesses in humans [1] . A previously unknown severe acute respiratory syndrome (SARS) [4] and manually edited using Bioedit program v7.0.5 [5] . Maximum likelihood (ML) methods were employed for the analyses because they allow for testing author/funder. All rights reserved. No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.24.915157 doi: bioRxiv preprint different phylogenetic hypotheses by calculating the probability of a given model of evolution generating the observed data and by comparing the probabilities of nested models by the likelihood ratio test. The best-fitting nucleotide substitution model was chosen by JModeltest software [6] . ML tree was reconstructed using Generalised time reversible (GTR) plus Gamma distribution and invariant sites (+G +I) as evolutionary model using MEGA-X. [7] Adaptive Evolution Server (http://www.datamonkey.org/) was used to find eventual sites of positive or negative selection. At this purpose the following tests has been used: fast-unconstrained Bayesian approximation (FUBAR) [8] . These tests allowed to infer the site-specific pervasive selection, the episodic diversifying selection across the region of interest and to identify episodic selection at individual sites [9] . Statistically significant positive or negative selection was based on p value < 0.05 [9] . Homology models have been built relying on the website SwissModel [10] . Structural templates have been searched and validated using the software available within the SwissModel environment and HHPred [11] . Homology models have been validated using the QMEAN tool [12] . Threedimensional structures have been analyzed and displayed using PyMOL [13] . From the selective pressure and structural analysis, mutations of surface proteins, as the spike protein S, and of nucleocapsid N protein conferring stability to the viral particle have been shown. The viral spike protein is responsible for virus entry into the cell after by binding to a cell receptor and membrane fusion, two key roles in viral infection and pathogenesis. The N protein is a structural protein involved in virion assembly, playing a pivotal role in virus transcription and assembly efficiency. Mutation of these proteins could determine two important characteristics of the coronavirus isolated during the 2019nCoV epidemic, a higher stability than the bat-like SARS coronavirus and a lower pathogenicity than SARS coronavirus. These features can explain the author/funder. All rights reserved. No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.24.915157 doi: bioRxiv preprint 2019-nCoV zoonotic transmission and its initial lower severity than SARS epidemic. These results doesn't exclude that further mutation due to positive selective pressure, lead by the epidemic evolution, could favor an ehancement of pathogenicity and transmission of this novel virus. Recently, Ji et al. described homologous recombination within the spike glycoprotein of 2019nCoV favoring cross-species transmission and suggested snake as probable virus reservoir for human infection because its Resampling Similarity Codon Usage (RSCU) bias is more similar to Bungarus multicinctus snake compared to other animals and humans [3] . In a previous article has been proven that compositional properties, mutation pressure, natural selection, gene expression and dinucleotides affect the codon usage bias of Bungarus species [14] . These data along with data from our study, enforcing bat origin of infection, could clarify the transmission dynamic of the 2019-nCoV supporting infection control policy during the ongoing epidemic. author/funder. All rights reserved. No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.24.915157 doi: bioRxiv preprint author/funder. All rights reserved. No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.24.915157 doi: bioRxiv preprint\",\n",
       "   '_search_score': 0.9573084999999999},\n",
       "  {'_id': 'a05179c57d5e1cdddc839ff3c0b70a614efbb6cb',\n",
       "   'insert_date_': '2020-11-04T18:47:29.781848',\n",
       "   'abstract': 'The recent outbreak of pneumonia in Wuhan, China caused by the 2019 Novel Coronavirus (2019-nCoV) emphasizes the importance of detecting novel viruses and predicting their risks of infecting people. In this report, we introduced the VHP (Virus Host Prediction) to predict the potential hosts of viruses using deep learning algorithm. Our prediction suggests that 2019-nCoV has close infectivity with other human coronaviruses, especially the severe acute respiratory syndrome coronavirus (SARS-CoV), Bat SARS-like Coronaviruses and the Middle East respiratory syndrome coronavirus (MERS-CoV). Based on our prediction, compared to the Coronaviruses infecting other vertebrates, bat coronaviruses are assigned with more similar infectivity patterns with 2019-nCoVs. Furthermore, by comparing the infectivity patterns of all viruses hosted on vertebrates, we found mink viruses show a closer infectivity pattern to 2019-nCov. These consequences of infectivity pattern analysis illustrate that bat and mink may be two candidate reservoirs of 2019-nCov.These results warn us to beware of 2019-nCoV and guide us to further explore the properties and reservoir of it. One Sentence Summary: It is of great value to identify whether a newly discovered virus has the risk of infecting human. Guo et al. proposed a virus host prediction method based on deep learning to detect what kind of host a virus can infect with DNA sequence as input.',\n",
       "   'title': 'Host and infectivity prediction of Wuhan 2019 novel coronavirus using deep learning algorithm',\n",
       "   'body': 'coronaviruses have strong potential to infect human. This method will be helpful in future viral analysis and early prevention and control of viral pathogens. The recently outbreak of pneumonia in Wuhan, China has brought the 2019 Novel Coronavirus Currently there is urgent need to identify the hosts potentially infected by the 2019-nCoV viruses. Large amounts of infectious viruses can be obtained from domesticated animals, birds, vector animals and environments. These viruses may pose a great threat to human since lots of viruses that are safe in animals may potentially infect human. In addition, owing to high mutation rate in viruses, there are also non-human-infectious viruses that are originally spread in animals, but mutate to infect human later (5) . Besides, any unexpected hosts of novel viruses are also concerning. To detect the potential host and pathogenicity of new viruses, the conventional way is based on their sequence similarity to known viruses, by either building phylogenetic tree or blast. However, this method seems to be rudimentary with several limitations. In fact, owing to the intrinsic heterogeneity within genome, different sequence fragments of the novel virus may be assigned to quite different known viruses with identical similarity by local alignment, which consequently leads to the ambiguous predictions of the hosts of novel viruses. In addition, the routine method, phylogenetic tree analysis, always utilizes the whole genomes assembled by shotgun reads, which is . CC-BY-NC-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.21.914044 doi: bioRxiv preprint frequently influenced by the sequencing and assembly mistakes. Moreover, when the potential hosts of novel viruses were predicted from a phylogenetic perspective, people usually accepted the assumption that the viruses with closer phylogenetic relationships have common hosts, pathogenicity and infection behavior. However, this assumption has proved unsustainable, because the abilities of infecting human are varied even within genus and most viruses are harmless to human. For example, Betacoronavirus, a genus in Coronavirinae, containing both viruses that can infect human, such as SARS, and viruses that are not reported to infect human, such as Rousettus bat coronavirus HKU9. Table 1, and Table S2 ). Besides, most of the reported human-infective Coronaviruses (9) (10) (11) (12) (13) (14) are assigned the lowest p-values predicted by the VHP method (Table S1 ). The similar probabilities of 2019-nCoV and other human Coronaviruses illustrates the high risk of 2019-nCoV. Part of the p-value results is shown in Table 1 , which is ranked by the decreasing order of the p-value for a virus to infect human. The prediction results warn us that the 2019-nCoV have strong potential of infecting human similar to SARS, Bat SARS-like Coronaviruses and MERS. While SARS was assigned slightly lower p-value on \"Human\", Bat SARS-like Coronaviruses were given closer p-value to 2019-nCoVs, which indicates that 2019-nCoV is probable to be less infectious within human than SARS but as infectious as Bat SARS-like Coronaviruses. Furthermore, we explore the reservoir of 2019-nCoVs with its infectivity pattern. As is shown in Table S3 , compared with the p-values of other host types, . CC-BY-NC-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.21.914044 doi: bioRxiv preprint To explore the differences of 2019-nCoVs, SARS and SARS-CoV-like Coronavirus, we then analyzed their core genes (more than 95% AAS identity) and nonsynonymous SNPs (single-nucleotide polymorphisms) (Table S3) The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.21.914044 doi: bioRxiv preprint from one origin. Besides, compared with the two Bat SARS-like Coronaviruses, 2019-nCoVs have 4, ~130, ~220 nonsynonymous SNPs on the three core genes respectively, which shows the differences among 2019-nCoV and SARS-CoV-like Coronaviruses. Except for 2019-nCoVs, our predictions for all coronaviruses indicate that Night-heron coronavirus HKU19, which is reported to infect a vertebrate, Night-heron, also has the strong potential to infect human. Coronaviruses are a large family of viruses which usually circulate among animals which will cause fatal diseases, once they mutate and evolve to be able to infect human. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.21.914044 doi: bioRxiv preprint GenBank. In order to develop a method expert in predicting the potential host types of novel viruses, the viral sequences data released before 2018 were used to build the training set, while those released after 2018 were utilized for testing. We integrated the hosts of viruses into five types, including plant, germ, invertebrate, vertebrate and human. The host subtypes contained in these five types are listed in Table 2 Medical College for genome sequencing sharing. We also acknowledge GISAID . CC-BY-NC-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.21.914044 doi: bioRxiv preprint',\n",
       "   '_search_score': 0.8702154000000002},\n",
       "  {'_id': 'a18236e9c75536c022502b9a9d0efdc6b0a3f0c2',\n",
       "   'insert_date_': '2020-11-04T18:49:30.516228',\n",
       "   'abstract': 'The ongoing global pandemic of infection disease COVID-19 caused by the 2019 novel coronavirus (SARS-COV-2, formerly 2019-nCoV) presents critical threats to public health and the economy since it was identified in China, December 2019. The genome of SARS-CoV-2 had been sequenced and structurally annotated, yet little is known of the intrinsic organization and evolution of the genome. To this end, we present a mathematical method for the genomic spectrum, a kind of barcode, of SARS-CoV-2 and common human coronaviruses. The genomic spectrum is constructed according to the periodic distributions of nucleotides, and therefore reflects the unique characteristics of the genome. The results demonstrate that coronavirus SARS-CoV-2 exhibits dinucleotide TT islands in the non-structural proteins 3, 4, 5, and 6. Further analysis of the dinucleotide regions suggests that the dinucleotide repeats are increased during evolution and may confer the evolutionary fitness of the virus. The special dinucleotide regions in the SARS-CoV-2 genome identified in this study may become diagnostic and pharmaceutical targets in monitoring and curing the COVID-19 disease.',\n",
       "   'title': 'Dinucleotide repeats in coronavirus SARS-CoV-2 genome: evolutionary implications',\n",
       "   'body': \"keywords: COVID-19, SARS-CoV-2, genomic spectrum, dinucleotide repeats, evolution fitness, virulence 1 Introduction The current global pandemic of COVID-19 caused by the novel coronavirus SARS-COV-2, formerly 2019-nCoV, has been a severe threat to public health and the economy since it emerged in Wuhan, China, December 2019. As of May 29, 2020, 5.7 million COVID-19 cases in the globe have been confirmed, and 357,688 deaths have occurred from COVID-19 disease (WHO, 2020) . SARS-CoV-2 is the aetiological agent and responsible for a large-scale outbreak of fatal disease. Understanding the notable features of the SARS-CoV-2 genome in zoonotic origin and the evolutionary trend is of importance for revealing the intervention targets, and disease control and prevention. Coronaviruses (CoVs) are the largest group of enveloped, positive-sense, single-stranded RNA viruses. Taxonomically, coronavirus SARS-CoV-2 belongs to Nidovirales order, Coronaviridae family, Coronavirinae subfamily, and beta-CoV genus (Fehr and Perlman, 2015) . SARS-CoV-2 is highly pathogenic with similar or lower pathogenicity as severe acute respiratory syndrome (SARS) coronavirus (SARS-CoV) in 2002 -2003 , and lower pathogenicity than Middle-East respiratory syndrome coronavirus (MERS-CoV) in 2012. Nevertheless, SARS-CoV-2 is highly transmissible in humans. To date, seven human CoVs (HCoVs) have been identified. Among them are alpha-CoVs, HCoV-229E, and HCoV-NL63. The remaining five beta-CoVs include HCoV-OC43, HCoV-HKU1, SARS-CoV, MERS-CoV, and SARS-CoV-2. Four common human CoVs, HCoV-229E, HCoV-OC43, HCoV-NL63, and HCoV-HKU1 usually cause mild symptoms, like the common cold and/or diarrhea (Su et al., 2016) . In contrast, SARS-CoV/MERS-CoV, and current SARS-CoV-2 are particularly pathogenic, causing SARS and COVID-19, respectively. Because of the resemblance of SARS-CoV-2 to SARS-like coronaviruses, bats are likely to act as natural reservoir hosts of the progenitor of SARS-CoV-2 (Andersen et al., 2020) . It is established that SARS-CoV was transmitted from palm civet to humans (Wang et al., 2006) and MERS-CoV from dromedary camels to humans, yet the zoonotic origin of SARS-CoV-2 is still unclear. The genome sequence analysis shows that SARS-CoV-2 has the closest relative SARS-like SLCoV/RaTG13, found in horseshoe bat (Rhinolophus affinis) from Yunnan, China (Zhou et al., 2020) . SARS-CoV-2 has 96.2% overall genome sequence identity to SARS-like coronavirus SLCoV/RaTG13 (Zhou et al., 2020) . Therefore, the natural reservoir of SARS-CoV-2 could be the horseshoe bat. Recent evidence suggests pangolins as host candidates (Lam et al., 2020) , however, the host pangolins are not firmly determined. Not all bat SARS-like CoV can infect humans. For example, bat coronavirus, swine acute diarrhoea syndrome coronavirus (SADS-CoV) found in 2017 (Zhou et al., 2018) , caused millions of piglet deaths, but no human cases. We may postulate that SARS-CoV-2 evolved from its progenitor through mutations and evolutionary fitness in the host-shift and adapting. SARS-CoV-2 are fast-evolving pathogens that continuously undertake mutations in the generations of infection of the host (Yin, 2020; Wang et al., 2020; Korber et al., 2020) . This fact suggests that SARS-CoV-2 had evolved mutations in critical proteins and genome structures prior to establishing human infection. To understand how SARS-CoV-2 jumps from animals to adaptively infect humans is important to the surveillance of virus evolution and diversity, therefore ultimately controlling the COVID-19 pandemic and preventing future SARS-like outbreaks. SARS-CoV-2 is an enveloped positive-strand RNA virus, having an exceptionally long (29.9kb) genome (Zhou et al., 2020) ,. The genome consists of 5' leader cap sequence along with a 3' poly (A) tail, genes encoding non-structural proteins (nsps), and structural proteins, as well as several accessory proteins. Approximate two-thirds of the genome comprises two large overlapping open reading frames (ORF1a and ORF1ab), encoding polyproteins that are subsequently cleaved by viral proteases to generate 16 non-structural proteins (nsp1 to nsp16). Non-structural proteins are essential for RNA replication, transcriptions, and immune evasion. Nsp3, a large multi-domain and multi-functional protein, plays essential roles in virus replication. The papain-like protease (PLpro) activity of nsp3 is responsible for the initial processing of OFR1a protein. In addition, nsp3, together with nsp4 and nsp6, recruits intracellular membranes to form double-membrane vesicles (DVMs) to support viral RNA replication. Nsp5 is a second viral protease (3C-like protease, 3CLpro) that splits both ORF1a and ORF1ab proteins. The downstream regions of the genome encode structural proteins, the spike (S) protein, the nucleocapsid (N) protein, the envelope (E) protein, and the membrane (M) protein. The four structural proteins are all required to produce a structurally complete viral particle. The S protein mediates viral attachment to the ACE2 receptor host, and the subsequent fusion between the viral and host cell membranes enables the virus to enter host cells. The nucleocapsid (N) protein, one of the most abundant viral proteins, can bind to the RNA genome and participate in processes of replication, assembly, host cell response during viral infection (McBride et al., 2014) . The SARS-CoV-2 genome has been sequenced and annotated, but little is known for the complex structure of the genome in light of evolutionary fitness and host infectivity. Conventionally, RNA viruses use encoded proteins to interact with the components in cellular response. However, numerous discoveries show that the virus RNA structures, determined by genome composition may also play central roles in maximizing virus replication and evolutionary fitness (Jensen and Thomsen, 2012) . For example, recent studies show that increased CG and TA dinucleotides in both coding and noncoding regions of echovirus 7 inhibit replication initiation during post-entry in several cell lines (Fros et al., 2017) . Therefore, RNA viruses simulate host mRNA composition, for example, the dinucleotide frequencies (Fros et al., 2017) . Animal genomes have a bias in their dinucleotide composition, and the heavy under-representation of CG and TA dinucleotides is especially well known. Most animal RNA and small DNA viruses suppress genomic CG and TA dinucleotide frequencies, apparently mimicking host mRNA composition (Di Giallonardo et al., 2017) . If a virus RNA composition or structure is very different from host mRNA, RIG-I (retinoic acid-inducible gene I)-like receptors may detect RNA molecules that are absent from the uninfected host (Goubau et al., 2013) . Detecting evolutionary microbial structures known as pathogen-associated molecular patterns (PAMPs) is an important feature of the innate immune system. Host-cells possess intrinsic defense pathways that prevent replication of viruses with increased CG and TA frequencies in mechanisms independent of codon usage (Belalov and Lukashev, 2013) . The genomic spectrum demonstrates dinucleotide, trinucleotide, and multi-nucleotide distributions. The dinucleotide distributions are often considered as the signature of a genome (Kariin and Burge, 1995) and the 3-periodicity patterns are distinguishing characteristics for the protein-coding regions (Tsonis et al., 1991) . The strengths of the 2-periodicity and 3-periodicity in a genome are determined by the perfect levels and copy numbers of dinucleotide and trinucleotide repeats, respectively. Because 2-periodicity and 3-periodicity are the essential characteristics of a genome, in this study, we only examine these two periodicities from the genomic spectrum when analyzing the genome. In this study, we present the genomic spectrum of SARS-CoV-2 and identify the dinucleotide repeats in ORF1a (nsps 3-6) that have been instrumental in interacting host immune systems and pathogenicity and host infectivity. These dinucleotide repeats are essential to the survival and infectivity of the microbe and can be considered as one of PAMPs in SARS-CoV-2. These genomic elements are evidence of the evolutionary fitness of SARS-CoV-2 in host-shift and adaption. Tracking the evolution of these elements may provide insights into the zoonotic origin of SARS-CoV-2 and the control of COVID-19 disease. To inspect the insightful traits of the SARS-CoV-2 genome, we utilize our periodicity analysis method to survey the nucleotide distributions and the rendered periodicities in the genome. We previously proposed the periodicity analysis method to quantitatively detect the nucleotide repeats and periodicities in a genome (Yin, 2017) . The method employs nucleotide distributions on periodic positions in a genome and identifies approximate repeat structures as the signatures of the genome. Because we have included more functionalities, such as smoothing the periodicity profile, from the original method, here we describe the method in detail though the technical algorithms had been delineated previously (Yin, 2017) . Our computer programs of the periodicity analysis of a genome are available to the public at GitHub repository https://github.com/cyinbox/DNADU. The nucleotide distributions at the periodic positions of a DNA sequence can be represented by a congruence derivative (CD) vector (Yin and Wang, 2016; Yin, 2017) . The CD vector of a nucleotide for a specific periodicity is constructed by the cumulative frequencies of the nucleotide at these periodic positions (Definition 2.1). Definition 2.1. For a DNA sequence of length n, let u  (k) = 1 when the nucleotide  appears at position k, otherwise, u  (k) = 0, where   {A, T, C, G} and k = 1,    , n. The congruence derivative vector of the nucleotide  of the DNA sequence for periodicity p, is defined as , where mod(k, p) is the modulo operation and returns the remainder after division of k by p, and f  = (f  (1), f  (2),    , f  (p)). Four congruence derivative vectors f  of periodicity p for nucleotides A, T, C and G form a congruence derivative (CD) matrix of size 4  p. The columns of the CD matrix indicate nucleotide frequencies at the periodic positions k = pt  q, where k is the position index of a DNA sequence, t = 1, 2, . . ., and q = p  1, . . . , 2, 1, 0. For example, consider the CD matrix of periodicity 5 for DNA sequence, the first column of the CD matrix shows the nucleotide frequencies at periodic positions k = 1, 6, 11, . . . , 5t  4; the second column of the matrix shows the nucleotide frequencies at periodic positions k = 2, 7, 12, . . . , 5t  3; the third column of the matrix shows the nucleotide distributions at periodic positions k = 3, 8, 13, . . . , 5t  2, and so on. The CD matrix of a DNA sequence describes nucleotide frequencies at all periodic positions and can be used to efficiently compute the Fourier power spectrum and determine periodicities in the DNA sequence (Yin and Wang, 2016) . Therefore, the CD vector reflects the arrangement of repetitive sequence elements and inner periodicities in the DNA sequence. Since the CD matrix contains the nucleotide frequencies on periodic positions, the variance of the matrix elements can measure the nucleotide distribution. For the CD matrix of periodicity p, the summation of 4p elements of the matrix is equal to the length n of the DNA sequence, and the mean of the elements of the CD matrix is n 4p . Therefore, to quantify the nucleotide distribution, we define the normalized distribution uniformity (NDU) of a DNA sequence using the CD matrix (Definition 3.2). Definition 2.2. For a DNA sequence of length n, let f i,j be an element of the CD matrix of periodicity p, the normalized distribution uniformity of periodicity p of the DNA sequence is defined as From Definition 2.2., we notice that the normalized distribution uniformity at periodicity p is an intuitive description for the level of unbalance of nucleotide frequencies on periodic positions. It depends on the quadratic function of the nucleotide frequencies, sequence, and periodicity length. NDU(p) can be used to indicate the existence of the periodicity p in a DNA sequence. This method offers an elaboration of the repetitive elements such as the repeat consensus, copy number, and the perfect level (Yin, 2017) . When using a sliding window along a genome, the periodicities of a range of 2 to 10 are calculated in each window segment. Therefore, a two-dimension periodicity spectrum is formed for the genome. The two-dimension spectrum can be considered as the genomics signature or the barcode signature of the genome. To locate the positions of a repeat region in a genome, we smooth and filter the corresponding sliding-window periodicity by moving average convolution (De Jong, 1989) . Then the peaks in the periodicity profile are detected using the Z-score algorithm (Brakel, 2020) . The peak positions are used to demarcate repeats in a genome. In a nutshell, to compute distribution uniformities of different periodicities of a DNA sequence, we first scan the sequence in different periodicity sizes, construct the congruence derivative matrix of each periodicity, and compute the NDU(p) of these periodicities p. The periodicity with the maximum distribution uniformity reflects the predominant pattern of repetitive elements. The NDU values of periodicities indicate the perfect levels, and copy numbers of corresponding repeat regions. This study depends on the complete genomes of coronaviruses, including SARS-CoV-2 (Wu et al., 2020) , Severe Acute Respiratory Syndrome (SARS) related coronavirus, Middle East Respiratory Syndrome coronavirus (MERS-CoV), and human infection coronaviruses (human-CoVs). These genome data are retrieved from the National Center for Biotechnology Information (NCBI) Gene Bank. The bat SLCoV /RaTG13 complete sequence was reported by (Zhou et al., 2020) , and downloaded from the GISAID repository (http://www.GISAID.org) (Shu and McCauley, 2017) . The genome data in this study are listed in the supplementary material. 3 Results and analysis 3.1 Genomic spectrum coronavirus SARS-CoV-2 reveals rich dinucleotide patterns in nsps To identify the signature features of the coronavirus SARS-CoV-2 genome, we employ the periodicity spectrum analysis to identify the characteristic periodicities in the genome. We create the genomic spectrum (barcode) of SARS-CoV-2 ( Fig.1(a) ) using the sliding window NDU method and compare it with the counterparts of SLCoV/RaTG13 ( Fig.1(b) ), SARS-CoV/Tor2 (Fig.1(c) ), and MERS-CoV ( Fig.1(d) ). From the spectrum comparison, we observe that SARS-CoV-2 and SLCoV/RaTG13 both have pronounced 2-periodicity in four regions while both SARS-CoV/Tor2 and MERS-CoV only have an extremely low level of 2-periodicity in the corresponding regions. The strong dinucleotide signal in SARS-CoV-2 encouraged us to investigate the causes in detail. To locate the regions of rich dinucleotide repeats, we verify that 2-periodicity and 3-periodicity are strong signals among all genomic periodicities ( Fig. 2(a, d) ), and detect the peaks of the slidingwindow periodicity profiles ( Fig.2 (b, c) ). The peak positions are used to demarcate the dinucleotide repeat regions in the genome. The dinucleotide repeat regions (dinucleotide islands) are in ORF1a and the corresponding genes are listed in Table 1 . From the genomic spectrum analysis, the relative abundance of the dinucleotide repeats, particularly dinucleotide TT, are mapped to the genes of ORF1a (nsp3, nsp4, and nsp6) ( Fig.3 and Table 1 .). However, these dinucleotide repeat signals are weak or imperceptible in the corresponding regions in the SARS-CoV and MERS-CoV genomes. In the coronavirus SARS-CoV-2 RNA genome, the gene for replicase of 20 kb encodes two overlapping polyproteins, ORF1a (replicase 1a) and ORF1ab (replicase 1ab). The genome structure and the corresponding dinucleotide regions identified are illustrated in Fig.3 . The two polyproteins are responsible for viral replication and transcription . The expression of the C-proximal portion of pp1ab requires (-1) ribosomal frame-shifting. The first dinucleotide repeat is in the coding region of Papain-like proteinase (PL proteinase, non-structural protein 3, nps3). Nsp3 is the largest essential component of the replication and transcription complex. The PL proteinase in nsp3 cleaves nsps 1-3 and blocks host innate immune response, promoting cytokine expression (Lei et al., 2018; Serrano et al., 2009) . The second dinucleotide repeat is in the coding region of non-structural protein 4 (nsp4). Nsp4 is responsible for forming double-membrane vesicles (DMV). The third dinucleotide repeat is in the coding region of the C-terminal 3CLPro protease (3 chymotrypsin-like proteinase, 3CLpro) and nsp6. 3CLPro protease is essential for RNA replication. The 3CLPro proteinase is responsible for processing the C-terminus of nsp4 through nsp16 for all coronaviruses (Anand et al., 2003) . Therefore, conserved structure and catalytic sites of 3CLpro may serve as attractive targets for antiviral drugs (Kim et al., 2012) . Together, nsp3, nsp4, and nsp6 can induce DMV (Angelini et al., 2013) . In summary, the dinucleotide repeat islands found in this study are located in the host-interaction regions of the genome of SARS-CoV-2. These special dinucleotide repeat regions in ORF1a most likely contribute to the adaptive immune response, therefore, implying evolution fitness. Coincidentally, previous work on MERS-CoV using co-evolution analysis revealed that nsp3 represents a preferential selection target in adaptive evolution for zoonotic MERS-CoV to a new host (Forni et al., 2016) . Our finding that nsp3 is involved in evolution fitness is consistent with the discovery in MERS-CoV. We investigate the correlation of dinucleotides AA and TA contents in SARS-CoV genomes and virulence. We examine the dinucleotide in the genomic regions (6 kb -12 kb). The increased 2-periodicity in the genomic regions of SARS-CoV-2 and SARS-like CoVs are the results of the unbalanced distributions of dinucleotides. To understand the evolutionary tendency of coronavirus genomes, we examine the genomic spectra of four major bats SARS-like coronaviruses (SLCoVs), pangolin-SLCoV, SLCoV/ZXC21, SLCoV/WIV1, and SLCoV/Shaanxi2011, all of which naturally live in bat Rhinolophidae horseshoe. Because pangolin-SLCoV was found similar to SARS-CoV-2, pangolin was exploratorily postulated as an intermediate animal host of SARS-CoV-2 (). SLCoV/ZXC21 is the second similar strain to SARS-CoV-2, with 82% similarity (Hu et al., 2018) . SLCoV/WIV1 was closely related to SARS-CoV/Tor2 in terms of genome identity and ACE2 binding in human cells (Ge et al., 2013) . SLCoV/Shaanxi2011 was found in 2011 . The results show that pangolin-SLCoV displays three major dinucleotide repeats in nsp3 and nsp4, but lacks the corresponding dinucleotide repeats in 3CLPro and nsp6 as found in SARS-CoV-2 ( Fig.4(a) ). So SARS-CoV-2 is mostly closed to SLCoV/RaTG13 ( Fig.1(b) ) and SLCoV/ZXC21 (Fig.4(b) ), not pangolin-SLCoV. If pangolins are the intermediary hosts of SARS-CoV-2, and SARS-CoV-2 was indeed evolved from pangolin-SLCoV, we may infer that pangolin-SLCoV would need to evolve dinucleotide repeats in 3CLPro and nsp6 during evolution fitness before infecting human hosts. We also observed the evolution trend between SLCoV/WIV1 (Fig.4(c) ) and SLCoV/Shaanxi2011 ( Fig.4(d) ). The genomic spectrum of SLCoV/Shaanxi2011 is similar to SLCoV/WIV1, but has additionally increased dinucleotide repeat in 3CLPro and nsp6. This new dinucleotide repeat in the region 3CLPro and nsp6 in SLCoV/Shaanxi2011 is consistent with the regions found in SARS-CoV-2, SLCoV/RaTG13, and SLCoV/ZXC21. Therefore, the dinucleotide repeat in the region 3CLPro and nsp6 probably play an important role in the evolutionary fitness of SARS-CoV-2 to the human hosts. The results show that only SARS-CoV and SARS-like CoVs have low dinucleotide repeats. The low-dinucleotide contents can be considered as in early evolution fitness when interacting with the human immune system, then low-dinucleotides may render high virus virulence because the virus has not adapted to the host immune system, and the host immune system acts intensely. To investigate the correlation of dinucleotide repeats and pathogenicity of coronaviruses, we produce and compare the genomic spectra of four common human coronaviruses (Fig.5) . spectrum analysis, we observe higher 2-periodicity in HCoV-229E than in HCoV-OC43 ( Fig.5 (a,c) ). These dinucleotide repeats regions correspond to the three sub-regions in SARS-CoV-2. High 2periodicity value may attenuate the virus replication and therefore reduce severe virulence. That is in an agreement with the correlation of dinucleotide repeats and pathogenicity previously. The spectra of HCoV-NL63 and HCoV-HKU1 demonstrate extremely high 2-periodicity, as well as 3-periodicity in the corresponding regions ( Fig.5 (b,d) ). HCoV-NL63 and HCoV-HKU1 are the most common human CoVs that cause only a mild cold symptom or no symptom (Pyrc et al., 2007) . Again, these high dinucleotide repeats may contribute to the light pathogenicity in these two viruses. We wish to know whether SARS-CoV-2 originally evolved from SARS-CoV, or SARS-CoV-2 would evolve to SARS-CoV. The answer to this question may help us to predict the evolution of SARS-CoV-2 virus for better disease prediction and control. Because the genomes of SARS-related coronaviruses over a long time period are rarely available, to infer the evolution of coronaviruses, we track the trend of HCoV-229E coronaviruses over the last six decades from the first human infected HCoV-229E identified in 1962 (Thiel et al., 2001) . HCoV-229E virus causes common cold but occasionally it can be associated with more severe respiratory infections in children, elderly, and persons with underlying illness. Using the measurements of dinucleotides in the genomic regions, the trend of HCoV-229 coronaviruses may infer the evolutionary stages of bat coronaviruses. In a similar method, we may then determine the origin SARS-CoV-2 if the trend of SARS-CoV-2 is compared with SARS-like coronaviruses. The human coronavirus HCoV-229E strains used in the dinucleotide trend analysis are from different historical periods. The reference genome HCoV-229 in the evolutionary analysis was obtained from the infectious HCoV-229E, the 1973-deposited laboratory-adapted prototype strain of HCoV-229E (VR-740). The HCoV-229E prototype strain was originally isolated in 1962 from a patient in Chicago. The first clinical HCoV-229E isolate from a US patient in 2012 was included in this study (Farsani et al., 2012) The result in the periodicity trends of the HCoV-229E coronaviruses demonstrates that both 2periodicity ( Fig.7(a) ) and 3-periodicity ( Fig.7(b) ) in the coronaviruses are increasing with evolutionary time. SARS-CoV-2 has relatively high 2-periodicity and 3-periodicity. This result suggests that the trends of 2-periodicity and 3-periodicity are increasing with time. The evolutionary origin of coronaviruses can be inferred by the trends of 2-periodicity and 3-periodicity. Therefore, we may compare the 2-periodicity and 3-periodicity in SARS-CoV-2 and SARS-like coronaviruses to understand the evolutionary origin of SARS-CoV-2. To investigate the correlation of dinucleotide repeats with virus virulence in SARS-related coronaviruses, we compare the spectra of the genomic region at coordinates 6k-12k bp of five SARS-related coronaviruses. The genomic regions contain abundant dinucleotide repeats. The region 6k-12k of the genome contains three dinucleotide repeats approximately located at 6k-8k, 8k-10k, and 10k-12k sub-regions. The five SARS-related coronaviruses have different levels of virulence. The most virulent virus is MERS-CoV, followed by SARS-CoV. The 2-periodicity magnitudes, which reflect the distribution of dinucleotides, are compared and shown in (Fig.8 (a,b) ). We may see that MERS-CoV genomic region has the lowest dinucleotide level in all three dinucleotide sub-regions. SARS-CoV also has a low dinucleotide level but is higher than MERS-CoV. We, therefore, may infer that the low dinucleotide level correlates with high virulence. The lower the dinucleotide level is, the higher virus virulence is. This postulation is supported by the observation of the dinucleotide in three SARS-related coronaviruses. Compared with the SARS-CoV, the SARS-like bats-SLOV/Rp3 has similar dinucleotide distributions in sub-regions 1 (6k-8k) and 3 (10k-12k), but higher dinucleotide distribution in sub-region 2 (8k-10k). SLCoV/Rp3 has lower virus virulence than SARS. The SARS-like SLCoV/ZXC21, which shares the highest sequence identity with SARS-CoV-2, shows slightly lower dinucleotide distributions in the two sub-regions 2 and 3, and slightly higher dinucleotide distribution than sub-region 1. We notice that trends of the 2-periodicity and 3-periodicity from MERS-CoV, SARS-CoV, SARSlike CoVs, and SARS-CoV-2 are increasing ( Fig.8 (a,b) ). Based on the previous analysis of the periodicity trends of coronaviruses at different times, we may infer that SARS-CoV-2 originates from the SARS-like CoVs. It is well studied that dinucleotide composition bias in RNA viruses may impact the virus replications, specifically, attenuating or strengthening virus virulence during evolution (Fros et al., 2017; Gu et al., 2019) . Clinical evidences have suggested that SARS-CoV-2 has lower virulence than SARS-CoV. To investigate if the increased dinucleotide repeats correlate with virus virulence, we compare the dinucleotide frequencies in the three dinucleotide rich islands of SAR-CoV-2 and SARS-CoV/Tor2. The result shows that both SARS-CoV-2 and SARS-CoV have an abundance of dinucleotides TT and TA in the whole genome, and three prominent dinucleotide repeat regions ( Fig.9 (a,b,c,d) ), and extreme CG deficiency in sub-region 3 ( Fig.9(d) ). The dinucleotides TT and TA are increased in SARS-CoV-2 compared with SARS-CoV ( Fig.9 (a,b,c,d) ). The role increased dinucleotides TT and TA in the SARS-CoV-2 genome is to possibly attenuate virus replication. One mechanism for attenuating virus by dinucleotide bias is that the dinucleotide regions fold a special structure as a target for cell RNA cleavage, which is a fundamental host response for controlling viral infections (Zhou et al., 1993) . The 2,5-oligoadenylate synthetase/RNase L system is an innate immunity pathway that responds to a pathogen-associated molecular pattern (PAMP) to induce degradation of viral and cellular RNAs, thereby blocking viral infection. In higher vertebrates, this process is often regulated by interferons (IFNs). Ribonuclease L (RNase L, L is for latent) is an interferon (IFN)-induced antiviral ribonuclease which, upon activation, destroys all RNA within the Table 1. cellular and viral (Silverman, 2007) . RNase L cleaves hepatitis C virus (HCV) RNA at single-stranded TT and TA dinucleotides throughout the open reading frame (ORF). An interesting discovery is that in bacterium Mycoplasma pneumonia, which is a respiratory infection agent, the genomes also have relative abundance extremes dinucleotides TT and TA (Karlin, 1998) . Therefore, we may postulate that the dinucleotides TT and TA regions in SARS-CoV-2 are possibly cleaved by RNase L during infection. In this study, we identify the unique dinucleotide repeats in the SARS-CoV-2 genome. The dinucleotide repeats in the genomic spectra are revealed by the periodicity analysis. We discover the strength of these repeats correlates with the evolutionary fitness of the virus to a human host for maximizing its survival in epidemics, instead of destroying the host. Therefore, RNA viruses simulate host mRNA composition such as the dinucleotide compositions (Fros et al., 2017) . Most vertebrate RNA and small DNA viruses suppress genomic CG and TA dinucleotide frequencies, apparently mimicking host mRNA composition. The abundance of dinucleotides TT and TA are most likely common pathogenicity islands in microbial genomes. This study on SARS-CoV-2 provides additional evidence that increased dinucleotides TT and TA in SARS-CoV-2 is the result of interaction with the host during virus evolution. We consider these three dinucleotide abundance regions as pathogenicity islands of the SARS-CoV-2 genome. In addition, these special regions may contribute RNA replications, and can be recognized by cell RNAase L for RNA degradation in the immune response. However, this study is only a theoretical analysis of the genomes. The actual functional consequences and the impacts on transmissibility and pathogenesis of these dinucleotide repeats should be determined by biochemical experiments and animal models. In humans and mammals, APOBEC (apolipoprotein B mRNA editing enzyme, catalytic polypeptide-like3) systems help protect the organisms from viral infections. For the origin of dinucleotide repeats in the SARS-CoV-2 genome, we speculate that the molecular mechanism of increased dinucleotide repeats during evolution fitness is possibly APOBEC3-mediated editing of viral RNA, in which Cytosine (C) is often mutated to Uracil (U) by deamination (Bishop et al., 2004) . Therefore, many dinucleotide TT repeats could be generated as the 2-periodicity island in the RNA genome by the APOBEC defense system. Cross-species transmission of coronaviruses from wildlife reservoirs may lead to disease outbreaks in humans, posing a severe threat to human health. To date, most studies on the zoonotic origin of SARS-CoV-2 primarily focus on the Spike protein, which is essential for the entry of virus particles into the cell. Mutations or acquisition of potential cleavage site for furin proteases in the Spike protein may confer zoonotic transmissibility of SARS-CoV-2 (Andersen et al., 2020; Hoffmann et al., 2020) . However, the spike protein is required but not sufficient for zoonotic coronavirus transmission. For instance, the Spike protein in SARS-like SHC014-CoV can only enable the chimeric virus SHC014-MA15 to infect human cells when the Spike protein is integrated into a wild-type SARS-CoV backbone (Menachery et al., 2015) . The SARS-CoV backbone contains ORF1ab replication components. Our study provides evidence for the importance of ORF1ab replication regions in evolutionary fitness. Consequently, these ORF1ab replication components are critical for zoonotic transmissions. This study on the genomic spectrum of SARS-CoV-2 reveals high dinucleotides TT regions in ORF1a might contribute to evolution fitness in host immune evasion. Accordingly, monitoring SARS-CoV-2 and developing antiviral drugs should envisage molecular characteristics and changes of nsps 3-6, which are vital components in interacting with human hosts. These special dinucleotide repeat regions should be investigated in detail for its functions and phenotype changes in SARS-CoV-2. Importantly, these dinucleotide repeat regions can possibly be the prophylactic and therapeutic targets for controlling COVID-19. \",\n",
       "   '_search_score': 0.8499500000000002},\n",
       "  {'_id': 'a45430e796de9329f33ca9a4ea936b3c569c5296',\n",
       "   'insert_date_': '2020-11-04T18:52:51.722127',\n",
       "   'abstract': 'The Middle East respiratory syndrome coronavirus was first identified in 2012 and has since then remained uncontrolled. Cases have been mostly reported in the Middle East, however travel-associated cases and outbreaks have also occurred. Nosocomial and zoonotic transmission of the virus appear to be the most important routes. The infection is severe and highly fatal thus necessitating rapid and efficacious interventions. Here, we performed a comprehensive review of published literature and summarized the epidemiology of the virus. In addition, we summarized the virological aspects of the infection and reviewed the animal models used as well as vaccination and antiviral tested against it.',\n",
       "   'title': 'Middle East respiratory syndrome coronavirus: a comprehensive review',\n",
       "   'body': 'Coronaviruses (CoV) became known to cause human disease in the twentieth century. HCoV-229E and HCoV-OC43 were discovered in the 1960s and shown to cause respiratory infections in humans [1, 2] . With the emergence of SARS-CoV in 2003 [3] , two other human coronaviruses were discovered, HCoV NL63, HCoV HKU1 [4] . In 2012, a new type of coronavirus was detected as the cause of severe respiratory illness in humans. The first case was a 60-year-old male from Saudi Arabia admitted to hospital with acute respiratory illness leading to pneumonia and acute renal failure. The virus initially named as human corona virus-EMC [5] , is currently known as the Middle East respiratory syndrome coronavirus (MERS-CoV) [6] . Classification and nomenclature of MERS-CoV Phylogenetically, MERS-CoV is a lineage C  coronavirus (-CoV) and is closely related to bat coronaviruses HKU4 and HKU5. The rooted phylogenetic analysis showed that MERS-CoV had an amino acid sequence identity less than 90% to all other known CoVs [7] . The virus initially named by many different working groups as novel coronavirus, human coronavirus EMC, human b coronavirus 2c EMC, human b coronavirus 2c England-Qatar, human b coronavirus 2C Jordan-N3, and b coronavirus England 1, which represented the places where the first complete viral genome was sequenced (Erasmus Medical Center, Rotterdam, The Netherlands) or where the first laboratory-confirmed cases were identified or managed (Jordan, Qatar, and England) was later named as MERS-CoV by the coronaviruses study groups of ICTV [5, 6, 8] . MERS-CoV is an enveloped virus with a positive sense RNA genome. Coronavirus genomes range between 25 to 32 kb in size. The complete sequence of HCoV-EMC-2012 resulted in 30 119 nucleotides sequence [7] . Coronavirus genomes are polycistronic with large replicase open reading frames ORF1a and ORF1b which are subsequently cleaved into 15 or 16 nonstructural proteins (NSPs). The region downstream of ORF1b encode smaller genes including the spike (S), envelope (E), membrane (M), and nucleocapsid (N) structural protein [9] [10] [11] . The functional receptor for MERS-CoV is the Dipeptidyl peptidase 4 (DPP4) which is present on human nonciliated bronchial epithelial cells surfaces [12] . The DPP4 protein displays high amino acid sequence conservation across different species, including the sequence that was obtained from bat cells. Cell lines susceptibility studies showed that MERS-CoV infected several human cell lines, including histiocytes as well as respiratory, kidney, intestinal, and liver cells [13] . The range of tissue tropism in vitro was broader than that for any other known human coronavirus [14] . MERS-CoV can also infect nonhuman primate, porcine, bat, civet, rabbit, and horse cell lines all possessing the DPP4 receptor [15] . The replication cycle of MERS-CoV consists of numerous steps as illustrated by Lu et al. [30] . The MERS-CoV S protein is a class I fusion protein composed of two subunits: the amino N-terminal receptor binding S1 and carboxyl C-terminal membrane fusion S2 subunits. The S1/S2 junction is a protease cleavage site which is responsible for membrane fusion activation, virus entry, and syncytium formation. The S1 C domain contains the receptor binding domain (RBD), and an N domain [13] . Neutralizing monoclonal antibodies against the RBD may inhibit virus entry into cells and receptor-dependent syncytium formation in cell culture, hence vaccines containing the RBD induced high levels of neutralizing antibodies in mice and rabbits [16] [17] [18] . DPP4 is the cell key functional receptor for the MERS-CoV S protein [19] . MERS-CoV is the first CoV that has been identified to use DPP4 as a receptor [19, 20] . DPP4 has important roles in glucose metabolism, T cell activation, chemotaxis modulation, cell adhesion, and apoptosis [19, 21] . The S2 subunit contains five domains: a fusion peptide, the heptad repeat 1 (HR1) and HR2 domains, a transmembrane domain, and a cytoplasmic domain, which form the stalk region of S protein that facilitates fusion of the viral and cell membranes [22, 23] . The binding of the S1 subunit to the cellular receptor triggers conformational changes in the S2 subunit, which inserts its fusion peptide into the target cell membrane to form a six-helix bundle fusion core between the HR1 and HR2 domains that approximates the viral and cell membranes for fusion. MERS-CoV utilizes many pathways for membrane fusion depending on available host proteases, such as transmembrane protease serine protease 2 (TMPRSS2), trypsin, chymotrypsin, elastase, thermolysin, endoproteinase Lys-C, and human airway trypsin-like protease. Proteases cleave the S protein into the S1 and S2 subunits to activate the MERS-CoV S protein for endosome-independent host cell entry at the plasma membrane [24] [25] [26] . In addition to the pervious fusion proteases furin has been identified recently to play an essential role in the MERS-CoV S protein cleavage activation into their biologically active forms [27, 28] . After cell entry, the virion particle disassembles to release the nucleocapsid and viral RNA into the cytoplasm for expression of viral polyproteins pp1a and pp1ab. Doublemembrane vesicles and convoluted membranes are formed by the attachment of the hydrophobic domains of the MERS-CoV replication machinery to the limiting membrane of auto-phagosomes [29] . The viral polyproteins pp1a and pp1ab are cleaved by papain-like protease and 3C-like protease into nsp1 to nsp16 [7, 30, 31] . These nonstructural proteins form the replication-transcription complex, which regulates transcription and viral protein expression [29] . After the production of abundant viral RNA and structural and accessory proteins, the N protein binds to the genomic RNA in the cytoplasm to form the helical nucleocapsid (viral core). The viral core is enveloped by budding through intracellular membranes between the endoplasmic reticulum and Golgi apparatus [32] . The S, E, and M proteins are transported to the budding virion, where the nucleocapsid probably interacts with M protein to generate the basic structure and complexes with the S and E proteins to induce viral budding and release from the Golgi apparatus [33] . MERS-CoV replication cycle is completed by releasing the progeny virions through the cell membrane via exocytosis pathway. Mice MERS-CoV strain HCoV-EMC/2012 was inoculated to three different mouse strains (immunocompetent BALB/c mice, 129S6/SvEv and innate immune-deficient 129/ STAT1 -/mice) intranasally. No significant weight loss was observed and infectious virus could not be detected in the lungs. Only moderate pathological lesions were observed in the lungs. Hence no viral replication was observed in these strains of mice [34] . Zhao et al. developed a mouse model transduced with a recombinant adenovirus vector expressing hDPP4 (Ad5-hDPP4) in lung tissue. Inoculation of MERS-CoV in these mice resulted in MERS-CoV replication but without mortality. Young mice cleared from MERS-CoV in 6-8 days and old mice in 10-14 days. Perivascular and peribronchial lymphoid infiltration was observed, with progression to an interstitial pneumonia postinfection [35] . In another study, transgenic mice expressing hDPP4 were susceptible to MERS-CoV infection. Infectious virus was isolated from lung and brain tissue and weight loss was observed [36] . Pascal et al. developed humanized transgenic mouse. No mortality or clinical signs was observed but interstitial pneumonia and significant lung disease were observed histopathologically, suggesting that humanized DPP4 mouse is a model for MERS-CoV infection in which pathological changes resembles MERS-CoV infection in humans [37, 38] . The rhesus macaque was the first animal model used for MERS-CoV infection as it possessed DPP4 receptor [38, 39] . In infected animals, an increase in respiratory rates, body temperature, cough and reduced appetite was observed with mild to moderate severity. Infectious virus isolated only from the lower respiratory tract. Viral RNA was detected in the conjunctiva, nasal mucosa, tonsils, pharynx, trachea, bronchus and lungs. Mild to marked interstitial pneumonia with dark red lesions appeared in lungs. Seroconversion of neutralizing antibodies began at 7 dpi and increased in titer with time. The development of a transient pneumonia, rapid replication, and tropism of MERS-CoV for the lower respiratory tract resembled the severity of the disease observed in humans [38, 40, 41] . Similarly, the common marmoset was shown to possess the DPP4 receptor [42] . Radiographic imaging showed mild to severe bilateral interstitial infiltration and extensive bronchointertitial pneumonia in infected animals. Infectious virus was detected in lower and upper respiratory tract tissue and viral RNA was detected in nasal mucosa, oropharyngeal swabs, blood, conjunctiva, lymph nodes, gastrointestinal tract, kidney, heart, adrenal gland, liver, spleen, brain and lungs [42] . Inoculation of Syrian hamsters and ferrets with MERS-CoV did not result in infection [12, 43] . Rabbits may be used as a model to study pathogenesis, transmission, and disease control strategies of MERS-CoV in vivo as they seroconvert and shed virus after inoculation [44] . In September 2012, a novel coronavirus infection was noted in ProMed Mail [45] . The virus was isolated from the sputum of a 60-year-old Saudi male, who was admitted to a hospital with pneumonia and acute kidney injury in June 2012. A few days later, another report appeared describing an almost identical virus detected in a patient in Qatar with acute respiratory syndrome and acute kidney injury. The patient had a recent travel history to Saudi Arabia and then traveled to UK for further medical care [5, 46, 47] . Two cases from Jordan (April 2012) were retrospectively diagnosed as MERS patients. Since that time, more than 1542 cases of MERS-CoV infection have been reported including 544 deaths [48] . The actual number of cases could be higher than those reported [49] . An outbreak of more than 180 confirmed cases including 36 deaths occurred in South Korea in May and June 2015. The median age of Korean cases were 55 years (range: 16 to 87 years), 60% were men, and 14% were health care professionals. The index case was a 68-year-old male who had recently traveled to several countries in the Arabian Peninsula [50] . Disease Control and Prevention (CDC), and the Ministry of Health of Saudi Arabia (MOHSA) as asymptomatic, mild, severely symptomatic, or mortal. Cases may be classified into suspected, probable, and confirmed [52, 53] . Any person with laboratory confirmation of infection with MERS-CoV irrespective of clinical signs and symptoms is considered as a confirmed case. WHO criteria for laboratory confirmation require detection of viral RNA or acute and convalescent serology. The presence of nucleic acid can be confirmed by positive results from at least two sequence-specific rRT-PCRs or a single sequence-specific rRT-PCR test and direct sequencing from a separate genomic target [54] . A case confirmation by serological methods requires demonstration of seroconversion in two samples collected at least 14 days apart using at least one screening assay (enzyme-linked immunoassay, immunofluorescence assay) and a neutralization assay. A probable case is defined by the following criteria, a febrile acute respiratory illness as pneumonia or acute respiratory distress syndrome, direct contact with a confirmed MERS-CoV case and unavailability of MERS-CoV testing or results being inconclusive for a single inadequate specimen. Any person who developed a fever and pneumonia or acute respiratory distress syndrome with a history of travel to countries in or near the Arabian Peninsula within 14 days before symptom onset or was in contact with a traveler from this region who developed a febrile respiratory illness is considered as a MERS-CoV suspected case. The WHO, CDC, and MOHSA recommended laboratory diagnostics for MERS-CoV infection [6, 9, 51, 55, 56] . MERS-CoV cases must be confirmed by at least two positive qRT-PCR tests on two different specific genomic regions or single positive qRT-PCR with a sequence of another positive genome fragment [57] . The WHO algorithm for testing MERS-CoV relies on qRT-PCR and sequencing [58] . Available real-time tests include an assay targeting the RNA upstream of the E gene (upE) as a highly sensitive screening assay and three confirmatory assays targeting open reading frames (ORF 1a and 1b) and/ or N gene. The ORF 1a assay is of equal sensitivity to the upE assay. The ORF 1b assay is less sensitive but is useful for confirmation. These assays are specific for MERS-CoV and have not shown cross-reactivity with other respiratory human coronaviruses. For sequencing, two target genes, the RNA-dependent RNA polymerase (RdRp, present in ORF 1b) and N genes are enough to confirm the existence of MERS-CoV RNA in the samples of a patient [57] . Several serologic assays including immunofluorescence assays, protein microarray assay, enzyme-linked immunosorbent assay (ELISA) have been developed for the detection of MERS-CoV antibodies [57, [59] [60] [61] . Any positive test by one of these assays should be confirmed with a neutralization assay. Single serological result may be valuable for definition of probable case and should be followed by further testing for confirmation of MERS-CoV infection [62] [63] [64] . Incubation period of MERS-CoV infections was studied by Assiri et al. in 2013. The median incubation period was 5.2 days (95% CI 1.9-14.7 days) [65] . In another report from France of a secondary case, a patient who shared a room with an infected patient, the incubation period was estimated at 9 to 12 days [66] . In the recent outbreak in South Korea during May/June 2015, the median incubation period was 6.3 days [67] . WHO and CDC recommended that individuals that returned from the Arabian Peninsula and other affected countries must be evaluated for MERS-CoV infections up to at least 14 days [68] . Clinical features of MERS-CoV infections range from asymptomatic cases to mildly ill, severe pneumonia, acute respiratory distress syndrome, septic shock and mortal with multi-organ failure (Table 1) [64, 65] . Many other clinical features such as gastrointestinal symptoms (anorexia, nausea, vomiting, abdominal pain, diarrhea), pericarditis, and disseminated intravascular coagulation were reported [65, 69, 70] . Specific clinical conditions (comorbidities) were apparently proportionate with high severity of MERS-CoV infections. A study by Assiri et al. in Saudi Arabia showed that of a total of 47 patients with MERS-CoV infection in 2013, 45 (96%) had underlying clinical conditions, including diabetes mellitus (68%), hypertension (34%), chronic cardiac disease (28%), and chronic kidney disease (49%) [65] . This high rate of comorbidities must be interpreted with some caution, since diabetes mellitus is common in Saudi Arabia, and because approximately half of those 47 were part of an outbreak in a hemodialysis unit, where rates of comorbidities might be high due to chronic kidney disease [65, 71] . In another study, being on dialysis, diabetes mellitus, and age > 50 years was associated with mortality [72] . In this study, testing positive for MERS-CoV in a plasma sample was a predictor of severe outcome [72] . Younger adults and children appeared to be less susceptible to MERS-CoV infection. Only one study described MERS-CoV infection in children [73] . All of those children were discovered during contact investigations of older patients. Only 2 of 11 children developed symptoms of MERS-CoV infection. These two children had underlying conditions (cystic fibrosis and Down syndrome). The other 9 children were asymptomatic. There are few reports of MERS-CoV infections in pregnant women. A five-month pregnant female developed vaginal bleeding and abdominal pain after one week, then delivered a stillborn infant [74] . Another case in the United Arab Emirates was near term phase, she gave birth to an apparently healthy baby, and died after delivery [52] . Mild and asymptomatic MERS-CoV infections have been reported, a majority of whom were identified among the contacts of patients [62, 75, 76] . In a report from MOHSA, more than 3000 contacts of patients were screened using qRT-PCR and seven healthcare workers with MERS-CoV infection were identified, two of whom were asymptomatic and five of whom had mild upper respiratory tract symptoms [75] . Epidemiological and virological studies were conducted in attempts to determine person to person transmission of MERS-CoV. They studied case clustering in household and hospital outbreaks in the UK, Tunisia, Italy, and in healthcare facilities in Saudi Arabia, France, Iran, and lately in South Korea. Those studies provided strong evidence that human-to-human transmission occurs [70, [77] [78] [79] [80] . The number of contacts infected by individuals with confirmed infections, however, appears to be limited [62] , except the outbreak of South Korea in May/ June 2015, where most cases were secondary and some cases were tertiary infections [67, 81] . Secondary cases often were milder or symptomless [62] . Possible modes of transmission include droplet and close contact transmission, air borne transmission, and fomite transmission [82] . The majority of all laboratory-confirmed secondary cases have been associated with healthcare settings [82] . The majority of cases of Jeddah, Saudi Arabia hospital outbreak during the spring of 2014 were acquired through human-to-human transmission due to systematic weaknesses in infection control [76] . Secondary transmission rates were assessed within households and the transmission rate was around 4%, suggesting that the actual number of infection is greater than reported [62] . During the outbreak in South Korea during May/June 2015, 25 secondary infections were associated with the index case, who was hospitalized from May 15 to May 17 and 11 were tertiary [83] . The median incubation period was six days for secondary cases and six days for tertiary cases. This outbreak also clearly demonstrated roles of \"superspreaders,\" who may be responsible for a high proportion of cases [83] . For instance, a single patient infected more than 70 other people while being treated in the emergency room of a hospital in South Korea for three days, 27-29 May 2015. Transmissibility and epidemic potential studies of MERS-CoV revealed that the reproduction number (R 0 ) of patients infected with MERS-CoV ranged between 0.6 to 0.69 [84, 85] . The finding of an R 0 < 1 suggests that MERS-CoV does not yet have pandemic potential. Other study suggested that R 0 values might reach to 0.8 to 1.3 in the absence of infection control [49] . Shedding periods of MERS-CoV in humans was reported to be long as viruses were detected in lower respiratory samples of symptomatic patients for more than two weeks [86] . At instances, prolonged shedding for 6 weeks was detected in an asymptomatic healthcare worker. These findings raise concerns that asymptomatic persons could transmit infection to others in a silent manner [87] . The majority of cases have occurred in Saudi Arabia and United Arab Emirates [88] [89] [90] . Many cases have also been reported outside the Arabian Peninsula in North Africa, Europe, Asia, and North America as shown in Table 2 . Almost all cases reported outside the Arabian Peninsula had a travel history to it. The first cluster was in October/November 2012 in four men of the same family in Riyadh, Saudi Arabia, two of whom died [75] . The second cluster was reported in Jordan in April 2012 involving 10 healthcare workers exposed to fatal patients. In addition, seven surviving hospital contacts seroconverted suggesting that they had MERS-CoV infection [91] . The third cluster was reported in UK during January/ February 2013. An English resident had a travel history to Saudi Arabia and Pakistan in January, developed a severe respiratory illness, and tested positive for both MERS-CoV and H1N1 influenza A, and died in March 2013 after infecting several contacts [92] . A cluster of 43 cases of MERS-CoV was reported in Al-Hasa in Saudi Arabia during April 2013. All those cases were directly linked to human to human contact in the same hospital. There were only two confirmed cases of healthcare workers, and three family members were detected by a survey of over 200 household contacts that visited this hospital [77] . In France, May 2013, an infection of MERS-CoV was reported in a patient who recently traveled to the United Arab Emirates. A second case who shared the hospital room with the first case tested positive. The first patient died and the second patient was critically ill. A survey of 100 healthcare workers found no other infections with MERS-CoV, despite the lack of use of personal protective equipment [70] . A surge in MERS-CoV cases was reported in Saudi Arabia and the United Arab Emirates during March and April 2014 [55, 76] . The majority of cases were associated with hospital-based outbreaks Jeddah, Riyadh, Tabuk, and Madinah in Saudi Arabia as well as in Al Ain, and Abu Dhabi in United Arab Emirates. Cases included several healthcare workers, visitors, patients, and ambulance staff. Person to person transmission was confirmed in > 75% of cases. The majority of infected health care workers developed mild symptomatic or asymptomatic infection, but about 15% had severe illness or died [93] . The recent outbreak of South Korea occurred in May 2015. The index case was a man who had recently traveled to Bahrain, the United Arab Emirates, Saudi Arabia, and Qatar [55] . As of late July 2015, > 180 secondary cases were reported including 36 death and many cases had been reported among household and hospital contacts [55, 67] . In China, one case occurred in a man who traveled to China from Korea following exposure to two relatives with MERS-CoV infection [55] . In spite of reporting of MERS-CoV infections throughout the year, some evidence on disease seasonality occurred. The first identified cases of MERS-CoV infection were reported in April and June 2012 [5, 46, 47] . A high increase in cases was reported in April and May 2013 followed by a surge in case reporting in April and May 2014. Increase in case reporting in March to May 2013 were attributed to infection from infected young camels [94, 95]  Sources and modes of transmission of MERS-CoV are still unclear. Initially, a bat origin of MERS-CoV was suggested based on the relation of genome sequences between MERS-CoV and bat coronaviruses [96] . Cell tropism studies showed that both bat coronavirus HKU4 and MERS-CoV shared the same cell type receptors, DPP4 [4, 7, 75] . MERS-CoV grows readily in several bat-derived cell lines [14] . There is no evidence for direct or indirect transmission of MERS-CoV from bats to humans. Virological studies performed in Europe, Africa, and Asia, including the Middle East, have shown that coronavirus RNA sequences are found frequently in bat feces. Some of the sequences were closely related to MERS-CoV sequences [97] [98] [99] . In a survey from Saudi Arabia, 823 fecal and rectal samples were tested by PCR for MERS-CoV, many coronaviruses sequences were detected [97] . Most of the detected sequences were unrelated to MERS-CoV, but one sequence of 190 nucleotide in the RNA-dependent RNA polymerase (RdRp) gene had a 100% identity with a MERS-CoV. This sequence was detected from feces of a Taphozous perforatus bat captured near the home of the index Saudi patient. Uncommon contact of humans with bats indicates that bats are not the intermediate host of MERS-CoV transmission but may be the reservoir of the virus [100] . Dromedary camels (Camelus dromedarus) appear to be the source of MERS-CoV. Other animals like sheep, goats, and cows tested negative to anti-MERS-CoV antibodies. Camel sera from Oman, Canary Islands, and Egypt were positive for anti-MERS-CoV antibodies in about 100%, 14%, and > 90% of the samples respectively [63, 101, 102] . Retrospective studies on archived human sera showed no evidence of infection with MERS-CoV before 2012 [103] , but anti-MERS-CoV antibodies were detected in archived camel sera in Saudi Arabia in 1993 [95] , and United Arab Emirates in 2003 [104] , indicating circulation of MERS-CoV in camels for many years. Bactrian camels in Mongolia tested negative for MERS-CoV antibodies [105] . Serologic studies from around the Middle East suggested that camels are one of the sources of MERS-CoV as > 90% of adult camels tested positive and had high titers of antibodies. Seropositivity was different in juvenile camels and was usually lower than in adults. These results suggested that MERS-CoV infections in camels occurred in young ages followed by frequent boosting [94, 95, 102, 104] . Camels in other parts of the world, far from the Middle East like in Europe, Australia, and the Americas do not have MERS-CoV antibodies and have no evidence of infection [106] . Table 3 summarizes camel serologic studies. In a study aimed to evaluate virus infectivity and shedding in camels, three adult dromedary camels were inoculated with MERS-CoV intratracheally, intranasally, and conjunctivally. Those camels shed large quantities of virus from the upper respiratory tract and infectious virus was detected in nasal secretions for 7 days post-inoculation and viral RNA for up to 35 days post-inoculation [113] . Human infections with MERS-CoV were linked to camels. The first evidence was a study in Saudi Arabia in which the MERS-CoV full genome sequences of isolates from a man with fatal infection and from one of his camels were identical. This patient had a direct contact with his deceased camels some days before the onset of symptoms. These results suggested that MERS-CoV can infect dromedary camels and can be transmitted from them to humans by direct close contact [86] . In other studies, phylogenetic analyses of camel and human isolates of the MERS-CoV genome demonstrated that the viruses were highly identical or in some cases were similar to each other [107, 108, 114] . Seroepidemiological studies shown low prevalence of MERS-CoV antibodies in humans in Saudi Arabia [103, 115] . A survey of 10 009 individuals representative of the general population of Saudi Arabia resulted in 15 seropositive subjects (0.15%), however, seropositivity increased 15-23 folds in camel-exposed individuals [78] . In a separate report, 7 of 87 camel shepherds and 140 slaughterhouse workers (3.1%) tested positive for MERS-CoV antibodies [103] . An overview of MERS-CoV transmission routes is illustrated in Fig.2 . The development of an effective vaccine is critical for prevention of a MERS-CoV pandemic. Some investigators have indicated that the RBD protein of MERS-CoV S protein is a good candidate antigen as a subunit vaccine. Various RBD fragments showed the highest DPP4 binding affinity and induced the highest-titer of IgG Ab and neutralizing Ab in mice and rabbits [17, 18, [116] [117] [118] [119] [120] . A robust neutralizing antibody response was elicited in BALB/c mice against MERS-CoV after immunization with purified full S protein nanoparticles produced in Sf9 cells infected with specific recombinant baculovirus containing the S gene [121] or a recombinant human adenoviral vectors (rAd5 or rAd41) containing the S or S1 genes [122, 123] . Vaccinia Ankara was encoded with full S protein and inoculated to BALB/c mice that developed high levels of neutralizing antibodies and had induction of humoral and cell-mediated immunity [124, 125] . Another study using Ad5-hDPP4-transduced BALB/c mice immunized with Venezuelan equine encephalitis virus replicon particles containing S protein elucidated a reduction of viral titers to nearly undetectable levels and increased neutralizing antibodies [35] . Recently, Wang et al. developed two candidate vaccines, a subunit (full S and S1 protein fraction) and a DNA vaccine (full S and S1 gene in a mammalian VRC8400 vector). The vaccine containing the full S DNA and S1 protein was the most efficacious in mice and rhesus macaques [126] . Using antibodies to deter MERS-CoV infection appears to have some promise. Transfer of sera containing anti-MERS-CoV-S protein to or seropositive camel sera to Ad5-hDPP4-transduced mice accelerated virus clearance, inhibited virus attachment, and reduced weight loss [35, 37, 127] . Recently, Corti et al. successfully isolated monoclonal antibodies from serum obtained from a MERS-CoV survivor after 200 days of infection [128] . Transduced Ad5-hDPP4 BALB/c mice were immunized with 15 mg/kg of the mAb and showed decreased lung [105] viral titers, no weight loss, and decreased peribronchial lymphoid infiltration [128] . No approved antivirals for use against MERS-CoV infection are yet available. The first approach performed when a new unknown virus like MERS-CoV emerges is testing drugs used as antiviral for similar viruses [29, 129, 130] . Type I interferons and ribavirine combination exhibited acceptable results in cell culture and rhesus macaques by decreasing the host inflammatory response, replication of virus, and improved clinical outcome [129, 131, 132] . A human cohort study in Saudi Arabia showed that treatment with combination of ribavirin and interferon-2b to 5 did not improve clinical outcomes but this may have been due to late treatment or due to the immunocompromised state of the patients [133] . In a retrospective study of 20 MERS-CoV infected patients treated with ribavirin and interferon -2a, results showed 14-day and 28-day survival was improved by 70% and 28% in the treated group as compared to an untreated group [134] . The second approach is screening of approved drugs with known safety profiles and transcriptional signatures in different cell lines. Several drugs, including antiparasitics, neurotransmitters, antibacterials, inhibitors of clathrinmediated endocytosis estrogen receptor, lipid or sterol metabolism, protein processing, and DNA synthesis or repair were tested on culture cells [119, [135] [136] [137] [138] [139] . Lopinavir-ritonavir combined with pegylated interferon and ribavirin therapy showed improved outcomes in infected marmosets [140] . The third approach involves in vitro inhibition of S protein to block virus entry into host cells using designed antiviral peptides targeting the HR2 domain of the S2 subunit of the MERS-CoV and preventing the interaction between the HR1 and HR2 domains required for the formation of the heterologous six-helix bundle in viral fusion core formation [22, 23] . Other drugs that act as inhibitors for viral proteases and helicase to suppress MERS-CoV infection were tested [141] [142] [143] [144] [145] . Other investigators studied inhibition of MERS-CoV infection by competitive inhibition of DPP4 cell receptor using compounds such as sitagliptin, vildagliptin, and saxagliptin [19, 146] . More than three years have passed since the first detection of MERS-CoV human infection and the virus, uncontrolled, continues to cause major outbreaks in the Middle East. The recent outbreak in Korea demonstrated that a single index case can lead to 185 more infections in a short period of time, hence raising questions about the accuracy of the number of cases being reported in the Middle East. Furthermore, the Korean outbreak confirmed the high fatality rate of MERS-CoV infection as being true rather than overestimated in case only the more severe cases are detected. In all, public health, veterinary health, and research efforts need to be consolidated in order to answer the following high priority questions: -What is the true extent of human infection with MERS-CoV? -What antivirals and vaccines are to be used in humans? -What infection control measures are needed in healthcare settings to prevent nosocomial outbreaks? -What measures need to be in place in order to prevent zoonotic infections from camels? -Is it possible to control the virus in the camel population and if so, how? -Are there other animal species involved in the MERS-CoV transmission cycle?',\n",
       "   '_search_score': 0.8114971999999998},\n",
       "  {'_id': 'a09090caa2498f6ff886f0fd7b10a7e45818751c',\n",
       "   'insert_date_': '2020-11-04T18:48:00.680441',\n",
       "   'abstract': 'Controversy remains over whether the coronavirus disease 2019 (COVID-19) virus may have infectivity during the incubation period before the onset of symptoms. The author had the opportunity to examine the infectivity of COVID-19 during the incubation period by conducting an epidemiological survey on a confirmed patient who had visited Jeju Island during the incubation period. The epidemiological findings support the claim that the COVID-19 virus does not have infectivity during the incubation period. This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (https://creativecommons.org/licenses/bync/4.0/) which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.',\n",
       "   'title': 'A Chinese Case of Coronavirus Disease 2019 (COVID-19) Did Not Show Infectivity During the Incubation Period: Based on an Epidemiological Survey',\n",
       "   'body': \"The World Health Organization declared the outbreak of novel coronavirus 2019 (2019-nCoV) on January 30, 2020 [1] . With the arrival of Chinese tourists and the domestic travel in conjunction with Chinese New Year's Day and Korean New Year's Day holidays, 24 cases were confirmed in Korea on February 8, 2020. As Jeju Island is a popular tourist destination, the risk of influx of coronavirus disease 2019 (COVID-19) and community spread is higher than any other region in Korea. Furthermore, there is still controversy over whether the virus is infectious during the incubation period before the onset of symptoms. Chinese authorities reported on January 26, 2020 that COVID-19 is infectious during the incubation period [2] . In addition, a news story from January 28, 2020 [3] and a research paper published on January 30, 2020 [4] reported that a con-pISSN 1975-8375 eISSN 2233-4521 firmed patient in Germany had contact with an asymptomatic woman from China two days before the onset of symptoms. There seem to be evidences supporting infectivity during incubation. If COVID-19 does have infectivity during the incubation period, it is a serious concern because it effectively neuters the current communicable disease control system. Because the starting time of infectivity in a confirmed case has not yet been defined. Fortunately, the author had the opportunity to observe the infectivity of COVID-19 during the incubation period through conducting an epidemiological survey on a confirmed patient who had visited Jeju Island during the incubation period. A 51-year-old Chinese woman, who lives in Wuhan, China, visited Jeju Island with her daughter through Yangzhou, China on January 21, 2020. They toured Jeju Island from the 22nd to the 24th and returned to Yangzhou on the 25th. The woman's fever started on the 26th, the day after returning home, and she was diagnosed on the 30th. The confirmation of infection was informally communicated to the Jeju Provincial Office on February 1, 2020 afternoon via fax from the airline on which she had traveled. Due to the confirmed diagnosis in China and the onset of symptoms after returning home, she was not subject to epidemiological investigations based on the guideline of the Centers for Disease Control and Prevention in Korea (KCDC). However, on the basis of the German case [3, 4] , Jeju Provincial Government decided to seek subjects who had face-to-face conversations with the patient on the 24th and 25th, two days before the day of symptom onset, and to take measures to quarantine up to 14 days of maximal incubation period. By using social networking service with the patient's daughter in China and checking the closed circuit television (CCTV) images installed in the hotel and throughout Jeju Island, the patient's travel route and persons with whom she came into close contact were confirmed [5] . Based on these, the author identified eleven persons who had talked directly with the patient for longer than 10 seconds and recommended quarantine to them. None of them showed any symptoms of COV-ID-19 infection through midnight, February 8, 2020, when the last contact was released from quarantine. During the quarantine period, it was suggested on February 3, 2020, that the Chinese woman who had contact with the first German patient had already developed symptoms and had been taking antipyretic analgesics during her stay in Germany [6] . This information was contrary to the claim that COV-ID-19 has infectivity during the incubation period. Additionally, according to the testimony of the doctor in Korea, most of the clinical progress is mild and the patient's condition is stable [7] . It is possible that if the onset of expression is not sudden and starts slowly, the manifestation date pointed to by the patient may be later than in actuality [8] . In light of this, it is possible to explain the confusing claims made by the Chinese authorities [2] . Accordingly, it would be reasonable to conduct an epidemiological investigation one day before the onset of confirmed patients in the 5th edition of the KCDC, which is effective beginning February 7, 2020 [9] . Conversely, if the symptoms are mild, the actual onset of the Chinese case may be earlier than January 26, 2020. However, the author concluded that her symptoms did not manifest during her stay in Jeju Island from the following facts. First, the third-parties such as hotel front desk staff, tour guides, and pharmacists, who had face-to-face contact with the patient, stated that she exhibited no ill symptoms such as dry coughing. Second, in the CCTV video, she walked for a long time and appeared to be a healthy tourist without any disease. These epidemiological findings support the claim that there is no infectivity during the incubation period. However, uncertainty about the possibilities of transmission of latent infection or asymptomatic patients remain, because the epidemiological spectrum of COVID-19 has yet to be accurately identified. This paper is a perspective so it did not need ethical consideration.\",\n",
       "   '_search_score': 0.7965499999999999},\n",
       "  {'_id': '00623268d0ce12f11d6b3acb0cc012821f592b93',\n",
       "   'insert_date_': '2020-11-04T18:31:40.943391',\n",
       "   'abstract': 'Coronavirus envelope (E) proteins are short ($100 residues) polypeptides that contain at least one transmembrane (TM) domain and a cluster of 2-3 juxtamembrane cysteines. These proteins are involved in viral morphogenesis and tropism, and their absence leads in some cases to aberrant virions, or to viral attenuation. In common to other viroporins, coronavirus envelope proteins increase membrane permeability to ions. Although an NMR-based model for the TM domain of the E protein in the severe acute respiratory syndrome virus (SARS-CoV E) has been reported, structural data and biophysical studies of full length E proteins are not available because efficient expression and purification methods for these proteins are lacking. Herein we have used a novel fusion protein consisting of a modified b-barrel to purify both wild type and cysteine-less mutants of two representatives of coronavirus E proteins: the shortest (76 residues), from SARS-CoV E, and one of the longest (109 residues), from the infectious bronchitis virus (IBV E). The fusion construct was subsequently cleaved with cyanogen bromide and all polypeptides were obtained with high purity. This is an approach that can be used in other difficult hydrophobic peptides.',\n",
       "   'title': 'Expression and purification of coronavirus envelope proteins using a modified b-barrel construct',\n",
       "   'body': \"Envelope (E) proteins are found in coronaviruses (family Coronaviridae, genus Coronavirus [1] ), a group of enveloped viruses that cause common cold in humans and a variety of lethal diseases in birds and mammals [2] . The species in the genus Coronavirus has been organized into three groups [3] . Group 1 includes the porcine Transmissible gastroenteritis virus (TGEV 1 ) and Human coronaviruses 229E (HCoV-229E) or NL63 (HCoV-NL63), Group 2 includes Murine hepatitis virus (MHV) and Human coronavirus OC43 (HCoV-OC43). Group 3 includes the avian Infectious bronchitis virus (IBV) and the Turkey coronavirus (TCoV). The virus responsible for the severe acute respiratory syndrome (SARS-CoV), a respiratory disease in humans [4] , is close to group 2 [5] . Coronavirus E proteins are minor components in virions, with varying lenghts of around $100 amino acids. All of them have at least one predicted a-helical transmembrane (TM) domain followed by a juxtamembrane cluster of 2-3 cysteines. The TM domain is flanked by a short hydrophilic N-terminus ($10 residues) and a C-terminal tail (39-76 residues) . E proteins are not essential for in vitro or in vivo coronavirus replication, but their absence results in attenuated viruses, as shown for SARS-CoV [6] and others [7] [8] [9] [10] [11] [12] . In some coronaviruses, absence of E protein results in deletereous morphological changes in the virion [13] [14] [15] [16] . E proteins has also been shown to have membrane permeabilizing [17] and ion channel [18] [19] [20] activity, although the purpose of this is not known. Biophysical or structural studies so far have been performed using synthetic peptides corresponding to the predicted a-helical TM domain [17, 21, 22] or with synthetic full length protein [23, 24] . In the latter case, however, the peptides were only partially purified and contained truncations or deletions. Thus, despite the evident biological interest of coronavirus E proteins, to our knowledge, none of the envelope coronavirus proteins has been efficiently expressed and purified. Our initial attempts to overexpress SARS-CoV E, using poly-Histidine, GST, ketosteroid isomerase (KSI) [25, 26] or maltose binding protein (MBP) fusion tags were not successful. While the first three resulted in very low expression levels, MBP could not be efficiently purified from SARS-CoV E (results not shown). Thus, we used a novel construct in which the E protein is Nterminally fused to an integral membrane protein, a modified eight strand b-barrel protein derived from the Escherichia coli outer membrane protein A (OmpA) [27] . This b-barrel, which has shortened extramembrane loops, has been previously characterized by NMR [28] , and is referred herein as b-barrel protein platform (BBP). A 6His-tag was added to the N-terminus of BBP to further facilitate purification of the fusion construct. BBP was linked to the N-terminus of the E protein with a flexible linker region which contained the restriction site for factor Xa. Although we obtained high expression levels (20 mg/L), proteolysis with factor Xa was not effective. Thus, the construct 6His-BBP-E was cleaved chemically with cyanogen bromide (CNBr), which cleaves polypeptides at methionine residues [29] [30] [31] . CNBr therefore cannot be used when internal methionines are present in the target protein, which fortunately is not the case for neither SARS-CoV E or IBV E. However, the b-barrel used as fusion protein contains four internal Met residues, and cleavage of the fusion construct with CNBr would produce in this case many fragments that could complicate purification. To avoid this, all four internal BBP Met residues were mutated to Ala (mutant BBP DMet ). To demonstrate that this method is efficient, we have expressed and purified two representatives of coronavirus E proteins, the shortest, SARS-CoV E (76 aa, 8.3 kDa), and the longest, IBV E (108 aa, 12.3 kDa). These two proteins, like other coronavirus envelope proteins, contain several (2-3) juxtamembrane cysteines of as yet unknown function that are conserved. It is known, however, that they can be palmitoylated in vivo [21, 32, 33] , and that removal of these cysteines in MHV E resulted in deformed viruses [34, 35] . The study of the role of these cysteines is therefore an objective in their own right. In addition, these cysteines could potentially produce inter or intra-molecular disulfide bonds, leading to sample heterogeneity. Thus, the potential negative effect of disulfide bond formation in the final samples was examined by also purifying these proteins without cysteines. The latter samples may also be more amenable for structural studies. Both envelope proteins and their cysteine-less mutants could be purified by reverse phase high performance liquid chromatography (RP-HPLC). This method should facilitate future detailed biophysical and structural studies of coronavirus envelope proteins. An additional benefit of this fusion protein is that BBP was originally engineered by inserting additional 12 amino acids in a loop region of the barrel, corresponding to loop 3 of calmodulin from X. laevis. This modified loop binds lanthanides [36, 37] , and the use of BBP can enable measurement of residual dipolar couplings (RDC) in membrane proteins [28] . Constructs pET3a/ 6H BBP DM expression vector was modified from the plasmid pET3a-BBP described in a previous work [28] . The mutated BBP (BBP DM ) was generated with its four internal methionine residues replaced with alanine using the QuickChange  SDM kit (Stratagene) with the relevant primers. Additionally, a hexa-His tag was introduced at the N-terminus of BBP to facilitate purification. The forward and reverse primers used to amplify the target gene SARS-CoV E and IBV-E are as follows: SARS-CoV E (NCBI Reference Sequence: NC_004718.3, protein ID NP_828854.1): forward primer 5 0 CAGGATCCATCGAAGGTCGTAT GTACTCATTCG3 0 and reverse primer 5 0 GAGGATCCTCAGACCAGA AGATCAGGAAC3 0 . IBV E (NCBI Reference Sequence: NC_004718.3, protein ID AAU14251): forward primer 5 0 CAGGATCCATCGAAGGTCGTATGT TTAATTTATTCC3 0 and reverse primer 5 0 GAGGATCCTTAGATATCA TCCACCTCTAATAG 3 0 . Both forward and reverse primers contain BamHI restriction sites. Factor Xa cleavage sequence was introduced at the N-terminus of the genes. The amplified inserts were digested with BamHI and then ligated to the similarly digested pET3a/ 6H BBP DM expression vector. The identity of the constructs was confirmed by DNA sequencing. Mutants of SARS-CoV E and IBV-E constructs at cysteine residues were generated using the QuickChange  SDM kit (Stratagene). In SARS-CoV E, three cysteine residues at position 40, 43 and 44 (represented by the first, second and third letters, respectively) were mutated to serine or alanine, generating triple Cys mutants (SSS and AAA). In IBV E, the two cysteine residues at positions 44 and 45 were mutated to Alanine, generating a doublecysteine mutant (AA). The plasmid containing the desired construct was transformed into the E. coli strain BL21-CodonPlus (DE3)-RIL (Stratagene). One colony was inoculated into LB media (3 ml) containing 50 lg/ml of ampicillin and 34 lg/ml of chloramphenicol and incubated overnight at 37C with shaking at 200-225 rpm. The culture was inoculated into 500 ml LB media in a 2.5 L flask containing the same concentrations of antibiotics. Protein expression was induced with 1 mM isopropyl b-d-thiogalactopyranoside (IPTG) at an OD 600 of 0.5-0.6. Expression of IBV AA mutant was much lower than the wild type, and best expression was obtained by using 0.4 mM IPTG. Cells were harvested by centrifugation when cell growth reached its stationary phase. Expression was checked by running the whole-cell lysate on SDS-PAGE. The fusion protein was found in inclusion bodies. For a 0.5 L culture, the cell pellet obtained after centrifugation was first washed with 20 mM Tris-HCl and 5 mM EDTA (TE buffer, pH 8) and then resuspended in 20 ml of 20 mM Tris-HCl and 5 mM EDTA. The resuspended cells was lysed by sonicating for 10-15 min in ice, followed by microfluidization 5-10 times and centrifugation at 8,000 rpm at 4C for 20 min. The white pellet was collected and resuspended in 20 ml TE buffer (pH 8) containing 2% Triton X-100, shaken at 37C for 1 h, and centrifuged at 8000 rpm at 4C for 30 min. The pellet was resuspended in TE buffer, incubated at 37C for 1 h and centrifuged at 8000 rpm at 4C for 20 min. The pellet was then resuspended completely in the binding buffer (8 M urea, 5 mM imidazole in 20 mM Tris pH 8.0) and centrifuged at 12,000 rpm at 4C for 30 min. The protein was allowed to bind overnight onto Ni-NTA resin (Bio-rad) pre-equilibrated with the binding buffer. After incubating, the resin was washed with 5 column volumes of washing buffer (8 M urea, 20 mM imidazole in 20 mM Tris pH 8.0) followed by elution (elution buffer: 8 M urea, 250 mM imidazole in 20 mM Tris pH 8.0). All fractions were analyzed by SDS-PAGE. The desired fraction was dialysed to remove urea, and the precipitate was collected and lyophilized for CNBr cleavage. The same protocol was followed for the expression of the 15 N labeled SARS-CoV E, although with some modifications. Initially, the cells were grown in normal LB medium and after the OD 600 reached 0.6, the cells were harvested by centrifugation and resuspended in minimal media. After two cycles of resuspension and centrifugation, cells were inoculated in M9 minimal medium enriched with 15 N labeled ammonium chloride (1 g/L, >99% 15 N) (Cambridge Isotopes, USA). In addition, the medium was supplied with 1% basic mineral elements solution (Sigma chemicals) which includes trace elements and vitamins. MALDI-TOF/MS (Applied Biosystems) was used for identification of the cleavage products. The protein was solubilized in 60% isopropanol, 35% acetonitrile, 5% water and 0.1% TFA and deposited onto a matrix of 2-(4-hydroxyphenylazo) benzoic acid (HABA). The lyophilized powder of purified construct was solubilized in 80% formic acid (to a few ml). In a different flask, five-fold molar excess of CNBr versus methionine content of the construct was also dissolved in 80% formic acid. After mixing both solutions the reaction proceeded in the dark under anaerobic conditions [38] . After 12-15 h, the solution was diluted and dialysed against water. The precipitate was lyophilized and solubilized by addition of 10 lL of trifluoroacetic acid (TFA). Acetonitrile was added to reach a 99.9:0.1 (acetonitrile to TFA) volume ratio. The solubilized CNBr cleavage products were injected into a Jupiter 5 l C4 300  analytical column with a linear gradient of water (Solvent A: 95% H 2 O, 5% acetonitrile, 0.1% TFA), and 2-propanol (Solvent B: 80% 2-propanol, 20% acetonitrile, 0.1% TFA). The presence and purity of SARS-CoV E or IBV E in the HPLC peaks was assessed by MALDI-TOF/MS using a HABA matrix in a 1:1 (v/v) ratio with HPLC fractions. Fractions with peaks matching the molecular weight of the protein were concentrated using a Scanvac centrifuge (Chemoscience) to remove excess organic solvents, diluted with H 2 O and lyophilized. Lyophilized SARS-CoV E or IBV E powder was solubilized in 20-30 ll of 1,1,1,3,3,3-hexafluoro-2-propanol (HFIP), and the concentration of the protein was estimated using its absorbance at 280 nm. SDS solubilizing buffer was added to the lyophilized protein to a final concentration of 1 mg/mL, with or without DTT. After mixing for 5 min, the sample was heated at 65C for 5 min and loaded. The loading volume was 10 ll (10 lg of peptide). Four percent to 12% NuPAGE Bis-tris gel (Invitrogen) was run at 150 V for 1 h with 2-(N-morpholino)-ethanesulfonic acid (MES)-SDS running buffer. After completion, the SDS-PAGE gel was stained with Coomassie blue. For PFO electrophoresis, peptides were dried from HFIP and dissolved in sample buffer containing 4% PFO to $1 mg/ml. Samples were heated at 65C for 4 min and loaded on a 4-12% NuPAGE -Bis-tris gel (Invitrogen). The gel was electrophoresed at 80 V for 2-3 h with 2-(N-morpholino)-ethanesulfonic acid (MES) PFO running buffer. After completion, the gel was washed twice with water, followed by Coomassie blue staining. Protein identity was confirmed by tryptic digestion after running SDS-PAGE. The protein band corresponding to SARS-CoV E or IBV E protein was excised and subjected to in-gel trypsin digestion. The gel fragments were diced into small fragments (1 mm 2 ) and washed three times with 200 ll 50% acetonitrile and 25 mM ammonium bicarbonate to remove the Coomassie blue dye. Gel fragments were then soaked in 100% acetonitrile for 10 min. Using the gel loading tip, acetonitrile was removed and residual acetonitrile was removed by Speedvac. Prior to enzymatic digestion, gel fragments were reduced with 10 mM DTT in 25 mM ammonium bicarbonate at 60C for 30 min. Alkylation was then performed with 55 mM iodoacetamide in 25 mM ammonium bicarbonate for 30 min at room temperature in the dark. The reduced and alkylated gel slices were rehydrated in 20 ll of 50 mM ammonium bicarbonate, pH 8.3, containing 200 ng of trypsin. Once this solution was fully absorbed by the gel slices, enzyme-free ammonium bicarbonate buffer was added until the gel pieces were completely covered. The samples were digested for 18 h at 37C [39] . After in gel tryptic digestion, the peptides were extracted and analyzed using MALDI-TOF/TOF-MS as described previously [40] . CD spectra were measured on a Chirascan CD Spectrometer (Applied Photophysics, UK). Data were collected at 20C at 1 nm interval, from 190 nm to 240 nm, on a 1 mm pathlength cell. SARS-CoV E and detergents were first dissolved in HFIP and dried, followed by addition of 1 mM phosphate buffer at pH 7. Final detergent concentrations were: 13 mM SDS and 10 mM DPC. The final protein concentration was approximately 10 lM. Spectra were smoothed with a Salvitsky-Golay algorithm. Infrared spectra were acquired as described previously on a Nicolet Nexus spectrometer (Madison, USA) [20] . The protein was incorporated in multilamellar liposomes by first dissolving a dry mixture of lipid and lyophilized peptide (50:1 M ratio) in HFIP. The solution was dried by a stream of N 2 , and the resulting peptide-lipid film was dissolved in phosphate buffer (1 mM, pH 7) followed by sonication. The compositions of lipid mixtures used were as follows: DMPC alone, DMPC:DMPG (1:1 M ratio) and POPC:SM:CH (2:1:1, w/w/w). The liposome solution was deposited on a germanium internal reflection element (IRE) and dried slowly by evaporation. NMR experiments were performed using a Bruker Avance-II 600 or 700 NMR spectrometers, equipped with cryogenic probes (Bruker BioSpin, GmbH), with three channels and triple resonance probes with shielded z-gradient coils. The carrier position was set to 4.70 ppm for 1 H and 119 ppm for 15 N. Sodium 2,2-dimethyl-2silapentane-5-sulfonate (DSS) was used as internal reference for 1 H nuclei. The chemical shifts of 15 N nuclei were indirectly calculated from the 1 H chemical shifts. SARS-CoV E protein was screened for solubilization in detergents DPC and SDS in phosphate buffer at pH 5.5 at various protein to detergent molar ratios, from 1:100 to 1:150. DTT was also used to reduce possible disulphide bonds. Sedimentation equilibrium experiments were performed at 25C using a Beckman XL-I analytical ultracentrifuge [41] in the presence or absence of 6 mM of reducing agent tris(2-carboxyethyl)phosphine (TCEP). Absorbance was measured at 230 nm, and for SARS-CoV E we used the estimated extinction coefficient e 230 = 27,000 M 1 cm 1 . The buffer composition was 50 mM Tris, 100 mM NaCl, pH 7.3 and 5 mM C14 betaine (c.m.c., 0.1 mM). To match the density of the detergent, D 2 O was added to the buffer to a final volume ratio of 29.4% [42] . The density-matched buffer was added to the dry peptide. The molar ratio peptide:detergent was 1:25, 1:50 and 1:100 for inner, medium and outer cell, respectively. The samples were centrifuged in three-compartment carbon-epoxy centerpieces with quartz windows for lengths of time sufficient to achieve equilibrium, tested with WinMatch. This typically was achieved $26 h after reaching the speeds at 38,000, 45,000 or 48,000 rpm. Data was processed and analyzed using Sedfit and Sedphat [43] . The monomeric molecular mass of the protein and its partial specific volume were calculated with SEDNTERP [44] after correction for partial hydrogen/deuterium exchange [41] : 8,229.84 Da and 0.7687 cm 3 /g. The log plot (ln A vs. r 2  r 2 0 ) was not linear, indicating the presence of more than one species at these peptide-to-detergent ratios. The data was fitted to several monomer-Nmer models. For IBV E, the conditions were identical to that of SARS-CoV E except that the concentration of detergent (C14 betaine) was 15 mM. The protein:detergent molar ratio was 1:300, 1:450 and 1:600. The samples were centrifuged at 28,000, 34,500, 42,000 or 48,000 rpm.  For the preparation of SARS-CoV E proteoliposomes, the protein was reconstituted into liposomes formed by DPhPC and cholesterol (10:1 M ratio, Avanti Polar Lipids), at a 1:100 protein/lipid (mol/ mol) ratio. Giant unilamellar vesicles were prepared by electroformation on a Nanion Vesicle Prep Pro setup (Nanion Technologies GmbH, Munich, Germany). Electrophysiological recordings were carried out with a Port-a-Patch automated patch clamp system (Nanion Technologies GmbH, Munich, Germany). First, a planar bilayer was obtained by applying 5 ll of GUV solution onto the borosilicate glass chips, $1 lm aperture, in presence of a slightly negative pressure (20 mbar). The formation of the planar lipid bilayer on the aperture provided a seal resistance of >10 GO. Then, proteoliposomes containing the protein were pipetted onto the planar lipid bilayer. Currents were amplified with Axon Patch 200B amplifier (Axon instruments, USA). The data were acquired by using Axon Digidata 1322A and analyzed with Clampfit (Axon instruments, USA). The experiment was performed in a symmetric solution of 150 mM NaCl, 5 mM HEPES at pH 5.1, varying the voltages across the lipid bilayer. To determine the optimal temperature for expression, yield was tested at 37, 30, 20 and 18C. For both SARS-CoV E (Fig. 1A) and IBV E (Fig. 1B) , highest expression was observed at 18C. Elution of the fusion construct containing SARS-CoV E ( Fig. 2A and B) , from a Ni-NTA column with 250 mM imidazole produced a relatively pure sample (Fig. 2C, lanes E1 and E2) . The yield of fusion construct after this step was approximately 20 mg/L of culture. After CNBr cleavage, the HPLC elution profile showed two distinct peaks labeled 1 and 2, respectively (Fig. 3A, insert) , eluting at 38% and 72% of solvent B. MALDI-TOF/MS analysis of peak 1 produced a mass of 18,415.1 Da corresponding to BBP DM (not shown), and peak 2 produced a main peak at 8,295.1 Da (Fig. 3A) , which corresponds to the monomeric SARS-CoV E (calculated mass 8,230 Da). Additional peaks at multiples of the monomeric form were observed, e.g., 16 ,623 Da, likely to represent dimers of SARS-CoV E. The 15 N labeled SARS-CoV E protein was observed at 8,436 Da (see Supplementary File, Fig. S1B ), whereas the calculated molecular weight was 8,316 Da. Like in unlabeled protein, multiples of this peak at 17,016, 25,532 and 34,901 Da were observed, corresponding to higher oligomers. We note that in both labeled and unlabeled protein, not only the observed molecular weight was higher than the calculated one, but the value obtained fluctuated between measurements. For example, in a total of five measurements, the average was 8,339  23 Da for unlabeled SARS-CoV E protein. These alliquotes originated from the same volume, and the machine was properly calibrated. In any case, we have further confirmed the identity of the purified protein by in-gel tryptic digestion and identified the fragments by MALDI-TOF (see Supplementary File, Fig. S2) . The final yield of non-labeled SARS-CoV E protein was 4 mg/L of culture, which is comparable or higher than other methods of expression for this class of proteins [45] , whereas the yield for 15 N-labeled protein was $15 mg/L. For cysteine-less mutants of SARS-CoV E, expression and purification results were similar (not shown), although in this case MALDI-TOF/MS mass spectra produced no multimers ( Fig. 4A and B) . It is thus likely that the presence of multimers in MALDI mass spectra is related to the presence of inter-molecular disulfide bonds. The fusion construct formed by 6His-BBP DM and IBV E ( Fig. 2A  and B) , also accumulated in inclusion bodies. The construct was eluted from a Ni-NTA column with 250 mM imidazole (Fig. 2D , lane E1) and the yield after this step was similar to that of SARS-CoV E. The HPLC chromatogram of this sample after CNBr cleavage (Fig. 3B) shows two main peaks. Peak 1 corresponds to BBP DM (not shown), whereas the MALDI-TOF/MS spectrum corresponding to peak 2 produced a peak at 12,196.4 Da corresponding to IBV E (calculated 12,116 Da). In this case, a peak at 6,075.6 Da, resulting from double protonation, was observed. The cysteine less mutant also could be purified (Fig. 4C) . The identity of IBV E was confirmed by tryptic digestion (see Supplementary File, Fig. S3 ). The fraction corresponding to peak 2 in the HPLC chromatograms (Fig. 3, inserts) and the equivalent peaks for other similar chromatograms was analyzed by SDS electrophoresis. Both wild type SARS-CoV E and IBV E contain cysteines, therefore the gels were run with or without reductant (DTT). The monomer of wild type SARS-CoV E migrated with its expected mobility in 4-12% Nu-PAGE ($8 kDa) together with several higher oligomers, and mutants SSS and AAA produced only monomers (Fig. 5A ). In the presence of DTT (Fig. 5B) , wild type SARS-CoV E produced only two bands consistent with monomers and trimers, whereas the mutants were unaffected. The difference between triple mutants and wild type in presence of DTT suggests that cysteine side chains are involved in SARS-CoV E inter-monomeric contacts, but this involvement does not necessarily imply the formation of disulfide bonds. The side chain of cysteine is more hydrophobic [46] than alanine or serine, and the latter residues may lead to different secondary structure propensities relative to cysteine [23, 24] . Wild type IBV E produced only one main band in SDS (Fig. 5C) , consistent with monomers, and only very faint bands corresponding to higher oligomers. Similar bands were observed for the double cysteine mutant AA, with or without DTT, which indicates that IBV E as prepared did not contain significant amount of intermonomeric disulfide bonds. To confirm that the proteins purified are able to form pentameric oligomers, we used perfluorooctanoic acid (PFO) electrophoresis. PFO is a detergent known to facilitate native interactions between transmembrane a-helices [47, 48] . We have shown previously [19] that the synthetic transmembrane domain of SARS-CoV E (SARS-CoV E-TM) forms pentamers in this detergent. However, neither full length wild type SARS-CoV E or the SSS mutant showed any band consistent with pentamers (Fig. 5D) , and produced instead aggregation well above 100 kDa. The AAA mutant, in contrast, showed a clear band consistent with pentamers, $40 kDa (Fig. 6D) . The difference in behavior of this mutant may be due to a change in secondary structure, possibly increased helicity, induced by substitution of Cys to Ala [50] . The presence of DTT did not have any effect on mobility in PFO gels (not shown). In the case of IBV E (Fig. 5E) , both for wild type and AA mutant, a large smear was observed around 66 kDa, which is consistent with a predominantly pentameric form. To further confirm the presence of a pentameric arrangement in these coronavirus envelope proteins, we used a milder, zwitterionic detergent, C14 betaine, in a sedimentation equilibrium (SE) experiment. For wild type full length SARS-CoV E no simple reversible association 'monomer:N-mer' model could be fitted to the data (N was tested from 2 to 15), probably because of the presence of disulfide bonds. However, in the presence of reductant TCEP, the data could be best fitted to a simple monomer-pentamer self-asso-ciation model (Fig. 6A-D) , with log K a = 17.5  0.2, similar to that obtained previously in similar conditions [19] for synthetic peptide TM in similar conditions. This suggests that the main contribution to pentamerization in SARS-CoV E is provided by the TM domain. The fact that we could fit the data in presence of TCEP to a monomer pentamer model demonstrates that disulfide bonds are not required to form pentamers. When we used the cysteine-less mutants (SSS or AAA) or IBV E and mutant AA, the data could still be fitted to a reversible monomer-pentamer equilibrium (not shown). For IBV E a good fit of the data could be obtained without the need of TCEP, consistent with the observation that no disulfide bonds were observed in SDS (Fig. 5C ). Purified SARS-CoV E has channel activity in lipid bilayers, although the film was unstable and recordings could only be obtained at two potentials ( Fig. 6E and F). After confirming that SARS-CoV E is pentameric and functinal for channel activity, we tested with independent techniques if it can be suitable for structural studies and can be properly incorporated in membranes. Thus we obtained CD and FTIR spectra to assess the secondary structure of SARS-CoV E. The CD spectrum of SARS-CoV E protein incorporated in anionic SDS or lipid-like DPC detergent micelles showed characteristic minima at 208 nm and 222 nm ( Fig. 7A) , typical of a-helical conformation. This is confirmed by the infrared amide I band envelope (Fig. 7B ) obtained after reconstitution of SARS-CoV E protein in lipidic membranes, which shows a predominant band centered at $1655 cm 1 , also characteristic of a-helices, although in some cases (e.g., DMPC) a small shoulder was observed consistent with some b-structure. Finally, 2D-HSQC NMR spectra were collected in DPC detergent (Fig. 7C ) or a mixture of DPC/SDS (Fig. 7D) . Although the spectra are not yet of sufficient quality for complete structure determination, the best spectra were obtained using 0.5 mM SARS-CoV E, 100 mM DPC, 100 mM SDS and 40 mM DTT, at a temperature of 328 K (Fig. 7D) . From visual inspection, about 70% of the 1 H-15 N peaks can be detected. Using the triple cysteine mutant did not improve the quality of the spectra (not shown). Nevertheless, the NMR spectrum, combined with CD and FTIR spectra, show that the protein is folded and after further optimization to improve signal intensity and peak dispersion, structural determination should be possible. We have also shown that these proteins form pentameric oligomers, are functional and are likely be properly folded when incorporated in detergent micelles or in lipid bilayers. We stress the fact that (1) this is the first report that shows unequivocally that coronavirus envelope proteins (full length) are pentameric, and (2) that the mass spectroscopy determination of these proteins is by no means routine. Indeed, the fluctuation between measurements observed for SARS-CoV E was not observed previously in TM fragments of SARS-CoV E [18, 19] , in proteolytic fragments after tryptic digestion (Supplementary File), or in similar viroporins, e.g., small hydrophobic (SH) protein from RSV [51] , which do not form large aggregates in SDS and PFO gels. Therefore, we attribute this variability to the high aggregation tendencies of these proteins. Overall, these results show that the two coronavirus E proteins could be expressed and purified using a novel fusion protein consisting of a modified transmembrane b-barrel. This is an approach that can be used to study the structural variations and functional characteristics of this so far unexplored family of membrane proteins. This approach can also be used in other small membrane proteins, as long as they do not contain internal methionines.\",\n",
       "   '_search_score': 0.7556745999999999},\n",
       "  {'_id': '008cf2148ee4383534191b3736096878cad82c67',\n",
       "   'insert_date_': '2020-11-04T18:33:08.595720',\n",
       "   'abstract': 'Virus vaccines have to be immunogenic, sufficiently stable, safe, and suitable to induce long-lasting immunity. To meet these requirements, vaccine studies need to provide a comprehensive understanding of (i) the protective roles of antiviral B and T-cell-mediated immune responses, (ii) the complexity and plasticity of major viral antigens, and (iii) virus molecular biology and pathogenesis. There are many types of vaccines including subunit vaccines, whole-inactivated virus, vectored, and live-attenuated virus vaccines, each of which featuring specific advantages and limitations. While nonliving virus vaccines have clear advantages in being safe and stable, they may cause side effects and be less efficacious compared to live-attenuated virus vaccines. In most cases, the latter induce longlasting immunity but they may require special safety measures to prevent reversion to highly virulent viruses following vaccination. The chapter summarizes the recent progress in the development of coronavirus (CoV) vaccines, focusing on two zoonotic CoVs, the severe acute respiratory syndrome CoV (SARS-CoV), and the Middle East respiratory syndrome CoV, both of which cause deadly disease and epidemics in humans. The development of attenuated virus vaccines to combat infections caused by highly pathogenic CoVs was largely based on the identification and characterization of viral virulence proteins that, for example, interfere with the innate and adaptive immune response or are involved in interactions with specific cell types, such as macrophages, dendritic and epithelial cells, and T lymphocytes, thereby modulating antiviral host responses and viral pathogenesis and potentially resulting in deleterious side effects following vaccination.',\n",
       "   'title': 'Molecular Basis of Coronavirus Virulence and Vaccine Development',\n",
       "   'body': 'There are four \"common\" human coronaviruses (CoVs) that are endemic in the human population: HCoV-229E, HCoV-OC43, HCoV-NL63, and HCoV-HKU1. The first two CoVs have been known since the 1960s, while the emergence of severe acute respiratory syndrome CoV (SARS-CoV) in 2002 led to an active search for novel CoVs and the identification of HCoV-NL63 and HCoV-HKU1 in 2004 , respectively (van der Hoek et al., 2004 Woo et al., 2005a) . The common human CoVs are generally associated with relatively mild clinical symptoms and cause a self-limiting upper respiratory tract disease (common cold) (Walsh et al., 2013) . In some cases, common CoVs may also be associated with more severe pathogenesis in the lower respiratory tract, such as bronchiolitis or pneumonia (Pene et al., 2003; van der Hoek et al., 2005; Woo et al., 2005b) . Human CoVs cause more serious disease in young, elderly, or immunocompromised individuals, and they may lead to exacerbation of preexisting conditions, such as asthma or chronic obstructive pulmonary disease, frequently requiring hospitalization (Mayer et al., 2016; Varkey and Varkey, 2008) . Considering that the prevalence of these viruses ranges between 3% and 16% and that around 70% of the population is infected during childhood, with recurrent infections occurring throughout life Zhou et al., 2013b) , common human CoVs represent a significant burden to public health. More recently, two previously unknown animal CoVs emerged that were shown to cause deadly disease in humans. The first, SARS-CoV, emerged in Southern China and spread around the globe in late 2002 and early 2003, infecting at least 8000 people and killing nearly 10% of the infected individuals (Lee et al., 2003) . The second, Middle East respiratory syndrome CoV (MERS-CoV), was first reported in 2012 in the Middle East and there are still ongoing reports of sporadic cases, particularly in Saudi Arabia and the United Arab Emirates. This virus has caused close to 1782 laboratory-confirmed cases, resulting in 634 deaths, and is the cause of an important outbreak in Korea that started in May 2015, leading to more than 186 confirmed cases with a death toll of 36, according to the World Health Organization (WHO) (http://www.who.int/emergencies/ mers-cov/en/). Current treatment strategies for SARS and MERS, and discussion of the discovery and development of new virus-based and host-based therapeutic options for CoV infection have been reviewed recently (Zumla et al., 2016) . In this chapter, we will mainly focus on the development of vaccines suitable to prevent infections caused by highly pathogenic CoVs, particularly SARS-CoV and MERS-CoV, in humans. To date, a wide range of vaccine candidates have been developed for these viruses, including subunit, wholeinactivated virus, DNA, and vectored vaccines (see reviews by Du and Jiang, 2015; Enjuanes et al., 2008; Zhang et al., 2014a) . However, in many cases, these vaccines were found to induce antibody-dependent enhancement of infectivity (ADEI) and eosinophilia. In contrast, live-attenuated vaccines have a long history of success and are the most frequently used vaccines in humans. This chapter will be focusing on a recently developed next generation of live-attenuated vaccines based on recombinant viruses. Attenuation of viruses generally relies on the previous identification of genes involved in viral virulence in specific hosts. Often, these genes encode proteins that antagonize the innate immune response, and their deletion leads to recombinant viruses that are attenuated in their virulence and, therefore, may be developed into candidate vaccines. The timeline from bench research to approved vaccine use is generally 10 years or more (Papaneri et al., 2015) . The use of genetically engineered viruses may significantly reduce both the time and costs required for vaccine development and production. In this chapter, specific features of promising vaccine candidates in meeting the earlier criteria will be reviewed. CoVs contain the largest genome known among RNA viruses, consisting of a single-stranded positive-sense RNA molecule of around 30 kb in length ( Fig. 1) (de Groot et al., 2012) . It is similar to cellular mRNAs, as it contains 5 0 -capped and 3 0 polyadenylated ends. The 5 0 -terminal two-thirds of the genome contain two overlapping open reading frames (ORFs): ORF1a and ORF1b (Fig. 1) . Translation of ORF1a yields polyprotein 1a (pp1a), and -1 ribosomal frameshifting allows translation of ORF1b to yield pp1ab (Ziebuhr, 2005) . Together, these polyproteins are co-and posttranslationally processed into 16 nonstructural proteins (nsps), most of them being involved in viral genome replication and subgenomic mRNA synthesis. The 3 0 -third of the genome encodes a series of structural proteins in the order 5 0 -S-E-M-N-3 0 and genus-specific proteins that vary in number among the different CoV members (Fig. 1) (de Groot et al., 2012; Sola et al., 2015) . Human HCoV-229E and HCoV-NL63 belong to the genus Alphacoronavirus, while HCoV-OC43, HCoV-HKU1, SARS-CoV, and MERS-CoV belong to genus Betacoronavirus. Although the overall genomic organization is conserved among all coronavirus genera, the members of this virus family encode a unique set of genus-specific proteins ( Fig. 1) that, in general, are involved in the modulation of pathogenesis. Neutralizing monoclonal antibodies (mAbs) represent a promising therapeutic strategy against emerging CoV infections. Fully human neutralizing antibodies may be developed using different technologies and applied as therapeutic or prophylactic agents (Jiang et al., 2014; Tang et al., 2014; ter Meulen et al., 2004; Ying et al., 2014; Zhang et al., 2005; Zhu et al., 2007) . Of special interest are the potent human cross-reactive neutralizing antibodies specific for SARS-CoV (Pascal et al., 2015) . Since most of these antibodies target the receptor-binding domain of the spike (S) glycoprotein, antigenic variability within the S gene among human and animal strains must be considered in the design of broad-spectrum neutralizing molecules (Ying et al., 2015b) . A combination of mAbs targeting different epitopes may be used to prevent the emergence of escape mutants and, potentially, generate a synergistic neutralizing effect (Ying et al., 2015a) . In line with this, passive immunotherapy with mouse or dromedary immune serum was shown to be protective in animal models against SARS-CoV (Subbarao et al., 2004) and MERS-CoV (Zhao et al., 2015b) , respectively, suggesting that this approach may be used prophylactically or therapeutically in infected patients. Neutralizing antibodies induced by the S glycoprotein provide complete protection from lethal CoV infections (Sui et al., 2005) . Moreover, an inverse correlation was observed between IgA secretion and MERS-CoV infectivity in patients, suggesting that virus-specific IgA production may be a suitable tool to evaluate the potency of candidate vaccines against MERS-CoV . However, the IgA response is short lived in patients. In contrast, virus-specific memory CD8 + T cells persisted for up to 6 years after SARS-CoV infection, at which time memory B cells and virus-specific antibodies were undetectable (Yang et al., 2006) . It has been shown that memory CD8 + T cells specific for an immunodominant epitope substantially protected aged mice from lethal SARS-CoV infection (Channappanavar et al., 2014) . After challenge, memory CD8 + T cells produced effector cytokines (interferon gamma, IFN-; tumor necrosis factor alpha, TNF-; and interleukin 2, IL-2) and cytolytic molecules, reducing viral loads in the lung. However, dysregulation of some of these inflammatory mediators, including type I IFN and inflammatory monocyte-macrophage responses, caused lethal pneumonia in SARS-CoV-infected mice and, therefore, should be considered during vaccine design to minimize immunopathology . In addition to the protective effect mediated by memory CD8 + T cells, SARS-CoV-specific CD4 + T cell and antibody responses are likely necessary for complete protection. The requirement of T-cell responses in MERS-CoV protection was also suggested by immunization experiments in macaques (Muthumani et al., 2015) . Therefore, for effective protection, CoV vaccines should elicit not only antibody responses but also specific memory CD4 + and CD8 + T cells. Information on the complexity of CoV serotypes is crucial for predictions on whether antibodies against a previous CoV infection or a specific vaccine may protect from reinfection, which has important implications for vaccine design and neutralizing antibody therapy. Antigenic variability in the S protein, the major target of neutralizing antibodies, is extremely low between different MERS-CoV strains (Drosten et al., 2015) . A recent serological study using infectious MERS-CoV isolates collected from patients in Saudi Arabia in 2014 showed no significant differences in serum neutralization, indicating that all these isolates belong to the same serotype . Based on these data, it seems likely that the S genes of all currently circulating MERS-CoVs are interchangeable in candidate vaccine formulations. The potential relevance of neutralizing antibodies directed against other envelope proteins remains to be studied. MERS-CoV and all the closely related viruses isolated from camels and bats belong to the same viral species, with bat viruses being at the root of the phylogenetic tree. Most likely, the virus circulating in camels was acquired from bats and represents the origin of viruses identified in humans over the past few years. Recombination events within the spike gene of viral ancestors were likely involved in the emergence of MERS-CoV . Therefore, camels may serve as reservoirs for the maintenance and diversification of the MERS-CoVs responsible for human infection (Sabir et al., 2016) . The resultant genetic variability may lead to additional antigenic diversity, with obvious consequences for vaccine design. Because SARS-CoV differs immunologically from other betacoronaviruses with little cross-reactivity of antiviral antibodies (Hou et al., 2010) , the development of vaccine candidates suitable to provide broad protection should address the diversity of the main immunogenic determinants of the S protein (Zhou et al., 2013a) . For example, SARS-CoV-specific domain in the S protein was found to contain an epitope (80R) that critically determines the sensitivity of a given virus to neutralizing antibodies specific for this epitope. Variants of this epitope have been found in SARS-like-CoVs from civet cats and in human SARS-CoVs that evolved during the epidemics. While the majority of SARS-CoVs from the first outbreak were sensitive to the 80R-specific antibody, the GD03 strain isolated from the index patient of the second outbreak was resistant, confirming the importance of the S protein\\'s natural antigenic variability in eliciting neutralizing antibody responses (Sui et al., 2005) . The SARS-CoV S glycoprotein is a major target of protective immunity in vivo. Two specific human mAbs recognizing the S protein exhibited potent cross-reactivity against isolates from the two SARS outbreaks and palm civets, but not bat strains (Zhu et al., 2007) . A combination of two neutralizing mAbs could prevent the emergence of neutralization escape mutants or at least attenuate viral virulence in vivo. However, although neutralizing mAbs directed against epitopes located at the interface between the viral S protein and its cellular receptor, angiotensin-converting enzyme 2 (ACE2), proved to have great potency and breadth in neutralizing multiple viral strains (Sui et al., 2014) , both the single and combined use of one or two mAbs, respectively, failed to prevent the emergence of antibody escape variants. Therefore, the use of one or two neutralizing mAbs that target a structurally flexible SARS-CoV epitope may be of limited value for in vivo immunotherapies and should be combined with neutralizing mAbs that bind a second conserved epitope with low structural plasticity. Suitable animal models that reproduce the pathology caused by human CoVs are required for studies of pathogenesis and vaccine testing. Unfortunately, no appropriate animal models have been developed to date for any of the four common human CoVs. A transgenic (Tg) mouse model expressing human aminopeptidase N was generated for HCoV-229E but was not suitable for pathogenesis studies as it was based on immunodeficient Stat1 -/mice (Lassnig et al., 2005) . A mouse model has been extensively used for studies of HCoV-OC43, as this human CoV causes lethal infections in mice (Jacomy and Talbot, 2003) . However, following inoculation of mice with respiratory isolates of HCoV-OC43, the virus was found to adapt rapidly to grow in brain tissue, while viral RNA remained nearly undetectable in the lung (Butler et al., 2006; St-Jean et al., 2004) , suggesting that the model does not reproduce the respiratory pathology seen in humans, limiting its value for studies of virus-induced pathology. The use of animal models for SARS-CoV and MERS-CoV for studying pathology in humans has been recently reviewed, including clinical symptoms, viral replication, and pathology in humans, nonhuman primates (NHPs), rabbits, ferrets, marmosets, hamsters, and mice (Gretebeck and Subbarao, 2015; van Doremalen and Munster, 2015) . Additional models for MERS-CoV based on dromedary camels and other animal species have also been reported (Falzarano et al., 2014; Haagmans et al., 2016; van Doremalen and Munster, 2015) . Their large size and the lack of clear clinical signs of disease make camels a less suitable model for studying MERS-CoV pathology. Marmosets show clinical signs following infection with MERS-CoV virus but, in this case, research animals and appropriate reagents suitable to characterize the immune response are scarce or not available, limiting the use of this model system (van Doremalen and Munster, 2015) . Hamsters cannot be naturally infected by MERS-CoV, largely preventing their use as an animal model. MERS-CoV S protein-mediated binding to its receptor, human dipeptidyl peptidase-4 (DPP4), involves interactions with 14 amino acid residues. Appropriate replacements of five residues that differ between hamster and human DPP4 render the hamster DPP4 a functional receptor for MERS-CoV (van Doremalen et al., 2014) . Thus far, Tg hamsters have not been used due to the lack of specific gene targeting tools. With the availability of the CRISPR-Cas9 system, the situation may now change and hamsters susceptible to MERS-CoV might be developed but their suitability as animal models of MERS-CoV-induced disease remains unclear at present. MERS-CoV is able to infect rabbits but does not cause histopathology or clinical symptoms although the virus can be detected in lungs. The virus is shed from the upper respiratory tract, providing a possible route of MERS-CoV transmission in this animal species (Haagmans et al., 2015) . Clearly, the large size of rabbits also limits their use in BSL-3 containment laboratories. Mice are an ideal model for pathogenesis studies of many viruses because of their small size and the availability of suitable genomic and immunological reagents. A key difference between SARS-CoV and MERS-CoV is that SARS-CoV infects several strains of mice, whereas MERS-CoV does not (Coleman et al., 2014b; Gretebeck and Subbarao, 2015) . A standard procedure was the adaptation of SARS-CoV and MERS-CoV to grow in mice and reproduce the disease caused in humans. This strategy was directly applied in the case of SARS-CoV using conventional mouse strains without the need for Tg mice expressing the human ACE2 receptor (Day et al., 2009; Frieman et al., 2012; Roberts et al., 2007) . Mouse-adapted SARS-CoV obtained by passing the virus 15 times in mice (SARS-CoV-MA15) has been an excellent model as it reproduces very well the pathology caused by SARS-CoV in humans, including mortality (DeDiego et al., 2011 (DeDiego et al., , 2014b . In contrast, in the case of MERS-CoV, the virus was grown in Tg or knockin humanized mice susceptible to MERS-CoV (K. Li, P. McCray, and S. Perlman, 2016, personal communication) . The first type of MERS-CoV-susceptible mice includes Tg mice that express hDPP4, the virus receptor, using promoters such as those from surfactant protein C, cytokeratin 18 , or cytomegalovirus . In these mice, the LD50 has been estimated to be <10 TCID 50 of MERS-CoV. Although MERS-CoV grows almost equally well in the lungs and the brain in these Tg mice, these animals proved to be very useful for protection studies Zhao et al., 2015a) . In some Tg mouse strains, in which the hDPP4 was expressed under control of the surfactant protein C promoter, the virus was found to grow primarily in the lung, which might extend their use to viral pathogenesis studies (C. Tseng, 2016, personal communication) . The other type of MERS-CoV-susceptible mice available to date includes knockin strains in which 3 or 13 exons of mouse DPP4 have been replaced with the homologous sequences from hDPP4 Li et al., 2016; Pascal et al., 2015) . A major difference to the Tg mice described earlier is that the mouse-adapted MERS-CoV only produced disease in knockin mice after the virus has been passed 30 times in the knockin mouse (K. Li, P. McCray, and S. Perlman, 2016, personal communication) , whereas the LD50 of MERS-CoV in these mice was around 10 4 pfu/ mouse. Another mouse lineage was generated using CRISPR-Cas9 technology by altering mDPP4 amino acid residues 288 and 330 that are known to interact with the MERS-CoV S protein, leading to mice that closely reproduced the disease observed in humans, including mortality. In this case, the LD50 was around 10 6 pfu/mouse (A. Crockrell and R. Baric, 2016, personal communication) , significantly higher than that observed in the knockin mice (Cockrell et al., 2014; Peck et al., 2015; van Doremalen and Munster, 2015) and (K. Li, P. McCray, and S. Perlman, 2016, personal communication) . Interestingly, all knockin mice and those generated with the CRISPR technology supported MERS-CoV replication in the lungs, making them extremely useful models for pathogenesis studies. An alternative approach for the rapid generation of a mouse model for MERS-CoV has been the transduction of mice with adenoviral vectors expressing the human host-cell receptor DPP4 . These mice developed a pneumonia characterized by extensive inflammatory cell infiltration, with virus clearance occurring 6-8 days after infection. Using these mice the efficacy of a therapeutic intervention (poly I:C) and a potential vaccine based on Venezuelan equine encephalitis (VEE) virus has been demonstrated ). An important advantage of this approach is that it may be rapidly adapted to other viruses that may emerge in the future, especially in cases in which a suitable mouse model is not available. Vaccines based on recombinant MERS-CoV S protein, in particular its RBD, have demonstrated partial efficacy in protecting immunized macaques from MERS-CoV infection, reducing pneumonia, and viral titers . Several fragments of the MERS-CoV S protein were found to induce MERS-CoV neutralizing antibody responses in mice and rabbits (Du et al., 2013; Jiang et al., 2014; Ma et al., 2014b; Mou et al., 2013) , similar to what has been shown previously for SARS-CoV (Du et al., 2008; Wang et al., 2012) . The fragment-containing residues 377-588 of MERS-CoV proved to be sufficient to protect Ad5/hDPP4transduced and hDPP4-Tg mice against MERS-CoV. The immunogenicity of this fragment was further improved, resulting in strong humoral and cellular immune responses, by linking the fragment to human Fc and using an adjuvant (Tang et al., 2015; Zhang et al., 2016) . These reports confirm that the MERS-CoV S protein is very well suited for the development of MERS subunit vaccines. The full-length S protein contains several nonneutralizing immunodominant domains that may compromise the immunogenicity of major neutralizing domains or induce harmful immune responses as demonstrated for the S protein of SARS-CoV (Weingartl et al., 2004) . Intranasal vaccination with the RBD domain of the MERS-CoV S protein induced much stronger local mucosal immune responses in the lung than subcutaneous immunization (Ma et al., 2014a) . Other studies showed that full-length monomeric or trimeric recombinant SARS-CoV and MERS-CoV S proteins are able to induce protective responses in mice (Honda-Okubo et al., 2015; Li et al., 2013) and the presentation of MERS-CoV full-length S protein as nanoparticles in combination with appropriate adjuvants elicits neutralizing antibodies in immunized mice (Coleman et al., 2014a) . Alternatively, DNA vaccines expressing full-length S protein or smaller fragments are effective against MERS-CoV infection. An optimized DNA vaccine encoding the full-length S protein of MERS-CoV elicited antigenspecific neutralizing antibodies in mice, camels, and rhesus macaques, with six of eight vaccinated macaques showing no radiographic evidence of infiltration after MERS-CoV challenge (Muthumani et al., 2015) . Potent antigen-specific cellular immune responses were induced in the immunized macaques, suggesting that T-cell responses may also play a role in MERS-CoV protection. Two companies actively working to develop MERS-CoV DNA vaccines (Inovio Pharmaceuticals Inc., Philadelphia, USA and Gen-eOne Life Science, Seoul, Korea) will soon perform a Phase I clinical trial for this DNA-based vaccine (Inovio News Release, http://ir.inovio.com/ news/news-releases/news-releases-details/2015/Inovio-Pharmaceuticals-Partners-with-GeneOne-Life-Science-for-MERS-Immunotherapy-Clinical-Development/default.aspx). In addition to the DNA-only strategy, DNA priming and protein boosting could be an alternative vaccination approach for MERS-CoV. It was shown that priming of mice and NHPs with DNA encoding the full-length S gene and boosting with the S1 subunit protein-induced robust neutralizing antibody responses against several MERS-CoV strains and protected NHPs from MERS-CoV challenge . Chimeric virus-like particles (VLPs) containing SARS-CoV S protein and influenza matrix protein 1 protected mice against challenge with SARS-CoV (Liu et al., 2011) and may induce strong immune responses due to its polymeric nature. To increase their efficacy, subunit vaccines or VLPs have to be administered together with adjuvants (Bolles et al., 2011; Tseng et al., 2012) . Subunit vaccines only include subviral components that do not represent the full antigenic complexity of the virus, resulting in limited protective efficacy due to limited Th1-mediated immune responses and, in some cases, unbalanced immune responses that may lead to immunopathology. Many vaccines based on chemically inactivated SARS-CoV virions have been evaluated in animal models such as hamsters, mice, ferrets, and NHPs (Bolles et al., 2011; Iwata-Yoshikawa et al., 2014; Roberts et al., 2010; Tseng et al., 2012) . In all cases, production of neutralizing antibodies with different levels of protection was observed. Challenge of mice with SARS-CoV was reported to cause Th2-type immunopathology, suggesting that SARS-CoV components induced hypersensitivity. Further studies suggested that the immunopathology leading to eosinophilia was linked, at least in part, to the viral nucleoprotein (Bolles et al., 2011) . Also, immunization with oligomers of the SARS-CoV S protein were shown to promote eosinophilia following viral challenge in different animal model systems (Tseng et al., 2012) , suggesting that potential side effects of vaccines based on inactivated SARS-CoV or MERS-CoV should be carefully evaluated prior to use in humans. Moreover, inactivated whole-virus vaccines raise biosafety concerns due to the risk of vaccine preparations containing infectious virus. To minimize this risk, genetically attenuated viruses may be used as the starting point for the production of killed virus inactivation. Vectored vaccines are generally based on vectors with a proven safety record. These vaccines allow production and release of immunogenic antigens from infected cells for a limited period of time. Vectors based on viruses from different families (poxvirus, adenovirus, measles, and togavirus (VEE)) have been used in the development of vaccines for CoVs. In the case of MERS-CoV, the most advanced and promising candidate is the modified vaccinia virus Ankara (MVA), a viral vector that does not replicate in mammalian cells and, therefore, holds great promise as a vaccine platform (Altenburg et al., 2014; Haagmans et al., 2016; Song et al., 2013; Volz et al., 2015) . Using this vector, S protein fragments of different length were expressed: full-length, extracellular S1 domain, or the RBD. In all cases, neutralizing antibodies and T-cell responses for MERS-CoV were induced. One of these vectors induced mucosal immunity and reduced the shedding of MERS-CoV by a factor of one thousand after challenge with the virulent virus in dromedary camels, thus preventing spread from the animal reservoir (Haagmans et al., 2016 ). An MVA-vectored vaccine will be entering clinical trials in 2016 supported by the German Center for Infection Research (DZIF) (Paddock, 2015) . A second, more advanced vectored vaccine is based on recombinant adenovirus expressing S protein fragments of different size Kim et al., 2014; Shim et al., 2012) . Sublingual immunization with a recombinant adenovirus encoding SARS-CoV S protein induced systemic and mucosal immunity in a mouse model system (Shim et al., 2012) . Systemic and mucosal immunity were also elicited in mice by single immunization with human adenovirus type 5 or 41 vector-based vaccines carrying the S protein of MERS-CoV . Whether or not, these immune responses confer protection against viral infection remains to be evaluated. Immunization with measles virus vectors expressing the SARS-CoV S protein induced neutralizing antibodies and strong Th1-biased responses, a hallmark of live-attenuated viruses and a highly desirable feature for an antiviral vaccine (Escriou et al., 2014) , though eradication of measles virus may represent an obstacle for the application of this type of vaccines. VEE replicon particles expressing the SARS-CoV S protein provided protection against lethal homologous and heterologous challenge in an aged mice model (Sheahan et al., 2011) . Therefore, in principle, several well-known vectors offer the possibility of protection against CoV infection. Although these vectors are based on live viruses with a reasonable record of safety, they are limited to presenting one or a reduced number of CoV antigens to the immune system, in contrast to live vaccines based on the whole, attenuated CoV. We consider it likely that the main strategy for producing more efficacious vaccines in the near future will be based on live-attenuated vaccines that lack specific virulence markers. The most rapid laboratory responses involved in vaccine generation is usually achieved by using subunit vaccines, whole-inactivated virus, or by the construction of vectored vaccines. Successful protection is frequently associated with the generation of a strong neutralizing immune response. Nevertheless, control of virus emergence with this type of vaccines frequently fails due to the relatively low titers and short-lived duration of neutralizing antibody responses and, at times, due to induction of immune pathogenesis including eosinophilia reactions. In contrast, live-attenuated vaccines have a long history of success and are generally more immunogenic than nonreplicating vaccines, because they produce a comprehensive spectrum of native viral antigens over a prolonged time span, presenting antigens to the immune system as in natural infections. For virus attenuation, we favor modern approaches that may overcome reversion to virulence, as described later. An alternative approach to the design of attenuated viruses as vaccine candidates that several laboratories have followed is the identification of virulence-associated viral genes (DeDiego et al., 2014a; Stobart and Moore, 2015; Totura and Baric, 2012) that, in many cases, encode nonessential immunomodulatory proteins. Modification or deletion of these genes leads to the generation of attenuated viruses that may be useful as vaccine candidates. Our laboratory has used this strategy extensively to generate CoV vaccines. Also, other groups have used this approach successfully to produce vaccine candidates for influenza, respiratory syncytial virus, measles, dengue, and mumps (Kirkpatrick et al., 2016; Stobart and Moore, 2015) . Engineering attenuated CoVs as vaccine candidates requires the availability of reverse-genetics systems suitable to introduce deletions of (or mutations in) virulence genes as well as appropriate animal models for the evaluation of efficacy and safety of these candidate vaccines. To date, no vaccines have been developed for common human CoV infections. Nevertheless, infectious cDNA clones have been engineered for the three human CoVs that can be efficiently propagated in tissue culture: HCoV-229E, HCoV-OC43, and HCoV-NL63 (Donaldson et al., 2008; St-Jean et al., 2006; Thiel et al., 2001) . Similarly, infectious cDNA clones have been generated for SARS-CoV (Almazan et al., 2000; Yount et al., 2003) and MERS-CoV (Almazan et al., 2013; Scobey et al., 2013) , providing an excellent basis for the rational development of live-attenuated vaccines based on recombinant viruses. Virulence of CoVs is generally associated with specific virulence genes that, in most cases, antagonize cellular innate immune responses but are nor required for efficient virus replication. CoV infection affects many host-cell pathways that modulate pathogenesis. Innate immune response is the first line of host-cell defense against viruses, and any viral factor modulating this pathway may have a strong impact on pathogenesis. CoV proteins nsp3, 4, and 6 are actively involved in the formation of host membrane vesicles and structures that affect CoV replication and evasion of the immune system by hiding the double-stranded RNA (dsRNA) generated during virus replication (Angelini et al., 2013) . Pathogen-associated molecular patterns (PAMPs), such as ssRNA, dsRNA, or viral proteins, trigger the activation of transcription factors leading to proinflammatory cytokines and type I IFN induction. PAMPs activate protein kinase RNA-activated, which leads to translation initiation factor eIF2 phosphorylation and inhibition of host translation, and 2 0 -5 0 oligoadenylate synthase (OAS), which triggers RNase L and RNA degradation (Fig. 2) . The activation of cytoplasmic sensors RIG-I and MDA-5 triggers the upregulation of IFN regulatory factors (IRF)-3 and IRF-7, and nuclear factor kappa-light-chain-enhancer of activated B cells (NF-kB) through mitochondrial antiviral-signaling protein (Fig. 2 ). In addition, Toll-like receptors (TLRs) activate the MyD88 response protein and adaptor molecule TRIF-dependent pathways, which also upregulate transcription factors IRF-3, IRF-7, NF-kB, and activator protein 1 (AP-1) (Fig. 2) (DeDiego et al., 2014a) . Some CoV proteins act as IFN antagonists by inhibiting its production or signaling (Fig. 2 ) (DeDiego et al., 2014b; Totura and Baric, 2012; Zhong et al., 2012) . Potent cytokines involved in controlling viral infections and priming adaptive immune responses are generated in response to CoV infection (Totura and Baric, 2012; Zhou et al., 2015) . However, uncontrolled induction of these proinflammatory cytokines can also increase pathogenesis and disease severity as described for SARS-CoV and MERS-CoV (DeDiego et al., 2011, 2014a,b; Selinger et al., 2014) . The cellular pathways mediated by IRF-3, IRF-7, activating transcription factor (ATF)-2/jun, AP-1, NF-kB, and nuclear factor of activated T cells, are the main drivers of the inflammatory response triggered after SARS-CoV infection, with the NF-kB pathway most strongly activated (DeDiego et al., 2011) . CoV proteins that elicit this type of innate immune response may be modified to generate attenuated viruses suitable for vaccine development. Human CoVs are able to suppress IFN induction to different extents and, in general, are sensitive to the addition of exogenous IFN (Kindler et al., 2013) . SARS-CoV in particular is highly resistant to the antiviral state induced by IFN treatment, suggesting that it possesses many mechanisms to counteract IFN-induced antiviral responses (Zielecki et al., 2013) . The betacoronavirus HCoV-OC43 is an exception for the sensitivity of human CoVs to IFN, as infection of certain cell types by this virus is favored in the presence of exogenous IFN. A possible explanation for this observation may be that HCoV-OC43 uses IFN-inducible transmembrane (IFITM) proteins 2 and 3 for cell entry (Zhao et al., 2014b) . Interestingly, IFITM3-mediated antiviral activity was observed for the alphacoronaviruses HCoV-229E and HCoV-NL63, but not for the betacoronaviruses SARS-CoV and MERS-CoV (Wrensch et al., 2014) , suggesting that IFITM proteins may have a positive effect on all betacoronavirus infections. A number of nsps have been implicated in human CoV pathogenesis, such as nsp1, nsp3, and nsp16, and, possibly, might be used to produce highly attenuated strains of common human CoVs as vaccine candidates. Nsp1 protein is only encoded by members of the Alphacoronavirus and Betacoronavirus genera, with similar functions in all cases despite low-sequence conservation. The nsp1 proteins of HCoV-229E and HCoV-NL63 inhibit protein expression, most likely via their association with 40S ribosomal subunits, and they also act as IFN antagonists Zust et al., 2007) . The nsp3 transmembrane replicase protein contains several functional domains (Fig. 3) that are conserved among all common human CoVs: two papain-like proteases (PLPs), two ubiquitin-like domains, and an ADP-ribose-1 00 -phosphatase (ADRP) domain. A role for ADRP in pathogenesis of human CoVs has been proposed, although it is not required for Fig. 3 Multidomain structure of CoV nonstructural protein nsp3. The approximate boundaries of each domain in SARS-CoV protein are indicated underneath by the amino acid numbers in the replicase polyprotein. Ac, Glu-rich acidic domain; ADPR, ADP-ribose-1 00 -phosphatase (also called macrodomain or X domain); NAB, nucleic acid-binding domain; SUD, SARS-unique domain; TM1-TM4, transmembrane domains forming an additional domain containing a metal-binding region (ZF); UBL, ubiquitin-like domain; Y1-Y3, Y domains preceding the C-terminal PLP cleavage sequence at nsp3/4. viral growth in cell culture. HCoV-229E and SARS-CoV mutants lacking ADRP activity grow to similar titers compared to wild-type viruses in MRC-5 and Vero cells, respectively (Kuri et al., 2011; Putics et al., 2005) . ADRP-deficient mutants of mouse hepatitis virus (MHV) were attenuated in vivo, most probably due to deficient induction of proinflammatory responses (Eriksson et al., 2008) . ADRP mutants of HCoV-229E and SARS-CoV were reported to be more sensitive to IFN compared to wild-type viruses, suggesting that ADRP activity is involved in counteracting IFN activity (Kuri et al., 2011) . The PLP domains of nsp3 have also been linked to the modulation of innate immune response. The HCoV-NL63 PLP2 domain has been extensively characterized and acts as an IFN antagonist (Clementz et al., 2010) , and apart from its PLP activity, it has deubiquitinating (DUB) and deISGylating activity (Chen et al., 2007; Clementz et al., 2010) . HCoV-NL63 PLP2 DUB activity seems to be involved in modulation of NF-kB activation (Clementz et al., 2010) and p53-mediated modulation of IFN production (Takaoka et al., 2003) . PLP2 deubiquitinates the cellular protein MDM2, leading to p53 proteasomal degradation and, as a consequence, decreased IFN production (Yuan et al., 2015) . Nsp16 has ribose 2 0 -O-methyltransferase (2 0 -O-MTase) activity that is required to produce the 5 0 cap1 structures on viral RNAs. The methylation proved to be important to avoid viral RNA to be recognized as being \"nonself\" by the host-cell sensor MDA5 and subsequent activation of cellular innate immune responses. HCoV-229E mutants that lack 2 0 -O-MTase activity replicated to 10 2 -fold lower titers than the wild-type virus in MRC-5 cells, confirming the relevance of nsp16 activity. Moreover, the 2 0 -O-MTase mutant was more sensitive to treatment with type I IFN, and macrophages infected with the mutant virus produced more type I IFN than those infected with the parental virus . The role of nsp16 in modulation of innate immunity seems to be conserved among all CoVs, as similar results were obtained for MHV and SARS-CoV, and a SARS-CoV mutant lacking 2 0 -O-MTase activity was attenuated in vivo . CoVs contain a variable number of genus-specific genes that have been implicated in modulation of pathogenesis. In fact, all common human CoVs contain a genus-specific gene located between the S and envelope (E) genes, named ORF4 (HCoV-229E and HCoV-HKU1), ORF3 (HCoV-NL63), or ns12.9 (HCoV-OC43) (Fig. 1) . These genes encode transmembrane proteins that differ significantly in both their sequences and predicted structures (Muller et al., 2010; Zhang et al., 2014b Zhang et al., , 2015 , but may have related functions, such as ion channel activity and morphogenesis (Donaldson et al., 2008; Zhang et al., 2014b Zhang et al., , 2015 . In fact, the SARS-CoV 3a protein, HCoV-NL63 protein 3, and the HCoV-229E 4a protein are each able to complement in trans the absence of HCoV-OC43 ns12.9 protein . The proteins are not essential for viral growth in cell culture but virus yield is increased between 6-and 25-fold if the protein is expressed . Also, there is evidence that these protein homologs may play an important role in vivo. For example, all clinical isolates of HCoV-229E contain a full-length ORF4, while in cell culture-adapted strains, this gene is spontaneously divided into two ORFs, 4a and 4b, or replaced by a truncated ORF encoding only the first transmembrane domain . Recombinant HCoV-NL63-3 virus grows to a 10-fold lower titer than the parental virus in primary human airway epithelium (HAE) culture (Donaldson et al., 2008) . Finally, a recombinant HCoV-OC43-ns12.9 virus was attenuated in vivo . These results suggest that ORF3 genes can be modified for the generation of attenuated viruses that may be used as vaccine candidates. The HCoV-OC43 ns2 gene is predicted to encode a 2 0 -5 0 -phosphodiesterase (PDE) based on its homology to the MHV ns2 gene (Roth-Cross et al., 2009) . The MHV ns2 protein-mediated PDE activity interferes with the OAS-RNase L pathway, affecting pathogenesis by counteracting cellular innate immune responses (Zhao et al., 2012) . Moreover, the absence of ns2 PDE activity can be complemented by other viral or cellular PDEs . Common human betacoronaviruses encode a hemagglutinin-esterase (HE) gene (Fig. 1) . The HE protein is a transmembrane protein that is incorporated in the viral envelope, acting as a receptor-destroying enzyme with a role in viral entry (Huang et al., 2015) . Interestingly, HCoV-OC43 mutants that lack the HE gene or encode an inactive HE protein produced replication-competent propagation-defective viruses (Desforges et al., 2013) . These viruses, in principle, may be useful as vaccine candidates. SARS-CoV encodes many known immunomodulatory proteins, such as nsp1, nsp3, nsp14, membrane (M), E, and nucleocapsid (N) (Fig. 2 ) (DeDiego et al., 2011 (DeDiego et al., , 2014a Totura and Baric, 2012; Zhong et al., 2012) . Modification or deletion of some of them may lead to attenuated viruses that could be used as live vaccine candidates. Many of these proteins also specifically inhibit cellular signaling pathways associated with the innate immune response. It is generally thought that deletion or genetic modification of SARS-CoV genes encoding proteins that have IFN antagonist activity or other viral defense mechanisms, such as nsp1, nsp3, nsp16, M, 3b, 6, and N, will result in attenuated viruses, as has been shown for SARS-CoV nsp1 and nsp16 mutants, respectively . As an alternative approach to produce attenuated CoVs to be used as vaccine candidates, the modification of viral replication fidelity has been proposed (Graham et al., 2012; Smith et al., 2013) . Live-attenuated RNA virus vaccines are efficacious but subject to reversion to virulence. Among RNA viruses, replication fidelity is recognized as a key determinant of virulence and escape from antiviral therapy, though reduced fidelity is detrimental for some viruses. The replication fidelity of CoVs has been estimated to be approximately 20-fold higher than that of other RNA viruses and is mediated by a 3 0 ! 5 0 exonuclease (ExoN) activity (Minskaia et al., 2006) that probably functions in RNA proofreading (Eckerle et al., 2010) . It has been demonstrated that engineered abrogation of SARS-CoV ExoN activity results in a stable mutator phenotype with profoundly decreased fidelity both in cell culture and in vivo, and attenuation of pathogenesis in young, aged, and immunocompromised mice (Graham et al., 2012) . The ExoN null genotype and mutator phenotype are stable and do not revert to virulence, even after serial passage or long-term persistent infection in vivo. ExoN inactivation thus has potential for broad applications in the stable attenuation of CoVs. Our laboratory has focused on the role of virulence of the SARS-CoV E and 3a proteins, and of the MERS-CoV E and 5 proteins, and on studying effects of their partial or complete deletion on virus attenuation and possible implications for vaccine development (DeDiego et al., 2007 (DeDiego et al., , 2011 Regla-Nava et al., 2015) . The safety of these vaccine candidates has been further enhanced by genetic modifications of the nsp1-coding sequence . The CoV E protein is 76-109 aa in length, depending on the virus, and has diverse functions in CoV morphogenesis and the secretory pathway (Westerbeck and Machamer, 2015) . The importance of E in different CoV species and genera varies greatly, ranging from being absolutely essential in some members of the genera Alphacoronavirus (TGEV) and Betacoronavirus (lineage C, MERS-CoV) to having nonessential functions in other betacoronaviruses (MHV (lineage A), SARS-CoV (lineage B)). In SARS-CoV, two strategies have been used to abrogate E function: complete deletion of E protein, and the introduction of small deletions of 8-12 aa at the E protein carboxyl-terminus . A similar strategy is currently being applied to MERS-CoV (J. Gutierrez, I. Sola, and L. Enjuanes, unpublished results) . The SARS-CoV E protein is nonessential for virus replication and dissemination, and its deletion has led to attenuated forms of SARS-CoV that are promising vaccine candidates (see later). Viral infections often induce endoplasmic reticulum (ER) stress, specifically the unfolded protein response (UPR). UPR induces three main signaling pathways to avoid the accumulation of proteins with altered folding in the ER. Stress in the ER is also interconnected with the innate immune response and other host-cell responses (Hetz, 2012) . Both HCoV-OC43 and HCoV-HKU1 induce ER stress by a mechanism that is associated with S protein expression leading to virulent CoVs in a mouse model (Favreau et al., 2009; Siu et al., 2014a) . Similarly, in the case of SARS-CoV, the E protein has been linked to virulence and the induction of UPR (DeDiego et al., 2011) . E protein has a major role in inflammasome activation and the associated exacerbated inflammation elicited by SARS-CoV in the lung parenchyma . This exacerbated inflammation causes edema leading to acute respiratory distress syndrome and, frequently, to the death of experimentally infected animals or human patients. The E protein is a viroporin that conducts Ca 2+ ions. Changes in intracellular Ca 2+ concentration, mediated by E protein ion channel activity, are responsible for inflammasome activation . Elimination of ion channel activity by the introduction of aa substitutions within the transmembrane domain of E protein led to virus attenuation . At its carboxyl-terminus, the E protein contains another virulence factor, a PDZ domain-binding motif (PBM) that, during SARS-CoV infection, could potentially target more than 400 cellular proteins containing PDZ domains with possible effects on viral pathogenicity. Interestingly, deletion or modification of the E protein PBM resulted in attenuated SARS-CoVs that are good vaccine candidates Regla-Nava et al., 2015) . Deletion of full-length E protein led to the attenuation of SARS-CoV in several animal models (Lamirande et al., 2008; Netland et al., 2010) . The immunogenicity and protective efficacy of a live-attenuated vaccine based on a recombinant SARS-CoV lacking the E gene (rSARS-CoV- E) was first studied using hamsters. After immunization with rSARS-CoV-E, hamsters developed high serum neutralizing antibody titers and were protected from challenge with homologous (SARS-CoV Urbani) and heterologous (GD03) SARS-CoV in the upper and lower respiratory tract. Deletion of the E protein modestly diminished viral growth in cell culture but abrogated virulence in mice (Netland et al., 2010) . We have shown that immunization with rSARS-CoV-E almost completely protected BALB/c mice from fatal respiratory disease caused by mouse-adapted SARS-CoV, and partly protected hACE2 Tg mice from lethal disease, although hACE2 Tg mice, which express the human SARS-CoV receptor, were extremely susceptible to infection. Furthermore, we also showed that rSARS-CoV-E induces antiviral T-cell and antibody responses. To improve vaccine efficacy, we engineered and adapted rSARS-CoV to efficiently grow in mice (rSARS-CoV-MA15). To this end, we incorporated six nucleotide substitutions into the SARS-CoV genome (Frieman et al., 2012; Roberts et al., 2007) using our reverse-genetics system based on bacterial artificial chromosomes. Using the rSARS-CoV-MA as a backbone genome, a second set of E-deletion vaccine candidates was generated (Fett et al., 2013) . rSARS-CoV-MA15-E was safe, causing no disease in 6-week-, 12-month-, or 18-month-old BALB/c mice. Immunization with this virus completely protected mice at three ages from lethal disease and induced a more rapid virus clearance. Compared to rSARS-CoV- E, rSARS-CoV-MA-E immunization elicited significantly greater neutralizing antibody titers and virus-specific CD4 + and CD8 + T-cell responses. After challenge, inflammatory cell infiltration, edema, and cell destruction were decreased in the lungs of rSARS-CoV-MA15-E-immunized mice, compared to those from rSARS-CoV-E-immunized 12-month-old mice, suggesting that this mouse-adapted virus is a safe candidate vaccine. To identify E protein domains that contribute to SARS-CoV-MA15- E attenuation, several rSARS-CoVs with mutations or deletions in the E protein were generated. Substitutions in the amino terminus or deletion of regions in the internal and carboxy-terminal regions of the E protein led to virus attenuation . Attenuated viruses induced minimal lung injury, diminished edema, limited neutrophil influx, and increased CD4 + and CD8 + T-cell counts in the lungs of BALB/c mice, compared to animals infected with the wild-type virus. The attenuated viruses completely protected mice against challenge with the lethal parental virus, considerably increasing the survival of infected animals and indicating that these viruses are promising vaccine candidates. Ideally, a vaccine should provide full protection, and this was the case for the SARS-CoV vaccine candidate that was engineered by deleting the fulllength E protein. Nevertheless, a good vaccine must also be genetically stable, and the initial rSARS-CoV-E vaccine was unstable both in cell culture and in vivo. In fact, this mutant virus regained fitness after serial passage in cell culture, resulting in the partial duplication of the membrane gene, and while the chimeric protein increased viral fitness in vitro, the virus remained attenuated in mice. When the full-length E gene was deleted or its PBM coding sequence mutated, revertant viruses either evolved a novel chimeric gene including PBM or restored the sequence of the PBM in the E protein, respectively . During SARS-CoV-E passage in mice, the virus incorporated a mutant variant of the 8a protein including a PBM, resulting in reversion to a virulent phenotype. These data, and additional evidence, led us to conclude that the virus requires a PBM on a transmembrane protein to compensate the removal of this motif from the E protein . Therefore, to increase the genetic stability of the vaccine candidate, we introduced small attenuating deletions in the E gene that did not affect the endogenous PBM, preventing the selection of revertants incorporating novel PBMs into the virus genome and leading to a genetically and functionally stable vaccine candidate. To further increase vaccine safety, we introduced additional attenuating amino acid substitutions in the nsp1 protein. Recombinant viruses including attenuating mutations in the E and nsp1-coding sequences maintained their attenuation after passage in vitro and in vivo. Furthermore, these viruses fully protected mice against challenge with the lethal parental virus, and are therefore safe and stable vaccine candidates for protection against SARS-CoV (Jimenez-Guardeno et al., 2015). The relevance of MERS-CoV proteins in modulating cellular innate immune responses varies depending on the experimental approach used. Two main strategies have been applied, either the expression of individual virus proteins or the deletion of appropriate protein-coding sequences from the full-length virus genome. Using the first approach, overexpressing MERS-CoV proteins in human 293T transfected cells, the structural, and accessory proteins M, 4a, 4b, and 5 were identified as potential IFN antagonists (Matthews et al., 2014; Yang et al., 2013) . In contrast, when the role of these virus proteins was analyzed in cells infected with MERS-CoV deletion mutants in which individual genes (3, 4a, 4b, and 5, respectively) were deleted, it was observed that, among these proteins, 4b was the major antagonist of the innate immune response induction (J. Canton, S. Perlman, L. Enjuanes, and I. Sola, 2016, unpublished results) . Deletion of ORF4a resulted in a MERS-CoV mutant with increased INF sensitivity. The effect may be associated with the dsRNA-binding domain of the 4a protein that was shown to interact with PACT in an RNA-dependent manner; thereby, reducing the activation of RIG-I and MDA5 in overexpression assays (Siu et al., 2014b) . Also, the combined deletion of the accessory genes 3, 4a, 4b, and 5 in a recombinant MERS-CoV (rMERS-CoV-ORF3-5) showed a 1-1.5 log reduction in viral titer compared with the full-length MERS-CoV (Scobey et al., 2013) . Therefore, engineering MERS-CoV with deletions in one or more genes could lead to promising vaccine candidates. The complete deletion of the E gene in MERS-CoV led to the generation of replication-competent propagation-defective viruses that are promising vaccine candidates (Almazan et al., 2013) . Because E protein induces cell apoptosis (An et al., 1999; DeDiego et al., 2011) , the establishment of stable cell lines constitutively expressing this protein to complement viruses lacking E protein has not been possible. To overcome this limitation, we have generated packaging cell lines that transiently express E protein in an inducible manner and used them to grow replication-competent propagation-defective MERS-CoV-E (J. Gutierrez-Alvarez, I. Sola, and L. Enjuanes, 2016, unpublished results) . Additionally, fully stable transformed mammalian cells were generated in which E protein expression was under the control of an optimized Tet-On system for doxycyclineinducible gene expression, drastically reducing leaky expression (Das et al., 2016; Markusic et al., 2005) and (J. Gutierrez, I. Sola, and L. Enjuanes, 2016, unpublished results) . In an alternative strategy, replication-and propagation-efficient rMERS-CoVs were generated that expressed a slightly shortened E protein lacking 8-12 aa at the carboxyl-terminus, as previously reported for SARS-CoV vaccine candidates (J. Gutierrez-Alvarez, S. Perlman, I. Sola, and L. Enjuanes, 2016, unpublished results) . Thus, the lessons learned from previous SARS-CoV vaccine biosafety studies, such as the use of attenuated viruses expressing a shortened E protein (see earlier), may also be applicable to engineering MERS-CoV vaccines. MERS-CoV replicase proteins nsp1, nsp3, and, possibly, nsp14, may also interfere with the signaling pathways associated with the innate immune response through different mechanisms Yang et al., 2014) . Therefore, modification or deletion of the replicase gene sequences encoding these proteins may also lead to the generation of attenuated viruses. Similar to nsp1 of SARS-CoV, which inhibits host gene expression at the translational level, it has been reported that MERS-CoV nsp1 exhibits a conserved function, inhibiting host mRNA translation and inducing host mRNA degradation. This information could be exploited to produce MERS-CoV vaccine candidates . As in the common human CoVs, SARS-CoV and MERS-CoV PLPs have deISGylating and DUB activities and act as interferon antagonists (Barretto et al., 2005; Chen et al., 2007; Clementz et al., 2010; Frieman et al., 2009; Ratia et al., 2006; Zheng et al., 2008) . Mutations introduced into the MERS-CoV PLP coding sequence to specifically disrupt ubiquitin binding without affecting viral polyprotein cleavage led to PLP variants without DUB activity that lost their wild-type ability to inhibit IFN- promoter activation in reporter assays. These findings directly implicate DUB and deISGylating functions of PLP in the inhibition of IFN- promoter activity, and such modification may lead to attenuated MERS-CoV. In addition, PLP catalytic activity was required by both MERS-CoV and SARS-CoV to reduce induction of endogenous proinflammatory cytokines in infected cells (Mielech et al., 2014) , consistent with the important functions of ubiquitination and modification of cellular proteins by interferon-stimulated gene 15 (ISG15) in regulating cellular innate immune pathways. On the other hand, it has been shown that the SARS-CoV PLP interferes with the formation of the signaling complexes including STING (stimulator of interferon genes), TRAF3, TBK1, and IKK; thus, preventing downstream phosphorylation, dimerization, and nuclear translocation of IRF3 mediated by STING and TBK1 (Fig. 2) Yang et al., 2014) . This inhibition is not dependent on the PLP catalytic activity . These studies provide valuable information on how MERS-CoV PLPmediated antagonism of the host innate immune response is orchestrated and offers additional attractive options for designing attenuated viruses as vaccine candidates. In general, subunit vaccines against SARS-CoV and MERS-CoV administered with and without adjuvants provide different degrees of protection (Haagmans et al., 2016; Jaume et al., 2012; Wang et al., 2014) . However, as mentioned earlier, these vaccines should be used with caution in humans because of possible ADEI mechanisms, especially when antibody levels are low. Highly concentrated antisera against SARS-CoV were shown to neutralize virus infectivity, whereas diluted mono-and polyclonal anti-S protein antibodies both caused ADEI in human promonocyte cell cultures, leading to cytopathic effects and increased levels of TNF-, IL-4, and IL-6 (Haagmans et al., 2016; Jaume et al., 2012; Wang et al., 2014) . In addition, it has been documented that immunization of mice with VLPs or inactivated virus, both in the presence or absence of adjuvant, induced eosinophilic immunopathology in young and aged mice (Bolles et al., 2011; Tseng et al., 2012) . Using a double-inactivated SARS-CoV vaccine, protection was observed after homologous and heterologous challenge. Protection against a nonlethal heterologous challenge was poor, and enhanced immune pathology was comparable with that seen in SARS-CoV N protein-immunized mice. Importantly, aged mice displayed increased eosinophilic immune pathology in the lungs, and mice were not protected significantly against virus replication. The induction of immunopathology was also observed during challenge in ferrets and NHPs immunized with candidate vaccines based on VLPs, the whole-inactivated SARS-CoV virus, and rDNA-produced S protein (Tseng et al., 2012) . The pulmonary damage after challenge with SARS-CoV was associated with a Th2-type immunopathology with prominent eosinophil infiltration, and upregulation of genes associated with the induction of eosinophilia was observed. Interestingly, this immunopathology in the lungs upon SARS-CoV infection could be avoided by administration of TLR agonist adjuvants (Iwata-Yoshikawa et al., 2014) . To increase the duration of immune responses elicited by vaccines and to prevent the CoV-induced lung immunopathology observed after challenge or natural infection, the effects of adjuvants have been studied. In immunizations either with recombinant CoV S protein or with inactivated wholevirus vaccines, the effects of different adjuvants including alum, CpG, and Adva, a new delta-inulin-based polysaccharide adjuvant, were analyzed (Honda-Okubo et al., 2015) . While all vaccines protected against lethal infection, addition of an adjuvant significantly increased serum neutralizing antibody titers and reduced lung virus titers on day 3 postchallenge. Whereas adjuvant-free or alum-formulated vaccines were associated with a significant increase in lung eosinophilic immunopathology on day 6 postchallenge, this was not seen in mice immunized with vaccines formulated with the deltainulin adjuvant. The absence of eosinophilic immunopathology in vaccines containing delta-inulin adjuvants was found to correspond to enhanced T-cell IFN--recall responses rather than reduced IL-4 responses, suggesting that immunopathology is primarily caused by an inadequate vaccineinduced Th1 response and illustrating the need to induce durable IFN- responses using appropriate adjuvants. The possibility of productive infection in macrophages (M) and dendritic cells (DCs) by MERS-CoV is under debate. Whereas some reports provided evidence for the infection of human Ms and DCs (Table 1 ) (Chu et al., , 2016 Scheuplein et al., 2015; Ying et al., 2016) , others have not been able to productively grow virus in Ms from Tg mice-expressing hDPP4 (C. Tseng, personal communication) . The different origin of the cells might explain these divergent results but more work is needed to clarify the susceptibility of Ms and DCs to MERS-CoV. Common human CoVs, such as HCoV-NL63, HCoV-OC43, and HCoV-HKU1, as well as the highly pathogenic SARS-CoV and MERS-CoV infect ciliated epithelial cells in HAE culture, whereas HCoV-229E infects nonciliated cells (Dijkman et al., 2013; Kindler et al., 2013) . Human CoVs do not elicit the production of proinflammatory cytokines in human primary respiratory epithelial cells or ex vivo human lung tissue culture, and the production of type I and III IFN was consistently found to be low (Chan et al., 2013; Kindler et al., 2013; Zielecki et al., 2013) , suggesting that all human CoVs have evolved mechanisms to antagonize the host innate immune response, mediated by a variety of viral proteins. Epithelial cells, Ms, and DCs play a role during virus replication in the lung, and may have an impact on pathogenesis. Both HCoV-229E and HCoV-OC43 can infect Ms, although only the former does it efficiently (Collins, 2002; Patterson and Macnaughton, 1982) . DCs serve as sentinels in the respiratory tract, and are the connection between innate and adaptive immunity in the lung. HCoV-229E efficiently infects and kills DCs, which has been proposed as a potential mechanism to delay host adaptive immunity, providing time to replicate in the host (Mesel-Lemoine et al., 2012) . The interaction of MERS-CoV with human Ms and DCs (Chu et al., , 2016 Scheuplein et al., 2015; Ying et al., 2016) induces CXCL-10, CCL-2, CCL-3, CCL-5, IL-8, and IL-12 . MERS-CoV also infects mouse monocyte-derived dendritic cells, inducing CXCL-10, CCL-5, IL-12, and IFN-, although no IFN- and only marginal IFN- levels were detected Zhou et al., 2015) . In contrast, in human plasmacytoid dendritic cells, MERS-CoV induced large amounts of type I and III IFNs, especially IFN- (Scheuplein et al., 2015) , although the infection was abortive. SARS-CoV also infects human Ms and DCs, but viral replication is abortive and no infectious virus particles are produced Law et al., 2005; Tseng et al., 2005; Yilla et al., 2005; Ziegler et al., 2005) . Despite the lack of productive infection in human Ms, SARS-CoV induced the expression of proinflammatory chemokines, including CXCL-10 and CCL-2 but, in contrast, antiviral cytokines such as IFN- and INF- were basically absent Law et al., 2005 Law et al., , 2009 Tseng et al., 2005) . In unproductive infections of DCs, SARS-CoV induced CXCL-10, CCL-2, CCL-3, CCL-5, and CXCL-10 \" CXCL-10 \" CXCL-10 # Apoptosis CCL-2, -36, -5 \" CCL-5 \" CCL-5 # Lymphopenia IL-8, -12 \" IL-12 \" IL-1 # IFN- \" IL-6, -8 # IFN-/ # IFN-/ # IFN-/ # IFN-III # SARS-CoV ABOR ABOR PROD ABOR CXCL-10 \" CXCL-10 \" CXCL-10 # Apoptosis CCL-2 \" CCL-5 \" CCL-5 # Lymphopenia IL-8, -12 \" CCL-2, -3 IL-1 # IL-12 \" IL-6, -8 # IFN-/ # IFN- \" IFN-/ # IFN-/ # IFN-III # TNF- (Law et al., 2005; Tseng et al., 2005) . Dysregulated type I IFN and inflammatory monocyte-M responses in SARS-CoV-infected mice resulted in high levels of cytokines and chemokines and impaired virusspecific T-cell responses leading to lethal pneumonia . Both MERS-CoV and SARS-CoV infect human primary T cells and induce massive apoptosis and lymphopenia, although they do not produce infectious virus in these cells (Chu et al., 2016; Gu et al., 2005; Zhou et al., 2015) . Infection of T cells may play a role in controlling the pathogenesis elicited by both MERS-CoV and SARS-CoV . Similarly to the field viruses, live SARS-CoV and MERS-CoV vaccines may interact with host Ms and DCs, promoting the synthesis of cytokines and chemokines that could lead to undesired side effects including imbalanced proinflammatory immune responses. The analysis of the type of interleukins and cytokines produced during vaccine administration should be included in safety studies of candidate vaccines and could involve murine leukocytes derived from susceptible Tg mice and human cells collected from healthy donors. Despite extensive efforts commencing with the SARS epidemic in 2003, no antiviral drugs suitable to treat CoV infections have been approved by the FDA (Barnard and Kumaki, 2011; Kilianski and Baker, 2014) . Nevertheless, there are several antiviral compounds in preclinical development that may be useful to control human CoV infections (Adedeji and Sarafianos, 2014; Zumla et al., 2016) . A number of these compounds target highly conserved replicase proteins, and therefore should be effective against a broad range of CoVs, including common human CoVs. Key enzymatic activities, such as the viral main protease, PLP, and helicase have been employed as targets for CoV-specific antiviral drugs . These conserved antiviral targets are generally thought to tolerate fewer mutations compared to structural protein genes due to higher fitness pressure; thus, possibly reducing the risk of resistant variants emerging rapidly during antiviral treatment. Other compounds inhibiting virus entry or morphogenesis have also been tested though, with few exceptions, these viral life cycle steps are poor antiviral targets, as escape mutants are easily recovered, especially when viral structural proteins are targeted . The cell attachment of HCoV-229E and HCoV-NL63 has been inhibited with different antibody combinations , and subsequent steps of cell entry have been inhibited by S protein-derived heptad-repeat 2 (HR2) peptides in the case of HCoV-NL63 . Similarly, inhibitors of vacuolar acidification were effective against HCoV-229E in cell culture and against HCoV-OC43 both in cells and in vivo (Keyaerts et al., 2009; Pyrc et al., 2007) . The identification of signaling pathways involved in SARS-CoV-mediated pathogenesis has provided selection systems for drugs that significantly increase the survival of infected mice. For example, the identification of NF-kB as the main signaling pathway leading to an exacerbated inflammatory response during SARS-CoV infection enabled the selection of an antiviral suitable to control this infection (DeDiego et al., 2014b) . Similarly, the identification of the increased phosphorylation of p38 MAPK by SARS-CoV E protein-activated syntenin also led to a dramatic increase (>80%) in the survival of infected mice treated with p38 MAPK inhibitors (Fig. 4) . . E, SARS-CoV envelope protein; ECM, extracellular matrix; FAK, signaling adhesion kinase protein; Mock, noninfected mice; P, phosphorylated residue; P38 MAPK, p38 MAP kinase; SB203580, inhibitor of p38 MAPK; Wt, mice infected with virulent SARS-CoV. New animal and human CoVs are constantly emerging or reemerging, as there are animal reservoirs that maintain them, including bats and birds present in high numbers and with high mobility. As a consequence, the development of technologies suitable to respond swiftly to newly emerging CoVs by producing vaccines is highly desirable. This goal could be achieved with the production of subunit or inactivated vaccines, but optimal combinations of antigen and adjuvant need to be established to minimize the risk of ADEI or eosinophilia that frequently occurred with CoV vaccines developed in the past. We believe that the vaccines of the future will be mainly based on live-attenuated viruses because of their superior potential to induce balanced Th1/Th2 immune responses, the potent and long-lasting immunity, and the comprehensive B and T-cell repertoire induced by this type of vaccines. However, the development of safe live-attenuated virus vaccines requires strong experimental support to confirm sufficient attenuation in the target species, possibly resulting in longer development and production times.',\n",
       "   '_search_score': 0.7367054999999998},\n",
       "  {'_id': 'a912f654028e787673b235c9ba2eb2d161e372aa',\n",
       "   'insert_date_': '2020-11-04T18:59:11.283733',\n",
       "   'abstract': 'Feline coronavirus (FCoV) is a pathogen causing a lethal infectious disease in cats, feline infectious peritonitis. It has two serotypes (type I FCoV and type II FCoV). According to our previous study, type I FCoV infection is inhibited by compounds inducing intracellular cholesterol accumulation, whereas type II FCoV infection is not inhibited. Intracellular cholesterol accumulation was reported to disrupt late endosome function. Based on these findings, types I and II FCoV are considered to enter the cytosol through late and early endosomes, respectively. We investigated whether the antiviral activities of a late endosome trafficking inhibitor and cholesterol-accumulating agents are different between the FCoV serotypes. The late endosome trafficking inhibitor did not inhibit type II FCoV infection, but it inhibited type I FCoV infection. Type I FCoV infection was inhibited by cholesterol-accumulating triazoles, but not by non-cholesterol-accumulating triazoles. These phenomena were observed in both feline cell lines and feline primary macrophages. This study provides additional information on the differences in intracellular reproductive cycle between type I and type II FCoV.',\n",
       "   'title': 'Endocytic Pathway of Feline Coronavirus for Cell Entry: Differences in Serotype-Dependent Viral Entry Pathway',\n",
       "   'body': \"Feline coronavirus (FCoV) is a positive-sense RNA virus in the species Alphacoronavirus 1 genus Alphacoronaivirus, of the subfamily Coronavirinae, in the family Coronaviridae [1] . It is divided into two serotypes (types I and II) based on the amino acid sequence of the spike (S) protein [2] . Type I FCoV is the dominant serotype (80-90%) in Europe and Asia [3] [4] [5] . FCoV can also be divided into two biotypes, feline infectious peritonitis virus (FIP virus, FIPV; virulent FCoV) and feline enteric coronavirus (FECV; avirulent FCoV), according to its pathogenesis in animals [6] . FIPV infection typically causes a lethal disease in Felidae known as FIP [7] . The hallmark pathological findings of FIP in domestic cats are serous fluid in peritoneal and pleural cavities, pyogranulomatous lesions in internal organs, and disseminated fibrinous on the serosal and organ surfaces [6] . The FIPV-infected cells in the lesion and serous effusion predominantly consist of macrophages. FIPV-infected macrophages have been suggested to exacerbate the disease condition [8] [9] [10] . FCoVs attach to the cell surface through feline aminopeptidase N (fAPN) as the viral receptor. FCoVs enter the cell through endocytosis after attachment to the fAPN [6] . It is speculated that low endosomal pH triggers the conformation change of the spike protein of FCoV that releases the virus from the endosome to the cytosol. However, the timing of cytosol entry of each serotype was unclear. FCoV infection is dependent on intracellular cholesterol [11] . Our previous study demonstrated that the triazole antifungal agent itraconazole blocks the intracellular trafficking of cholesterol and inhibits type I FCoV infection in a feline cell line (felis catus whole fetus-4 cells: fcwf-4 cells) [12] . Similar results were observed with U18666A, the widely studied cholesterol transport inhibitor [11] . Other researchers reported that itraconazole and U18666A blocked cholesterol egress from late endosomes [13] [14] [15] . Furthermore, cholesterol accumulation was reported to disrupt late endosome function [16, 17] . Based on this, cholesterol transport inhibitors are hypothesized to inhibit the release of type I FCoV into the cytosol by indirectly disrupting the late endosome function. However, as described above, the site of endosomal release of both FCoV serotypes has not been investigated. In this study, we demonstrated that type I FCoV cannot replicate in cells unless it reaches late endosomes, whereas type II FCoV can even though it does not reach late endosomes. Moreover, we investigated the effects of cholesterol-accumulating triazoles (itraconazole and posaconazole) and non-cholesterol-accumulating triazoles (voriconazole and fluconazole) [18] on the infectivity of FCoV in established feline cell lines (Felis catus whole fetus-4 cells (fcwf-4 cells)) and feline primary macrophages. Cytotoxicity assay was performed to determine the non-toxic concentration of compounds against fcwf-4 cells ( Figure 1 ). The concentrations used in this study were selected because they had low cytotoxic effects (cell viability: >83-90%) on feline cells.  To confirm whether FCoV accesses late endosomes for cell entry, we investigated the effects of 25-hydroxycholesterol (25-HC), a non-specific late endosome trafficking inhibitor [19] , on FCoV. We assessed whether 25-HC inhibits late endosome function using intracellular cholesterol accumulation as an index. The 25-HC affected cholesterol accumulation in a dose-dependent manner (Figure 2A Figure 2C ), suggesting that type I FCoV enters via late endosomes. To confirm this hypothesis, we examined the effects of viral infection after 4-bromobenzaldehyde N-(2,6-dimethylphenyl) semicarbazone (EGA) treatment, which specifically inhibits transport from early to late endosomes [20] . EGA significantly suppressed FCoV-I KU2 infection, but not FCoV-II 79-1146 infection ( Figure 3 ).  Based on the above results and our previous study [11, 12] , different FCoV serotypes may enter feline cells through different endocytic routes: FCoV-I KU2 is released from late endosomes into the cytosol, whereas FCoV-II 79-1146 is released from early endosomes into the cytosol. Cholesterol accumulation was previously reported to disrupt late endosome function [16, 17] . Therefore, we investigated the effects of cholesterol-accumulating triazoles (itraconazole and posaconazole) and non-cholesterol-accumulating triazoles (voriconazole and fluconazole) on the infectivity of FCoV.  We next investigated whether findings similar to those observed in fcwf-4 cells are observed in macrophages, the target of virulent FCoV (FIPV). The concentration of triazole antifungals disrupting late endosome function in feline primary macrophages was confirmed, i.e., the concentration at which triazole antifungals accumulate intracellular cholesterol was measured. Both itraconazole and posaconazole clearly induced cholesterol accumulation at 20 M (Figure 7 ). In contrast, voriconazole and posaconazole did not induce cholesterol accumulation even at the concentration limit not exhibiting cytotoxicity (Figure 7) . We then examined whether FCoV can replicate in primary macrophages with disrupted late endosome function. In this experiment, only type I FCoV was used in consideration of the limited number of cats. In addition, no cytopathic effects are observed, and no virus is produced in the culture supernatant in type I FCoV-infected primary macrophages. Taking these conditions into consideration, viral infection of macrophages was judged based on the detection of FCoV N protein. As a result, N protein levels of FCoV-I KU2 were reduced by itraconazole and posaconazole, but not by voriconazole or fluconazole, in primary macrophages, similar to in fcwf-4 cells (Figure 8 ).  Viruses employ several entry strategies. After endocytosis, virus entry occurs at early or late endosomes [21] [22] [23] [24] [25] . According to previous studies, there are two cell entry types in coronavirus infection: Cytosol entry through early endosomes (e.g., MERS-CoV, infectious bronchitis virus, and human coronavirus NL-63) and late endosomes (e.g., SARS-CoV) [26, 27] . Burkard et al. reported that FCoV (serotype is unclear) enters the cytosol through late endosomes [26] . We demonstrated that types I and II FCoV enter the cytosol through late and early endosomes, respectively ( Figure 9 ). To our knowledge, viruses that enter the cytosol at different times among serotypes have not been reported other than FCoV observed in this study. The FCoV serotypes are distinguished based on the amino acid sequence of S protein [2] , i.e., timing of the serotypes to transit to the cytosol is dependent on S protein. Indeed, cleavage of coronavirus S protein in endosomes plays an important role in cytosol entry [28] [29] [30] . Cathepsin L is an endosomal protease involved in S protein cleavage of coronaviruses [31] [32] [33] . Cholesterol accumulation in late endosomes is observed in cells with inhibited cathepsin L activity [34] . Of note, the inhibition of cathepsin L activity does not influence the multiplication of FCoV-II 79-1146 [35] . Further investigation is necessary to clarify whether S protein cleavage by cathepsin L or cholesterol transport influences cytosol entry by FCoV. The 25-hydroxycholesterol is oxysterol produced by cholesterol-25-hydroxylase induced by interferon [36] . Although 25-hydroxycholesterol inhibits several viral infections [36] [37] [38] [39] [40] [41] , many points remain unclear with regard to its antiviral activity. Civra et al. reported that 25-hydroxycholesterol accumulates cholesterol in late endosomes and inhibits rotavirus infection [19] . Based on their report, 25-hydroxycholesterol exhibits its antiviral activity through disrupting late endosome function. We confirmed that 25-hydroxycholesterol inhibits type I FCoV infection at the cholesterol-accumulating concentration, i.e., type I FCoV infection was blocked by 25-hydroxycholesterol through the same mechanism as that of rotavirus infection inhibition. Based on this finding, 25-hydroxycholesterol can be applied as a potent antiviral agent for FCoV. However, oxysterols, such as 25-hydroxycholesterol, induce arteriosclerosis [42, 43] . Therefore, further investigation is necessary to use 25-hydroxycholesterol as an antiviral agent for FIP. High-concentration (100 M) 25-hydroxycholesterol inhibited type II FCoV infection by 60%. The reason for this is unclear, but a large amount of 25-hydroxycholesterol incorporated by endocytosis may have physically blocked type II FCoV release from early endosomes. Both FCoV serotypes use the endosomal pathway for cell entry [11, 44] . However, the timing of cytosol entry of each serotype was unclear. EGA prevents transport from early endosomes to late endosomes [20] . Type II FCoV infection was not inhibited by EGA, demonstrating that type II FCoV enters the cytosol before it reaches late endosomes. In contrast, EGA inhibited type I FCoV infection. Thus, type I FCoV requires late endosomes to enter the cytosol. We tried to directly demonstrate that type I FCoV reaches late endosomes, i.e., we investigated colocalization of a late endosome marker, Rab7, and the virion, but we were unable to find a commercial anti-Rab7 antibody that reacts with feline Rab7. It is necessary to produce an antibody reacting with feline Rab7 and confirm the hypothesis described above. Based on previous reports, triazole antifungals are classified into two types: cholesterolaccumulating triazoles (itraconazole and posaconazole) and non-cholesterol-accumulating triazoles (voriconazole and fluconazole) [18] . To our knowledge, the reason for these differences has not been clarified. These differences may be due to the structure of each drug. The structure of itraconazole is similar to that of posaconazole but different from those of voriconazole and fluconazole. Trinh et al. reported that itraconazole and posaconazole bind the sterol-sensing domain (SDD) of Niemann-Pick disease type C1 (NPC1) [45] . U18666A and 25-hydroxycholesterol, which induce cholesterol accumulation similarly to itraconazole, bind SDD [46, 47] . Unlike these drugs, voriconazole and fluconazole may be unable to bind SDD. On the other hand, it was suggested that itraconazole, posaconazole, U18666A, and 25-hydroxycholesterol interfere with the binding of type I FCoV to NPC1, in addition to inducing cholesterol accumulation, by binding SDD [48] . However, in cells treated with SDD-binding cholesterol, type I FCoV infection increased. Thus, inhibition of type I FCoV infection by the above drugs is not related to binding of the virus to the SDD of NPC1. FCoV-infected macrophages play an important role in the development of FIP [8] [9] [10] . We previously analyzed the mechanism of FCoV infection in feline primary macrophages [44] . Type I and type II FCoV are incorporated by endocytosis when they attach to the cell surface of macrophages, but it is unclear why FCoV that entered macrophages was not degraded. Considering these results, FCoV incorporated by macrophages may escape degradation by macrophage lysosomes by escaping into the cytosol from early/late endosomes. In general, the infectivity of type I FCoV in feline cells is lower than that of type II FCoV. Recently, O'Brien et al. established fcwf-4 cells (fcwf-4 CU) that permit type I FCoV replication [49] . They noted lower interferon sensitivity as a factor enabling type I FCoV infection of this cell line [49, 50] . As described above, 25-hydroxycholesterol inhibiting type I FCoV infection is induced in cells by interferon, suggesting that 25-hydroxycholesterol is not induced in fcwf-4 CU even though interferon is produced after infection. If this hypothesis is correct, the reason for increased infection by type I FCoV of fcwf-4 CU may be reduced 25-hydroxycholesterol production. In this study, we indirectly demonstrated that type I FCoV cannot enter the cytosol unless it reaches late endosomes, whereas type II FCoV can even though it does not reach late endosomes. Our findings warrant further studies to elucidate the mechanism of cell entry by FCoV. The fcwf-4 cells (kindly supplied by Dr. M. C. Horzinek of Utrecht University) were grown in Eagle's minimum essential medium (MEM) containing 50% L-15 medium, 5% fetal calf serum (FCS), 100 U/mL of penicillin, and 100 g /mL of streptomycin. For the primary macrophages, feline primary macrophages were selected. Feline alveolar macrophages were obtained from four specific-pathogen-free (SPF) cats aged 3-5 years by bronchoalveolar lavage with Hank's balanced salt solution. Feline primary macrophages were maintained in D-MEM supplemented with 10% FCS, 100 U/mL of penicillin, and 100 g/mL of streptomycin. SPF cats were maintained in a temperature-controlled isolated facility. The experiment using animals was approved by the President of Kitasato University through the judgment of the Institutional Animal Care and Use Committee of Kitasato University and performed in accordance with the Guidelines for Animal Experiments of Kitasato University. Sample sizes were determined based on our previous study, and the minimum number of cats was used. The FCoV-I KU-2 was isolated in our laboratory. The FCoV-II 79-1146 was kindly provided by Dr. M. C. Horzinek of Utrecht University. These viruses were grown in fcwf-4 cells at 37  C. The 25-HC (cholesterol transport inhibitor), EGA (late endosome trafficking inhibitor), posaconazole, voriconazole, and fluconazole were purchased from Sigma Aldrich Japan (Tokyo, Japan). Itraconazole was purchased from Janssen Pharma-central K. K. (Tokyo, Japan). The 25-HC was dissolved in methanol. EGA, Posaconazole, voriconazole, and fluconazole were dissolved in dimethyl sulfoxide (DMSO). Itraconazole was dissolved in 40% (w/v) hydroxypropyl-beta-cyclodextrin (HbCD) solution. All compounds were adjusted to 10 mM with solvent, aliquoted, and stored at-30  C until use. On the day of the experiments, these compounds were diluted to the desired concentrations in maintenance medium. The fcwf-4 cells were seeded on 96-well plates. The compounds were added to the wells in triplicate. After incubation for defined periods, the culture supernatants were removed, WST-8 solution (Kishida Chemical, Osaka, Japan) was added, and the cells were returned to the incubator for 1 h. The absorbance of formazan produced was measured at 450 nm using a 96-well spectrophotometric plate reader, as described by the manufacturer. Percentage viability was calculated using the following formula: Cytotoxicity (%) = [(OD of compound-untreated cellscompound-treated cells)/(OD of compound-untreated cells)]  100. Confluent fcwf-4 cell monolayers were cultured in medium containing compounds at the indicated concentrations in 24-well multi-plates at 37  C for 24 h. Cells were washed and the virus (MOI 0.01) was adsorbed into the cells at 37  C for 1 h. After washing, cells were cultured in carboxymethyl cellulose (CMC)-MEM. The cell monolayers were incubated at 37  C for 48 h, fixed and stained with 1% crystal violet solution containing 10% buffered formalin, and the resulting plaques were then counted. The plaque reduction percentage was calculated using the following formula: Plaque reduction percentage (%) = [(plaque number of compound-treated cells)/(plaque number of compound-untreated cells)]  100. Feline cells (fcwf-4 cells or feline primary macrophages) were cultured in each well of an 8-well Lab-Tek Chamber Slide (Thermo Fisher Scientific, Waltham, MA, USA). Semi-confluent cell monolayers were cultured in medium containing each compound at 37  C for 24 h. Cells were washed and the virus (MOI 0.01) was adsorbed into the cells at 37  C for 1 h (3 h in the case of macrophages). After washing, cells were cultured in MEM. The cell monolayers were incubated at 37  C. After 24 h, N protein levels were measured by immunofluorescence assay (IFA), as described previously [51] . To recognize FCoV N protein, mAb E22-2 (mouse IgG1) prepared by our laboratory was used [51] . Nuclei were stained with 4 ,6-diamidino-2-phenylindole (DAPI; Dojindo Laboratories, Kumamoto, Japan). The cellular cholesterol content of fcwf-4 cells was evaluated by the Cholesterol Cell-Based Detection Assay Kit (Cayman chemical, USA) according to the manufacturer's instructions. Briefly, fcwf-4 cells were grown on an 8-well Lab-Tek Chamber Slide (Thermo Fisher Scientific, Waltham, MA, USA). Semi-confluent fcwf-4 cell monolayers were cultured in medium containing compounds at 37  C for 24 h. After fixing and staining, filipin III-binding cells were analyzed using a Leica DM4B microscope and LAS X integrated imaging system (Leica Microsystems, Wetzlar, Germany). Data from only two groups were analyzed by the Student's t-test (Welch's t-test or Bartlett's test), and data from multiple groups were analyzed by one-way ANOVA followed by Tukey's test. \",\n",
       "   '_search_score': 0.7334266},\n",
       "  {'_id': 'a17525ba0365d376469cb2e8faeb0bad6d6ed682',\n",
       "   'insert_date_': '2020-11-04T18:48:58.911948',\n",
       "   'abstract': 'Background: The novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is the cause of a rapidly spreading illness, Coronavirus Disease 2019 (COVID-19), affecting thousands of people around the world. Urgent guidance for clinicians caring for the sickest of these patients is needed. We formed a panel of 36 experts from 12 countries. All panel members completed the World Health Organization conflict of interest disclosure form. The panel proposed 53 questions that are relevant to the management of COVID-19 in the ICU. We searched the literature for direct and indirect evidence on the management of COVID-19 in critically ill patients in the ICU. We identified relevant and recent systematic reviews on most questions relating to supportive care. We assessed the certainty in the evidence using the Grading of Recommendations, Assessment, Development and Evaluation (GRADE) approach, then generated recommendations based on the balance between benefit and harm, resource and cost implications, equity, and feasibility. Recommendations were either strong or weak, or in the form of best practice recommendations. The Surviving Sepsis Campaign COVID-19 panel issued 54 statements, of which 4 are best practice statements, 9 are strong recommendations, and 35 are weak recommendations. No recommendation was provided for 6 questions. The topics were: (1) infection control, (2) laboratory diagnosis and specimens, (3) hemodynamic support, (4) ventilatory support, and (5) COVID-19 therapy.',\n",
       "   'title': 'Surviving Sepsis Campaign: guidelines on the management of critically ill adults with Coronavirus Disease 2019 (COVID-19)',\n",
       "   'body': 'At the end of 2019, a novel coronavirus, named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), resulted in an acute respiratory illness epidemic in Wuhan, China [1] . The World Health Organization (WHO) termed this illness Coronavirus Disease 2019 . By the time this guideline panel was assembled, the COVID-19 had become a pandemic and had affected over 120,000 individuals in more than 80 countries, and resulted in more than 5000 deaths worldwide [2] . The WHO and the United States Center for Disease Control and Prevention (CDC) have issued preliminary guidance on infection control, screening and diagnosis in the general population, but there is limited guidance on the acute management of critically ill patients with severe illness due to COVID-19. This guideline provides recommendations to support hospital clinicians managing critically ill adults with COVID-19 in the intensive care unit (ICU). The target users of this guideline are frontline clinicians, allied health professionals, and policymakers involved in the care of patients with COVID-19 in the ICU. The guideline applies to both high and low-middle income settings. The Surviving Sepsis Campaign (SSC) COVID-19 subcommittee selected panel members in such a way as to obtain a balance of topic expertise, geographic location and, as far as possible, gender. The SSC COVID-19 panel was assembled and worked within very tight timelines in order to issue recommendations in a timely manner. The panel included experts in guideline development, infection control, infectious diseases and microbiology, critical care, emergency medicine, nursing, and public health. The panel was divided into four groups: (1) infection control and testing, (2) hemodynamic support, (3) ventilatory support, and (4) therapy. The Guidelines in Intensive Care Development and Evaluation (GUIDE) group provided methodological support throughout the guideline development process. All panel members completed a conflict of interests (COI) form prior to joining the guideline panel [3, 4] . We used the GRADEpro guideline development tool (GDT) online software (http://gdt.guide lined evelo pment .org) to administer WHO COI disclosure forms to participating panel members. Direct financial and industry-related COIs were not permitted and were considered disqualifying. The development of this guideline did not include any industry input, funding, or financial or non-financial contribution. No member of the guideline panel received honoraria or remuneration for any role in the guideline development process. The guideline development process is summarized in Fig. 1 . All actionable guideline questions were structured in the Population, Intervention, Control, and Outcome(s) (PICO) format, with explicit definitions, whereas descriptive questions were not. Content and methods experts in each group participated in developing the guideline questions. The PICO format provided the basis for defining inclusion and exclusion criteria for the literature searches (where performed) and for identification of relevant studies. To facilitate rapid development of recommendations, we did not perform a novel systematic prioritization of outcomes, but used the outcome prioritization informed by the ongoing SSC guideline 2020 work and expert input [5] . Accordingly, we focused on hospital mortality and serious adverse event outcomes for most questions, and for some included other outcomes deemed critical for decision making. For some questions, with help of professional medical librarians, we electronically searched major databases, i.e. Cochrane Central and MEDLINE, to identify relevant systematic reviews, randomized controlled trials (RCTs), observational studies, and case series. These electronic searches were performed looking for studies published in English from inception to March 2020. To inform the recommendations on hemodynamic and ventilatory support, we used recently published systematic reviews and asked experts to identify any new relevant studies. For selected PICO questions, a pair of reviewers screened titles and abstracts retrieved from the bibliographic databases; for each PICO question, all potentially eligible studies were assessed for eligibility according to prespecified criteria. Content experts were asked to indicate any additional studies not identified by the search. Subsequently, pairs of reviewers independently abstracted relevant data on the corresponding PICO questions, and items relevant to risk of bias. We obtained intention-to-treat data whenever available; otherwise we used complete case data, i.e. ignoring missing data [6] . We used the Grading of Recommendations, Assessment, Development and Evaluation (GRADE) approach to assess the quality of evidence [7] , i.e. our confidence in the estimate of the effect to support a recommendation [8] . The quality of evidence was rated as high, moderate, low, or very low [9] . We used the GDT online software (http://gdt.guide lined evelo pment .org) to generate the evidence profiles (evidence summaries) [10] . Given the recent emergence of COVID-19, we anticipated that there would be a scarcity of direct evidence, and therefore used a predefined algorithm to decide whether indirect evidence could inform a specific question ( Figure S1 -2). The SSC COVID-19 panel decided which population to extrapolate evidence from based on the context of the recommendation, and the likelihood of the presence of an effect modifier ( Figure S3 ). Accordingly, we used, as sources of indirect evidence, data on Middle East Respiratory Syndrome Coronavirus (MERS-CoV), Severe Acute Respiratory Syndrome (SARS), and other coronaviruses; in the same way, we considered, as indirect evidence, published data on supportive care in the ICU from studies on influenza and other respiratory viral infections, acute respiratory distress syndrome (ARDS) and sepsis. We used the principles outlined in the evidence to decision framework (EtD) to formulate recommendations, but because of the tight timelines we did not complete the online EtD tables [11] . The EtD framework covers the following domains: priority setting, magnitude of benefit and harm, certainty of the evidence, patient values, balance between desirable and undesirable effects, resources and cost, equity, acceptability and feasibility. Each of the four subgroups drafted the preliminary recommendations. We use the wording \"we recommend\" for strong recommendations and \"we suggest\" for suggestions (i.e. weak recommendations). The implications of the recommendation strength are presented in Table 1 . The final list of recommendations was developed by panel discussion and consensus; voting on recommendations was not required. We present the guideline statements and recommendations in Table 2 . We will have periodic automated electronic searches sent to assigned panel members every week to identify relevant new evidence as it emerges. Accordingly, we will issue further guideline releases in order to update the recommendations, if needed, or formulate new ones. A recent report from the Chinese Center of Disease Control and Prevention described 72,314 cases of COVID-19 from China, of which 44,672 were laboratory confirmed. Among laboratory-confirmed cases, 1716 (3.8%) were healthcare workers, most of whom, 63% (1080 of 1716), acquired the infection in Wuhan. The report describes that 14.8% (247 of 1668) of infected healthcare workers had severe or critical illness, and that 5 died [12] . In Italy, as of March 15, 2020 , there are 2026 documented COVID-19 cases among healthcare workers [13] . Although incidence data is not available, these data point to a considerable burden of infection among healthcare workers. The risk of patient-to-patient transmission in the ICU is currently unknown, therefore, adherence to infection control precautions is paramount. Healthcare workers should follow the infection control policies and procedures already in place at their healthcare institutions. We provide the following recommendations and suggestions as considerations rather than a requirement to change institutional infection control policies. 1. For healthcare workers performing aerosol-generating procedures* on patients with COVID-19 in the ICU, we recommend using fitted respirator masks (N95 respirators, FFP2, or equivalent), as opposed to surgical/medical masks, in addition to other personal protective equipment (i.e. gloves, gown,and eye protection, such as a face shield or safety goggles) Best practice statement. 1 For healthcare workers performing aerosol-generating procedures * on patients with COVID-19 in the ICU, we recommend using fitted respirator masks (N95 respirators, FFP2, or equivalent), as opposed to surgical/medical masks, in addition to other personal protective equipment (i.e. gloves, gown, and eye protection, such as a face shield or safety goggles) Best practice statement  Negative pressure rooms are an engineering control intended to prevent the spread of contagious airborne pathogens from room to room (e.g. measles, and tuberculosis). The main goal is to avoid the accidental release of pathogens into a larger space and open facility, thereby protecting healthcare workers and patients in a hospital setting. Negative air pressure is created in the patient\\'s room to keep the pathogen inside and avoid its diffusion. By adopting this precaution when aerosol-generating procedures like tracheal intubation, bronchoscopies, or non-invasive positive pressure ventilation (NIPPV) are performed within the room, there is a lower risk of crosscontamination among rooms and infection for staff and patients outside the room. Negative pressure is created and maintained by a ventilation system that allows extra air to enter the isolated room by differential pressure, and be exhausted directly to the outside or be filtered through a high-efficiency particulate air (HEPA) filter directly before recirculation. Moreover, the presence of unnecessary staff in the room should be avoided. Negative pressure rooms have proven to be an effective measure that helped to avoid cross-contamination during the SARS epidemic [15] . Accordingly, for aerosolgenerating procedures, the WHO guidance on COVID-19 recommends the use of negative pressure rooms with a minimum of 12 air changes per hour or at least 160 L/ second/patient in facilities with natural ventilation [16] . Bronchoscopies are among the procedures at highest risk of aerosolization, and their use should be minimized. Non-invasive ventilation is also at high risk of aerosolization, and strategies have been described to contain the risk [17] of virus spread, also according to a previous report on SARS infection [18] . Where this is not feasible, a portable HEPA filter should be used in the room wherever possible. A HEPA filter is a mechanical air filter, used for isolation where maximum reduction or removal of submicron particulate matter from air is required. HEPA filters have been demonstrated to reduce virus transmission in simulated settings [19] . 3. For healthcare workers providing usual care for non-ventilated COVID-19 patients, we suggest using surgical/medical masks, as opposed to respirator masks, in addition to other personal protective equipment (i.e. gloves, gown, and eye protection, such as a face shield or safety goggles). Weak recommendation, low-quality evidence. 4. For healthcare workers who are performing non-aerosol-generating procedures on mechanically ventilated (closed circuit) patients with COVID-19, we suggest using surgical/medical masks, as opposed to respirator masks, in addition to other personal protective equipment (i.e. gloves, gown, and eye protection, such as a face shield or safety goggles). Weak recommendation, low-quality evidence. Our recommendations are in line with the WHO guidance, and with the current evidence, which suggests that surgical/medical masks are probably not inferior to N95 respirators for providing protection against laboratory confirmed seasonal respiratory viral infections (e.g. influenza, but not measles). We updated the most recent systematic review and meta-analysis of RCTs [20] , and identified one new RCT [21] . Overall, 4 RCTs (5549 individuals) randomized healthcare workers to N95 respirators or medical masks [21] [22] [23] [24] [25] . The use of medical masks, as opposed to N95 respirators, did not increase laboratory-confirmed respiratory infection (OR 1.06, 95% CI 0. 90-1.25) . Although the point estimates suggest that use of medical masks was associated with increased risk of influenza-like illness (OR 1.31, 95% CI 0.94, 1.85) and clinical respiratory infection (OR 1.49, 95% CI 0.98-2.28), the differences were not statistically significant. A No recommendation 48 There is insufficient evidence to issue a recommendation on the use of recombinant rIFNs, alone or in combination with antivirals, in critically ill adults with COVID-19 No recommendation 49 There is insufficient evidence to issue a recommendation on the use of chloroquine or hydroxychloroquine in critically ill adults with COVID-19 No recommendation 50 There is insufficient evidence to issue a recommendation on the use of tocilizumab in critically ill adults with COVID-19 No recommendation recent systematic review and meta-analysis reached similar conclusions [26] . Only one RCT reported on coronavirus. On testing for seasonal coronavirus (OC43, HKU1, 229E, NL63) by means of PCR in this non-cluster RCT, 4.3% (9/212) of nurses in the medical mask group had RT-PCR confirmed coronavirus infection as compared with 5.7% (12/210) in the N95 respirator group [22] . When making these recommendations, the panel considered the lack of convincing evidence that N95 respirators improve clinical outcomes, the cost and resources associated with N95 mask use, and the need to preserve the N95 respirator supply for aerosol-generating procedures. Therefore, the panel issued a suggestion to use medical masks in this context. However, SARS-CoV-2 appears to be more easily transmissible and lethal than seasonal influenza. Specifically, an early estimate of the reproductive number (R 0 ) of SARS-CoV-2, the average number of people an infected person subsequently infects as a function of biological properties of the pathogen in combination with social and environmental factors, is 2.3 [27] . By comparison, the estimated average R 0 for the 1918 influenza pandemic that resulted in an estimated 50 million deaths globally was 1.8, and the estimated average R 0 for seasonal influenza is 1.28 [28] . Therefore, a minimum of a surgical/medical mask is recommended for healthcare workers caring for non-ventilated COVID-19 patients and for healthcare workers who are performing non-aerosol-generating procedures on mechanically ventilated (closed circuit) patients with COVID-19. When scarcity is not an issue, use of a fitted respirator use of a fitted respirator mask is a reasonable option. patients with COVID-19, we suggest using video-guided laryngoscopy, over direct laryngoscopy, if available. Weak recommendation, low-quality evidence. There is no direct evidence comparing the use of videolaryngoscopy with direct laryngoscopy for intubation of patients with COVID-19. While SARS-CoV-2 appears to be predominantly spread by large respiratory droplets, intubation is likely a small particle (less than 5 m) aerosol-generating procedure, which increases the risk of transmission to healthcare workers [29] . Intubation is particularly risky given the close contact of healthcare workers with the patient\\'s airway and respiratory secretions. Thus, techniques that can reduce the number of attempts at endotracheal intubation and the duration of the procedure and minimize the proximity between the operator and the patient, should be prioritized, potentially reducing the risk of complications in hypoxic COVID-19 patients. In a systematic review including 64 studies and 7044 patients, video-laryngoscopy reduced the risk of failed intubation (OR 0.35, 95% CI 0.19-0.65), without a significant impact upon the proportion of successful first-pass attempts (OR 0.79, 95% CI 0.48-1.3), hypoxia (OR 0.39, 95% CI 0.1-1.44), or time for tracheal intubation [30, 31] . In patients with difficult airways, the first-attempt success rate may be improved with video-laryngoscopy [32] . Thus, in settings where video-laryngoscopy is available and staff are skilled in its use, we suggest that it be used, in preference to direct laryngoscopy, to maximize the chances of success. Recognizing that not all centers will have rapid access to video-laryngoscopy or skilled users, this recommendation is conditional. For COVID-19 patients requiring endotracheal intubation, we recommend that endotracheal intubation be performed by the healthcare worker who is most experienced with airway management in order to minimize the number of attempts and risk of transmission. Best practice statement. Similar to the reasoning above, factors that maximize the chances of first pass success should be used when intubating patients with suspected or confirmed COVID-19. Thus, we recommend that the healthcare operator with the most experience and skill in airway management should be the first to attempt intubation. The WHO recently declared a COVID-19 pandemic. Accordingly, every critically ill patient arriving with evidence of respiratory infection should be considered potentially infected with SARS-CoV-2. Real-time polymerase chain reaction (RT-PCR) is the gold standard for similar viral infections, including SARS [33] . Notably, COVID-19 poses several diagnostic challenges due to an extended incubation period (approximately 2 weeks) that includes a prolonged interval (approximately 5 days) of viral shedding prior to the onset of symptoms. Moreover, the duration of asymptomatic shedding is not only variable but may also differ based on the anatomic level (upper versus lower) of the infection in the respiratory system [1, 34] . Accordingly, the performance of biomolecular assay may vary by site of sampling. 7.1 For diagnostic testing, we suggest obtaining lower respiratory tract samples in preference to upper respiratory tract (nasopharyngeal or oropharyngeal) samples. Weak recommendation, low-quality evidence. 7.2 With regard to lower respiratory samples, we suggest obtaining endotracheal aspirates in preference to bronchial wash or bronchoalveolar lavage samples. Weak recommendation, low-quality evidence. COVID-19 diagnosis is based on RT-PCR testing of respiratory samples from nasopharyngeal and oropharyngeal swabs, and of lower respiratory tract samples whenever possible. Bronchoalveolar lavage should be limited and performed only if indicated and with adequate precautions, due to the risk of aerosolization and consequent exposure of healthcare professionals. Similarly, sputum induction should be avoided due to increased risk of aerosolization. Tracheal aspirate specimens appear to carry a lower risk of aerosolization, and can sometimes be obtained without disconnecting the patient from the ventilator. The procedures involved in laboratory RT-PCR testing for SARS-CoV-2 using a number of assays currently in use are well described [35] . Despite the generally high sensitivity and specificity of RT-PCR-based assays [36] , it may not be enough to rely on oropharyngeal swabs specimens alone for SARS-CoV-2 diagnosis due to their low negative predictive value. In a recent study, only 9 out of 19 (47%) oropharyngeal swabs from COVID-19 patients tested positive by RT-PCR [37] . Similar data were reported using RT-PCR during the 2002-2003 SARS epidemic [38] . Using seroconversion as the \"gold standard\" for SARS diagnosis, RT-PCR assays performed on nasopharyngeal and throat specimens were positive only 65 and 70% of the time, respectively. However, no false positives were observed indicating assay specificity of 100%. Similarly, in a study accounting for CT scan findings among suspected COVID-19 cases, 48% with negative oropharyngeal or nasal swabs were considered highly likely cases, and 33% were considered probable cases [39] . Consequently, a single negative swab from the upper airway does not rule out SARS-CoV-2 infection and repeated sampling from multiple sites, including the lower airway, will increase diagnostic yield. Similarly, given that coinfection with other viral pathogens has been observed, a positive test for another respiratory virus does not rule out COVID-19, and should not delay testing if there is a high suspicion of COVID-19 [40] . Given this high specificity, a single positive swab confirms the diagnosis of COVID-19 and is enough to trigger infection control precautions and appropriate treatment of the patient. Lower respiratory tract specimens are considered to give a higher diagnostic yield than upper respiratory specimens in patients with pneumonia, consistent with what was observed for SARS [41] , and should therefore be obtained whenever possible. The reported prevalence of shock in adult patients with COVID-19 is highly variable (from 1 to 35%), depending on the patient population studied, the severity of illness, and the definition of shock. In a recent report summarizing the epidemiological characteristics of 44,415 Chinese patients with COVID-19, 2087 (5%) were diagnosed as critical cases, defined as severe hypoxemia and/or the presence of other organ failure, including shock [12] . In another Chinese study of 1099 patients with COVID-19 with similar severity of illness, only 12 (1.1%) developed shock [1] . In hospitalized patients, the incidence is likely higher [42] (Table 3) , and may reach 20-35% among patients in the ICU [42, 43] . Cardiac injury (elevation of cardiac injury biomarkers above the 99th percentile upper reference limit) has been reported in 7-23% of patients with COVID-19 in Wuhan, China [42] [43] [44] [45] . While the prevalence of cardiac injury may correlate with the prevalence of shock, a lack of systematic screening for cardiac dysfunction in hemodynamically stable patients means that this association cannot be taken as certain (Table 3) . The prognosis of patients with COVID-19 and shock has not been systematically reported. In a study of 150 patients from 2 hospitals in Wuhan, China, shock was a major reason for death in 40%, and may, at least in part, be due to fulminant myocarditis [46] . Studies on risk factors associated with shock in patients with COVID-19 are lacking. The majority of those that are available report unadjusted estimates [12, 42, 46] . Despite methodological limitations, these studies suggest that older age, comorbidities (especially diabetes and cardiovascular disease including hypertension), lower lymphocyte count, higher D-dimer level, and possibly cardiac injury are risk factors to consider. Recommendation 8. In adults with COVID-19 and shock, we suggest using dynamic parameters skin temperature, capillary refilling time, and/or serum lactate measurement over static parameters in order to assess fluid responsiveness. Weak recommendation, low-quality evidence. There is no direct evidence addressing the optimal resuscitation strategy in patients with COVID-19 and shock, therefore the panel based this recommendation on indirect evidence drawn from critically ill patients in general. In a systematic review and meta-analysis of 13 RCTs (n = 1652) examining the effect of dynamic assessment of fluid therapy on important patient outcomes in adult ICU patients requiring fluid resuscitation [47] , the use of dynamic assessment to guide fluid therapy was found to reduce mortality (RR 0.59, 95% CI 0.42-0.83), ICU length of stay (MD  1.16 days, 95% CI  1.97 to  0.36) and duration of mechanical ventilation ( 2.98 h, 95% CI  5.08 to  0.89). Of note, only one trial focused on patients with septic shock. Dynamic parameters used in these trials included stroke volume variation (SVV), pulse pressure variation (PPV), and stroke volume change with passive leg raising or fluid challenge. Among the examined dynamic parameters, passive leg raising followed by PPV and SVV appears to predict fluid responsiveness with highest accuracy [48] . The static parameters included components of early goal-directed therapy, e.g. central venous pressure (CVP) and mean arterial pressure (MAP). The use of serum lactate levels to guide resuscitation of patients with shock has been summarized in a systematic review and meta-analysis of 7 RCTs (n = 1301) [49] . Compared with central venous oxygen saturation (ScVO2) guided therapy, early lactate clearance-directed therapy was associated with a reduction in mortality (RR 0.68, 95% CI 0.56-0.82), shorter ICU length of stay (MD 1.64 days, 95% CI  3.23 to  0.05), and shorter duration of mechanical ventilation (MD  10.22 h, 95% CI  15.94 to  4.50). However, a high lactate level does not always indicate hypovolemia; it may also be caused by mitochondrial dysfunction, liver failure, beta-agonists, mesenteric ischemia, or epinephrine. In the ANDROMEDA-SHOCK trial, capillary refill testing (CRT) every 30 min was associated with a nonsignificant reduction in mortality (HR 0.75, 95% CI 0.55-1.02) compared with serum lactate measurement every 2 h [50] . CRT is a simple and easy test that can be used in almost any setting. Given the possible improvements in mortality, length of stay, and duration of mechanical ventilation that they may produce, as well as their availability, we suggest using dynamic parameters skin temperature, capillary refilling time, and/or lactate measurement over static parameters to assess fluid responsiveness in patients with COVID-19 and shock. For the acute resuscitation of adults with COVID-19 and shock, we suggest using a conservative over a liberal fluid strategy. Weak recommendation, low-quality evidence. No direct evidence exists on patients with COVID-19 and shock, therefore the panel used indirect evidence from critically ill patients with sepsis and ARDS to inform this recommendation. A recent systematic review of 9 RCTs (n = 637 patients) comparing restricted versus liberal fluid volumes in the initial resuscitation of patients with sepsis found no statistically significant difference in mortality (RR 0.87, 95% CI 0.69-1.10) and serious adverse events (RR 0.91, 95% CI 0.78-1.05) [51] . However, all assessed outcomes favored conservative fluid therapy (lower volumes). Importantly, the quantity and quality of evidence were both judged to be very low, suggesting that more research is needed. Correspondingly, in a 2017 meta-analysis of 11 RCTs (n = 2051 patients), adults and children with ARDS or sepsis managed according to a conservative fluid strategy in the post-resuscitation phase of critical illness had more ventilator-free days and shorter ICU stays than patients managed according to a liberal fluid strategy [52] (see section on respiratory support for more details). In 2011, a large RCT of 3141 febrile African children (FEAST) found that children randomized to fluid boluses with saline or albumin had increased mortality compared with children not receiving fluid boluses [53] . In the absence of data demonstrating a benefit of the use of liberal fluid strategies in critically ill patients with sepsis or ARDS, and considering that the majority of COVID-19 patients in the ICU develop ARDS, we suggest an initial conservative approach to fluid resuscitation in patients with COVID-19 and shock. we recommend using crystalloids over colloids. Strong recommendation, moderate quality evidence. Since there exists no direct evidence on shock in patients with COVID-19, the panel based this recommendation on indirect evidence from critically ill patients in general. In a systematic review of 69 RCTs (n = 30,020 patients) that compared the use of crystalloids versus colloids in critically ill patients [54] , no outcomes favored the use of colloids. Considering that some colloids are harmful (see below), all colloids are more costly than crystalloids, and availability of colloids is limited in some settings (e.g. some low-and middle-income countries), we recommend using crystalloids for fluid resuscitation in patients with COVID-19 and shock. we suggest using buffered/balanced crystalloids over unbalanced crystalloids. Weak recommendation, moderate quality evidence. No direct evidence addresses this question in patients with COVID-19 and shock; the panel therefore based this recommendation on indirect evidence from critically ill patients in general. A systematic review and meta-analysis of 21 RCTs (n = 20,213 patients) comparing intravenous buffered (balanced) crystalloid solutions versus 0.9% saline for resuscitation of critically ill adults and children [55] reported no significant differences in hospital mortality (OR 0.91, 95% CI 0.83-1.01) or acute kidney injury (OR 0.92, 95% CI 0.84-1.00) between the treatments. However, the point estimates for both outcomes suggest a potential for benefit from buffered crystalloid solutions. In the absence of apparent harm, and considering the roughly equivalent costs, we suggest using buffered crystalloid solutions over unbalanced crystalloid solutions for resuscitation of patients with COVID-19 and shock. In settings with limited availability of buffered solutions, 0.9% saline remains a reasonable alternative. For the acute resuscitation of adults with COVID-19 and shock, we recommend against using hydroxyethyl starches. Strong recommendation, moderate quality evidence. Given the absence of direct evidence on patients with COVID-19 and shock, the panel based this recommendation on indirect evidence from critically ill patients in general. A systematic review of 69 RCTs (n = 30,020 patients) compared the use of crystalloids versus colloids in critically ill patients; 24 of these RCTs (n = 11,177 patients) compared the use of crystalloids with the use of starches [54] . When the data were pooled, no statistically significant difference in mortality was observed at the end of follow-up (RR 0.97, 95% CI 0.86-1.09), within 90 days (RR 1.01, 95% CI 0.90-1.14), or within 30 days (RR 0.99, 95% CI 0.90-1.09). The authors, however, reported an increased risk of blood transfusion (RR 1.19, 95% CI 1.02-1.39) and renal replacement therapy (RRT) with starches (RR 1.30, 95% CI 1.14-1.48). Given the risk of clinically significant harm and of the apparent absence of benefits from the use of hydroxyethyl starches, we recommend against their use for resuscitation of patients with COVID-19 and shock. we suggest against using gelatins. Weak recommendation, low-quality evidence. Since no study has evaluated this question in patients with COVID-19 and shock, the panel based this recommendation on indirect evidence from critically ill patients in general. In a systematic review of 69 RCTs (n = 30,020 patients) comparing crystalloid versus colloid use in critically ill patients, crystalloids were compared with gelatins in 6 RCTs (n = 1698) [54] . No statistically significant difference in all-cause mortality was observed at the end of the follow-up (RR 0.89, 95% CI 0.74-1.08), within 90 days (RR 0.89, 95% CI 0.73-1.09), or within 30 days (RR 0.92, 95% CI 0.74-1.16), although point estimates favored the use of crystalloids. Considering the absence of any benefit of gelatins, and their higher costs, we suggest against using gelatins for resuscitation of patients with COVID-19 and shock. For the acute resuscitation of adults with COVID-19 and shock, we suggest against using dextrans. Weak recommendation, low-quality evidence. Given the absence of direct evidence on patients with COVID-19 and shock, the panel based this recommendation on indirect evidence from critically ill patients in general. A systematic review and meta-analysis on crystalloid versus colloid use in critically ill patients identified 19 trials comparing crystalloids with dextrans (n = 4736) [54] . It reported similar mortality rates at the end of follow-up (RR 0.99, 95% CI 0.88-1.11) and within 90 days (RR 0.99, 95% CI 0.87-1.12), but a possibly increased risk of blood transfusion in the dextran arm (RR 0.92, 95% CI 0.77-1.10). In view of a possible increased risk of blood transfusion (bleeding) and higher costs associated with dextrans, we suggest against their use for resuscitation of patients with COVID-19 and shock. we suggest against the routine use of albumin for initial resuscitation. Weak recommendation, moderate quality evidence. Since there is no direct evidence on patients with COVID-19 and shock, the panel based this recommendation on indirect evidence from critically ill patients in general. A systematic review and meta-analysis identified 20 RCTs (n = 13,047) comparing albumin with crystalloid use [54] . It demonstrated no significant difference in all-cause mortality at the end of the follow-up (RR 0. 98 In the absence of a benefit of albumin, and considering its cost and limited availability, we suggest against its routine use for the initial resuscitation of patients with COVID-19 and shock. Recommendation 16 . For adults with COVID-19 and shock, we suggest using norepinephrine as the first-line vasoactive agent, over other agents. Weak recommendation, low-quality evidence. There is no direct evidence on patients with COVID-19 and shock, therefore the panel based this recommendation on indirect evidence from critically ill patients in general. A systematic review of 28 RCTs (n = 3497 patients) and a clinical practice guideline from 2016 summarized the available body of evidence on the best first-line vasopressor for patients with shock [56, 57] . As norepinephrine is the most widely studied vasoactive agent with a low a priori risk of undesirable effects, we suggest using norepinephrine as the first-line vasoactive agent in patients with COVID-19 and shock. 17. If norepinephrine is not available, we suggest using either vasopressin or epinephrine as the first-line vasoactive agent, over other vasoactive agents, for adults with COVID-19 and shock. Weak recommendation, low-quality evidence. In the absence of direct evidence on patients with COVID-19 and shock, the panel based this recommendation on indirect evidence from critically ill patients in general. In a systematic review of 28 RCTs (n = 3497 patients) norepinephrine was compared with both vasopressin and epinephrine, but no trials directly compared the two options [57] . If norepinephrine is not available, we suggest using either vasopressin or epinephrine, as both agents have been assessed in RCTs without showing clear evidence of harm. Factors determining the choice between vasopressin and epinephrine may include availability and contraindications to the two agents. With vasopressin, digital ischemia may be a concern; with epinephrine, tachycardia and excess lactate production may be considerations. using dopamine if norepinephrine is available. Strong recommendation, high quality evidence. Because no direct evidence addresses this question in patients with COVID-19 and shock, the panel based this recommendation on indirect evidence from critically ill patients. A 2016 Cochrane systematic review found 6 RCTs (n = 1400) comparing norepinephrine and dopamine in patients with shock [57] . When pooled, the results showed no significant difference in all-cause mortality, but the point estimate favored norepinephrine (RR 1.07, 95% CI 0.99-1.16), and an increased risk of arrhythmias (RR 2.34, 95% CI 1.46-3.78) was found in the dopamine arm. On the basis of an increased risk of harm, including a possible increased risk of mortality in patients treated with dopamine, we recommend against using dopamine in patients with COVID-19 and shock where norepinephrine or alternatives are available (see recommendation 17). For adults with COVID-19 and shock, we suggest adding vasopressin as a second-line agent, over titrating norepinephrine dose, if target mean arterial pressure (MAP) cannot be achieved by norepinephrine alone. Weak recommendation, moderate quality evidence. In the absence of data on patients with COVID-19 and shock, the panel based this recommendation on indirect evidence from critically ill patients in general. In a recent clinical practice guideline, the use of vasopressin and vasopressin analogs in critically ill adults with distributive shock was assessed [58] . Analyzing 25 RCTS (n = 3737 patients), the authors found low certainty of a reduction in mortality (RR 0.91, 95% CI 0.85-0.99), high certainty of a reduction in atrial fibrillation (RR 0.77, 95% CI 0.67-0.88), and moderate certainty of an increased risk of digital ischemia (RR 2.56, 95% CI 1. 24-5.25 ) with the addition of vasopressin or its analogs to catecholamines. Another recent systematic review reached similar conclusion [59] . In view of these findings, we suggest adding vasopressin as a second-line agent, over titrating norepinephrine dose, if target MAP cannot be achieved by norepinephrine alone in patients with COVID-19 and shock. For adults with COVID-19 and shock, we suggest titrating vasoactive agents to target a MAP of 60-65 mmHg, rather than higher MAP targets. Weak recommendation, moderate quality evidence. No direct evidence informs this recommendation; it is based on indirect evidence from critically ill patients. A recent individual patient-data meta-analysis of 2 RCTs (n = 894 patients) comparing higher versus lower blood pressure targets for vasopressor therapy in adult patients with shock reported no significant difference in 28 [60] . The risk of arrhythmias was increased in patients allocated to the higher target group (OR 2.50, 95% CI 1. 35-4.77) . Correspondingly, the recently published 65 trial reports an absolute risk difference in mortality of 3% (RR 0.93, 95% CI 0.85-1.03) in favor of a MAP target of 60-65 mmHg (lower target), as compared to a standard of care MAP target (higher target) [61] . With an indication of improved outcome with lower MAP targets (and no firm indication of harm), we suggest titrating vasoactive agents to a target of 60-65 mmHg. For adults with COVID-19 and shock with evidence of cardiac dysfunction and persistent hypoperfusion despite fluid resuscitation and norepinephrine, we suggest adding dobutamine, over increasing norepinephrine dose. Weak recommendation, very low-quality evidence. In the absence of direct evidence in patients with COVID-19 and shock, the panel used indirect evidence from critically ill patients to inform this recommendation. In a clinical practice guideline from 2018 assessing the optimal inotropic agent in patients with acute circulatory failure (shock), no RCTs comparing dobutamine vs. placebo or no treatment were identified [62] . Based on a physiological rationale, we suggest adding dobutamine, over no treatment, in patients with COVID-19 and shock with evidence of cardiac dysfunction and persistent hypoperfusion despite fluid resuscitation and high doses of norepinephrine. The use of dobutamine in shock, including in COVID-19 patients with shock, is a research priority. For adults with COVID-19 and refractory shock, we suggest using low-dose corticosteroid therapy (\"shock-reversal\"), over no corticosteroid therapy. Weak recommendation, very low-quality evidence. Remark: A typical corticosteroid regimen in septic shock is intravenous hydrocortisone 200 mg per day administered either as an infusion or intermittent doses. Since no data exist on the use of steroids in patients with COVID-19 and shock, the panel based this recommendation on indirect evidence from critically ill patients in general. As time to resolution of shock and length of stay (especially in ICU) are important cost considerations, we suggest using low-dose corticosteroid therapy in patients with COVID-19 and refractory shock. Below, we provide further guidance on patients with COVID-19 and respiratory failure in the absence of refractory shock. The prevalence of hypoxic respiratory failure in patients with COVID-19 is 19% [12] . Recent reports from China showed that 4-13% of COVID-19 patients in these studies received non-invasive positive pressure ventilation (NIPPV), and that 2.3-12% required invasive mechanical ventilation (Table 3 ) [1, 12, 42, 43, 65] . Although the true incidence of hypoxic respiratory failure in patients with COVID-19 is not clear, it appears that about 14% will develop severe disease requiring oxygen therapy, and 5% will require ICU admission and mechanical ventilation [12] . Another study reported on 52 critically ill COVID-19 patients; 67% of these patients had ARDS, 33 (63.5%) received high-flow nasal cannula (HFNC), 56% invasive mechanical ventilation, and 42% NIPPV [42] . Risk factors associated with respiratory failure requiring mechanical ventilation are not clearly described in published reports, although from the limited available data, risk factors associated with a critical illness/ICU admission included older age (> 60 years), male gender, and the presence of underlying comorbidities such as diabetes, malignancy, and immunocompromised state [1, 12, 42, 43] . The CDC reported an overall case-fatality rate (CFR) of 2.3%, with a CFR of 14.8% in patients aged 80 years or older. In critically ill patients, the CFR was 49.0%, and it was higher than 50% in those who received invasive mechanical ventilation. The presence of pre-existing comorbid conditions such as cardiovascular disease, diabetes, chronic respiratory disease, hypertension, and cancer were associated with higher risk of death [12] . In adults with COVID-19, we suggest starting supplemental oxygen if the peripheral oxygen saturation (SpO 2 ) is < 92% (weak recommendation, low-quality evidence), and recommend starting supplemental oxygen if SpO 2 is < 90%. Strong recommendation, moderate quality evidence. 24 . In adults with COVID-19 and acute hypoxemic respiratory failure on oxygen, we recommend that SpO 2 be maintained no higher than 96%. Strong recommendation, moderate quality evidence. A recent study described the disease course of 1009 patients with COVID-19 in China and showed that 41% of all hospitalized patients and over 70% of those with severe disease required supplemental oxygen [1] . In critically ill patients, hypoxia can be detrimental and is associated with poor outcomes [66] . There are no randomized or non-randomized studies on the use of oxygen in adults with COVID-19. However, the panel used indirect evidence from the acutely ill population to inform the recommendations. A systematic review and meta-analysis of 25 RCTs (16,037 patients) showed that a liberal oxygen strategy is associated with increased risk of hospital mortality (RR1.21, 95% CI 1.03-1.43) in acutely ill patients [67] . Furthermore, a meta-regression showed a linear association between risk of death and higher SpO 2 targets [67] . The median SpO 2 in the liberal oxygen group was 96% (IQR 96-98) across all trials. A recent clinical practice guideline recommended that SpO 2 be maintained no higher than 96% [68] . Subsequent trials provided further guidance on oxygenation targets. The ICU-ROX trial randomized 1000 critically ill patients to receive either conservative oxygen (based on a protocol to dial down oxygen) or usual care. This trial showed no difference in 180-day mortality between the two groups (OR 1.05, 95% CI 0.81-1.37) [69] . The ICU-ROX trial did not compare hyperoxia with a conservative oxygen strategy; instead it compared usual care with a conservative oxygen strategy. The recent LOCO2 trial randomized patients with ARDS to a conservative oxygen arm (target SpO 2 88% to 92%) or a liberal oxygen arm (target SpO 2  96%). The trial was stopped early for futility and possible harm after 61 deaths had occurred in 205 included patients for 28-day mortality [risk difference (RD) 7.8%, 95% CI; 4.8-20.6] [70] . At 90 days, the conservative oxygen arm had a higher risk of death (RD 14.0%, 95% CI, 0.7-27.2). Considering the associated patient harm at the extremes of SpO 2 targets and the increased cost of liberal oxygen use, as well as the potential to reduce equity if oxygen resources are depleted, the panel issued a strong recommendation against using oxygen to target SpO 2 > 96%, and a strong recommendation to avoid lower values (SpO 2 < 90%). Therefore, a reasonable SpO 2 range for patients receiving oxygen is 92-96%. despite conventional oxygen therapy, we suggest using HFNC over conventional oxygen therapy. Weak recommendation, low-quality evidence. As there is no direct evidence on patients with COVID-19, the panel used indirect evidence from the critically ill population to inform this recommendation. In an RCT comparing HFNC with conventional oxygen therapy in patients with acute hypoxic respiratory failure, HFNC resulted in reduced 90-day mortality (OR 0.42, 95% CI 0.21-0.85), but did not reduce the risk of intubation [71] . A systematic review and meta-analysis of 9 RCTs (2093 patients) showed that HFNC reduces intubation compared with conventional oxygen (RR 0.85, 95% CI 0.74-0.99), but does not affect the risk of death or ICU length of stay [72] [73] [74] . Even though the evidence on mortality and length of stay was not as strong, the reduction in the need for intubation is an important finding, particularly from the perspective of pandemics such as COVID-19, where resources such as critical care beds and ventilators may become limited. In addition, in SARS, there are reports of increased transmission of disease to healthcare workers, especially nurses, during endotracheal intubation (OR 6.6, 95% Cl 2.3-18.9) [29, 75, 76] . Although this is a finding based mostly on retrospective observational studies, HFNC does not seem to confer an increased risk of transmission of disease. In studies evaluating bacterial environmental contamination, HFNC presented a contamination risk similar to that of conventional oxygen [77] . In SARS, healthcare workers exposed to HFNC were not at increased risk of developing disease [75] . Finally, patients may find HFNC more comfortable than, or at least as comfortable as, conventional oxygen therapy [71, 74] . Although some authors advised avoiding the use of HFNC in patients with COVID-19 due to the fear of disease transmission, studies supporting this advice are lacking [78] . Although some have proposed that patients wear face masks while on HFNC therapy, we are uncertain about the efficacy and safety of this approach. This question could be addressed in future studies. we suggest using HFNC over NIPPV. Weak recommendation, low-quality evidence. In adults with COVID-19 and acute respiratory failure, we suggest the use of HFNC over NIPPV. In an RCT comparing HFNC with NIPPV in patients with acute hypoxic respiratory failure, HFNC resulted in reduced mortality at 90 days (HR 2.50, 95% CI 1.31-4.78), but did not significantly affect the need for intubation (50% failure rate in NIPPV vs 47% in conventional oxygen and 40% in HFNC groups; p = 0.18) [71] . Another meta-analysis comparing HFNC with NIPPV showed HFNC to decrease the need for intubation of patients, yet without significantly reducing mortality or ICU length of stay [72] . Additionally, patients may find HFNC more comfortable than NIPPV [71] . Given the evidence for a decreased risk of intubation with HFNC compared with NIPPV in acute hypoxemic respiratory failure, and studies suggesting that NIPPV may carry a greater risk of nosocomial infection of healthcare providers, we suggest HFNC over NIPPV. However, any patients receiving HFNC or NIPPV should be monitored closely and cared for in a setting where intubation can be facilitated in the event of decompensation, as the failure rate may be high and emergency intubation in an uncontrolled setting may increase the risk of nosocomial infection of healthcare providers [79, 80] . if HFNC is not available and there is no urgent indication for endotracheal intubation, we suggest a trial of NIPPV with close monitoring and short-interval assessment for worsening of respiratory failure. Weak recommendation, low-quality evidence. 28 . We were not able to make a recommendation regarding the use of helmet NIPPV compared with mask NIPPV. It is an option, but we are not certain about its safety or efficacy in COVID-19. 29. In adults with COVID-19 receiving NIPPV or HFNC, we recommend close monitoring for worsening of respiratory status, and early intubation in a controlled setting if worsening occurs. Best practice statement. In adults presenting with hypoxic respiratory failure from COVID-19, there is no direct evidence to support the use of NIPPV; furthermore, some prior studies suggested that it may be associated with an increased risk of infection transmission to healthcare workers. Metaanalyses of RCTs showed reductions in both intubation and mortality risks with NIPPV in hypoxic respiratory failure. However, these meta-analyses included studies focused on immunocompromised, acute cardiogenic pulmonary edema, or post-operative patients; their findings may therefore be less applicable to COVID-19 patients, in whom acute hypoxemic respiratory failure and ARDS are more common presentations. [43, [81] [82] [83] In acute hypoxemic respiratory failure with an etiology other than cardiogenic pulmonary edema, NIPPV has a high failure rate. In one RCT, failure was reported in 49% of patients with hypoxic respiratory failure ventilated with NIPPV; these patients therefore required intubation [71] . In addition, patients with hypoxic respiratory failure randomized to NIPPV had higher mortality (28%, 95% CI 21-37%) than those treated with conventional oxygen therapy (23%, 95% CI 16-33%) or HFNC (13%, 95% CI 7-20%) (p = 0.02). In a cohort of Middle East Respiratory Syndrome (MERS) patients, NIPPV was not associated with improved mortality or length of stay, compared with patients who were intubated without trying NIPPV [79] . However, NIPPV was associated with a high failure rate (92.4%), leading to intubation. Patients who received NIPPV prior to intubation had increased inhaled nitric oxide requirements and increased mortality [79] . Failure rates in other pandemics, such as influenza, H1N1 and SARS, range from 10 to 70%, while demonstrations of efficacy mainly come from case series and observational studies rather than RCTs, leading to practice variation. In China, the use of NIPPV for pandemic respiratory infection is common, whereas guidelines from Europe, Hong Kong, and the US advise against NIPPV as a first-line therapy in H1N1 [84] . There are additional concerns over the use of NIPPV in respiratory pandemics like COVID-19: NIPPV may aggravate severe forms of lung injury as a result of injurious transpulmonary pressures and large tidal volumes [85, 86] , and may delay initiation of invasive mechanical ventilation, leading to emergency or more unstable intubations that can increase the risk of transmission to the healthcare team [85] . In addition, NIPPV is an aerosol-generating procedure that can increase the risk of transmission of disease to healthcare workers [29] . Several other studies and meta-analyses of SARS have also highlighted the risk of nosocomial spread of the disease with NIPPV [76, 87] . The balance between benefit and harm when using NIPPV in adults with COVID-19 is unclear. If, in certain COVID-19 patients, other forms of respiratory failure, such as acute hypercapnic respiratory failure or acute cardiogenic pulmonary edema, are known to be the cause of respiratory failure, NIPPV may be beneficial [88, 89] . However, because limited experience with NIPPV in pandemics suggests a high failure rate, we recommend that any patient receiving NIPPV be monitored closely and cared for in a setting where intubation can be facilitated in the event of decompensation [79, 80] . However, when resources become stretched, there may be insufficient ability to provide invasive ventilation, and even a moderate chance of success with NIPPV may justify its use. If NIPPV is used, helmet NIPPV is an attractive option, if available. A single-center RCT showed decreased intubation and improved mortality from NIPPV delivered by helmet in ARDS patients [90] . Of particular importance in the setting of a pandemic such as COVID-19, NIPPV by helmet has also been shown to reduce exhaled air dispersion, whereas face masks were insufficient [91] . However, helmet NIPPV is more expensive, and without direct evidence of benefit in COVID-19 patients, resources should not be utilized to acquire this equipment if is not already available. Figure 2 summarizes the recommendations on HFNC and NIPPV in patients with COVID-19.  Recommendation 30 . In mechanically ventilated adults with COVID-19 and ARDS, we recommend using low tidal volume (Vt) ventilation (Vt 4-8 mL/kg of predicted body weight), over higher tidal volumes (Vt > 8 mL/kg). Strong recommendation, moderate quality evidence. Currently there are no studies addressing mechanical ventilation strategies in COVID-19 patients. However, the panel of experts believes that mechanically ventilated patients with COVID-19 should be managed similarly to other patients with acute respiratory failure in the ICU. While mechanical ventilation is a potentially life-saving intervention, it can worsen lung injury and, through ventilator-induced lung injury (VILI), contribute to multiorgan failure in patients with ARDS [86] . One of the main ventilator strategies to minimize VILI is low Vt ventilation. A systematic review and meta-analysis of RCTs found an inverse association between larger Vt gradient and mortality [92] . In addition, authors found that using a protocolized low Vt strategy with high PEEP (9 RCTs and 1629 patients) reduced the risk of death (RR, 0.80, 95% CI, 0.66-0.98) [92] . Our analysis of 5 RCTs (1181 patients) showed a reduction in hospital mortality with low Vt ventilation (RR 0.73, 95% CI 0.63-0.85) [93] [94] [95] [96] [97] [98] . On the basis of the available body of evidence, several guidelines recommended using low Vt (4-8 mL/kg of predicted body weight) in patients with ARDS [99, 100] . The panel judged the magnitude of benefit to be moderate, the cost to be low, and the intervention to be acceptable and feasible to implement, and they therefore issued a strong recommendation to use low Vt (4-8 mL/ kg predicted body weight) when ventilating patients with ARDS (Fig. 3) . The ARDSNet study protocol set the initial Vt at 6 mL/kg which can be increased to 8 mL/kg if the patient is double triggering or if inspiratory airway pressure decreases below PEEP [95] . Strict adherence to target Vt in spontaneously breathing patients with ARDS is a challenge; patient-ventilator dyssynchrony is not uncommon [101] . recommend targeting plateau pressures (Pplat) of < 30 cm H 2 O Strong recommendation, moderate quality evidence. There are no clinical trials examining the effect of plateau pressure (Pplat) limitation on COVID-19 induced ARDS. However, there is a large body of indirect evidence in patients with ARDS. Along with low Vt ventilation, Pplat limitation is a lung protective strategy to limit VILI. A systematic review and meta-analysis of RCTs found that using a lung protective strategy including protocolized low Vt and Pplat < 30 cmH 2 O (9 RCTs and 1629 patients) reduced the risk of death (RR, 0.80, 95% CI 0.66-0.98) [92] . A subsequent meta-analysis of RCTs comparing ventilatory strategies with low and high Pplat in patients with ARDS (15 studies) found that short-term mortality was higher in patients with Pplat > 32 cmH 2 [99, 100] . The panel judged the magnitude of benefit to be moderate, the cost to be low, the patients\\' values to be consistent, and the intervention to be acceptable and feasible to implement, and therefore, issued a strong recommendation to keep Pplat < 30 cmH 2 O when ventilating patients with ARDS. The ARDSNet study protocol set the initial Vt at 6 mL/kg, and then measured Pplat (after a 0.5 s inspiratory pause) [95] . If the Pplat > 30 cmH 2 O, Vt could be reduced in 1 mL/kg (to 4 mL/kg) steps until Pplat was within range. severe ARDS, we suggest using a higher PEEP strategy, over a lower PEEP strategy. Weak recommendation, low-quality evidence. Remarks: If using a higher PEEP strategy (i.e. PEEP > 10 cm H 2 O), clinicians should monitor patients for barotrauma. In ARDS, extrinsic PEEP is used to prevent repeated opening and closing of alveoli (i.e. atelectotrauma), and therefore to reduce VILI. In addition, PEEP increases and sustains alveolar recruitment, which improves oxygenation and reduces oxygen requirement. There are no clinical trials examining the effect of PEEP on coronavirus-induced ARDS. However, there is a large body of indirect evidence in patients with ARDS. An individual patient data meta-analysis (IPDMA) of the 3 largest trials (2299 patients) of high PEEP [103] [104] [105] found no difference in in-hospital mortality in all patients (RR 0.94, 95% CI 0.86-1.04) [106] . However, in patients with ARDS, a higher PEEP strategy resulted in lower ICU mortality (RR 0.85, 95% CI 0.76-0.95), lower in-hospital mortality (RR 0.90, 95% CI 0.81-1.0), and a reduction in the use of rescue therapies (RR 0.63, 95% CI 0.53-0.75), at the expense of a possible increase in the risk of pneumothorax [106] . A recent systematic review and meta-analysis of 9 RCTs (3612 patients) examined the effect of a higher PEEP strategy on patient-important outcomes [107] . Overall, a higher PEEP strategy did not reduce hospital mortality (RR 0.92, 95% CI, 0.79-1.07). However, in a subgroup of trials that enrolled patients with oxygenation response to PEEP (6 RCTS, 1888 patients), the use of high PEEP significantly reduced in-hospital mortality, compared with a lower PEEP strategy (RR 0.83, 95% CI 0.69-0.98). Although the body of evidence suggests a beneficial effect of higher PEEP in selected patients, the results are likely to be confounded by the fact that low Vt ventilation was not used in the control arm of these trials [108] . There is no clear and agreed upon definition of higher PEEP; moreover, the optimal PEEP level in ARDS patients is unknown, and is likely to vary based on the extent of disease, lung compliance, and other factors. In the aforementioned IPDMA, the median PEEP level in the high PEEP arm was 15.3 and 13.3 cm H 2 O on days 1 and 3, respectively, compared with median values of 9 and 8.2 cm H 2 O on days 1 and 3 in the low PEEP arm [106] . Although arbitrary, clinicians could consider PEEP levels > 10 cm H 2 O to constitute a higher PEEP strategy, and PEEP levels < 10 cm H 2 O as a lower PEEP strategy. Because the IPDMA combined different strategies to set higher PEEP, a reasonable starting point would be to implement a strategy used in the large RCTs that were included (i.e. ALVEOLI, LOV, and ExPRESS) [103] [104] [105] . After increasing the PEEP level, clinicians should monitor their patients for evidence of barotrauma. Importantly, higher PEEP may result in higher Pplat, which is associated with its own risks and benefits when Pplat > 30 cmH 2 O. Clinicians can use the ARDS Network protocol strategies to determine the optimal PEEP level. Other available strategies include decremental PEEP strategy, the esophageal balloon technique, and electrical impedance tomography. However, the effect of using these techniques on clinical outcomes is unknown. For mechanically ventilated adults with COVID-19 and ARDS, we suggest using a conservative fluid strategy over a liberal fluid strategy. Weak recommendation, low-quality evidence. The optimal fluid strategy in COVID-19 is not known, however, it is plausible that these patients will respond to fluid similarly to other ARDS patients. The limited data available on COVID-19 show that cardiac failure, alone or in combination with respiratory failure, was the cause of 40% of COVID-19 deaths [46] . Another study showed that 44% of COVID-19 patients had arrhythmia [43] . The data suggest the presence of myocardial injury in some patients with COVID-19. Few RCTs have been published that compare conservative or deresuscitative with liberal fluid strategies in ARDS. A recent systematic review included 5 RCTs enrolling 1206 patients with ARDS. The risk of death was similar in both groups: 28% in the conservative fluid strategy group and 31.1% in the liberal strategy group (RR 0.91, 95% CI 0.77-1.07) [52] . This study included RCTs in critically ill patients with or without ARDS, and the authors found that a conservative fluid strategy increased ventilator-free days (MD 1.82 days; 95% CI 0.53-3.10 days) and reduced ICU length of stay (MD  1.88 days, 95% CI  0.12 to  3.64 days), compared with a liberal fluid strategy. There was no difference in harm, including renal failure between the two groups. The landmark trial in ARDS patients (FACTT) found a significant reduction in the duration of mechanical ventilation with a conservative fluid strategy [109] . Furthermore, the majority of patients with COVID-19 in the ICU are elderly, and may develop myocardial dysfunction that could limit their ability to handle large fluid volumes [46] . In view of the moderate benefit observed in other ARDS populations, the possible reduced cost of administering less fluids, and the feasibility of the intervention, the panel issued a weak recommendation favoring conservative fluid strategy in patients with COVID-19 and ARDS to severe ARDS, we suggest prone ventilation for 12-16 h, over no prone ventilation. Weak recommendation, low-quality evidence. In a series of 81 patients with COVID-19, radiographic features progressed over the first 1-2 weeks after symptom onset from predominant ground glass opacities to a mixed pattern of predominant basilar consolidation. This latter pattern may suggest a role for prone ventilation [110] . Prone positioning theoretically makes ventilation more homogeneous by decreasing ventral alveolar distention and dorsal alveolar collapse [111] . This may reduce the difference between the dorsal and ventral transpulmonary pressures, in addition to reducing lung compression [112] and improving perfusion [113] . A recent study that described the clinical course of COVID-19 in the ICU showed that prone ventilation was used in 11.5% of patients (6 out of 52) [42] . However, there are no studies available that describe the clinical course of patients with COVID-19 who were ventilated in the prone position. A recent systematic review and meta-analysis of 9 RCTs (2129 patients) showed that prone ventilation for at least 12 h in patients with moderate to severe ARDS reduced mortality (5 RCTs; RR 0.74, 95% CI 0.56-0.99), but had no effect on mortality in studies that used prone ventilation for < 12 h (3 RCTs; RR 1.03, 95% CI 0. 88-1.20) . On the other hand, prone ventilation increased the risks of pressure sores (RR 1.22, 95% CI 1.06-1.41) and endotracheal tube obstruction (RR 1.76, 95% CI 1.24-2.50) [114] . Other systematic reviews reached similar conclusions [115] [116] [117] . We have moderate certainty that prone ventilation for more than 12 h in patients with moderate to severe ARDS reduces mortality, but may increase the risk of pressure sores and endotracheal tube obstruction. Healthcare workers proning patients with COVID-19 should be trained in the proper technique for proning and take infection control precautions in the event of accidental endotracheal tube disconnection from the ventilator. Proning itself is not associated with significant cost, and we believe that it may provide significant benefit. Further, proning can be implemented in low-and middle-income settings, and efforts should be made to provide the necessary training and education of healthcare workers to facilitate the practice (https ://www.youtu be.com/watch ?v=E_6jT9R 7WJs). A protocol for proning should be used at all institutions, based on the available resources and level of training. If prone ventilation is used, healthcare workers should be aware of complications such as pressure sores, vascular line and endotracheal tube displacement, facial edema, transient hemodynamic instability, corneal abrasions, brachial plexus injury, and hemodialysis vascular access flow issues. In addition, clinicians should be familiar with the absolute contraindications for prone ventilation, such as unstable spine, open abdomen or open chest (i.e. surgery or trauma). Enteral nutrition via nasogastric or nasoduodenal tube can be continued during proning [118, 119] .  Several professional societies have issued recommendations on the use of NMBAs in ARDS [100, [120] [121] [122] [123] . Most issued recommendations favoring the use of an NMBA infusion in patients with moderate to severe ARDS. These recommendations were mostly based on the pooled estimates from 3 RCTs (431 patients) showing a reduction in 90-day mortality with an NMBA infusion as compared with no NMBA infusion [124] . However, the results of the Re-evaluation of Systemic Early Neuromuscular Blockade (ROSE) trial challenged those of previous trials. The ROSE trial investigators randomized 1006 patients with moderate or severe ARDS to receive either an infusion of NMBA for 48 h or intermittent NMBA boluses on an as needed basis [125] . The ROSE trial showed that a continuous infusion of cisatracurium did not improve any patient important outcomes. Due to differences in design between the ROSE trial and the earlier trials, we did not perform a meta-analysis for mortality outcome, although the pooled estimate for barotrauma favored continuous NMBA infusion (RR 0.55, 95% CI 0.35-0.85). The panel suggests that a continuous NMBA infusion should be reserved for patients who have an indication for ongoing paralysis in which intermittent dosing may not suffice, such as patients with persistent ventilator dyssynchrony, and patients needing ongoing deep sedation prone ventilation, or persistently high plateau pressures. The effect of NMBAs on longterm outcomes is unclear. In mechanically ventilated adults with COVID-19 ARDS, we recommend against the routine use of inhaled nitric oxide. Strong recommendation, low-quality evidence. 37 . In mechanically ventilated adults with COVID-19, severe ARDS and hypoxemia despite optimizing ventilation and other rescue strategies, we suggest a trial of inhaled pulmonary vasodilator as a rescuetherapy; if no rapid improvement in oxygenation is observed, the treatment should be tapered off. Weak recommendation, low-quality evidence. There are no studies that describe the use of pulmonary vasodilators in COVID-19 patients. A Cochrane review identified 13 RCTs (1243 patients) on inhaled nitric oxide in ARDS; this treatment showed no significant effect on mortality (RR 1.04, 95% CI 0.9-1.19), and was associated with an increased risk of acute kidney injury (RR 1.59, 95% CI 1.17-2.16). Inhaled nitric oxide results in a transient improvement in oxygenation. The subgroup of studies reporting PaO 2 /FiO 2 (mm Hg) values up to 24 h after the intervention showed a statistically significant difference in favor of inhaled nitric oxide, which was not present beyond 24 h. No study assessed the use of inhaled nitric oxide as a \"rescue\" therapy [126] . Because of the possible harm from inhaled nitric oxide and the absence of a clear mortality benefit, the panel issued a strong recommendation against its routine use in patients with ARDS. However, in view of the finding of improved oxygenation, a trial of inhaled nitric oxide as a \"rescue\" therapy, after trying other options, is reasonable if available. If inhaled nitric oxide is used without a good response in terms of oxygenation, it should be tapered off to avoid rebound the pulmonary vasoconstriction that can occur with prolonged use and abrupt discontinuation. No adequately powered RCTs have evaluated inhaled prostacyclins such as ilioprost, therefore, we could not recommend against or for their use in severe ARDS. despite optimizing ventilation, we suggest using recruitment maneuvers, over not using recruitment maneuvers. Weak recommendation, low-quality evidence. 39. If recruitment maneuvers are used, we recommend against using staircase (incremental PEEP) recruitment maneuvers. Strong recommendation, low-quality evidence. No studies have assessed the role of recruitment maneuvers (RMs) in patients with ARDS secondary to COVID-19. RMs aim to improve oxygenation by increasing transpulmonary pressure to open atelectatic alveoli [127] . However, exposure to high levels of positive pressure may lead to barotrauma, as well as cause transient hypotension in already critically ill and unstable patients. We assessed 8 indirect RCTs assessing RMs in ARDS patients, including patients with sepsis due to bacterial or viral pneumonia. Varying strategies were used to help recruit atelectatic lungs, however two strategies, in particular, were common in the 8 RCTs included in this meta-analysis. Traditional RMs are described as prolonged inspiratory holds for a set duration of time on higher levels of CPAP, most commonly 35-40 cm H 2 O for 40 s [93, 104, 128, 129] . Incremental PEEP titration RMs are described as incremental increases in PEEP from 25 to 35 to 45 cm H 2 0 for 1-2 min each [130] [131] [132] [133] . In a systematic review and meta-analysis of 6 RCTs (1423 patients), RMs reduced mortality and the use of rescue interventions, and improved oxygenation at 24 h without increasing the risk of barotrauma [134] . Similarly, we identified 8 RCTs (2544 patients) that reported on in-hospital mortality. In these studies, RMs were not associated with reduced mortality (RR 0.90, 95% CI 0.78-1.04). However, subgroup analyses suggested that traditional RMs significantly reduced mortality (RR 0.85, 95% CI 0.75-0.97), whereas incremental PEEP titration RMs increased mortality (RR 1.06, 95% CI 0. 97-1.17) . While the effects of RMs on oxygenation may be transient, the studies showed a significant improvement in oxygenation after 24 h. Trials used different PEEP strategies in intervention and control arms; RMs are best combined with a higher PEEP strategy. Patients with severe ARDS and hypoxemia may benefit from traditional recruitment maneuvers along with higher levels of PEEP, but evidence specific to COVID-19 is needed. Patients receiving RMs should be monitored closely for severe desaturation, hypotension or barotrauma. RMS should be stopped if they lead to patient deterioration. hypoxemia despite optimizing ventilation, use of rescue therapies, and proning, we suggest using venovenous (VV) ECMO if available, or referring the patient to an ECMO center. Weak recommendation, low-quality evidence. Remark: Due to the resource-intensive nature of ECMO, and the need for experienced centers and healthcare workers, and infrastructure, ECMO should only be considered in carefully selected patients with COVID-19 and severe ARDS. There are no clinical trials of ECMO in COVID-19 patients. A recent report from China suggested that 11.5% of COVID-19 cases in the ICU received ECMO [42] , but the clinical courses and the outcomes of these patients have not been reported yet. The Ministry of Health in Saudi Arabia established an ECMO program during the MERS-CoV epidemic. In a retrospective cohort study of 35 patients with MERS-CoV and refractory hypoxemia, the group of patients who received VV ECMO had lower in-hospital mortality (65 vs. 100%, p = 0.02) [135] . However, this cohort study is at high risk of selection bias given its retrospective design. Only two RCTs have evaluated ECMO vs. conventional mechanical ventilation in severe ARDS. Guidelines published in 2017 were unable to provide specific guidance on the use of ECMO, and further research was recommended [99] . Although the most recent RCT (EOLIA) was stopped early for futility [136] , a re-analysis of this trial using a Bayesian approach provided a more favorable interpretation, suggesting lower mortality with ECMO in severe ARDS [137] . A recent systematic review including two RCTs (429 patients) found a reduction in 60-day mortality with ECMO (RR 0.73, 95% CI 0.58-0.92), but the risk of major bleeding was higher with ECMO [138] . ECMO is a resource-intensive technique restricted to specialized centers, and it remains an extremely limited resource. Therefore, its use as a rescue therapy should be reserved for carefully selected patients [139] . Future studies describing the outcomes of COVID-19 patients on ECMO and the mechanisms of death will advance our understanding and guide practice. In this section we will discuss possible treatment options for SARS CoV-2 and its complications, including antiviral agents, immunosuppressive agents, immunomodulators and other therapies. Cytokine storm syndrome is a hyperinflammatory state that is characterized by fulminant multi-organ failure and elevation of cytokine levels. A recent study from China showed that COVID-19 is associated with a cytokine elevation profile that is reminiscent of secondary hemophagocytic lymphohistiocytosis (HLH) [44] . Some authors even suggest that we screen critically ill COVID-19 patients for secondary HLH using the Hscore [140] , and that corticosteroids and other immunosuppressive agents can be used in patients with a high likelihood of HLH [141] . More evidence is needed before we can make recommendations on the treatment options for cytokine storm. In mechanically ventilated adults with COVID-19 and respiratory failure (without ARDS), we suggest against the routine use of systemic corticosteroids. Weak recommendation, low-quality evidence. 42 . In mechanically ventilated adults with COVID-19 and ARDS, we suggest using systemic corticosteroids, over not using corticosteroids. Weak recommendation, low-quality evidence. Remark: The majority of our panel support a weak recommendation (i.e. suggestion) to use steroids in the sickest patients with COVID-19 and ARDS. However, because of the very low-quality evidence, some experts on the panel preferred not to issue a recommendation until higher quality direct evidence is available. There are no controlled clinical trials on the use of corticosteroids in COVID-19 patients or other coronaviruses. A published, but not peer-reviewed, report of 26 patients with severe COVID-19 reports that the use of methylprednisolone at 1-2 mg/kg/day for 5-7 days was associated with shorter duration of supplemental oxygen use (8.2 days vs. 13.5 days; p < 0.001) and improved radiographic findings [142] . Although interesting, we judged these preliminary reports to be an insufficient basis for formulating recommendations, due to the risk of confounding. Therefore, we used indirect evidence from community acquired pneumonia, ARDS, and other viral infections to inform our recommendation. There are several RCTs on the use of systemic corticosteroids in hospitalized patients with community-acquired pneumonia, mostly non-ICU patients, some with sepsis or septic shock. A systematic review and meta-analysis of RCTs showed that using corticosteroids may reduce the need for mechanical ventilation (5 RCTs; 1060 patients; RR 0.45, 95% CI 0.26-0.79), ARDS (4 RCTs; 945 patients; RR 0.24, 95% CI 0.10-0.56) and the duration of hospitalization (6 RCTs; 1499 patients; MD  1.00 day, 95% CI,  1.79 to  0.21), but increase the risk of hyperglycemia requiring treatment [143] . However, these trials included different populations, the effect on mortality outcome was unclear, and they used different drugs and dosing regimens. In addition, there are some concerns about corticosteroid use in viral pneumonias. Therefore, the results may not be generalizable to the COVID-19 population. There are many published observational studies on the use of steroids in viral pneumonias (i.e. influenza virus, coronaviruses, and others), but they are prone to confounding, as sicker patients usually receive corticosteroids. We updated a recent Cochrane review on the use of corticosteroids in influenza [144] and searched for studies on other coronaviruses. We included a total of 15 cohort studies on influenza and 10 on coronaviruses. Our meta-analysis of adjusted ORs showed an association between corticosteroid use and increased mortality (OR 2.76, 95% CI 2.06-3.69), but the effect in the patients with other coronaviruses was unclear (OR 0.83, 95% CI 0. 32-2.17) . Also, these studies are limited by significant heterogeneity. We found significant homogeneity between observational studies on the use of corticosteroids in ARDS caused by coronaviruses and in general viral ARDS (I 2 = 82% and 77% respectively). Furthermore, in both cases, the summary statistic tended toward harm with the use of steroids. We updated a recent Cochrane review [145] and identified an additional RCT [146] dealing with ARDS. Overall, we included 7 RCTs enrolling 851 patients with ARDS. The use of corticosteroids reduced mortality (RR 0.75, 95% CI 0.59-0.95) and duration of mechanical ventilation (MD  4.93 days, 95% CI  7.81 to  2.06). However, these trials were not focused on viral ARDS, which limits the generalizability of their results to COVID-19 patients. In addition, we reviewed observational studies on corticosteroid use in viral ARDS, and identified 4 cohort studies. Although the point estimate showed increased mortality, the CI included substantial harm and benefit (OR 1.40, 95% CI 0. 76-2.57) . In a recent RCT (INTEREST trial), the use of recombinant interferon 1b (rIFN 1ba) did not reduce mortality in ARDS patients, but in the subgroup of patients receiving corticosteroids, rIFN 1ba use was associated with increased mortality (OR, 2.53, 95% CI 1.12-5.72) [147] . The only direct evidence comes from a retrospective cohort study of 201 patients with COVID-19 pneumonia. This study showed an association between corticosteroid use and lower mortality in patients with COVID-19 and ARDS (HR 0.38, 95% CI 0.20-0.72). However, the estimate was not adjusted for confounding factors [148] . The effect of corticosteroids in COVID-19 patients with sepsis or septic shock may be different. Recent systematic reviews and meta-analyses of RCTs in sepsis showed small improvements in mortality and faster resolution of shock with corticosteroid use, compared with not using corticosteroids [63, 149, 150] (see the previous section on hemodynamic support). It is widely recognized that corticosteroids have a range of adverse effects. In viral pneumonia in the ICU, several studies showed increase in viral shedding with corticosteroid use [151] [152] [153] , potentially indicating viral replication, but the clinical implication of increased viral shedding is uncertain. Considering the above, the panel issued a suggestion against the routine use of systemic corticosteroids for respiratory failure in COVID-19, and a suggestion to use corticosteroids in the sicker population of COVID-19 with ARDS. If clinicians use corticosteroids in ARDS, they should use lower dosing and shorter treatment courses. In mechanically ventilated patients with COVID-19 and respiratory failure, we suggest using empiric antimicrobials/antibacterial agents, over no antimicrobials. Weak recommendation, low-quality evidence. Remark: if the treating team initiates empiric antimicrobials, they should assess for de-escalation daily, and re-evaluate the duration of therapy and spectrum of coverage based on the microbiology results and the patient\\'s clinical status. There are no controlled clinical trials evaluating the use of empiric antimicrobials in COVID-19 patients or other coronaviruses. This recommendation is therefore based upon extrapolation of data from other viral pneumonias, particularly influenza [154] . Identifying bacterial coinfection or superinfection in patients with COVID-19 is challenging, as the symptoms may be similar to those of the underlying viral infection. The diagnostic difficulty is reflected in high rates of intravenous antibiotics administered in Wuhan: 53% with non-severe disease and > 90% of patients admitted to hospital or the ICU [1, 42, 43] . Data on the prevalence of bacterial superinfection in patients with COVID-19 are limited, as in larger case studies clinicians were often too overwhelmed to systematically obtain high-quality samples [1] . In critically ill patients with MERS, 18% had bacterial and 5% viral co-infections [155] . Co-infection with Staphylococcus aureus is common with influenza pneumonia and can be especially virulent [154] . Recent clinical practice guidelines recommend initiating empiric antibacterial therapy in adults with community-acquired pneumonia who test positive for influenza [154] . Data from critically ill patients demonstrate secondary infection in about 11% of cases, although the numbers are small. Isolated organisms included gram-negative organisms such as K. pneumoniae, P. aeruganosa, and S. marcescens. On the basis of these limited data it is difficult to determine patterns of superinfection, including the risk of S. aureus infection, commonly seen in influenza. In patients with COVID-19 and hypoxic respiratory failure requiring mechanical ventilation, the panel suggest empiric antimicrobial treatment, on the basis that superinfection is reasonably common in this population and may to lead to a substantial increase in mortality, as in pandemic influenza [156] [157] [158] . Therefore, critically ill patients with suspected or confirmed COVID-19 should be treated with empiric antimicrobial therapy in accordance with the clinical syndrome (e.g. community-acquired or hospital-acquired pneumonia). Secondary infections occur in patients with COVID-19, but the incidence is unknown given the very limited data [159] . These infections should be treated according to clinical and microbiological data. For critically ill adults with COVID-19 who develop fever, we suggest using acetaminophen/paracetamol for temperature control, over no treatment. Weak recommendation, low-quality evidence. The majority of patients with COVID-19 develop fever during hospitalization (92% of those with severe disease). In the largest report from China, the median temperature across 1099 patients was 38.3 C (IQR 37.8-38.9) [1] . Data from critically ill patients in general are available. We reviewed the literature and identified 12 RCTs (1785 patients) that examined the effect of fever control in the critically ill population, excluding neurological indication for temperature control [160] [161] [162] [163] [164] [165] [166] [167] [168] [169] [170] [171] ; active temperature management (pharmacologic or non-pharmacologic) did not reduce the risk of death (RR 1.03, 95% CI 0.81-1.31), ICU length of stay (MD  0.07 days, 95% CI  0.70-0.56), but it was effective in reducing body temperature (MD  0.36 C, 95% CI  0.42 lower to  0.29). Given the safety of acetaminophen and lack of harm in the body of evidence, increasing patient comfort through fever management maybe important. Therefore, we issued a suggestion for clinicians to consider using pharmacologic agents for controlling fever in COIVD-19 patients. The use of non-steroidal anti-inflammatory drugs to treat fever in patients with COVID-19 continues to be debated. Until more evidence is available, we suggest using acetaminophen/paracetamol to treat fever. In critically ill adults with COVID-19, we suggest against the routine use of standard intravenous immunoglobulins (IVIG). Weak recommendation, low-quality evidence. The use of intravenous immunoglobulin (IVIG) has been reported in several series of COVID-19 patients, but no efficacy data are available [172] . In the absence of adequate titers of neutralizing antibodies, standard intravenous immunoglobulin is unlikely to have a biologic effect in COVID-19. While IVIG may have immunomodulatory actions, its use can, rarely, also be associated with an increased risk of serious adverse events including anaphylactic reactions, aseptic meningitis, renal failure, thromboembolism, hemolytic reactions, transfusion-related lung injury, and other late reactions [173] . Preparations of anti-SARS-CoV-2 polyclonal or monoclonal antibodies are being developed. However, data from recent trials on the use of antibody-based therapies (immune plasma, hyperimmune globulin, monoclonal antibody to hemagglutinin stalk) [173] in hospitalized seasonal influenza patients did not demonstrate improvement in outcomes [174] [175] [176] . In critically ill adults with COVID-19, we suggest against the routine use of convalescent plasma. Weak recommendation, low-quality evidence. Convalescent plasma obtained from patients who have recovered from COVID-19 has been suggested as a potential therapy that may provide passive immunity from SARS-CoV2-specific antibodies [177] . Convalescent plasma has been used to treat several other viral infections, including those caused by SARS coronavirus, avian influenza A (H5N1) virus, and influenza A (H1N1) pdm09 virus [178] [179] [180] [181] [182] . A recent meta-analysis of observational studies using passive immunotherapy for the treatment of severe acute respiratory infections of viral etiology suggests that convalescent plasma therapy was associated with reduction in mortality (OR 0.25, 95% CI 0.14-0.45) [183] . During the current outbreak in China, convalescent plasma was used in some patients with COVID-19 [184] . However, data on the efficacy and safety of convalescent plasma are limited, and the target for sufficient levels of neutralizing antibody titers against SARS-CoV-2 is unknown. A study on MERS concluded that use of convalescent plasma might be feasible but was challenging due to a small pool of potential donors with sufficiently high antibody titers [185] . An RCT in patients with confirmed Ebola virus disease showed that convalescent plasma, with unknown levels of neutralizing antibodies, was not associated with improvement in survival [186] . Another RCT in patients with seasonal influenza treated with high-titer versus low-titer anti-influenza immune plasma was terminated for futility because of the lack of effect on the primary outcome measured by a 6-point ordinal scale of clinical status on Day 7 [187] . Given the lack of convincing evidence from RCTs and the uncertainty surrounding the optimal preparation of convalescent plasma and its safety, we suggest that it should not be routinely used in treating patients with COVID-19 until more evidence is available. In critically ill adults with COVID-19: 47.1. We suggest against the routine use of lopinavir/ritonavir (weak recommendation, low-quality evidence). There is insufficient evidence to issue a recommendation on the use of other antiviral agents in critically ill adults with COVID-19. The prolonged detection of SARS-CoV-2 RNA in the respiratory tract and sometimes other sites of seriously ill COVID-19 patients provides the rationale for administration of antiviral agents to reduce replication in efforts to improve clinical outcomes [45] . At present, no direct-acting antivirals have been proven to inhibit replication or provide clinical benefit in COVID-19 or MERS patients. A considerable number of agents approved for other indications have been proposed for use, but the comments below address the most promising ones. Several others are undergoing testing (e.g. arbidol [umifenovir] , favipiravir, ribavirin, traditional Chinese medicines, inhaled interferons), alone or in combinations, and in one or more countries. Lopinavir is an antiretroviral protease inhibitor used in combination with ritonavir to ensure adequate lopinavir exposure for the treatment of human immunodeficiency virus (HIV) infection [188] . Because it was found to show in vitro activity against SARS-CoV, lopinavir/ritonavir was administered, in combination with high-dose oral ribavirin and a tapering course of systemic corticosteroids, in a cohort of 41 patients with SARS, and was found to be associated with significantly fewer adverse clinical outcomes (ARDS or death) compared with ribavirin alone used in 111 historical controls that received ribavirin and corticosteroids [189] . In a high-throughput screening for antiviral compounds, lopinavir inhibited replication of MERS-CoV in vitro [190] . In an animal model of MERS-CoV infection, treatment with lopinavir/ritonavir or IFN-1b was associated with virologic, histologic and clinical improvement versus placebo [191] . Lopinavir/ritonavir in combination with interferon beta 1-b is being tested in an RCT in MERS-CoV patients [192] . This combination was considered the second candidate in a WHO research prioritization list of therapeutic agents [193] . The drug has a generally good safety profile, but may have interactions with many drugs commonly used in critically ill patients (http://www.covid 19-drugi ntera ction s.org/). A recent RCT compared the use of lopinavir/ritonavir to usual care in 199 hospitalized patients with COVID-19 in China [194] . In this trial, lopinavir/ritonavir did not significantly reduce 28-day mortality (RD  5.8%; 95% CI  17.3 to 5.7) or time to clinical improvement (MD 1.31 days, 95% CI 0. 95-1.80 ). In addition, lopinavir/ritonavir was associated with more adverse events [194] . This trial is the only available direct evidence on the use of lopinavir/ritonavir in patients with COVID-19, however, it has several limitations. The trial was unblinded and it enrolled a small number of patients (n = 199) with a small number of events (44 deaths in total), which limits our confidence in its results. Nevertheless, the routine use of lopinavir/ritonavir in critically ill patients is probably not warranted, and a weak recommendation against the routine use of lopinavir/ritonavir in critically ill COVID-19 patients is reasonable. Lopinavir/ritonavir is one of the arms in a planned WHO core treatment protocol for hospitalized patients with COVID-19, and in the REMAP-CAP (Randomized, Embedded, Multifactorial Adaptive Platform Trial for Community-Acquired Pneumonia) trial (NCT02735707) The results of ongoing trials will help increase the precision of estimates and the certainty in the evidence. Remdesivir is the prodrug of an adenosine analog, which incorporates into nascent viral RNA chains and results in premature termination. It was considered the most promising drug in an informal consultation on research prioritization of candidate therapeutic agents by WHO [195] . Currently, there are published case reports but no published trials on the use of remdesivir in COVID-19. Remdesivir demonstrated effective inhibition of SARS-CoV-2, MERS-CoV, and SARS-CoV in in vitro studies [196] . Furthermore, studies in animal models of MERS-CoV showed that it was more effective than control and superior to lopinavir/ritonavir combined with systemic IFN- [197, 198] . Although intravenous remdesivir appears to adequately tolerated, a recent RCT showed that it was less effective than several antibody therapies in Ebola virus disease [199] . There are several ongoing RCTs that aim to examine the efficacy and safety of intravenous remdesivir for severe COVID-19 (clinicaltrials.gov NCT04257656) and for mild and moderate COVID-19 (clinicaltrials.gov NCT04252664). Another trial sponsored by the National Institute of Allergy and Infectious Diseases is recruiting patients in USA (clinicaltrials.gov NCT04280705). We will update our guidelines as new evidence emerges. There is insufficient evidence to issue a recommendation on the use of recombinant rIFNs, alone or in combination with antivirals, in critically ill adults with COVID-19. Recombinant interferon, often combined with ribavirin therapy, has been used in patients with MERS and SARS [179, [200] [201] [202] . Different preparations of recombinant rIFNs (rIFN-2a, rIFN-2b, rIFN-1a and rIFN-1b) have shown activity against MERS-CoV in Vero and LLC-MK2 cells, and in a rhesus macaque model of MERS-CoV infection [200, 201, 203] . The largest cohort of critically ill patients with MERS showed that rIFN-2a, rIFN-2b, rIFN-1a and ribavirin were not associated with lower mortality (OR 1.03, 95% CI . 73-1.44) or reduced viral clearance when adjusted for time-varying covariables [204] . The relative effectiveness of different interferons against SARS-CoV-2 is unknown at this point. In vitro data showed that rIFN- displayed the strongest MERS-CoV inhibition among different rIFN preparations (rIFN-2b, rIFN-, rIFN-universal, and rIFN-2a , rIFN-), at 41 times lower than the previously reported 50% inhibitory concentration (IC50) of rIFN-2b [203, 205] . An RCT to examine the effect of a combination of lopinavir/ritonavir and rIFN--1b on mortality of hospitalized patients with MERS is currently recruiting patients [206] . Unpublished data indicate that IFN- inhibits SARS-C0V-2 in cell culture, and IFNs have been prioritized for study in COVID-19 by the WHO. There is insufficient evidence to issue a recommendation on the use of chloroquine or hydroxychloroquine in critically ill adults with COVID-19. Chloroquine and its metabolite, hydroxychloroquine, are antimalarial agents that have demonstrated antiviral effects on SARS-CoV and SARS-CoV-2 in vitro [196, 207, 208] . Prior studies found inhibitory effects of chloroquine for multiple RNA viruses in vitro, but RCTs in treatment of dengue and chikungunya virus infections and of influenza prophylaxis failed to demonstrate antiviral or clinical benefits [209] . In one non-human primate model of chikungunya infection, it was shown that chloroquine\\'s immunomodulatory effects were associated with delayed immune responses, higher levels of viral replication, and worse illness [210] . A news briefing suggested that its use in more than 100 patients showed \"that it was superior to the control in inhibiting the exacerbation of pneumonia, improving lung imaging findings, promoting a virus negative conversion, and shortening the disease course\", but the data have not been published yet [211] . A recent consensus document recommended chloroquine phosphate 500 mg twice daily for minimum of 5 days, with dose modifications if severe gastrointestinal side effects occur [212] . Since chloroquine is not available in some countries, hydroxychloroquine is an alternative. A recent study in China explored various dosing regimens of chloroquine and hydroxychloroquine using physiologicallybased pharmacokinetic models [208] . The study found hydroxychloroquine to be more potent than chloroquine in inhibiting SARS-CoV-2 in vitro. Based on these models, a hydroxychloroquine loading dose of 400 mg twice daily followed by 200 mg twice daily for 4 days was recommended [208] . A recent systematic review found no published studies in COVID-19 patients [213] . Pending the results of ongoing trials, we were unable to issue a recommendation for or against chloroquine. There is insufficient evidence to issue a recommendation on the use of tocilizumab in critically ill adults with COVID-19. Tocilizumab is a humanized immunoglobulin that functions in the immune response and blocks IL-6 receptor binding to IL-6. It has been approved for CRS and other inflammatory conditions related to IL-6 related inflammation, such as rheumatoid arthritis and juvenile idiopathic arthritis [214] [215] [216] [217] . Severely ill patients with COVID-19 may have an extreme immune response leading to severe respiratory failure. In such cases, inhibition of IL-6 may help attenuate the cytokine release syndrome by reducing cytokine concentrations and acute phase reactant production [218] . Ongoing trials of tocilizumab will help address the safety and efficacy of this therapy in COVID-19. From the rheumatoid arthritis literature, a systematic review and meta-analysis of 6 RCTs (3 with 8/mg dose and 3 with 4 mg/kg dose) showed an increased risk of adverse events compared with control treatment (OR 1.53, 95% CI 1.26-1.86), and an increased risk of infections (OR 1.30, 95% CI 1.07-1.58) [219] . Another systematic review and meta-analysis of RCTs on tocilizumab in rheumatoid arthritis found an increased risk of infectious respiratory adverse events (RR 1.53, 95% CI 1.04-2.25) [220] . Since we have no data on the safety or efficacy of tocilizumab in COVID-19, we were unable to issue a recommendation. Nafamostat is a synthetic serine protease inhibitor and a potent inhibitor of MERS CoV. Nitazoxanide is an antiprotozoal agent with antiviral potential against several respiratory viruses including influenza, parainfluenza, respiratory syncytial virus, and rhinovirus. An in vitro study showed that both nafamostat and nitazoxanide inhibited SARS-CoV-2 [196] . An RCT in patients with acute uncomplicated influenza demonstrated that the use of nitazoxanide reduced the duration of symptoms [221] . However, in hospitalized patients with severe acute respiratory infection in Mexico, nitazoxanide was not found to be superior to placebo [222] .',\n",
       "   '_search_score': 0.7328115},\n",
       "  {'_id': '007075cfdca03250c97761eb0e9c8e1bc8a94d6c',\n",
       "   'insert_date_': '2020-11-04T18:32:12.847262',\n",
       "   'abstract': 'Background: Since the outbreak of Coronavirus Disease 2019 in December 2019, many studies have reported the presence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in the conjunctival sac of patients infected with this virus, with several patients displaying symptoms of viral conjunctivitis. However, to our best knowledge, there is no in-depth report on the course of patients with COVID-19 complicated by relapsing viral conjunctivitis or keratoconjunctivitis. Case presentation: A 53-year-old man confirmed with COVID-19 developed symptoms of viral conjunctivitis in the left eye approximately 10 days after the onset of COVID-19. The results of a nucleic acid test were positive for SARS-CoV-2 in the conjunctival sac of the left eye. The symptoms were relieved 6 days after treatment. However, the patient was subsequently diagnosed with viral keratoconjunctivitis in both eyes 5 days after the symptoms in the left eye were satisfactorily relieved. The disease progressed rapidly, with spot staining observed at the periphery of the corneal epithelium. Although SARS-CoV-2 could not be detected in conjunctival secretions, the levels of inflammatory factors, such as interleukin-6, were increased in both eyes. Both eyes were treated with glucocorticoids, and symptoms were controlled within 5 days. There was no recurrence. Conclusions: In this case report, the pathogenesis, clinical manifestations, treatment, and outcome of a case with COVID-19 complicated by relapsing viral keratoconjunctivitis is described, and the involvement of topical cytokine surge in the pathogenesis of COVID-19 as it relates to viral keratoconjunctivitis is reported.',\n",
       "   'title': 'Relapsing viral keratoconjunctivitis in COVID-19: a case report',\n",
       "   'body': 'Since the outbreak of Coronavirus Disease 2019 in December 2019, the number of infections and deaths has increased exponentially. At the time of this writing, more than 2 million people have been infected with SARS-CoV-2 worldwide [1] . On 11 February 2020, the disease caused by the novel coronavirus was designated COVID-19 by the World Health Organization [2] , and the causative pathogen was identified as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by the International Committee on Taxonomy of Viruses [3] . SARS-CoV-2 is the seventh species of -coronavirus to infect humans, with 85% homology to bat-SL-CoVZC45 [4] . In February 2020, Lu et al. suggested that the ocular surface might be a route of SARS-CoV-2 transmission, and studies on the association between the virus and the ocular surface were initiated [5] . In March 2020, Deng et al. reported that SARS-CoV-2 can infect rhesus macaques through the conjunctiva [6] . Subsequently, Zhou et al. demonstrated the presence of SARS-CoV-2 in the conjunctival sac of patients with COVID-19 [7] [8] [9] . In addition, Chen et al. reported an association between conjunctival congestion, the main COVID-19-related ocular symptom, and conjunctivitis, suggesting that SARS-CoV-2 can cause conjunctivitis [10] . At the time of this writing, there is only one case report on the course of a patient with COVID-19 complicated by viral conjunctivitis [11] , and the pathogenesis of SARS-CoV-2-related conjunctivitis was not sufficiently addressed in their case report. In view of the ongoing COVID-19 pandemic, a recent study has reported elevated inflammatory factor levels in severely ill patients, indicating that cytokine surge is involved in the pathogenesis of COVID-19 [12] . However, the significance of topical cytokine surge in SARS-CoV-2-related conjunctivitis/keratitis is unknown. Therefore, the pathogenesis, clinical manifestations, treatment, and outcome of a case with COVID-19 complicated by relapsing viral keratoconjunctivitis are shared here, thus providing a guidance for future research. A male patient, 53 years old, was admitted to the First Affiliated Hospital of Zhejiang University on 8 February 2020 for a history of 12-day dry cough, 6-day fever with a maximum body temperature of 38.0C, and 3-day disease aggravation with shortness of breath. Three days before admission, nasopharyngeal swab specimens were obtained and tested for SARS-CoV-2; the results of nucleic acid tests were positive twice. Thoracic computed tomography (CT) scans showed thin exudate in both lungs. The patient was diagnosed with COVID-19 and transferred to our hospital. The body temperature was 36.6C and it remained normal after admission. A CT scan indicated extensive inflammation in both lungs (Fig. 1, left panel) , and no conjunctivitis was noted in neither eye. The patient was treated with oral arbidol (200 mg, tid), recombinant interferon spray (100,000 U, qid), oral darunavir (800mg, qd) as well as oxygen and other symptomatic supportive treatments. A CT scan revealed improvement (Fig. 1 , middle and right panels), and there were no other complaints. On 16 February 2020, the patient complained of general discomfort and stabbing pain in the left eye. And he did not report visual loss in either eye. An eye examination revealed redness and edema of lower eyelid, bulbar conjunctival hyperemia and edema, accompanied by a large quantity of watery secretions and a small quantity of thin viscous secretions. The cornea was clear, without infiltration. The right eye was relatively normal, except that slight redness and edema of the lower eyelid were obvious (Fig.  2a) . On 17 February 2020, binocular tear and conjunctival secretion specimens were obtained using swabs and tested for SARS-CoV-2. The results of nucleic acid tests were positive (for details, see Reference 8 [8] ); specifically, the left eye was positive for the virus, whereas the right eye was negative. The patient stated that he had a tendency of not washing his hands and rubbing his eyes, and he was advised to reverse his bad habit. On 18 February 2020, the left eye was treated with levofloxacin hydrochloride (drops, tid) and 0.1% sodium hyaluronate (drops, tid). One day after treatment, specimens were obtained and tested as indicated, and the left eye was positive for the virus, whereas the right eye was negative. On 22 February 2020, the symptoms in the left eye subsided. The results of nucleic acid tests using left eye tear and conjunctival secretion specimens, as well as sputum and fecal specimes, were all negative for SARS-CoV-2. From 19 to 23 February 2020, left eye tears and conjunctival secretion specimens were repeatedly obtained for virus isolation and culture, and the results were all negative. The treatment of the left eye was halted, and the patient was discharged and quarantined at home. On 27 February 2020, the patient phoned the clinic and stated that he had sudden discomfort in both eyes with conjunctival hyperemia and edema. However, the patient denied recently rubbing his eyes. Both eyes were treated with ganciclovir ophthalmic gel, levofloxacin hydrochloride drops, and 0.1% sodium hyaluronate drops. On 1 March 2020, the patient was re-examined. However, the symptoms of both eyes were not relieved; instead, they were aggravated, with spot staining observed at the periphery of the corneal epithelium of both eyes (Fig. 2b) . Binocular tear and conjunctival secretion specimens were obtained and tested for SARS-CoV-2, adenovirus, and herpes simplex, as well as for inflammatory factors. The results of nucleic acid tests were all negative for the viruses; however, the levels of inflammatory factors, such as interleukin-6 (IL-6), increased ten-fold of the upper normal reference range in the left eye (Table 1 ). Both eyes were treated with 0.1% fluorometholone (drops, qid). Four days after treatment, binocular symptoms were relieved. On 8 March 2020, the patient was re-examined. Binocular keratoconjunctivitis was completely relieved, and the cornea was transparent (Fig. 2c) . The timeline of the disease course is shown in Fig. 3 . In this case report, the diagnosis, disease course, and treatment of a patient with COVID-19 complicated by relapsing viral keratoconjunctivitis are described. The patient developed symptoms of conjunctivitis in the left eye approximately 10 days after the onset of COVID-19. The results of nucleic acid tests were positive for SARS-CoV-2 in the conjunctival sac of the left eye. However, the results of tests for other common viruses that cause conjunctivitis, such as adenovirus and herpes simplex virus, were all negative. Therefore, we concluded that viral conjunctivitis was related to SARS-CoV-2 infection in this case. In this patient, the main symptom of conjunctivitis were general discomfort and stabbing pain. The typical symptoms of conjunctivitis are bulbar conjunctiva congestion and edema, accompanied by a large quantity of watery secretions and a small quantity of thin viscous secretions, an absence of follicles or pseudo-membranes formation, a lack of the corneal infiltration at the initial stage of infection, no decrease in vision, and no swelling of the lymph nodes near the anterior and posterior ear. At late stages of infection, the symptoms of conjunctivitis were similar with those at early stages, and corneal infection involved peripheral infiltration with epithelial involvement. In this patient, the initial course of conjunctivitis was less than 1 week. The patient consistently wore a mask, although the upper edges of the mask did not fit closely with the nose root. Therefore, it is very likely that infectious respiratory droplets entered the eyes. In addition, the patient had a bad habit of eye rubbing, which led us to speculate that the conjunctival infection was due to external factors. In addition, the results of repeated nucleic acid tests revealed that the virus could not exist in the conjunctival sac for a long time, because it could not be detected in the conjunctival sac after three days, and these results are consistent with those of Chen et al. [10] . However, we could not rule out the presence of residual virus in the conjunctival sac at late stages of the infection, because conjunctival secretion specimens are hard to obtain, quantities are little also, which results in the relatively high specificity and low sensitivity of the polymerase chain reaction assays method [13] . The initial progression of conjunctivitis might have been related to the proliferation rate of SARS-CoV-2 in the conjunctival sac. SARS-CoV-2 infection requires binding to the angiotensin converting enzyme 2 (ACE2) receptor [14, 15] . ACE2 is widely expressed by a variety of cells in the human body, and its expression is highest in pulmonary capillary endothelial cells. Another study has also reported ACE2 expression in the human conjunctiva [16] , suggesting that SARS-CoV-2 might exist in conjunctival secretions. In our patient, conjunctival secretion specimens were repeatedly obtained for virus isolation and culture, but the results were negative, indicating that SARS-CoV-2 might not proliferate in the conjunctiva and cause conjunctivitis through direct invasion and destruction. Symptoms of viral keratoconjunctivitis re-appeared in both eyes approximately 5 days after the symptoms in the left eye were satisfactorily relieved. Symptoms were similar to those at early stages of infection, except that disease progressed more rapidly and corneal involvement at the periphery of the cornea was obvious, which was indicative of reactive disease. The results of nucleic acid tests for related viruses in the conjunctival sac were all negative, although these findings might have been caused by a history of topical antiviral drug use. However, the levels of a variety inflammatory factors increased significantly. Therefore, we speculate that the disease was not caused by the direct invasion and destruction of SARS-CoV-2, but rather \"topical cytokine surge\" caused by the autoimmune response. Cytokine and chemokine responses generated against viruses play important roles in mediating crosstalk among the various immune cells and in connecting innate and adaptive immunity. Upon infection, cytokine levels are elevated. Subsequently, inducible inflammatory chemokines direct effector leukocytes to specific tissues, which can upset the immune regulatory network through the constant self-amplification of the positive feedback loop, thereby resulting in cytokine surge [17, 18] . A recent study has reported that the levels of inflammatory factors, such as G-CSF, TNF-, IL-6, and IP-10, are significantly higher in severely ill COVID-19 patients than those in less severely ill patients, indicating that cytokine surge is involved in the pathogenesis of the disease [12] . In infectious immune corneal injuries, bilateral peripheral ulcerative keratitis caused by the herpes simplex virus [19] and bilateral disciform keratitis complicated by adenoviral conjunctivitis [20] due to the induction of inflammatory reactions or immunological responses in the corneal stroma by immune-degrading complexes have also been reported. However, these delayed corneal infiltrations display different clinical signs, and therefore, they are often misdiagnosed. In this patient, the clinical signs of peripheral infiltration with epithelial involvement were similar to those of viral infection; however, this may have been caused by the presence of different immune cells within tissues [21] . Based on the findings of the aforementioned studies, we conclude that viral infections of the eyes, together with increased levels of serum cytokines, tend to associate with corneal infiltration with epithelial involvement or ulceration, resulting in the recruitment of NK cells. On the other hand, viral infections of the eyes tend to associate with corneal infiltration with stromal involvement, CD4+ T cell reactivity [22] , and antigen deposition [23] . In this patient, the serum IL-6 level was two-fold higher than the normal reference range 4 days after hospital discharge. This pro-inflammatory cytokine can diffuse into the conjunctiva, which is rich in blood vessels. In addition, SARS-CoV-2-infected epithelial cells of the conjunctiva may express inflammatory chemokines, thereby recruiting more inflammatory cells and producing more pro-inflammatory cytokines [24] . As a result, the IL-6 level in the left eye was much higher than that in the right eye, and the IL-6 level in the right eye was similar to that in the blood. Therefore, we conclude that conjunctivitis or keratoconjunctivitis caused by SARS-CoV-2, especially the symptoms at late stages (7 days or later) of the infection, was not due to the local invasion and destruction of the virus; instead, it was due to cytokine surge. The extensive pro-inflammatory cytokine and chemokine response caused by the virus in epithelial cells of the conjunctiva and corneal limbus eventually resulted in apoptosis of epithelial cells of the peripheral cornea and reactive inflammatory injury rather than apoptosis of epithelial cells of the central cornea. This might explain why the early stages of the infection involved only one eye, whereas cytokine surge involved both eyes. According to the treatment guidelines of epithelial keratitis, glucocorticoids are not usually recommended for punctiform viral epithelial keratitis, because they are believed to promote viral replication. However, in this case, cytokine surge, rather than viral invasion, played a major role. Therefore, glucocorticoid could successfully control symptoms within 5 days, with no recurrence, indicating that hormone therapy was effective, although the symptoms involves corneal epithelial defects. In addition, our findings indicated that cytokine surge might play an important role in the delayed aggravation. In cases of severe corneal inflammation, it might be feasible to inhibit IL-6 production [25] , because this cytokine has important roles in inflammation. Our findings indicated that SARS-CoV-2 could induce viral conjunctivitis through an exogenous mechanism. At early stages of the disease, local invasion and inflammation contributed to the localized nature of the disease, and the disease course did not exceed 1 week. At late stages of the disease, keratoconjunctivitis was aggravated and more widespread, which was related to cytokine surge. Therefore, we conclude that aggressive medicinal intervention, longer and more frequent follow-up, and proper use of topical glucocorticoids can diminish cytokine surge. ',\n",
       "   '_search_score': 0.7250315999999999},\n",
       "  {'_id': '0022361a636d679fc3493cc0b20bb6127bf63a6a',\n",
       "   'insert_date_': '2020-11-04T18:28:56.299471',\n",
       "   'abstract': 'The novel Coronavirus disease has increased rapidly in the Wuhan city of China in December 2019. This fatal virus has spread across the whole world like a fire in different stages and affecting millions of population and thousands of deaths worldwide. Therefore, it is essential to classify the infected people, so that they can take the precaution in the earlier stages. Also, due to the increasing cases spread of Coronavirus, there are only limited numbers of polymerase change reaction kits available in the hospitals for testing Coronavirus patients. That why it is extremely important to develop artificial intelligence-based automatic diagnostic tools to classify the Coronavirus outbreak. The objective of this paper is to know the novel disease epidemiology, major prevention from spreading of Coronavirus Severe Acute Respiratory Syndrome, and to assess the machine and deep learning-based architectures performance that is proposed in the present year for classification of Coronavirus images such as, X-Ray and computed tomography. Specifically, advanced deep learning-based algorithms known as the Convolutional neural network, which plays a great effect on extracting highly essential features, mostly in terms of medical images. This technique, with using CT and X-Ray image scans, has been adopted in most of the recently published articles on the Coronavirus with remarkable results. Furthermore, according to this paper, this can be noted and said that deep learning technology has potential clinical applications. [copyright information to be updated in production process]',\n",
       "   'title': 'Journal Pre-proofs Review on Machine and Deep Learning Models for the Detection and Predic- tion of Coronavirus Review on Machine and Deep Learning Models for the Detection and Prediction of Coronavirus',\n",
       "   'body': 'Viruses are a submicroscopic agent, which are made up of genetic substances inside of a protein coating and it can be found anywhere such as air, water, and soil. A virus can cause different infectious types of diseases such as common flu, cold, and warts. They also cause severe illnesses such as Acquired Immunodeficiency Syndrome (AIDS), Ebola, Smallpox [1] , etc. Viruses are akin to hijackers; they enter normal cells and from those cells to replicate themselves. This process destroys or damages the cells and makes us sick, and this kind of virus is called \"Pandemic virus\". Various viruses attack and harm certain cells of the human body such as our respiratory system, liver, or blood. When we get a virus, it will not always affect us to get ill from the virus. the immune system of us may be capable to fight with it. For most epidemiologic infections, the only method to help is the treatment of symptoms as you should wait for the immune system to confront against or fight off the virus. For viral infections, the antibiotics cannot work, there are antiviral drugs and medicines to cure or heal some of the viral infections. The vaccines help to prevent us from getting various viral diseases. A pandemic is an occurrence of a disease or wide-scale outbreak of any infectious disease, which can rapidly increase mortality and morbidity rate on a large-scale, over a large area. Even crossing the international boundaries, which can disturb a large population of the world and that can cause a significant social, economic, and political disturbance. Previous studies indicate that the chances of pandemics have risen in the past centuries [2] . This pandemic outbreak can happen when a novel virus turns out to be capable of circulating or spreading rapidly globally, wherein an epidemic can be in a specific region or city. In one and other cases, there is a serious illness that can rapidly spread out from one to another individual quickly. Evidence shows that the death rate of the epidemic is generally lesser than a pandemic outbreak. For example, the unfavorable pandemic in the history that caused the death of more than a hundred million people, was the Spanish flu [3] . Coronavirus has rapidly spread to almost all countries of the world to date. Coronavirus is a big family of viruses, which causes illness or sickness starting from the very common cold to very more acute or severe disease. SARS-CoV-2 showed about 80% identity of SARS-CoV-1 and 50% to the Middle East Respiratory Syndrome (MARS-CoV) from the genetic sequence and both have their origin from bats. According to the phylogenetic reports and genetic sequence, Coronavirus is adequately similar to SARS-CoV. Most likely this disease has spread from bats, as proof supports that this virus has a high grade of homology of ACE2 (Angiotensin Converting Enzyme 2) receptor from a variety of animal species [4] . The common sign of the Coronavirus infection consists of high and consistent fever, Persistent dry cough, respiratory syndromes such as breathing difficulties, and breath shortness [5] . The Coronavirus infection has spread like a fire and transitioned into a globally pandemic in which, as if now, know medical researchers found any therapeutic vaccine or drug. Because of this serious issue, it is very significant to identify the diseases at its early stage, and quickly isolate people who are infected from the non-affected population. The Coronavirus should be the starting of an exciting time or decade in science and medicine because with the development and improvement of numerous digital technologies they can be applied and used to tackle different diseases and major clinical problems. These are the technologies that include Big-data analytics, IoT (Internet of things), Blockchain, AI which uses machine learning (ML) and deep learning (DL), etc. These are the advanced technologies that are being used in many aspects of healthcare, especially AI and machine learning systems to predict the outcomes and understanding healthcare trends [6] and how AI-based technology and deep learning can help and enhance in the diagnosis of this Coronavirus? By using a different kind of radiographical images. CT and X-ray scans of the chest can be used in diagnosing the pneumonia. There are AI-based automated CT or X-ray images analysis techniques for the detecting, monitoring, and quantifying of Coronavirus, and to find out the patients from the healthy person have developed [7]. As of 14 April 2020, the total number of confirmed cases is 1 776 867, the confirmed number of deaths is 111,828 and the overall number of countries affected by this virus is 213 as reported by WHO. The USA, China, Italy, Spain are the most affected countries. Initially, in European countries, the very fast and worst affected country was Italy, where the public health departments, hospitals, emergency medical systems are struggling to deal with the surge of affected patients. The fatality rate of the Coronavirus was increasing in the outbreak region of China. The first 17 deaths reported by the National Health Commission of China was on 22nd January 2020 and 56 deaths on 25th January 2020 [4] . From the initial case in Dec 2019 to the rise of new incidents outside the city of Wuhan by 13th January 2020, 41 number of cases or incidents were confirmed. The epidemiologic analysis has shown which already in this early stage, human-to-human space and transmission have been happened by close interaction. On 13 January 2020, in Thailand, the first case which was outside China had been reported. This was caused by ax resident of Wuhan who traveled to Thailand, and 19 cases were reported in January from outside Wuhan City, in Beijing City [8] . Furthermore, 85.8% of 37 269 confirmed cases had either lived in or traveled to Wuhan or had close contact with persons who had been to Wuhan. The Figure 1 depicted above shows the outbreak of the disease till 9th April [8] . The Coronavirus varies widely, from the range of infection asymptomatic to severe condition and serious pneumonia along with the high number of fatality rates. The main Disease Control Centers of the Chinese reported that the majority of the number of infected people were classified as a symptom of mild to moderate condition, 13.8% as severe condition, and only 4.7% classified as critical condition. The total rate of mortality for Coronavirus confirmed cases were detected higher in males as compared to the female patients and high risk of death in both cases with rising age. The maximum fatality rate of the aged started from the 80 and above [10] .  It was very important to control the epidemic and lift the quarantine, but only if specific measures could have been taken as much as possible usage of facial masks and restricting public contact is the most effected precaution that people can take until the vaccine is not available. Apart from these, WHO announced other very significant measures to be taken by everyone such as frequent hand washing, use of disinfectants, avoid from touching the eye, face, nose and mouth, use gloves to name a few which are adopted worldwide to reduce the probability of 2nd wave of this pandemic. In the recent report of WHO, 78% of infected cases were because of household spread or transmission and in the recent clinical report, 5% of hospital-acquired infections from patients and medical staff. Until the vaccine is made available, the studies recommended that keeping a reduction in person to person contact (less than the pre-quarantine level) is very important to contain the spread of the virus. The study has shown that from the previous experience, Coronavirus symptoms are mainly highlighted by checking the travel history rather than radiography of chest. The treatment and detection for the isolation of Coronavirus patient cases at an early stage are very important. All hard works are being made to decrease the fast spreading of the disease and to provide time for better preparing of healthcare systems and also to the general public. It is important to clearly characterize Coronavirus to guide and help general public health recommendations and to build vaccines, therapeutics, and timely diagnostics. Lastly, with the improvement and advancement of internet communication, it increases the availability and broadcasting of knowledge. The Internet has the possibilities in terms of development and the spreading of fake news or misinformation. So, the government sector must be responsible for providing and delivering the accurate information and clarifying misinformation to help the general public to face this Coronavirus infection. The below figure depicts the percentage of the Coronavirus symptoms.  In this article, we have selected publications for review of literature that have assessed machine and deep learning algorithms applied in medical images to solve a clinical problem and compared each algorithm regarding their output and performance. Machine learning is the branch of AI which is based on how the system can learn from previous data, recognize patterns, and to make decisions with minimum human intervention, for example, algorithms includes Support Vector Machine (SVM), logistic regression, clustering, etc. Deep learning (DL) is a subset of machine learning, and in terms of medical images, it can be defined as a computational model [20, 21] which is composed of several processing layers to learn data representation and extract the features with multiple abstraction levels. An example of a deep learning type is CNN [11] . In [12] has proposed a deep learning-based technique called deep transfer learning, which can predict patients with Coronavirus disease automatically. It uses images of chest X-ray obtained from patients with Coronavirus and from a healthy person. Dataset of 50 patients, with Coronavirus, images of X-Ray were taken from a shared GitHub repository and 50 X-ray images of healthy humans have taken from a repository in Kaggle. In this study, results showed that the pre-trained model, ResNet50, yielded 98% accuracy among the other three models. They have also stated that we believe in our findings and this can help the public health care workers to make decision in clinical practice because of its high performance and accuracy. Also, for patients with Coronavirus, early prediction of the infection can avoid the rapid spreading of the disease. This paper study [13] presented automated techniques are used for classifying the X-Ray of chest into pneumonia and the class of disease-free by using nine architectures of deep learning, which includes the following, Experiments have been conducted using a CT scan and X-Ray dataset that includes 5856 images with 1583 normal and 4273 pneumonia, and the performance was evaluated using different performance metrics. The result shows that the MobileNet_V2, Inception_Resnet_V2, and Resnet50 have given more accurate results, which is more than 96%. In addition, they have suggested that with the bigger datasets and more sophisticated techniques of feature extraction based upon deep learning image segmentation may improve the performance. In [14] established a deep learning paradigm for the screening of Coronavirus patients at an early stage. The main aim of this paper is to distinguish Coronavirus from Influenza-A viral pneumonia and normal cases with the use of CT images. CT samples have taken from three hospitals designated to Coronavirus from China, Zhejiang Province. The total number of 618 samples was collected, which includes 219 from 110 Coronavirus patients, 224 samples of CT from patients with the viral pneumonia Influenza-A and CT samples of 175 healthy people. The experiments result of this study shown an overall 86.7% accuracy from the viewpoint of CT cases as a whole. They demonstrated that it can be a promising accompanying diagnostic tool for the clinical frontline doctors. As in [15] introduced a new approach of deep learning called COVIDX-Net to help radiologists to automatically diagnose patients with Coronavirus using X-ray images. This technique is built on seven Deep CNN classifiers, which consists of DenseNet121, VGG19, ResNetV2, InceptionResNetV2, Xception, InceptionV3 and MobileNetV2. In this model, they have successfully experimented and evaluated the result which is based on 80% training phase, 20% testing phase of X-ray images. The VGG19 and the Dense CNN (DenseNet) models have shown a better and similar functioning of automated classification of Coronavirus with 0.89 f1-scores for normal and 0.91 for Coronavirus, respectively. X-ray images of the public dataset used in this work, which consists of 50 images, divided into two classes as normal cases of 25 and positive Coronavirus images of 25. Another paper implements an automated detection system of Coronavirus diseases using Deep learning Convolutional neural network and chest X-ray images. In this paper, they have implemented three different CNN based models such as ResNet50, Inception-ResNetV2, and InceptionV3 for the identification of Coronavirus cases using X-ray images. The dataset of 50 Coronavirus patients has been taken from the open-source GitHub repository (Dr. Joseph Cohen) and 50 images of normal chest X-rays have been selected from a repository in Kaggle (Chest X-Ray images). The overall performance out of three models shows that the best results obtained from the pre-trained model of ResNet50 with an accuracy rate of 98% and the other two models accuracy rate from Inception-ResNetV2 and InceptionV3 are 87% and 97%. In future work, they have mentioned that the different CNN models classification performance can be tested by adding the greater number of images in the dataset [7] . In this study [16] they present earlier detection of Coronavirus using CT Scan images by using machine learning methods and the dataset is taken from \"Societa-Italiana di Radiologia Medica-e Interventistica\", which belongs to the 53 Coronavirus cases and data involving CT abdominal images of 150 and 150 CT images. Then, patch regions of the images were cropped, and four different subsets of the patch were created. The feature extraction techniques used in this study are the following, \\uf0b7 Grey -Level Co-Occurrence Matrix, \\uf0b7 Local Directional Pattern \\uf0b7 Grey-Level Size-Zone Matrix \\uf0b7 Grey-Level Run-Length Matrix \\uf0b7 Last-Discrete Wavelet Transform The SVM classifier was applied to classify the extracted features and the best classification outcomes were obtained from GLSZM future extraction techniques with an accuracy rate of 99.68%. In future work more classification and segmentation, studies should be done on Coronavirus disease by increasing the dataset. As in [17] used different deep learning-based techniques for Coronavirus detection diseases by using X-ray images. Different classification models, based on SVM, using deep features of various deep learning architectures as follow, \\uf0b7 XceptionNet are used for detecting the Coronavirus patients. For experimental work they have been conducting, they use an X-ray images dataset that includes 50 images with 25 Coronavirus cases from GitHub (Dr. Joseph Cohen) and 25 normal cases from Kaggle (X-ray Images of Pneumonia). The results show that SVM + ResNet50 obtained high accuracy (FPR=95.52%, F1 score= 95.52%, MCC = 91.41% and Kappa= 90.76%) for detecting the Coronavirus patients as compare to the other models. Various ML and DL-based algorithms and techniques used for classification of the novel disease known as Coronavirus 2019 have been studied and reviewed. Different papers are studied in which the majority of the related paper used different deep learning architecture. According to the literature review, it is demonstrated that deep learning with convolutional neural networks might have remarkable impacts on the automatic detection and automatic extraction of highly essential features from chest images which is related to the diagnosing of Coronavirus. The below figures 3, 4 demonstrated the accuracy of different models using CT, X-Ray images used for the detection of this virus. As in the following table on the next page, we have categorized each paper with their what technique, data modality, data source they have used, and the result they obtained, respectively with the given references.  For the treatment of Coronavirus patients, there is no effective drug present until now and due to the rapid increase in the number of cases of Coronavirus patient\\'s effectual medicinal approach is urgently needed to treat the patients worldwide [18] . Earlier detection and prediction of Coronavirus cases can decrease the spread of diseases [7] . Worldwide patients of Coronavirus data will be a great source and useful for the researchers working on AI and ML to develop an automatic diagnostic tool, therapeutic strategy against Coronavirus patients, and for the same type of pandemics for the coming future [18, 19] . In this study, we have summarized different AI-based approaches and the current situation of spreading Coronavirus. According to the results and studies reviewed, it is indicated that using convolutional neural networks, Due to the early stage of Coronavirus, there are still some limitations of the related studies that can be conquered or overcome in the future researches. Specifically, an in-depth analysis requires a lot more patient data, particularly those patients suffering from the novel Coronavirus. So, the main point of focus for research work in the future could be distinguishing the patients indicating mild symptoms, instead of pneumonia symptoms, whereas these symptoms might not be visualized on X-rays more accurately, or even not at all to be visualized. As more authentic dataset will be available in the near future, a more accurate AI-based prediction model can be formed. We are working in this direction to procure a more authentic dataset as on today after reviewing all the papers mentioned in Table 1 , we are working to develop CNN based deep learning methods for the prediction of Coronavirus patients. AI-based approaches are very useful for the automatic detection of Coronavirus patients using X-ray and CT images. The important point is to increase the number of datasets for Coronavirus patients and using advance deep learning algorithms to achieve better performance for the detection and prediction of Coronavirus. Also, in spite of the fact that proper treatment or cure cannot be determined only from an X-Ray or CT image, these techniques would be useful as an initial screening of the patients. Appropriate use of quarantine actions should be implemented on the positive samples, until a specific treatment and a complete examination procedure is followed. ',\n",
       "   '_search_score': 0.7245579000000002},\n",
       "  {'_id': 'a1949f80e726d045fe1cccb24a0e911cc0b912b9',\n",
       "   'insert_date_': '2020-11-04T18:49:30.521226',\n",
       "   'abstract': 'Porcine epidemic diarrhea virus (PEDV) and porcine deltacoronavirus (PDCoV) are emerged and reemerging viruses in pigs, and together with transmissible gastroenteritis virus (TGEV), pose significant economic concerns to the swine industry. These viruses infect epithelial cells of the small intestine and cause watery diarrhea, dehydration, and a high mortality in neonatal piglets. Type I interferons (IFN-/) are major antiviral cytokines forming host innate immunity, and in turn, these enteric coronaviruses have evolved to modulate the host innate immune signaling during infection. Accumulating evidence however suggests that IFN induction and signaling in the intestinal epithelial cells differ from other epithelial cells, largely due to distinct features of the gut epithelial mucosal surface and commensal microflora, and it appears that type III interferon (IFN-) plays a key role to maintain the antiviral state in the gut. This review describes the recent understanding on the immune evasion strategies of porcine enteric coronaviruses and the role of different types of IFNs for intestinal antiviral innate immunity.',\n",
       "   'title': 'Immune evasion of porcine enteric coronaviruses and viral modulation of antiviral innate signaling',\n",
       "   'body': \"Coronavirus enteritis is a highly contagious viral disease in pigs characterized by severe diarrhea, vomiting, and dehydration with a high mortality, especially in neonatal piglets less than 2 weeks of age. Transmissible gastroenteritis virus (TGEV) is a frequent cause for endemic and epidemic viral enteritis in neonates and older pigs (Kim et al., 2000; Laude et al., 1993) , and porcine epidemic diarrhea virus (PEDV) has become a major concern in the swine industry in the US since its emergence in April 2013. As with TGEV, PEDV causes severe enteritis in newborn piglets but the mortality can be much higher and may reach up to 100% during epidemics (Debouck and Pensaert, 1980; Song and Park, 2012) . PED was first described in the UK in the early 1970s and has since become endemic in some parts of Europe. Highly virulent PEDV has emerged in Asia and subsequently in the US (Chen et al., 2014a,b; Marthaler et al., 2013; Mole, 2013; Stevenson et al., 2013) . PEDV has since quickly spread to most states and as of April 2016, 36 states have become endemic for PEDV, raising a significant economic concern (www.aphis.usda. gov/animal-health/secd). PDCoV is a newly emerged swine enteric coronavirus causing enteritis and severe diarrhea in piglets, which was first described in a surveillance study 2012 in Hong Kong, China (Woo et al., 2012) . PDCoV is an additional enteric virus emerged in the US in February 2014 . PDCoV has been identified in the feces and intestinal samples of pigs experiencing severe diarrhea without PEDV and TGEV . Retrospective studies show that PDCoV has been present in the US at least since August 2013 (McCluskey et al., 2016; Sinha et al., 2015) . PDCoV has also spread to at least 20 states of the US (www. aphis.usda.gov/animal-health/secd), Korea, and China Lee and Lee, 2014) . PDCoV infection seems clinically milder with a lower death rate of 30-40% in neonates than typical PEDV infection , but co-infection with PEDV, TGEV, or porcine rotavirus is common and result in a severe form of disease (Marthaler et al., 2014; Wang et al., 2014) . Clinical symptoms by PDCoV are reproducible in both gnotobiotic pigs and conventional pigs and present diarrhea accompanied by severe histopathological lesions such as villi atrophy in the absence of other pathogens Jung et al., 2015c; Ma et al., 2015) . Although the clinical signs and pathological lesions are indistinguishable among PEDV, TGEV, and PDCoV (Jung et al., 2014) , these coronaviruses are antigenically distinct and thus the cross-protection is absent (Lin et al., 2015) . Pathogenesis of these coronaviruses may include destruction of enterocytes and villous atrophy of the intestinal mucosa in the jejunum and ileum (Debouck and Pensaert, 1980; Jung et al., 2014) . Coronaviruses are enveloped viruses containing the genome of single-stranded positive-sense RNA in the family Coronaviridae. The coronavirus genome ranges of 28-30 Kb in length and is currently the largest known viral RNA genome. Coronaviruses are grouped in the Coronavirinae subfamily which is comprised of 4 genera; Alpha-, Beta-, Gamma-and Delta-coronaviruses (http://www.ictvonline. org/virustaxonoym.asp [ICTV, 2014] ). Evolutionary studies suggest that Alphacoronavirus and Betacoronavirus are originated from bats while Gammacoronavirus and Deltacoronavirus are from birds (Woo et al., 2012) . In humans, coronaviruses infect the respiratory tract and cause symptoms ranging from common colds to severe pneumonia and acute respiratory distress. Severe acute respiratory syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome coronavirus (MERS-CoV) belong to Betacoronavirus, and cause lower respiratory tract infections of significant public concerns (Gralinski and Baric, 2015) . Among swine enteric coronaviruses, TGEV and PEDV belong to Alphacoronavirus and share a similar genome organization, whereas PDCoV belongs to Deltacoronavirus (http://www.ictvonline.org/virustaxonomy.asp). The PEDV genome is approximately 28 Kb in length with the 5 -cap and 3 -polyadenylated tail. The genome is arranged with ORF1a-ORF1b-S-ORF3-E-M-N in order with both termini flanked with untranslated regions (UTRs) (Duarte et al., 1993) . ORF1a codes for a large polyprotein pp1a, while ORF1b is expressed as the pp1ab fusion protein via the ribosomal frameshifting. These polyproteins are proteolytically cleaved to 16 nonstructural proteins, nsp1 through nsp16, by the proteinase activity of nsp3 and nsp5. The TGEV genome is similar to that of PEDV and arranged as ORF1a-ORF1b-S-ORF3a-ORF3b-E-M-N-ORF7 in order (Alonso et al., 2002) . For PEDV, ORF3 encodes the sole accessory protein, whereas for TGEV, three accessory proteins are encoded by ORF3a, ORF3b, and ORF7 (Brian and Baric, 2005) . The PDCoV genome is the smallest of the three coronaviruses with 26 Kb in length, and includes two accessory genes of NS6 and NS7 with the gene order of ORF1a-ORF1b-S-E-M-NS6-N-NS7 (Woo et al., 2012) . Notably, PDCoV lacks of the nsp1 gene and thus codes for only 15 nsps in total. Genomic similarities and dissimilarities for -, -, -, and -coronaviruses are illustrated in Fig. 1 . Upon virus infection, a host reacts quickly to invading virus by producing type I interferons (IFN-/) and elicits an antiviral state in infected cells and uninfected neighbor cells. Three different types of IFNs are known: type I (IFN-/), type II (IFN-), and type III (IFN-) IFNs (Table 1) . For humans, type I IFNs contain 13 subtypes of IFN- and a single subtype for IFN-, IFN-, IFN-, and IFN-. For pigs, IFN- is produced from as many as 17 functional genes. For type II IFN, only a single subtype of IFN- is reported for pigs, which is produced largely in immune cells. Type III IFNs include IFN-1 (interleukin 29 ), 2 (IL-28A), 3 (IL-28B), and IFN-4 (Kotenko et al., 2003; Prokunina-Olsson et al., 2013; Sheppard et al., 2003) . Different types of IFNs signal through different receptors. IFN-/ signal through a heterodimeric complex composed of a single chain of IFN- receptors 1 (IFNAR1) and 2 (IFNAR2), which are believed to be expressed on the cell surface of all nucleated cells (Gibbert et al., 2013; Pestka et al., 2004) . IFN- forms a homodimer for signaling through the IFN- receptor (IFNGR) complex, which is composed of dimer of two transmembrane-spanning receptors [IFN- receptors 1 (IFNGR1) and 2 (IFNGR2)] with broad tissue distributions. IFN-signals through heterodimers of interleukin-10 receptor 2 (IL10R2) and IFN-receptor 1 (IFNLR1). While IL-10R2 is widely distributed in different cell types, IFNLR1 is largely restricted to epithelial cells (Sommereyns et al., 2008) , and thus subtypes of IFN-limit their antiviral functions to epithelial cells. All IFNs activate the Janus kinase (JAK)-signal transducer and activator of transcription (STAT) signaling pathway to establish an antiviral state. Since IFNGR1/2 are widely distributed, most cell types are capable of responding to IFN- (Valente et al., 1992) . While the IFN- has pleiotropic effects on immune cells, a direct antiviral activity is rather limited. In contrast, both IFN-/ and IFN-induce a potent antiviral state by regulating the expression of hundreds of interferon stimulating genes (ISGs). Even though IFN-/ and IFN-utilize different receptors for downstream signaling, their induction and antiviral mechanism are similar, and in vivo studies for IFN-show a high degree of redundancy of these two IFN systems in lung epithelial cells (Ank et al., 2008; Mordstein et al., 2008; Mordstein et al., 2010) , raising a question that why two seemingly redundant antiviral systems exist. IFN signaling cascades are summarized in Fig. 3 . Swine enteric viruses are mostly transmitted via the fecal-oral route and a possible aerosol transmission and some of them are highly infectious. TGEV infects and replicates primarily in small intestinal enterocytes and to a lesser extent in the respiratory tract (Kim et al., 2000) . PEDV and PDCoV also infect the small intestinal enterocytes (Jung et al., 2015c; Jung and Saif, 2015) , and thus intestinal epithelial surface is the first line of host defense to prevent enteric viral infections. Due to the distinctive features of the intestinal epithelial mucosal surface, induction and signaling of IFNs are unique. Coronaviruses have a large genome and have evolved to carry redundant mechanisms to counteract the host innate immune response. For SARS-CoV, at least 11 viral proteins have been identified as type I IFN antagonists (Kindler and Thiel, 2014; Shi et al., 2014a; Totura and Baric, 2012) , and a recent study for PEDV shows that at least 10 viral proteins are type I IFN antagonists . By contrast, acute infection of TGEV induces a high level of IFN- in newborn pigs (La Bonnardiere and Laude, 1981) . Despite the variation of genome organization, swine enteric coronaviruses seem to utilize similar but distinct mechanisms for activation and evasion of host innate immunity on the intestinal epithelial surface. A wealth of information is available for IFNs about their mechanisms of induction, signaling cascades, and functions, but many fundamental questions still remain to be answered, especially for their roles for intestinal antiviral innate immunity. The current review discusses the activation and modulation of innate immune responses mediated by swine enteric coronaviruses. More specifically, the mechanisms of viral proteins that modulate host innate antiviral defense and the roles of different types of IFNs in the intestinal innate immunity are described. The host innate immune system utilizes PRRs (patternrecognition receptors) to sense and respond to PAMPs (pathogenassociated molecular patterns) of invading viruses (Kawai and Akira, 2011) . This recognition of the membrane-bound PRRs triggers the activation of IFN induction pathway. Toll-like receptors (TLRs) are among the best-characterized groups of PRRs (Kawai and Akira, 2011) . TLRs constitutes a family of single transmembrane proteins with ectodomains containing the leucine-rich repeats for recognition of PAMPs and a cytosolic TIR (toll/IL-1 receptor) domain as a key domain transducing signals to downstream adaptors including TRIF (TIR-containing adaptor protein inducing IFN-) and MyD88 (myeloid differentiation primary response gene 88) (Kawai and Akira, 2010) . Among TLRs, TLRs 3, 7, 8, 9, and 13 are involved in endosomal nucleic acid sensing. TLR3 is the dsRNA sensor and responds to poly(I:C), a synthetic RNA analog (Kariko et al., 2004; Okahira et al., 2005) . TLR7 and TLR8 recognize ssRNA derived from RNA viruses (Diebold et al., 2004; Heil et al., 2004) , whereas TLR9 senses CpG DNA motifs in the genome of DNA viruses (Vollmer et al., 2004) . TLR13 has recently been identified as the sensor for bacterial 23 S ribosomal RNA (Oldenburg et al., 2012) . The RLR (RIG-I-like receptor) family is responsible for the sensing of cytosolic RNA. The RLR members include RIG-I (activated retinoic acid-inducible gene I), MDA5 (melanoma differentiation gene 5), and LPG2 (laboratory of genetics and physiology 2). RIG-I and MDA5 respond to poly(I:C), with their specificities based on the length of dsRNA. RIG-I senses shorter dsRNA (300 bp) containing a 5 -triphosphate panhandle structure, whereas MDA5 recognizes long dsRNA ( > 4 kb) fragments (Kato et al., 2008; Schlee et al., 2009; Schmidt et al., 2009) . Cytoplasmic DNA sensors have been addressed including STING (stimulator of IFN genes) and cGAS (cyclic GMP-AMP synthase) for cytosolic DNA sensing (Ishikawa and Barber, 2008; Sun et al., 2013; Wu et al., 2013) . For type I IFNs production, activated RIG-I or MDA5 binds to the mitochondrial adaptor protein MAVS/IPS-1 and recruits TRAF3 (TNF receptor-associated factor 3) and TRAF6. TRAF3 activates downstream IKK (IB kinase) related kinases such as TBK1 (TANK-binding kinase 1) and IKK for induction of IRF3/IRF7-dependent type I IFN production (Fitzgerald et al., 2003; Sharma et al., 2003) . TRAF6 leads to TAK1 activation, followed by NF-B activation for type I IFN and other cytokine productions (Rajsbaum and Garcia-Sastre, 2013) . Within the IFN- promoter, there are four regulatory cis elements, namely, the positive regulatory domains (PRDs) I, II, III, and IV. PRD I/III, PRD II, and PRD IV  are binding sites for IRFs, NF-B , and ATF-2/c-Jun (AP-1), respectively (Honda et al., 2006) . The expression of IFN- requires the assembly of these regulatory factors on PRDs to form an enhanceosome complex, which recruits CBP for IFN production (Randall and Goodbourn, 2008) . Following production and secretion, IFNs bind to the cell surface receptors to trigger the activation of JAK-STAT pathway for the IFN-signaling cascade. The three different IFN systems signal through distinct IFN receptors. Type I IFNs and type III IFNs trigger the formation of ISGF3 complex for the production of hundreds of ISGs. In contrast, type II IFN triggers the phosphorylation of STAT1 and subsequent formation of IFN- activation factor (GAF) for antiviral genes expression (Schneider et al., 2014) . Expression of the membrane-bound PRRs is cell typedependent. Specific molecular features of the PRRs and the cell type that recognizes them are two main factors for the specificity in IFN subtype production (Hoffmann et al., 2015) . Nearly all nucleated cells are capable of producing IFN- through activation of IRF3 and NF-B, but IFN- subtypes are primarily produced by leukocytes (Cantell et al., 1981) . Plasmacytoid dendritic cells (pDCs) are the most potent type I IFN producers with up to a hundred to a thousand times more IFN- than other cell types (Siegal et al., 1999) . Activated pDCs migrate to the lymph nodes and potentially influence various cell types through the secreted IFNs. How IFN responses vary during virus infection is still unclear. A recent study shows that different IFN responses depend on the activation of pDCs by interactions with infected cells or virus alone (Frenz et al., 2014) . Levels of total IFN- production in pDCs vary greatly with invading virus, time, and amount of inoculum (Thomas et al., 2014) . Another study shows that pDCs produce a highly uniformed IFN- profile regardless of the PAMP (Szubin et al., 2008) . The ratios between different IFN subtypes may be optimized by evolution in a specific cell type. Similar to IFN-/, IFN-can be induced in different cell types by various viruses. IFN-can be expressed in a variety of cell types of the hematopoietic lineage, which also produces IFN-/ in abundance. Among the nonhematopoietic cells, epithelial cells are potent producers for IFN-. The IFN-response is largely restricted to the epithelium such that lung epithelial cells respond to IFN-/ and IFN-and intestinal epithelial cells respond exclusively to IFN- (Mordstein et al., 2010; Pott et al., 2011; Sommereyns et al., 2008) . Small intestinal epithelial cells (IECs) form a physical barrier to segregate the mucosal immune system and commensal microbial communities in the lumen. The Paneth and goblet cells secrete mucus and antimicrobial proteins to exclude invading bacteria and viruses from the epithelial surface, and renewal of the epithelial cell layer is controlled by the intestinal stem cells in the crypts. Secreted IgA (sIgA) also contributes to the barrier function. The lamina propria is made of stromal cells, B cells, T cells, dendritic cells, and macrophages. Intraepithelial lymphocytes (IELs) are localized between IECs, and these cells constitute a large and highly conserved T cell compartment, which serves through cytolysis of dysregulated IECs and renewal of IECs. Microfold (M) cells are found in the gut-associated lymphoid tissues (GALT) of the Peyer's patches and the mucosa-associated lymphoid tissue (MALT) of other parts of the gastrointestinal tract, and they are specialized for antigen sampling and initiation for mucosal immune responses (Peterson and Artis, 2014) . Unlike the lung epithelium, the intestinal epithelium has to maintain the tolerance to the symbiotic gastrointestinal microflora while mounting an effective immune response during infection. The intestinal epithelial barrier is depicted in Fig. 2 . Table 2 Modulation of innate immune response and signaling by PEDV. Suppression of IFN- induction upon poly(I:C) stimulation and inhibits IRF3 nuclear translocation. Vero E6 cells Xing et al. (2013) Suppression of type I IFN induction and degrades CBP in the nucleus. MARC-145 cells Zhang et al. (2016) Lack of poly(I:C)-mediated IFN- induction and blocking activation of IPS-1 in RIG-I-mediated pathway. Intestinal epithelial cells (IECs) Cao et al. (2015a) Ativation of NF-B pathway through TLR2, TLR3, and TLR9. IECs Cao et al. (2015b) Activation of extracellular signal-regulated kinase (ERK) pathway early during infection. (2015) PEDV infection causes undetectable natural killer (NK) cell activity in suckling piglets compared to weaned pigs. Pigs Annamalai et al. (2015) A marked induction of serum IFN- in suckling pigs at 1 day post infection, but it is declined significantly to the base level at 5 days post-infection. Pigs Annamalai et al. (2015) 3 Type I IFNs (IFN-/) are major cytokines to restrict viral infections. Recent studies however shed a new light on the importance of IFN-for restricting viral infection in the intestinal epithelium. In the rotavirus model, IFN-/ rather play a minor role for restricting viral infections, and instead IFN-play a major role to inhibit viral replication in the intestinal epithelium (Angel et al., 1999; Pott et al., 2011) . IECs strongly respond to IFN-but only marginally respond to IFN-/ in vivo. The effects of IFN-/ and IFN-are compartmentalized with the major effect of IFN-/ on the intestinal lymphocytes and IFN-on the epithelial cells (Pott et al., 2011) . Distinct roles of IFN-/ and IFN-for controlling systemic and local dissemination of rotavirus, respectively, have been suggested. IFN-/ produced by mucosal pDCs can enhance B cell response to promote viral clearance and protection against re-infection (Deal et al., 2013) . For rotavirus, significance of IFN-/ seems to vary depending on the viral strains, host, and synergy with other types of IFNs, but IFN-determines localized intestinal epithelial antiviral defense. A recent finding shows that activated IELs upregulate the IFN-/ and IFN-expression to mount a rapid antiviral state for protection of IECs (Swamy et al., 2015) . Activation of IELs in vivo rapidly provokes ISGs expression through the IFN- and IFNreceptors. Thus, activation of IELs offers an overt means to promote an innate antiviral potential of the intestinal epithelium. Resistance of invading viruses in the gut epithelium relies mainly on IFN-, not IFN-/, which raises a question on the ubiquitous expression of receptors for IFN-/. A recent study shows that mouse IECs minimally express functional type I IFN receptors and thus do not respond to type I IFNs, explaining why the IFN-/ system is unable to compensate the IFN-deficiency during enteric epitheliotropic viral infections . This study shows that virus-infected IECs are potent producers of IFN-, suggesting that the gut mucosa possesses a compartmentalized IFN system in which epithelial cells respond predominantly to IFN-, whereas other cells of the gut rely on IFN-/ for antiviral defense. Timely production of IFN-in IECs drives rapid clearance of intestinal viral infections and limits viral shedding in the feces. The intestinal epithelium responds exclusively to IFN-in a non-redundant fashion to control epitheliotropic viruses (Pott et al., 2011) . For reovirus, IFN-restricts initial viral replication and reduces viral shedding in the feces, whereas IFN-/ are critical for the prevention of systemic infection. Similarly, norovirus infection also requires IFN-to control persistent enteric infection, although IFN-/ restrict the systemic spread . In the gut, the protective effect of the innate immunity, in particular IFN-, is counteracted by intestinal commensal bacteria . PEDV infection causes the decrease of commensal bacteria normally present in the healthy gastrointestinal tract and induces an unbalanced shift of gut microbiota (Koh et al., 2015; Liu et al., 2015b) , suggesting a possible modulation of IFNresponse by PEDV. The border surface of the intestinal epithelium is the major entry site for enteric viruses and specific mechanisms are needed for the fail-safe protection of such multi-layered surfaces. Although IFN-production by IECs is required, it is not sufficient for protection of the epithelium from rotavirus-induced cell damages. The production and release of IL-1 by IECs following rotavirus infection enhances IL-22 production. A recent study identifies a cooperation of IFN-and IL-22 to synergistically curtail mucosal viral infection . The receptors for IL-22 and IFN-are restricted to epithelial cells and simultaneous engagement of both receptors on IECs is required for restriction of rotavirus infection, suggesting a second layer control is necessary . Downregulation of IFNLR1 and interference of the STAT1 signaling may be an important strategy for viral evasion (Sen et al., 2014) . After stimulation with poly(I:C) and infection with reovirus, both IECs and hematopoietic cells in the intestinal epithelium express IFN-strongly and quickly but do not express IFN-/ genes . Thus, mechanisms seem to have evolved to produce IFN-specifically and potently in the intestinal epithelium. Since common pathways are believed to be utilized for induction of both IFN-/ and IFN- (Onoguchi et al., 2007) , the IFN-/ pathway is specifically blocked in IECs by an unknown mechanism. Another study shows that peroxisome-bound MAVS may preferentially activate IFN-expression (Odendall et al., 2014) . Nlrp6 (nucleotide-binding oligomerization domain-like receptor 6) is an intracellular protein specific for tissues and cell-types with the highest expression in IECs (Elinav et al., 2011) . Nlrp6 binds to viral RNA via the RNA helicase Dhx15 as a RNA sensor and triggers MAVS-dependent expression of IFN-/ and IFN-plus ISGs, and thus plays an important role for intestinal antiviral response (Wang et al., 2015b) . Using gene-knockout mice for specific IFN receptors, IFN-has been identified to play an important role in the defense against SARS-CoV and other respiratory viral infections (Mordstein et al., 2010) . Fecal shedding of SARS-CoV can be detected from mice lacking both receptors for IFN-/ and IFN-but not from mice lacking single receptors, suggesting that IFN-controls viral infections in epithelial cells of both respiratory and gastrointestinal tracts. Relative contributions of the various IFN subtypes for restricting enteric coronavirus infections remain elusive. Both TGEV and PEDV can infect and replicate in PAMs in the respiratory tract in vivo (Park and Shin, 2014) . PEDV has also been shown to infect macrophages infiltrated in the lamina propria and modulates immune responses after infection of monocyte-derived DCs (MoDCs) and intestinal DCs Lee et al., 2000) . In summary, IFN-seems to be critical to efficiently restrict enteric coronavirus infections in Fig. 4 . Coronavirus nsp1 is a major IFN antagonist. (A) Degradation of CREB-binding protein (CBP) by PEDV nsp1. Cells were seeded on coverslips and transfected with PEDV nsp1 gene. Porcine reproductive and respiratory syndrome virus (PRRSV) nsp1 is known to degrade CBP in the nucleus and thus included as a control. Cells were stained at 24 h post-transfection for indirect immunofluorescence assay (IFA) using rabbit anti-FLAG polyclonal antibody (red) and mouse anti-CBP monoclonal antibody (green). Nuclei were stained with DAPI (blue). White arrows indicate nsp1-expressing cells. Yellow arrows indicate control cells. (B) PEDV nsp1 does not alter the expression of p300. Cells were grown to 90% confluence and HA-tagged p300 gene was co-transfected with PEDV nsp1 gene or empty vector pXJ41. Cells were stained at 24 h post-transfection for detection of nsp1 (red) using rabbit anti-FLAG polyclonal antibody and for p300 (green) using mouse anti-HA monoclonal antibody. Nuclei were stained with DAPI (blue). (C) Suppression the intestinal epithelial surface, including PEDV, TGEV, and possibly PDCoV infections, and in turn, these viruses may modulate the host IFN-response. It is of great interest to investigate how such enteric coronaviruses regulate the IFN-response in pigs. A fulllength infectious cDNA clone is available for PEDV, and it will be useful to study individual gene functions using replicating mutants in vivo. As such, we will be able to better understand the pathogenic mechanisms of porcine enteric coronaviruses and host responses to such infections (Beall et al., 2016; Jengarn et al., 2015) . For coronaviruses, S protein and its interaction with a receptor on susceptible cells determine the host range and viral tropism (Bosch et al., 2003; Gallagher and Buchmeier, 2001) . Unlike many other coronaviruses, cell culture adaptation of PEDV triggers mutations in the S protein sequence (Lawrence et al., 2014) and requires the supplement of an exogenous protease, typically trypsin, to activate the S protein to induce membrane fusion after binding to the cellular receptor (Hofmann and Wyler, 1988; Park et al., 2011; Wicht et al., 2014) . The trypsin dependency of PEDV in vitro is consistent with the tissue tropism of trypsin-rich small intestine in pigs. Some of cell-adapted PEDV strains lose the trypsindependency (Nam and Lee, 2010; Wicht et al., 2014) . PEDV-infected cells produce distinct cytopathic effects including cell fusion, multinucleated syncytia formation, vacuolation, and detachment due to apoptotic cell death (Kim and Lee, 2014) . A recent study shows that heparan sulfate on the cell-surface may function as an attachment factor for PEDV (Huan et al., 2015) . As with TGEV, PEDV also utilizes aminopeptidase N (APN; CD13) as a functional receptor. APN is a heavily glycosylated transmembrane protein and is expressed by most cells of myeloid origin including monocytes, macrophages, granulocytes, and their hematopoietic precursors (Chen et al., 1996; Mina-Osorio, 2008; Riemann et al., 1999) . It is also abundantly expressed in the brush border of epithelial cells from the small intestine and renal proximal tubules and some other cell membranes. In the small intestine, it plays a role in the final digestion of peptides generated from hydrolysis of proteins by gastric and pancreatic proteases (Kruse et al., 1988) . TGEV utilizes APN as receptor for entry into swine testis (ST) cells, which however cannot render efficient propagation of PEDV (Delmas et al., 1992) . This differentiation may due to the relatively low level expression of APN on ST cells . Over-expression of APN enhances PEDV infection, and an increased APN density enhances the PEDV susceptibility in cultured ST cells Nam and Lee, 2010; Oh et al., 2003) . Porcine intestinal tissues are more sensitive for high-titer virus production for PEDV natural infection, which may due to the abundant expression of APN in porcine intestinal brush border membrane . Collectively, the high level expression of APN at the intestinal barrier correlates with the efficient replication of both TGEV and PEDV, and infectivity of PEDV is more dependent on APN density. Additionally, PEDV also infects cultured cells of pigs, humans, monkeys, and bats, which suggests that PEDV may utilize APN of other species origin or other unidentified receptors or co-receptors for efficient entry (Liu et al., 2015a; Zhang et al., 2016) . Both PEDV and TGEV infect alveolar macrophages besides small intestinal epithelial cells (Laude et al., 1984; Park and Shin, 2014) . PEDV also infect infiltrated macrophages in the lamina propria in vivo (Lee et al., 2000) . Even though the clinical signs of epidemic PEDV and TGEV infections are indistinguishable, it is of great interest to investigate whether the cell and tissue tropism contributes to the differential pathogenesis in vivo. Porcine enteric coronaviruses replicate in IECs and are transmitted via a fecal-oral route. The viral entry and release for TGEV are restricted to the apical surface of polarized epithelial cells (Rossen et al., 1994) . Experimental infection of newborn piglets with TGEV induces a strong and early IFN- production in serum and intestinal secretions (La Bonnardiere and Laude, 1981) . The blocking of the viral receptor still induces IFN- response, indicating that the entry is not needed for IFN induction in leukocytes (Riffault et al., 1997) . Early and strong IFN- production can also be induced in nonimmune cells after exposure to TGEV-infected epithelial cells (Charley and Laude, 1988) . The rapid and massive release of IFN- in serum is mediated by M and E proteins of TGEV (Charley and Laude, 1988; Laude et al., 1992; Riffault et al., 1997) . A specific point mutation in the M protein results in lower IFN- induction as compared to wildtype virus (Splichal et al., 1997) , and the virus-like particles made of M and E proteins induce IFN- production in porcine leukocytes as almost efficiently as fully infectious TGEV (Baudoux et al., 1998) . However, reconstituted virosomes lose the ability to induce IFN- production, indicating that the native envelope structure is essential (Riffault et al., 1997) . Immunohistochemistry analysis of the tissues from TGEV-infected piglets show that IFN--producing cells (IPCs) are almost exclusively detected in the intestinal tissues and mesenteric lymph nodes (MLNs) at an early time of infection such as 6 h and disappeared by 24 h (Riffault et al., 2001) . The majority of IPCs are localized between enterocytes in the epithelial layer, the lamina propria, around the Peyer's patches, and are accumulated in the MLNs (Riffault et al., 2001) . The number of IPCs in spleen or peripheral lymph nodes is limited, and thus circulating IFN- is likely originated from the gut and MLNs triggered by virus (Riffault et al., 2001) . IPCs in pigs are identified as pDCs (Summerfield et al., 2003) . Interestingly, gut-associated IPCs express swine leucocyte antigen II (SLA II) and are distinct from TGEV-positive cells, suggesting that these cells may be the mucosal counterparts of the intestinal pDCs (Riffault et al., 2001) . Contrary to pDCs that produce high levels of IFN- and TNF- after infection, porcine monocytederived DCs do not respond to TGEV (Guzylack-Piriou et al., 2006) . The frequency of intestinal pDCs however is low comparing to other DCs, and thus the limited populations of intestinal pDCs may not be a major subset of antigen-presenting DCs. The production of IFN- by intestinal pDCs in MLNs may flood the T cell area to influence the immune outcome (Charley et al., 2006) . In addition to the production of IFN-/, TGEV induces phosphorylation and nuclear translocation of STAT1 in virus-infected cells and enhances the expression of ISGs . TGEV infection also promotes the p53 signaling to regulate apoptosis, and the viral nucleocapsid protein is responsible for this action and the cell cycle arrest (Ding et al., 2014a; Huang et al., 2013) . IFN- has been shown to inhibit TGEV protein expression without reducing viral RNA in TGEV-infected cells (Jordan and Derbyshire, 1995) . Infection of TGEV upregulates the expression of type I IFN induction by nsp1 of bovine coronavirus (BCoV) and transmissible gastroenteritis coronavirus (TGEV). HeLa cells were seeded in 24-well plates and cotransfected with pIFN--luc along with individual nsp1 gene and pRL-TK at a ratio of 1:1:0.1. PRRSV nsp1 is a known IFN suppressor and was included as a control. At 24 h post-transfection, cells were stimulated with poly(I:C) (0.5 g/ml) for 12 h and the luciferase activities were measured. The reporter experiments were repeated three time, each in triplicate. Asterisks indicate the statistical significance. Statistical analysis was performed by Student's t-test. *, P < 0.05, **, P < 0.01, and ***, P < 0.001. of 2 -5 -oligoadenylate (2 -5 A) synthetase for inhibition of viral replication (Bosworth et al., 1989; Jordan and Derbyshire, 1995) . Swine IFN- has been shown to enhance the protection from TGEV and alleviate clinical signs in piglets when orally co-administered with Salmonella enterica serovar Typhimurium expressing swine IFN- with or without cytokine IL-18 (Kim et al., 2010; Lee et al., 2011) . The viral load is significantly reduced in intestinal segments of piglets when treated with an IFN inducer before challenge, but no reduction in viral yields is seen when simultaneously inoculated with TGEV and IFN-, suggesting that IFN may be more useful prophylactically (Jordan and Derbyshire, 1994) . PEDV is cytolytic with an acute necrosis in IECs, leading to villous atrophy in the small intestine (Jung et al., 2014) . The high mortality of PEDV in piglets is associated with extensive dehydration resulting from severe villous atrophy. The severe dehydration may due to the extensive structural destruction and disorganization of tight junctions and adherens junctions during infection in nursing piglets (Jung et al., 2015b) . The slower turnover of the enterocytes, which is mediated by the regeneration of crypt stem cells in nursing piglets compared to weaned piglets, is probably responsible for the age-dependent resistance to PED (Jung et al., 2015a) . PEDV antigens are mainly observed in villous enterocytes of the small and large intestines. PEDV also infect macrophages infiltrated in the lamina propria in vivo (Lee et al., 2000) . Experimental infection of TGEV shows that the lung is the tract sustaining viral replication and IFN synthesis (La Bonnardiere and Laude, 1983) . TGEV can replicate in porcine alveolar macrophages (PAMs) in vitro and induces a marked synthesis of type I IFNs (Laude et al., 1984) . Similarly, PEDV can also infect and replicate in PAMs in vivo (Park and Shin, 2014) , suggesting an extra-intestinal replication of PEDV. Additionally, PEDV has been shown to induce apoptotic cell death in vitro and in vivo through the caspase-independent mitochondrial apoptosisinducing factor (AIF) pathway (Kim and Lee, 2014) . Another study claims that PEDV may not induce the death of enterocytes in the small intestine through apoptosis pathway in infected pigs . After binding to APN, PEDV is internalized into target cells by direct membrane fusion, and the genomic RNA is released into the cytosol for replication. The sensing of viral nucleic acids by PRRs in the endosome or cytosol initiates the innate immune signaling. Like other coronaviruses, PEDV likely replicates in the double membrane vesicles to hide the viral RNA from sensing by PRRs and to attenuate the innate immune response (Angelini et al., 2013; Snijder et al., 2006) . A recent study shows that TLR2, TLR3, and TLR9 may contribute to NF-B activation in response to PEDV infection in IECs in vitro but does not cause RIG-I activation (Cao et al., 2015b) . TLR2 on the plasma membrane responds to viral glycoproteins on the envelope to induce production of IFNs and inflammatory cytokines, while TLR3 and TLR9 are localized largely in the endosome to sense dsRNA and CpG DNA, respectively (Boehme and Compton, 2004) . This suggests that PEDV may initiate the innate immune response through its surface glycoprotein and viral nucleic acids in the endosomes. PEDV activates the ERK (extracellular signal-regulated kinase) pathway early during infection independently from viral RNA replication . Knockdown or inhibition of ERK suppresses viral transcription, protein expression, and progeny production, which suggest that the activation of ERK pathway is beneficial and required for PEDV replication . Proteomics analysis of PEDV-infected cells shows differential expression of proteins involved in apoptosis, signal transduction, and stress responses , and ERK1 is significantly downregulated during PEDV infection. Thus, the ERK pathway modulation by PEDV is of further inter-est to study. In one study, six different signaling pathways have been shown to be differentially regulated by PEDV, which includes the RLR, Rap1, PI3K-Akt, MAPK, JAK-STAT, and TLR signaling pathways , suggesting the modulation of host innate immunity by PEDV. For PDCoV, little is known about the viral modulation of host immunity. The PDCoV N protein forms non-covalently linked oligomers localized in the nucleolus . The N protein alters the expression of cellular proteins relevant to cell division, metabolism, stress response, protein biosynthesis and transport, cytoskeleton networks, and cell communication . Notably, PDCoV N protein upregulates two members of the heat shock protein 70 (HSP70) family. In two lines of swine cells (LLC-PK1 and ST), which are used to isolate and propagate PDCoV (Jung et al., 2016) , apoptosis is induced in vitro, but no apoptosis is observed in vivo in PDCoV-infected intestinal enterocytes (Jung et al., 2016) . This suggests that a better cell system is needed to mimic in vivo conditions for PDCoV infections, and for this purpose IECs, intestinal enteroids, or organoids may be useful (Finkbeiner et al., 2012; Jung et al., 2016) . PEDV infection in suckling pigs does not alter the natural killer (NK) cell activity compared to weaned pigs that show significantly higher NK cell activities in the blood and ileum (Annamalai et al., 2015) . Interestingly, PEDV infection seems to stimulate a marked induction of serum IFN- in suckling pigs at 1-day post-infection which then declines to the basal level by 5-day post-infection (Annamalai et al., 2015) . In weaned pigs however, PEDV infection causes a slight induction of serum IFN- at 3-day post-infection and a decrease to the basal level at 5-day post-infection. The peak production of serum IFN- coincides with the peak of viral RNA shedding in the feces and serum of suckling pigs or serum in weaned pigs and the onset of diarrhea. PEDV infection differentially modulates inflammatory cytokines in both suckling and weaned pigs and induces a similar level of TNF- and IFN- (Annamalai et al., 2015) . According to this study, PEDV is capable of modulating IFN- production especially in weaned pigs. Since PEDV infection occurs in the small intestine as with TGEV, it is of interest to examine IFN- in the intestinal secretions and how PEDV modulates type I IFN induction in the intestinal pDCs. Intestinal DCs are highly effective antigen-presenting cells distributed beneath the epithelial lining. Unlike TGEV, PEDV infects MoDCs and produces high levels of IL-12 and INF- without alteration of the IL-10 expression . Interaction between PEDV and MoDCs affects the stimulation of T cell response in vitro. PEDV may also enhance the ability of porcine intestinal DCs to sample antigens and activate T-cell proliferation in vivo . How the intestinal pDCs respond to PEDV infection and modulate innate immune signaling remains to be determined. IFN-/ plays an important role in restricting viral infections, and enteric coronaviruses have evolved redundant mechanisms to counteract the host innate immunity for optimal viral adaptation and replication. Even though TGEV activates IFN-/ production, its accessory protein 7 modulates IFN-/ expression by interacting with PP1 and prolongs viral dissemination for increased survival . Accumulating evidence shows that PEDV also has an ability to evade host IFN response. PEDV does not activate IFN- expression even after poly(I:C) stimulation in virusinfected cells (Xing et al., 2013; Zhang et al., 2016) . It appears that PEDV infection inhibits IRF3 nuclear translocation and thus blocks IRF3-mediated IFN induction. Using MARC-145 cells that are IFNcompetent and PEDV-permissive, our laboratory has investigated the innate immune modulatory role of PEDV . In our study, the culture supernatants from PEDV-infected cells do not suppress the growth of VSV-GFP, providing evidence for sup-pression of IFN-/ by PEDV. Further studies show that PEDV has an ability to degrade the CREB-binding protein (CBP) in the nucleus and thus inhibits the enhanceosome assembly resulting in the suppression of IFN transcription . IECs are major target cells of PEDV in pigs, and in these cells, PEDV blocks the activation of MAVS/IPS-1 and IRF3 nuclear translocation in the RIG-I-mediated pathway and do not induce IFN- expression (Cao et al., 2015a) . IPEC-J2 cells are epithelial cells derived from the jejunum of a colostrum-deprived neonatal pig of 12 h of age (Zhao et al., 2014; Dr. A. Blikslager, North Carolina State University) . These cells however appear to be non-permissive for PEDV infection (unpublished data), and thus they are less useful than the initial anticipation for the study of PEDV innate immune evasion. We have individually cloned all coding sequences from PEDV and expressed them to screen for IFN antagonists by luciferase reporter assays. Of 23 viral proteins, 10 proteins are found to suppress IFN induction, which includes both structural proteins and nonstructural proteins: nsp1, nsp3, nsp7, nsp14, nsp15, nsp16, E, M, N, and ORF3 (Zhang et al., 2016) . These proteins antagonize the IFN- and IRF3 activities. Among these, nsp1 is the most potent IFN antagonist, and further studies show that nsp1 does not interfere the IRF3 phosphorylation and nuclear translocation, and rather interrupts the interaction of IRF3 with CREB-binding protein (CBP) and thus inhibits the IFN enhanceosome assembly. PEDV nsp1 triggers CBP degradation in the nucleus in a proteasomedependent manner, which seems to be the key mechanism for nsp1-mediated IFN suppression . PEDV dramatically decreases the level of ubiquitin (Ub)-conjugated proteins in the cytoplasm. Papain-like proteinase 2 (PLP2) in nsp3 of PEDV contains a deubiquitinase (DUB) activity to process the cytosolic K48-and K63-linked polyubiquitin chains (Xing et al., 2013) . PLP2 in nsp3 of both -CoV and -CoV could process both lysine-48and lysine-63-linked polyubiquitin chains of the cytosolic proteins (Chen et al., 2014a,b; Clementz et al., 2010) . SARS-CoV PLP2 reduces the levels of ubiquitinated forms of RIG-I, STING, TRAF3, TBK1, and IRF3 in the STING-TRAF3-TBK1 complex for inhibition of the type I interferon signaling pathway (Chen et al., 2014a,b) . Similarly, PEDV PLP2 interacts with deubiquitinated RIG-I and STING to antagonize IFN response. PLP2 of SARS-CoV specifically deubiquitinates MDM2 (mouse double minute 2 homolog) and degrades p53 to suppress IFN signaling (Yuan et al., 2015) . Similarly, PEDV upregulates MDM2 expression, degrades p53, and significantly inhibits IRF7 expression, suggesting that PEDV PLP2 may also target the p53 pathway for innate immune evasion and inhibit p53-dependent apoptosis (Yuan et al., 2015) . Collectively, PEDV differentially modulates ubiquitination level in the cytoplasm and nucleus for modulation of innate immune responses. PEDV infection leads to activation of the NF-B pathway in IECs (Cao et al., 2015b) . PEDV stimulates nuclear localization of exogenous p65 at 24 h post infection, and increases the nuclear localization of endogenous p65 at 12 h through 48 h post-infection. In another study, PEDV normally induces activation of NF-B and AP-1 following poly(I:C) stimulation at 24 h post-infection (Xing et al., 2013) . These studies are relatively late stage for PEDV infection. PEDV may inhibit NF-B activation at an early stage for optimal survival and then activates it at late stage for inflammation. For porcine reproductive and respiratory syndrome virus (PRRSV), which is a porcine arterivirus of the Nidovirales order, NF-B is activated 30 min after infection (Fu et al., 2012) . Thus, how PEDV inhibits NF-B during an early time of infection is of interest and needs to be explored. For SARS-CoV, the N protein activates NF-B (Liao et al., 2005) whereas the M protein suppresses NF-B, probably through a direct interaction with IKK (Fang et al., 2007) . For PEDV, the E and N proteins cause ER stress leading to activation of the NF-B pathway for up-regulation of IL-8 and Bcl-2 (Xu et al., 2013a; Xu et al., 2013b) , and the M protein of PEDV is reported to induce a cell cycle arrest at the S phase via the cyclin A pathway (Xu et al., 2015) . In other study, the N protein blocks the IFN- production and ISGs expression, leading to suppression of the IRF3 and NF-B activities (Ding et al., 2014b) . The N protein is an RNA-binding protein and shuttles between the cytoplasm and nucleus (Shi et al., 2014b) . Furthermore, N interacts with TBK1 and sequesters its association with IRF3 to block the phosphorylation and nuclear translocation of IRF3 leading to inhibition of IRF3-mediated IFN production (Ding et al., 2014b) . The nsp5 protein of PEDV is a 3C-like proteinase and has been shown as an IFN antagonist. The cysteine proteinase activity of nsp5 cleaves NEMO which is an essential NF-B modulator and disrupts the RIG-I/MDA5 signaling (Wang et al., 2015a) . Modulation of innate immune response and signaling by PEDV infection is summarized in Table 2 . For SARS-CoV, nsp14 is an exoribonuclease and nsp15 is an endoribonuclease, and both function through a specific digestion of dsRNA and subsequent removal of RNA-PAMPs, leading to suppression of IFN responses (Kindler and Thiel, 2014) . SARS-CoV nsp16 is a 2 -O-methlytransferase that modifies the 5 cap structure of viral RNA and functions to avoid detection by the host immune system (Totura and Baric, 2012) . PEDV nsp14, nsp15, and nsp16 correspond to nsp14, nsp15, and nsp16 of SARS-CoV and are presumed to share similar functions for innate immune evasion, which needs to be verified experimentally. The M protein plays an important role in viral assembly and induction of PEDV neutralizing antibodies, and a recent study shows that M is an additional IFN antagonist . Since M protein of TGEV is responsible for abundant expression of IFN-, it is of great interest to determine how the PEDV M protein can suppress IFN response and functions differently from the TGEV M protein. ORF3 is the sole accessory protein of PEDV and its role for pathogenesis remains enigmatic. Some strains of cell-adapted PEDV display internal deletions in the ORF3 gene (Park et al., 2008) , suggesting that ORF3 relates to adaptation and pathogenicity. ORF3 has also been shown to function as an ion channel protein and thus the suppression of ORF3 expression inhibits viral production . Additionally, ORF3 delays the S phase in dividing cell and suppresses the cell cycle progression (Ye et al., 2015) . A contradictory data exists that ORF3 causes an inability to recover progeny virus from a full-length infectious clone, suggesting that ORF3 negatively modulates PEDV replication in vitro (Jengarn et al., 2015) . However, the deletion of entire ORF3 by targeted RNA recombination has been possible and ORF3 appears nonessential for PEDV replication in vitro . ORF3 inhibits type I IFN production and facilitates the vesicle formation in ORF3-expressing cells (Ye et al., 2015) , implicating an important role of ORF3 for innate immune evasion. These viral antagonists may target different signaling pathways to modulate the host innate immunity synergistically and efficiently during infection. The modulation of type I IFN responses by PEDV and the IFN antagonism by PEDV proteins is shown in Fig. 3 . Coronavirus nsp1 is the N-terminal cleavage product of polyproteins pp1a and pp1a/b processed by the papain-like proteinase 1 (PLP1) activity of nsp3 (Ziebuhr, 2005) . The nsp1 gene is one of the most divergent genes among four different genera. Only the genome of -CoV and -CoV codes for nsp1, whereas the genome of -CoV and -CoV lack the nsp1 gene (Woo et al., 2010; Woo et al., 2012; Ziebuhr, 2005; Ziebuhr et al., 2007) . The nsp1 gene of -CoV is much smaller than that of -CoV and the sequence homology is also limited. The PEDV nsp1 gene encodes for 110 amino acids with the predicted molecular migration of 12 kD. The protein sequence of nsp1 does not harbor any known cellular functional motifs (Connor and Roper, 2007) and thus is a genus-specific genetic marker (Snijder et al., 2003) . However, nsp1 of -CoV and -CoV share similar functions. Coronavirus nsp1, especially -CoV nsp1, regulates host cell and viral gene expression (Narayanan et al., 2014) . The nsp1 protein of -CoV and -CoV is a potent IFN antagonist, and among all coronaviruses, SARS-CoV nsp1 is best characterized. SARS-CoV nsp1 inhibits the reporter gene expression under the constitutive promoters as well as the IFN- promoter (Kamitani et al., 2006) . As with SARS-CoV nsp1, murine hepatitis coronavirus (MHV) nsp1 and HCoV-229E nsp1 affect the cellular gene expression by suppressing IFN-, IFN-stimulated response element (ISRE), and SV40 promoter (Zust et al., 2007) . MHV nsp1 facilitates viral replication in mice by suppressing IFN-/ through an unknown mechanism. MHV nsp1 and SARS-CoV nsp1 are distributed in the cytoplasm of virus-infected cells (Brockway et al., 2004; Kamitani et al., 2006) , and SARS-CoV nsp1 suppresses IFN mRNA accumulation without altering the IRF3 dimerization (Kamitani et al., 2006) . SARS-CoV nsp1 blocks the NF-B, IRF3, and IRF7 activities. SARS-CoV nsp1 also inhibits IFN-dependent signaling by interfering STAT1 phosphorylation (Wathelet et al., 2007) . Bat-CoV nsp1 inhibits type I IFNs and ISGs production (Tohya et al., 2009) , HCoV-229E nsp1 and HCoV-NL63 nsp1 also inhibits the IFN-, ISRE, ISG15 activities as well as the SV40, HSV-TK and CMV promoters Zust et al., 2007) . PEDV nsp1 has shown to degrade the CBP in the nucleus to antagonize the type I IFN induction Fig. 4A ). The p300 protein is a transcriptional co-activator protein for CBP, which functions as histone acetyltransferase and plays a central role in coordinating and integrating multiple signal-dependent events for appropriate level of gene transcription. PEDV nsp1 does not alter the expression level for p300 (Fig. 4B) , which further confirm the specificity of degradation of CBP. To examine whether the nsp1 protein of other coronaviruses also possessed an activity for type I IFN suppression, nsp1 genes were cloned from TGEV and bovine coronavirus (BCoV) and examined for an IFN suppressive activity using an IFN- luciferase assay. Similar to PEDV nsp1, TGEV nsp1 and BCoV nsp1 also significantly suppress the IFN- promoter activities (Fig. 4C ). TGEV nsp1 has been shown to inhibit host protein synthesis without binding to 40 S ribosomal subunit (Huang et al., 2011) , and BCoV infection activates host innate defense via TLR7/8-dependent pathways and inhibits type I IFNs and pro-inflammatory cytokines (Aich et al., 2007) . It is of interest to investigate the detailed mechanism for TGEV-nsp1-and BCoV-nsp1-mediated IFN suppression. Two emerging enteric coronaviruses, PEDV and PDCoV, have caused epidemic and endemic infections in pig populations in many countries and have become a major economic threat to the pork industry. The intestinal epithelial surface is the first line of defense to restrict the enteric viral infections, and both IFN-/ and IFNare important antiviral cytokines that viruses have evolved to counteract. For PEDV, at least 10 viral IFN-/ antagonists have been identified in our laboratory. Since enteric coronaviruses infect intestinal epithelial surface, several questions still remain to be answered (Fig. 2) . Firstly, whether PEDV and PDCoV selectively activate and/or modulate IFN-production and whether IFN-plays a protective role on the intestinal surface. Secondly, how different cell types such as IECs, pDCs, and IELs in the intestinal barrier induce different types of IFNs and function for mucosal defense. Thirdly, the intestinal epithelium constantly interacts with commensal bacteria and opportunistic viral infections, and thus IFN systems must be able to deal with these organisms without severe inflammation and tissue damages. Thus, better research models for cells and animals are needed to truly reflect viral evasions from innate immune defense. Lastly, specific cytokine mediators are produced in IECs, and how they respond to porcine enteric viral infections is of an additional interest.\",\n",
       "   '_search_score': 0.7034755000000001},\n",
       "  {'_id': 'a03517f26664be79239bcdf3dbb0966913206a86',\n",
       "   'insert_date_': '2020-11-04T18:47:02.655617',\n",
       "   'abstract': 'Background: Bat-borne virus surveillance is necessary for determining inter-species transmission risks and is important due to the wide-range of bat species which may harbour potential pathogens. This study aimed to monitor coronaviruses (CoVs) and paramyxoviruses (PMVs) in bats roosting in northwest Italian regions. Our investigation was focused on CoVs and PMVs due to their proven ability to switch host and their zoonotic potential. Here we provide the phylogenetic characterization of the highly conserved polymerase gene fragments. Results: Family-wide PCR screenings were used to test 302 bats belonging to 19 different bat species. Thirty-eight animals from 12 locations were confirmed as PCR positive, with an overall detection rate of 12.6% [95% CI: 9.3-16.8]. CoV RNA was found in 36 bats belonging to eight species, while PMV RNA in three Pipistrellus spp. Phylogenetic characterization have been obtained for 15 alpha-CoVs, 5 beta-CoVs and three PMVs; moreover one P. pipistrellus resulted co-infected with both CoV and PMV. A divergent alpha-CoV clade from Myotis nattereri SpA is also described. The compact cluster of beta-CoVs from R. ferrumequinum roosts expands the current viral sequence database, specifically for this species in Europe. To our knowledge this is the first report of CoVs in Plecotus auritus and M. oxygnathus, and of PMVs in P. kuhlii. Conclusions: This study identified alpha and beta-CoVs in new bat species and in previously unsurveyed Italian regions. To our knowledge this represents the first and unique report of PMVs in Italy. The 23 new bat genetic sequences presented will expand the current molecular bat-borne virus databases. Considering the amount of novel bat-borne PMVs associated with the emergence of zoonotic infections in animals and humans in the last years, the definition of viral diversity within European bat species is needed. Performing surveillance studies within a specific geographic area can provide awareness of viral burden where bats roost in close proximity to spillover hosts, and form the basis for the appropriate control measures against potential threats for public health and optimal management of bats and their habitats.',\n",
       "   'title': 'Coronavirus and paramyxovirus in bats from Northwest Italy',\n",
       "   'body': \"Bats (order Chiroptera) represent at least one-fifth of existing mammals, consisting of over 1300 known species of which at least 44 are present in Europe [1] and 34 in Italy [2] . Species diversity is expected to increase as some taxa, i.e. Myotis nattereri complex, are in the processes of being defined as cryptic species using molecular approaches rather than using morphological characteristics [3] . Bats are grouped into two suborders: the fruit-eating megabats (Megachiroptera), or flying foxes consisting of the single family Pteropodidae, and the echolocating insectivorous microbats (Microchiroptera) comprising 16 bat families [4] . Bat borne viruses are arousing increased interest since viral infections in bats have been associated with zoonotic disease outbreaks in humans and domestic animals, including livestock. Rabies virus, Hendra and Nipah viruses, Severe Acute Respiratory Syndrome (SARS) and Middle East Respiratory Syndrome (MERS) coronaviruses, as well as Filoviruses exemplify the role of bats in spreading viruses [5] [6] [7] . In the last fifteen years, at least two widespread outbreaks have been caused by novel coronaviruses jumping the species barrier, SARS in 2002-2003 and MERS starting from the Arabian Peninsula since 2012 [6, 7] . Genetic similarities between the viral sequences detected during outbreaks and CoV sequences in bats suggest the viruses originated in flying mammals and presumably passed to humans through a previous adaptation in intermediate hosts, i.e. civet cats and dromedaries [8] . Coronaviruses (family Coronaviridae, subfamily Coronavirinae) are divided into four main genera: Alphacoronavirus (alpha-CoV) and Betacoronavirus (beta-CoV) found mainly in mammals, Gammacoronavirus detected in birds and marine mammals and Deltacoronavirus found mainly in birds. Several alpha and beta-CoVs have been described worldwide in different bat species (e.g. [9] [10] [11] [12] [13] [14] [15] [16] [17] ). From the first report in China, Rhinolophus species have been specifically associated with SARS-like CoVs [18] [19] [20] , belonging to the lineage b of beta-CoV genus. Further investigations are needed to clarify the origin of all mammalian coronaviruses, assumed to be from viral ancestors residing in bats [21] , untill the recent discovery of a new and highly divergent CoV (i.e. WESV) from house shrews in China [22] . As of 2010, the circulation of CoV in Italian bat population has been notified in only few published studies: SARS-like beta-CoVs have been identified in Rhinolophus species [23] and CoVs sequences are available only for Italian Pipistrellus kuhlii, Hypsugo savii, Nyctalus noctula, Epseticus serotinus, Myotis blythii and R. hipposideros species from fecal samples [24, 25] . Despite the rapid accumulation of bat CoV sequences in the last decade, any viral isolation trial, on different mammalian and bat cell lines failed till 2013, when the first isolation of SARSlike CoV from bat fecal samples succeeded in China [26] . On the list of emerging zoonoses there is a broad diversity of bat-borne paramyxoviruses (PMV), belonging to the wide Paramyxoviridae family, as the emergent Nipah virus and Hendra virus (Henipaviruses) and rubulaviruses (e.g. Menangle virus, Tioman virus and Tuhoko virus 1, 2 and 3) (e.g. [27] [28] [29] and references therein). Detection and isolation of paramyxoviruses from tissues and urine have been obtained mainly from flying foxes of the genus Pteropus in Africa, Asia, and South America (e.g. [27, 30, 31] ) and in Australia (e.g. [32] [33] [34] ), but also microbat species not previously indicated as PMV reservoirs tested positive for PMV RNA in Africa and Europe [27, [35] [36] [37] . Moreover, the ever-increasing attention paid to bat-associated pathogens, has led to the discovery of numerous novel and yet unclassified PMV, revealing an unexpected genetic diversity in the Paramyxovirinae subfamily [36] . PMV identification has been reported in only few studies in insectivorous bats in Europe from Germany, Bulgaria, Romania and Luxembourg, with none of the novel viruses closely related with highly or human pathogenic paramyxoviruses [16, 17, 27, 36] . Following the increasing need of surveillance for bat-borne viruses and the wide range of bat species potentially representing reservoirs for known or unknown pathogens, this study aimed to estimate the viral diversity and distribution in the bat population resident in Northwest Italy. Our investigation was focused on coronaviruses and paramyxoviruses due to their proved ability to switch host and their zoonotic potential. Here we provide the phylogenetic characterization of viral polymerase gene fragments, which are highly conserved within the viral families under investigation. Since all bat species in Europe are protected under the Habitats Directive of the European Union [38] and the Agreement on the Conservation of Populations of European Bats [39] , samples collection and bat species identification were performed by expert chiropterologists authorized by the Italian Ministry of Environment (authorization number DPN/2010/0011879 and 000882/PNM/08052014). Bats were captured, during the three years of surveillance (2013-2016) in the Northwestern Italian regions of Piedmont and Liguria, following ethical and safety recommendations [40] . Samplings were conducted from mid-June to October, a period that approximately corresponds to the pregnancy, lactation, dispersion and mating activity of European bats. To minimize animal disturbance, bats were caught soon after parturition with nylon mist-nets of mesh size of 16 to 19 mm positioned at 10-20 m from the reproductive and temporary roost along flight paths towards foraging and drinking areas. During autumn catches were focused particularly at swarming sites in caves where individuals from different colonies meet to mate [41] . All nets were checked every 10 min and captured bats were removed carefully from nets as soon as possible to minimize injury, drowning, strangulation, or stress and individually placed into disposable cloth bags awaiting species identification, collection of biometric data and biological samples. Species identification was carried out according to Dietz & Kiefer [1] and individual details such as age, class, sex, reproductive status, forearm length, and body mass were recorded. Saliva and urine drops, when present, were collected directly on the animal by swabbing, while feces were recovered, when present, from the cotton bag. All bats were released in the same place of capture after minimal manipulations and were not tagged. Based on the results of the first two years of surveillance, an increase in feces collection was performed in 2016 setting up random, non-invasive feces samplings underneath single-species reproductive roosts. Briefly, plastic films were left on the ground under different areas of each reproductive colony, then 15 min later single fresh droppings were collected with clean disposable forks, placed in 1 ml of buffered peptone water and kept at 4C till analyses. Dead animals in good post-mortem conditions were also collected and stored at 20C for further analyses. Swabs and feces were maintained in 1 ml of UTM Viral Transport Medium (Catalog Number: 360C; Copan Diagnostics, Corona, California) and stored at 20C. Before any further analyses took place, the presence of the rabies virus antigen was investigated on dead animals by direct immunofluorescent staining in a BSL3 Laboratory, after necropsy. Once rabies infection has been excluded, samples underwent a pre-treatment before being submitted to automatic nucleic acid purification with magnetic beads. Pre-treatment for tissues involved the preparation of a tissues pool composed by heart, lung, spleen and intestine from individual animals. The pools were homogenized at a ratio of 1:10 w/V in 1 ml of DEPC-treated PBS in a TissueLyser (Qiagen, Hilden, Germany). Tissue homogenates were then clarified at 13,000g for 10 min at 4C, then 200 l of tissues pool supernatant were incubated at 56C for 10 min with 180 l of ATL buffer and 20 l of Qiagen protease provided by the EZ1 Virus Mini Kit v2.0 (Qiagen, Hilden, Germany).To avoid any biosafety risk, the pre-treatment for swabs (saliva and urine) and feces suspensions involved the direct inactivation of 200 l of each suspension in 200 l of ATL buffer under a BSL3 hood. Nucleic acid purification (RNA/ DNA) was finally accomplished on the EZ1 Advanced XL Instrument using an amount of 400 l as sample input and a final elution volume of 60 l of RNase-DNase free water, following the manufacturer's guidelines. RNA was stored at 80C until amplification protocols were performed. cDNA was synthetized from 5 l of each RNA/DNA sample with the Transcriptor First Strand cDNA Synthesis Kit (Roche Diagnostics, Mannheim, Germany), according to manifacturer's instructions. For coronavirus detection, 2 l of cDNA were amplified with an end-point PCR assay targeting a conserved RNA-dependent RNA polymerase (RdRp) gene fragment (537 bp), as described by Poon et al. [42] . The amplification was set up in a 25 l reaction mixture containing 0.2 mM deoxynucleoside triphosphates, 1.5 mM MgCl2, 0.2 M of IN-6 and IN-7 primer and 1 U of Platinum Taq Polymerase (Invitrogen, Carlsbad, CA). The cycling conditions were 94C for 2 min, 40 cycles at 94C for 1 min, 48C for 1 min, 72C for 1 min and final elongation step at 72C for 7 min.The annealing temperature of primer was modified from 58C to 48C. Upon amplification, 20 l of PCR products were run in 1.5% agarose gel electrophoresis and visualized by GelGreen Nucleic Acid Gel Stain (Biotium) staining; bands of the expected size were excised from the gel for sequencing. For paramyxovirus detection, a broadly reactive seminested PCR assay specific for the RNA polymerase (L)gene (538 bp) of the Paramyxovirinae subfamily was applied. 0 2 l of cDNAs were amplified using the PAR primers designed by Tong et al. [43] and the protocol optimized with Taguchi method by Kurth et al. (36) . Briefly for first round, the final concentration of the 25 l reaction mix was: 0.1 mM deoxynucleoside triphosphates, 10 mM MgCl2, 0.12 M of PAR F1 and PAR R primers and 1.25 U of Platinum Taq Polymerase (Invitrogen, Carlsbad, CA). The cycling conditions were 94C for 2 min, 40 cycles at 94C for 15 s, 50C for 30 s, 72C for 30 s and a final elongation step at 72C for 7 min. Then 1 l first round PCR product was used in the second round with the same concentrations except for the MgCl 2, set up at 1 mM and the use of PAR F2 and PAR R primers, cycling parameters were identical to the first round. PCR products (20 l) were run and recovered from a 1.5% agarose gel, as described before. Amplicons were purified by gel extraction with the QIAquick Gel Extraction kit (Qiagen, Hilden, Germany), according to the manufacturer's instructions. After elution, nucleic acid quantification of the recovered DNA was done using Thermo Scientific Nanodrop spectrofotometer and submitted for direct sequencing to BMR Genomics, Padua, Italy. The obtained chromatograms were manually checked for unclear base calls and edited using Geneious R7.1.7 software (Geneious, Auckland, New Zealand). The sequences were aligned using Muscle (implemented in Geneious software) and the alignment was used to evaluate the best evolutionary model (Modeltest ver 3.7) and to draw a bayesian phylogenetic tree (MrBayes ver. 3.1.2). Consensus tree was created after at least 1 million of heuristic search generations and after eliminating the first 25% of evaluated tree topologies (burnin = 25%). A total genomic DNA extraction was performed only for PCR positive individuals starting from the original swab suspensions using the QIAmp DNA Mini kit (Qiagen, Hilden, Germany) and following the manufacturer protocol. To confirm species identification by genetic determinations, the complete mitochondrial Cytochrome b gene (Cytb) was amplified as in Puechmaille et al. [3] . PCR products were submitted for direct sequencing to BMR Genomics, Padua, Italy. The obtained chromatograms were manually checked for unclear base calls and edited using Geneious R7.1.7 software (Geneious, Auckland, New Zealand). Species identification was conducted by comparing the obtained sequences to on-line available reference sequences (BLAST alignment, NCBI web site). Table 1 . No animal captured during the active surveillance showed signs of disease. During necropsies, no macroscopic lesions referring to infectious diseases were observed, and all the examined bats were negative in the rabies virus antigen IF test. CoV and PMV positive sample types included feces (33/ 158; 21%) and urine swabs (6/49; 12.2%). None of the tissue pools from dead bats or oral swabs were PCR positive. A significantly greater percentage of female bats, 11.5% (9/78), were PCR-positive than males, 4.3% (5/117), (p = 0.05). Coronavirus and/or paramyxovirus RNA was found in 38 animals belonging to eight bat species (Table 1) . Specifically, CoV RNA was detected in 36 bats from 12 sampling sites in Piedmont and one in Liguria, while PMV RNA in three animals from three sampling sites in Piedmont; a map showing the positive sites is presented in Fig. 1 . In our sample set, the detection rate of CoV was 12% (36/302; 95% confidence interval [CI] = 9.6-17) ranging between 3.6% for P. kuhlii, despite representing the most abundant species in our sample, and 47.4% for R. ferrumequinum. Phylogenetic analysis was performed on 20 unique sequences obtained from 36 samples that yielded a PCR product of the expected size after the CoV PCR screening. The positive samples were collected from: M. nattereri (n = 3), M. myotis (n = 2), M. oxygnathus (n = 1), P. kuhlii (n = 1), P. pipistrellus (n = 3), P. auritus (n = 1) and R. ferrumequinum (n = 9). Any new sequences identified were submitted to GenBank and the accession numbers assigned are given in Table 2 . The PMV strains were detected in three different provinces from two P. kuhlii at CN2 and AL10 sites and one P. pipistrellus at VC8 site; moreover, phylogenetic analysis based on the L-gene fragment was possible for all the three strains retrieved in this study. Interestingly, one P. pipistrellus from VC8 site was coinfected by both CoV and PMV as PCR positive results were obtained from the same urine sample. Details of positive sequenced samples are displayed in Table 2 . RdRp phylogeny is presented in Fig. 2 and shows that 15 CoV strains from this study clustered in the alphacoronavirus genus and 5 in the beta-coronavirus genus. As shown in Fig. 2  PMV phylogeny based on representative L-gene sequences available from GenBank is presented in Fig. 3 . The new PMV strains were detected in three different locations from one P. pipistrellus at VC8 site and two P. kuhlii bats at CN2 and AL10 sites (80 km distance). Our 1015 P. pipistrellus strain revealed a 97% nucleotide identity with the E95 PMV strain (AN: JN086951) detected in the same bat species in Germany in 2009, but is more than 23% divergent from any other known PMV sequence. The two PMV sequences from P. kuhlii (600 site CN2 and 621 site AL10) are 97% similar to each other and cluster separately from previously known PMV sequences (18-20% divergence) in the L-gene fragment phylogenetic tree. To our knowledge, the P. kuhlii species was never previously implicated as paramyxovirus host. Recently, emerging disease surveillance programs have intensified to investigate the role of bats in the evolution and spillover of zoonotic pathogens from wildlife. Our study involved three years of active and passive surveillance to characterize the viral diversity of the Northwestern Italian bat population. Using viral family-wide PCRs we identified and phylogenetically characterized 20 new CoVs and 3 PMVs strains. To date, studies on bat CoVs phylogeny are mainly based on datasets of short sequences (i.e. 440 bp) (e.g. [9, 10, 13, 14, 24, 25, 44] ) due to the difficulties of obtaining isolates and good quality viral RNA from bats, but ideally long sequence fragments would be beneficial to infer more reliable phylogenies. The high prevalence of positive fecal samples (21%) in our study is in concordance with other studies, which identified feces as the best sample type for CoVs detection in bats [9, 18] . Rather than collecting samples from individually caught bats, which is time consuming and labor intensive, collecting single fecal droppings under mono-species roosts turned out to be a reliable and non-invasive method for virological surveillance of bat roosts during their reproductive period. Moreover, urine is confirmed as the most suitable and appropriate sample types for detection of paramyxoviruses in bat populations [33] , considering that 2 out of the 3 PMV positive samples from our study were urine swabs. In 2013-2014 coronavirus circulation was identified in at least four species-specific reproductive roosts of Piedmont: TO7 site for M. nattereri, VC8 site for P. pipistrellus, CN2 and AL10 for P. kuhlii. Unfortunately, attempts to re-test the same roosts in 2016 failed since the VC8 colony moved due to the effect of human disturbance (i.e. robbery of the copper roof cover used as refuge by P. pipistrellus bats), and the other three colonies located in private buildings were inaccessible due to logistical reasons. The likelihood of roost disturbance should be taken into account when putting in place bat surveillance plans to enable a steady follow up of the colonies over time. Bats social behavior could explain the significantly higher infection rate detected in our study for female bats all sampled in August near maternity roosts. Previous studies documented higher virus detection rates in females and juveniles captured near maternity roosts in summer, supporting the hypothesis that virus amplification occurs mainly in reproductive roosts [11, 45] . The identification of the same CoV strains (100% identical) in different roosts of the same bat species (i.e. R. ferrumequinum and P. pipistrellus) located also at over 100 km distance, seems to confirm that most bat-CoVs appear species-specific and thus more closely associate with the host species than the sampling location [11, 15, 20] . Interestingly, we identified a divergent alpha-CoV lineage in M. nattereri SpA, representing a cryptic lineage within the Myotis nattereri species complex in the Mediterranean region. The lineage is known to be present in Italy, however no information is available for Germany and Hungary [46] . Following the host-virus coevolution theory based on their close phylogenetic concordance [47] , the small divergence (from 3.5 to 5%) between our M. nattereri SpA CoV strains and the German or Hungarian M. nattereri ones could indicate that they all reside in the M. nattereri SpA host, considering that molecular species identification for those specimens is lacking. The detection of identical alpha-CoV sequences in two different species belonging to the Myotis genus (M. oxygnathus and M. myotis) from two distinct roosts (sites SV12 and CN4) 90 km apart could be due to the expansion and overlapping of habitats and foraging areas of Myotis spp. through the Maritime Alpine chain and valleys. To our knowledge this is the first report of CoV in the M. oxygnathus species. The compact cluster of almost identical beta-CoV (lineage b) strains from two separate R. ferrumequinum roosts gives further indications that the Rhinolophus genus may represent the specific host for SARS-like CoVs and gives an important contribution in terms of available beta-CoV sequences from this species in Europe. To our knowledge, this is the first report of CoV in the P. auritus species. This sequence clusters separately within the beta-CoV (lineage c) showing a 14% divergence with a MERS strain identified from a camel in Egypt. The detection of highly divergent alpha-CoV strains within one P. pipistrellus reproductive roost, the circulation of both alpha and beta-CoVs within one R. ferrumequinum roost and the co-infection of P. pipistrellus with both CoV and PMV provide further evidence that bats are able to carry more than one virus. While infection with multiple CoVs in the same species/bat/colony is well known, and has been previously reported in China [26, 48, 49] and Europe [19] , apart from metagenomic studies notably biased towards the identification of sequences from dsDNA viruses, to our knowledge the coinfection of different ssRNA viral families in the same animal was so far reported only in one study in Europe from P. pygmaeus in Hungary [44] . In the specific, the coinfection with two ssRNA viral families within the same host may be explicable in the light of the IFN inhibition used by paramyxoviruses to circumvent host' innate immune response [50] . This mechanism, known as IFN antagonism, may be exploited by other viruses able to escape the adaptive immunity, e.g. CoVs, to be introduced and proliferate in the same host, as observed in mallards [51] . By the increased viral surveillance, a considerable number of novel paramyxoviruses has been discovered in pteropoid and non-pteropoid species, but to date the number of bat PMV sequences for Europe is very scarce and only from few bat species [16, 17, 27, 36] . The three new PMV strains, two in P. kuhlii and one in P. pipistrellus species, couldn't be classified within any of the current seven known PMV genera, but cluster in the crowded, unassigned PMV clade, which comprises several bat derived strains. Our report represents the first identification of PMVs in the P. kuhlii species worldwide. The two sequences, retrieved from two roosts located 90 km apart, are divergent from previously known PMV clusters, which may indicate a stronger association to the host species rather than the geographic area also for paramyxoviruses. This viral tropism is also strongly supported by the high similarity of our P. pipistrellus sequence to that of the one other E95 PMV sequence retrieved in Germany from the same bat species. In support of this hypothesis, a study on renal tissues from African bats underlined how paramyxovirus divergence in pteroid and non-pteroid bats correlates with bat taxonomy, suggesting a strong association with bat genera [37] . Because the L-gene fragment used as genetic marker in the aforementioned study is not overlapping with the sequence we used, we couldn't phylogenetically compare them. Nevertheless, given the high similarity our P. pipistrellus sequence shows with the E95 PMV strain, our findings support this association. Moreover, an extensive collection of urine samples from the colony would be necessary to facilitate PMVs isolation, which remains a critical requirement for full genome and pathogenic characterization of the strains detected. Compared to previous studies published in Italy [24, 25] , we detected alpha and beta-CoVs in not previously surveyed Italian regions and in new bat species; moreover, this report represents the first and novel identification of PMVs in Italy. The 23 new bat genetic sequences will fill gaps and expand the current molecular bat-borne virus databases. Considering the amount of novel bat-borne PMVs associated with the emergence of zoonotic infections in animals and humans in the last years define the virus diversity within European bat species is needed. Performing surveillance studies within a specific geographic area can provide awareness of viral burden where bat roosts are in close proximity to spillover hosts, and can form the basis for the appropriate control measures to curb potential threats for public health and optimal management of bats and their habitats. Financial support for this study and its publication was provided by the Italian Ministry of Health in the context of Ricerca Sanitaria Corrente 2013 (Code: IZS PLV 09/13 RC). Data generated or analyzed during this study are included in this paper and can be made available by the corresponding author upon a reasonable request. Sequence data obtained in this study are deposited in GenBank database: full sequence name and GenBank accession n o are indicated in Table 2 . Authors' contributions MLM, SR1. and RO participated in the conception and coordination of the study; RT and PC performed all the captured and bat species identification in the field; FR and AL performed the experiments and FR wrote the paper; LB and KE analyzed the genetic data; SZ and AD performed the necropsies; AK and SR2 contributed reagents, materials and analysis tools. All authors read and approved the final manuscript. Ethics approval and consent to participate Samples collection and bat species identification were performed by expert chiropterologists authorized by the Italian Ministry of Environment (authorization number DPN/2010/0011879 and 000882/PNM/08052014). All field operations were performed in compliance with the Habitats Directive of the European Union [41] and the Agreement on the Conservation of Populations of European Bats [45] , and respecting ethical and safety recommendations [46] . Not applicable\",\n",
       "   '_search_score': 0.7024539000000001},\n",
       "  {'_id': 'a911adbaf5828c2f84558d48e2e72e00810069aa',\n",
       "   'insert_date_': '2020-11-04T18:59:11.280295',\n",
       "   'abstract': 'Objective Severity of the coronavirus disease 2019 (covid-19) has been assessed in terms of absolute mortality in SARS-CoV-2 positive cohorts. An assessment of mortality relative to mortality in the general population is presented. Design Retrospective population-based study. Setting Individual information on symptomatic confirmed SARS-CoV-2 patients and subsequent deaths from any cause were compared to the all-cause mortality in the Swiss population of 2018. Starting February 23, 2020, mortality in covid-19 patients was monitored for 80 days and compared to the population mortality observed in the same time-of-year starting February 23, 2018. Participants 5 160 595 inhabitants of Switzerland aged 35 to 95 without covid-19 (general population in spring 2018) and 20 769 persons tested positively for covid-19 (spring 2020). Measurements Sex-and age-specific mortality rates were estimated using Cox proportional hazards models. Absolute probabilities of death were predicted and risk was assessed in terms of relative mortality by taking the ratio between the sex-and age-specific absolute mortality in covid-19 patients and the corresponding mortality in the 2018 general population. Results A confirmed SARS-CoV-2 infection substantially increased the probability of death across all patient groups, ranging from nine (6 to 15) times the population mortality in 35-year old infected females to a 53-fold increase (46 to 59) for 95 year old infected males. The highest relative risks were observed among males and older patients. The magnitude of these effects was smaller compared to increases observed in absolute mortality risk. Male covid-19 patients exceeded the population hazard for males (hazard ratio 1.20, 1.00 to 1.44). Each additional year of age increased the population hazard in covid-19 patients (hazard ratio 1.04, 1.03 to 1.05). Limitations Information about the distribution of relevant comorbidities was not available on population level and the associated risk was not quantified. Conclusions Health care professionals, decision makers, and societies are provided with an additional population-adjusted assessment of covid-19 mortality risk. In combination with absolute measures of risk, the relative risks presented here help to develop a more comprehensive understanding of the actual impact of covid-19.',\n",
       "   'title': 'Relative Coronavirus Disease 2019 Mortality: A Swiss Population-based Study',\n",
       "   'body': 'Early reports from China and Italy 1-7 on disease severity of covid-19 caused unprecedented public health interventions around the world, ranging from social distancing measures or school and university closings to complete lockdowns of societies. The absolute mortality in patients diagnosed with covid-19, i.e., the case-fatality rate, has been the main entity used for communicating risks associated with the disease 2-6 . Risk factors for mortality risk, most prominently higher age, being male, and preexisting medical conditions, have become publically known 1, 3, 4 . On the current occasion of liberalisation of the most stringent public health interventions in many countries, an assessment of the actual impact of the covid-19 pandemic is called-for. Health care professionals, politicians, and societies at large currently engage in a discussion about the appropriateness of the mitigation measures taken, and the first scientific contribution on the matter has arisen 8 . The probability of death estimated from hundreds of thousands of covid-19 patients are constantly reported from many countries [2] [3] [4] 9 . These numbers can, however, be hard to compare, due to differences in testing regimes, varying ascertainment of mortality, different age structures of societies, or different health-care and public health systems 10 . As an alternative risk difference of disease impact, excess numbers of deaths has been reported for some populations, i.e., the number of observed all-cause deaths during the time of the covid-19 pandemic (end of February to mid-May 2020 in most European countries) minus the expected number of deaths in the given population. Reports on the number of excess deaths observed since the onset of the pandemic are available from Portugal 11 , Spain 12 , northern Italy 13, 14 , various other European countries 15 , and the United States 16 . However, cross-country comparisons are again difficult, because the success or failure of public health interventions and possible overruns of hospital capacities will be reflected in the presence and magnitude of excess mortality 14 . The emergence of mature data on the course of the covid-19 pandemic from many countries allows its actual impact on societies and health-care systems to be discussed in light of the shortterm relative mortality. This relative risk compares the absolute all-cause mortality observed in patients diagnosed with covid-19 during the spring 2020 outbreak with the absolute all-cause mortality in the uninfected population of earlier years during the same calendar time of year. The population mortality varies between females and males, and over attained age, with males and older individuals experiencing a higher short-term risk of dying. How much of the increased mortality reported for male and older covid-19 patients 1, 4, 6, 7 can be attributed to the increase in population mortality risk in general is an important question awaiting an answer. Furthermore, sex-and age-specific relative covid-19 mortality allows a stratified assessment of risk-increase caused by a SARS-CoV-2 infection 17 . We report age-adjusted relative covid-19 mortalities for females and males based on an analysis of individual population level and covid-19 death records from Switzerland covering the time 3 . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint this version posted June 12, 2020. . https://doi.org/10.1101/2020.06.10.20127670 doi: medRxiv preprint between 2020-02-24 and 2020-05-14. For this population-based study, Swiss general population data from 2014-2018, including Official SARS-CoV-2 testing in Switzerland was performed by polymerase chain reaction (PCR) only, based on lower and upper respiratory tract samples from symptomatic persons. Individuals experiencing the following symptoms were eligible for testing 18 : cough, sore throat, muscle pain, dyspnea (with or without fever), and acute anosmia or ageusia. Testing of asymptomatic persons was only recommended to control local outbreaks in hospitals or nursing homes. All positive and negative SARS-CoV-2 test results were directly reported to the Federal Office of Public Health, patients were followed-up subsequently. The study population consists of two cohorts. The first cohort consists of persons with a SARS-CoV-2 positive PCR test between 35 and 95 years of age at time of testing. Individuals tested post-mortem or after hospitalisation were not included as the aim was to study relative mortality in a cohort of newly infected people who did not have a short-term increased mortality risk. We refer to this cohort as the \"Swiss covid-19\" cohort in the sequel. The second cohort was defined as all inhabitants of Switzerland alive on February 23, 2018 aged 35 to 95 years. Sex-and age-specific hazard ratios (HRs) with 95% confidence intervals (CIs) were estimated using a stratified Cox proportional hazards model, allowing for separate baseline rates in the Swiss covid-19 and Swiss 2018 population cohorts, with 65-year old females as reference. Sex and age were modelled with main effects only for the Swiss 2018 cohort, whereas for the Swiss covid-19 cohort, interaction effects between SARS-CoV-2 status and sex and age were included to capture the additional mortality effects among the patients. P-values and 95% confidence intervals for hazard ratios were adjusted for multiplicity. To quantify the impact on mortality associated with a SARS-CoV-2 diagnosis, the 60-day probability of death was predicted from the fitted model estimates, which captures deaths occurring within 60 days of February 23 or positive test, for the Swiss 2018 population cohort and Swiss covid-19 cohort, respectively. Using these probabilities, the sex-specific relative mortality (and associated 95% confidence bands) was calculated by taking the ratio between the two, along an age gradient between 35 and 95 years. The relative mortality incorporated uncertainty in the Swiss covid-19 cohort only 19 . The assumption of proportional hazards was assessed by fitting models allowing for time-varying effects. Potential deviations from the linear age effect were assessed in a Cox model allowing nonlinear effects of age. The main-effects only model was compared to a model including sex  age interactions. As a sensitivity analysis, all models were re-fitted using the Swiss 2014 to 2017 general populations as reference. Further details on the statistical analyses performed can be found in the Online Supplement. All analyses were performed in the R system for statistical computing 20  The daily number of deaths observed during the 80-day study period (2020-02-24 to 2020-05- 14) in the Swiss covid-19 cohort increased rapidly from mid-March and peaked during the first days of April, in both males and females ( Figure 1 ). The numbers reduced to less than ten reported deaths per day during the last week of the study period. [ Figure 1 about here.] 5 . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review) The copyright holder for this preprint this version posted June 12, 2020. . https://doi.org/10.1101/2020.06.10.20127670 doi: medRxiv preprint Characteristics of the Swiss covid-19 cohort, the Swiss 2018 population cohort, and the earlier Swiss population cohorts (2014) (2015) (2016) (2017) are presented in Table 2 . The ratio of females to males indicate that patients in the Swiss covid-19 cohort were more likely female. The mean age was similar between the two cohorts, with SARS-CoV-2 positive patients being between one and a half (females) and up to three years (males) older than individuals in the Swiss population cohort. The similarity was further observed when comparing the whole age distribution among females and males, between cohorts ( Figure 2 ). However, there was a slight over-representation of older people, between 85 and 95 years of age, in SARS-CoV-2 positive individuals. [ Table 2 The all-cause mortality rate among 65-year old males from the Swiss 2018 population cohort was 1.50 times that among females of the same age. The mortality rate further increased by a factor of 1.09 for each additional year of age (Table 3 ). In 2018, males experienced the same mortality as females aged 4.74 years older. The additional increment in the mortality rate for individuals in the Swiss covid-19 cohort were smaller than the sex-and age-effects observed in the Swiss 2018 population cohort (difference in log-HRs between population and patients 0.22, 95% CI 0.02  0.41, for the sex effect and 0.04, 95% CI 0.03  0.05, for the age effect). Being a male covid-19 patient was associated with a HR of 1.20, relative to males in the general population ( Table 3 ). The hazard ratio comparing the hazard of males to females in the Swiss covid-19 cohort was 1.50  1.20 = 1.80 (HR, 95% CI: 1.58  2.06). Each additional year of age increased the covid-19 mortality rate by a factor of 1.09  1.04 = 1.14 (HR, 95% CI: 1.13  1.14). [ Table 3 about here.] The 60-day sex-and age-specific probability of death was considerably larger in the covid-19 cohort ( Figure 3 55-year old infected males were associated with a probability of death around 1%, equivalent to the 95-year old not infected males in the general population. The probability of death in infected individuals was above 10% for females older than 82 and males older than 76 years. The relative mortality in 80-year old covid-19 patients was approximately three times higher than the relative mortality in 40-year old covid-19 patients. . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review) The probabilities of death implied a nearly log-linear function of age, for both females and males, and in both cohorts of the study population. Hence, it follows that the relative risk comparing the mortality in young patients to the mortality in patients aged 20 years older, for example, would be constant regardless of the age of younger group of patients. The slope of the probabilities of death as a function of age was larger in the Swiss covid-19 cohort due to the models\\' age effect attributable to the infection (Table 3 , covid-19 Age  65). [ Figure 3 about here.] The sex-and age-specific relative mortality (Figure 3 bottom panel and Table 4 ) ranged from 9. 26 (35 year old females) to 53.76 (92 year old males). This is interpreted as a 35 years old infected female was 9.26 times more likely to die within 60 days after the positive test than a 35 years old uninfected female was likely to die within 60 days following February 23, 2018. Fatalities were 53.76 times more likely in 92-year old infected versus non-infected males. However, the 95% confidence bands demonstrated substantial uncertainty in the estimates of this relative risk; the lower confidence band for 35-year old persons was as large as 6.03 for females and 7.99 for males. The relative mortality was largest for 95 year old females and 92 year old males. [ Table 4  The nine to 53-fold increase in probability of death found in female and male, young and old, symptomatic covid-19 patients from Switzerland in comparison to the Swiss population of 2018 provides novel sex-and age-specific information on the severity of this pandemic. Covid-19 mortality has never previously been reported as a relative risk in direct relation to the mortality in the general population. The comparison of absolute mortalities and case-fatality rates between risk groups of covid-19 patients without any population adjustment are likely to over-estimate the increased mortality in males and older people. As presented here, more than half of the hazard increase observed in male and older patients could be attributed to the generally increased population mortality in these risk groups. 7 . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint this version posted June 12, 2020. . https://doi.org/10.1101/2020.06.10.20127670 doi: medRxiv preprint The population-based setting with matched calendar time allows for estimation of covid-19 related mortality in an unselected cohort of symptomatic and diagnosed patients, without the need to differentiate between deaths caused or not caused by covid-19 in terms of death certificate information 28 . Seasonal effects were implicitly accounted for by comparing cohorts over the same time-of-year. Unlike other population studies estimating excess mortality, the probability of death in these cohorts could not be confounded by ongoing public health interventions or testing coverage 28 . Using data from Switzerland was especially useful for this type of analysis. The borders to northern Italy and Austria caused covid-19 outbreaks early in the pandemic, broad and uniform symptom-based testing overseen by federal authorities was implemented quickly, and all test results were reported. A symptom-based testing regime was carried out, with testing of asymptomatic persons only recommended to control local outbreaks in hospitals or nursing homes, however, a number of hospitals started to test all patients admitted to the hospital at different time points regardless of symptoms. In contrast to reports from northern Italy 14 , the number of covid-19 patients in need for hospitalisation never exceeded health-care capacities, and every patient received the best possible treatment under the circumstances. Absolute mortalities in the Swiss covid-19 cohort were smaller than those reported for Italian and Chinese covid-19 patients 3-5 between 40 and 80 years old. The numbers are expected to be higher than in Germany, where the overall case fatality rate was only 1.2% and more tests were performed in younger patients with mild symptoms 29 . Due to differences in testing protocol, substantially higher or lower relative mortality may therefore be expected in other countries. The Swiss covid-19 cohort excluded persons with known increased mortality risks (those tested posthumously or while being hospitalised), as well as very old and thus a priori frail persons. The figures presented here can nevertheless inform models developed for computing prognoses on the number of expected deaths in real or hypothetical populations, because relative mortality is not affected by public health interventions which lead to a reduced or even nonexistent excess mortality in many European countries. For the UK, prognoses assumed a 1-year relative mortality risk not larger than two, uniformly for females and males of all ages 30 . A short-term 60-day relative mortality larger than nine, as found here, suggests that the actual risk is larger than assumed based on prognostic models. However, the more general question of the true covid-19 relative mortality will be lower than that reported in this study, as the inclusion criteria for the covid-19 cohort was defined on being symptomatic and testing positive for SARS-CoV-2. The true proportion of asymptomatic cases and the actual prevalence are ongoing questions 31 . The sex-and age effects on all-cause mortality attributable to covid-19 are, however, still relevant and males and older persons were associated with higher risk. Males had a 20% higher mortality rate than females, however, the uncertainty around this hazard ratio was large and the effect was absent in populations of earlier years and in a model with sex  age interactions (see Online Supplement). The relative mortality (ratio of the probabilities of death) was approximately three times higher among 80-year old than 40-year old covid-19 patients. This population-adjusted comparison of the risk between young and old patients suggest a less drastic relative impact of age on 8 . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint this version posted June 12, 2020. . https://doi.org/10.1101/2020.06.10.20127670 doi: medRxiv preprint to justify based on the continuously increasing age-specific relative mortalities reported here. The results suggest that covid-19 risk assessment in terms of case-fatality rates and excess mortalities should be complemented by population-adjusted relative mortalities such that a more complete picture can emerge, potentially leading to improvements for age-based risk stratification. Contributions TH, MB, and MJC designed the study, preprocessed, and analysed the data. TH prepared an initial draft, all authors reviewed and improved early versions of the manuscript and approved the final version. Funding Swiss National Science Foundation. The funder of this study had no role in study design, data collection, data analysis, data interpretation, or writing of the final report. Competing interests TH has been paid for consulting, lectures or presentations from Novartis, Roche, and PricewaterhouseCoopers. HFG has received unrestricted research grants from Gilead Sciences and Roche; fees for data and safety monitoring board membership from Merck; consulting/advisory board membership fees from Gilead Sciences, Merck and ViiV Healthcare; and grants from SystemsX, and the National Institutes of Health. MJC has been paid for consulting, lectures or presentations from Roche and Reinsurance Group of America UK Services Limited. Data sharing All results can be reproduced from study cohorts, R and Stata computer code publically available from https://gitlab.switch.ch/torsten.hothorn/relative_covid-19_ mortality. 9 . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint this version posted June 12, 2020.  . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint this version posted June 12, 2020. . https://doi.org/10.1101/2020.06.10.20127670 doi: medRxiv preprint . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint this version posted June 12, 2020. 12 . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint this version posted June 12, 2020. 13 . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint this version posted June 12, 2020. 14 . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint this version posted June 12, 2020. 15 . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint this version posted June 12, 2020. 16 . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint this version posted June 12, 2020. 17 . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint this version posted June 12, 2020. . 18 . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint this version posted June 12, 2020. .  . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint this version posted June 12, 2020. . 20 . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint this version posted June 12, 2020. . For the population cohort of a given year, all inhabitants of Switzerland alive and aged between 35 and 95 years old at February 23 enter the risk set. Inhabitants surviving 80 days were censored after 80 days, and those who died within 80 days after February 23 were recorded as an event. To be able to evaluate mortality rates and ratios on the same timescale, allowing for seasonal effects, for the covid-19 cohort, a delayed entry approach was taken, as patients do not become at risk of death until their date of positive test. All deaths in this cohort were treated as events. Individuals without a death record were administratively right-censored 80 days after February 23. We therefore modelled time-to-death from any cause within 80 days after February 23 (the first positive SARS-CoV-2 test was recorded February 24, 2020; the database was closed on 2020-05-14). The mortality rate in the Swiss covid-19 cohort was compared to the mortality rate in the Swiss 2018 population cohort during the same time of year. Hazard ratios were estimated by a Cox proportional hazards model expressing the distribution of time-to-death after February 23 in days t > 0. We fitted a Cox model allowing separate baseline hazard functions, assuming where h 2018 (t) is the baseline mortality rate at time t, for a 65 year old female in the Swiss 2018 population cohort, h covid-19 (t) describes the mortality rate in the Swiss covid-19 cohort for (on the same calendar timescale to account for seasonal effects), and  takes the value of 1 if the individual is in the covid-19 cohort, and 0 if the individual is in the 2018 cohort. The linear predictor is parameterised in such a way that we can quantify the effects of age and being male in the general population by the hazard ratios exp( 1 ) and exp( 2 ), respectively, and the additional increase/decrease in the hazard attributable to age and being male in the covid-19 cohort, described by the exp( 3 ) and exp( 4 ), respectively. The proportional hazards assumption was assessed by comparing the in-sample log-likelihood of the model described above to the in-sample log-likelihood of a Cox model with time-varying sex-and age-effects. The simpler model assuming proportional hazards maximised the log-likelihood at 538664 and the model not assuming proportional hazards at 537964, suggesting that mortalities were appropriately modelled under proportional hazards. For the estimation and comparison of absolute risk, i.e. the probability of dying by time t, we model the mortality rate in the covid-19 cohort on the time since diagnosis timescale. Therefore, the model for the 2018 cohort remains the same, and we are interested in the probability of death 60 days after February 23, 2018. In the covid-19 cohort, we are interested in the probability of death 60 days after diagnosis. Two separate Cox models were fitted to the cohorts with the appropriate timescales. Relative mortality (RM), at time t, is then defined as, RM(t | sex, age) = P(t | sex, age, 1) P(t | sex, age, 0)  . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint this version posted June 12, 2020. . https://doi.org/10.1101/2020.06.10.20127670 doi: medRxiv preprint Supplementary Table 1 : Swiss 2018 population and covid-19 cohorts. Log-hazard ratios (log-HR) and hazard ratios (HR) expressing the risk of being male (\"Male\") and each year of age (\"Age65\") compared to the baseline hazard in 65 year old females, based on a Cox proportional hazards model with sex  age interaction. Estimates are given with standard errors (SEs) for log-HRs and 95% confidence intervals (CI) for HRs, the latter and P -values were adjusted for multiplicity. log-HR SE10 P -value HR 95% CI Female which is the ratio of the probability P(t | sex, age, 1) to die within t days after a positive SARS-CoV-2 test and the probability P(t | sex, age, 0) to die within t days after February 23, 2018 without an infection for female or male persons of a specific age. We used a time horizon of t = 60 days. A relative mortality RM(60 | female, 65) = X means that the probability to die within 60 after an SARS-CoV-2 infection is X times larger than the population probability to die within 60 days after February 23, 2018, for 65 year old females. Proportional hazards models for absolute risk were estimated with log-cumulative baseline hazard functions parameterised in terms of a Bernstein polynomial 22 , that is, using a parametric equivalent of the semiparametric Cox model. In all models, the assumption of a linear age effect was assessed by comparing the linear Cox proportional hazards model to a Cox proportional model featuring a nonlinear additive effect of age on the log-hazard ( Figure 1 ). For the 2018 population cohort, differences were marginal and justified the use of a linear age term. For the Swiss covid-19 cohort, the nonlinear model suggested a stronger age effect up to 80 years, which then levelled-off for older people. The discrepancy between the nonlinear and the linear Cox model in terms of the estimated probability of death at day 80 can be inferred from Figure 2 . For the 2018 population cohort, the probability of death from both models were very similar, with some variation for people older than 90 years, which can be explained by the sharp increase of the corresponding loghazard ratio (Figure 1 ). In the age range between 60 and 90 years, the two models were relatively similar (25%), however, the nonlinear model suggested the probability of death was 50% reduced compared to probability of death from the linear model. This means that the relative mortalities assuming a linear effect of age on the log-hazard scale ( Figure 3 and Table 4 ) could be up to two times smaller for people younger than 60 and up to two times larger for persons around 80 years old in a more complex model allowing a nonlinear effect of age. We also considered an interaction between sex and age, but found no difference (Table 1 ). . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint this version posted June 12, 2020.  Computations were performed in R version 4.0.1 20 . Cox proportional hazards models were fitted semiparametrically to estimate log-hazard ratios using the survival 21 package. Absolute mortalities were obtained from Cox models, parametrically parameterised with flexible logcumulative baseline hazard function, using package mlt 23, 33 . Confidence bands were computed using the multcomp package. The primary analyses were conducted in R; however, to ensure validity and robustness, all analyses were replicated independently in Stata by using a flexible parametric survival model 25 using the stpm2 26 and merlin 27 commands, which can be considered a parametric equivalent to the Cox model but use restricted cubic splines to directly model the baseline (log cumulative) hazard function. We found complete agreement across software platforms, and modelling approaches. Furthermore, the sensitivity of the main findings reported was assessed by conducting the same analyses, but using the population data from 2014 to 2017, independently, as the comparative populations, instead of the 2018 cohort. Age distributions and baseline cumulative hazards for these study populations are given in Figures 3, 5 , 7, and 9. Absolute and relative mortalities can be obtained from Figures 4, 6 , 8, and 10. Hazard ratios for all study populations are presented in Table 2 . The deviations from the main results were marginal. . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint this version posted June 12, 2020.  . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint this version posted June 12, 2020. . https://doi.org/10.1101/2020.06.10.20127670 doi: medRxiv preprint . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint this version posted June 12, 2020.  . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint this version posted June 12, 2020. 28 . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint this version posted June 12, 2020.  . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review) The copyright holder for this preprint this version posted June 12, 2020.  . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review) The copyright holder for this preprint this version posted June 12, 2020.  . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review) The copyright holder for this preprint this version posted June 12, 2020.  . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review) The copyright holder for this preprint this version posted June 12, 2020.  . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review) The copyright holder for this preprint this version posted June 12, 2020. . https://doi.org/10.1101/2020.06.10.20127670 doi: medRxiv preprint',\n",
       "   '_search_score': 0.6969185000000002},\n",
       "  {'_id': 'a007977dad90a07b3beb9f689e3be8b3f7d2a7f6',\n",
       "   'insert_date_': '2020-11-04T18:47:02.644581',\n",
       "   'abstract': 'Hepatitis C virus (HCV) infection is an important public health problem worldwide. HCV exploits complex molecular mechanisms, which result in a high degree of intrahost genetic heterogeneity. This high degree of variability represents a challenge for the accurate establishment of genetic relatedness between cases and complicates the identification of sources of infection. Tracking HCV infections is crucial for the elucidation of routes of transmission in a variety of settings. Therefore, implementation of HCV advanced molecular surveillance (AMS) is essential for disease control. Accounting for virulence is also important for HCV AMS and both viral and host factors contribute to the disease outcome. Therefore, HCV AMS requires the incorporation of host factors as an integral component of the algorithms used to monitor disease occurrence. Importantly, implementation of comprehensive global databases and data mining are also needed for the proper study of the mechanisms responsible for HCV transmission. Here, we review molecular aspects associated with HCV transmission, as well as the most recent technological advances used for virus and host characterization. Additionally, the cornerstone discoveries that have defined the pathway for viral characterization are presented and the importance of implementing advanced HCV molecular surveillance is highlighted. Viruses 2015, 7 1154',\n",
       "   'title': 'Advanced Molecular Surveillance of Hepatitis C Virus',\n",
       "   'body': 'Hepatitis C virus (HCV) infection is an important global public health problem. Approximately 180 million people are currently infected with HCV [1] , and an alarming number of new infections occur annually [2, 3] . The prevalence of hepatitis C varies significantly worldwide, imposing an important burden in highly endemic countries [2] . HCV infection is commonly associated with chronic liver disease, which frequently results in the advanced stages of cirrhosis and hepatocellular carcinoma (HCC) following years of silent infection [4, 5] . HCV is a single-stranded, positive-sense, enveloped flavivirus. The viral RNA genome is ~9.6 kb in length and contains a single open reading frame encoding a large polyprotein. The polyprotein is processed by viral and host proteases, resulting in three structural proteins and seven nonstructural proteins ( Figure 1A ) [6, 7] . HCV molecular evolution plays an important role in virus transmission, dictating in many ways the outcome of disease and therapy. Based on the nucleotide variability in the viral genome, seven HCV genotypes and multiple subtypes have been recognized ( Figure 1B) [8] . These genotypes exhibit a characteristic distribution [9] : genotypes 1-3 are distributed worldwide, while genotypes 4 and 5 are mainly found in Africa and genotype 6 is endemic in Asia [9] [10] [11] [12] [13] [14] [15] . The high mutation rate is characteristic of HCV replication, resulting in a high degree of intrahost genetic diversity [16] [17] [18] . The molecular plasticity of HCV allows rapid rearrangement of the intrahost viral population under different selection pressures [19, 20] . This remarkable genetic variability is one of the main factors that have prevented the development of a successful vaccine. shown. The most common regions used for molecular analyses and genetic relatedness studies are highlighted; (B) HCV representative strains belonging to all seven genotypes and all different subtypes are depicted. Sequences from the NS5B region were chosen to illustrate the maximum diversity within a subtype. The neighbor joining tree was constructed using maximum composite likelihood nucleotide distances using MEGA5 and 1000 replications for bootstrapping. Molecular surveillance of hepatitis C is of the utmost importance for identification of HCV transmission networks. Molecular surveillance is a critical component of HCV outbreak investigations because tracking of infections is necessary for the identification of sources of infection and accurate identification of cases associated with transmission networks [21] . Additionally, molecular surveillance can provide important information about the capacity of HCV lineages to cause disease [21] . In addition to imposing important challenges for vaccine development, HCV genetic variability also represents a major issue for the establishment of genetic relatedness between cases associated with a common source of infection. The rapidly evolving nature of the virus results in significant changes in genetic makeup of the virus, which can prevent our ability to link cases by genetic similarity [16] . However, a high degree of genetic variability is also required to track infections and facilitates genetic relatedness studies which otherwise would be hampered [22] . Here, we present a historical review of the keystone discoveries that have propelled our understanding of the mechanisms controlling HCV transmission. Additionally, we discuss the most recent information on molecular aspects related to viral spread and novel approaches designed to assess HCV intrahost genetic variability for the establishment of genetic relatedness between cases. Finally, the implementation of advanced HCV molecular surveillance as an integral component of hepatitis C global epidemiology is highlighted. Since its discovery 25 years ago, the study of HCV has seen a number of cornerstone developments that have improved our understanding of HCV-related disease pathogenesis ( Figure 2 ). The field of viral hepatitis initially began in the late 1950s with the coining of the term \"infectious\" or \"serum\" hepatitis [23] . This event was followed by the discovery of the hepatitis A (HAV) [24] and hepatitis B viruses (HBV) [25] in the late 1960s and early 1970s. However, our understanding of HCV infection dates to the late 1980s when researchers from the Centers for Disease Control and Chiron Corporation, led by Daniel Bradley and Michael Houghton, respectively, identified the virus in samples from experimentally infected chimpanzees [26] . Prior to its identification, the advent of serologic testing for HAV and HBV in the mid-1970s led to the realization that most infectious hepatitis cases were non-A, non-B (NANBH). These findings prompted the use of the chimpanzee as an animal model for the passage of the unidentified agent responsible for NANBH [27] . The subsequent development of serologic tests for the detection of HCV infection in the early 1990s allowed screening of the blood supply and successfully prevented transmission via transfusion [28] . HCV molecular divergence was recognized shortly after the discovery of the virus when Japanese strains were shown to be genetically distant from American strains [29] . Subsequently, the complete sequencing of the HCV genome allowed the elucidation of its organization [30] [31] [32] . Molecular characterization of the viral enzymes during the early 1990s served as the foundation for the development of successful anti-HCV therapies [18, [33] [34] [35] [36] [37] and resulted in the constant reshaping of anti-HCV therapy based on interferon (IFN). The initial approval of alpha IFN (IFN) for the treatment of HCV in 1991 was followed by the licensing of consensus IFN in 1997 and ribavirin (RBV) in 1998. In 2001, the introduction of pegylated IFN, a compound with significantly prolonged half-life in blood, increased the likelihood of a sustained virological response (SVR). Full recognition of the degree of genetic heterogeneity among HCV isolates was achieved during the mid-1990s [38, 39] , resulting in the proposed classification of Hepaciviruses as an independent genus within the Flaviviridae family representing a distant relative of the Flavivirus and Pestivirus genera [40] . Importantly, the fact that HCV diversity played an important role in interferon (IFN)-based therapy highlighted the relevance of assessing the genetic variability of the viral population in vivo [41, 42] . An important advance for drug development was the generation of human hepatoma cell lines capable of replicating HCV [43] [44] [45] [46] . Over the years, other important discoveries have been reported by several groups, especially relating to the characterization of viral proteins and their role in HCV replication [34, [47] [48] [49] . Additionally, the characterization of several cellular receptors [50] [51] [52] [53] , and entry factors [54, 55] has been reported. The initial release of the first next generation sequencing (NGS) commercial platform in 2005 completely revolutionized the field of genetics. However, initial studies reporting the use of NGS approaches to assess HCV intrahost genetic variation emerged in late 2011 [56, 57] . Since this time, a wealth of information has been generated using NGS to address a multitude of issues related to HCV transmission [58] [59] [60] [61] [62] [63] [64] [65] [66] [67] [68] [69] [70] [71] . The challenge now resides in the study of whole genome (WG) genetic variation, which still represents a daunting task for the study of HCV molecular evolution since reconstruction of original haplotypes from NGS data is difficult [72] [73] [74] . In 2009, Ge et al. reported a strong association between polymorphisms in the promoter region of the interleukin-28B gene (also known as IFN-3) and the antiviral response based on IFN [75] . Subsequently, work conducted by other groups further confirmed the relationship between the IL-28B genotype and sustained viral response (SVR) [76, 77] . The relevance of these reports cannot be understated, because they resulted in extensive work on human genetics and became an important component in the study of the pathogenesis of HCV-related disease. The arrival of the second generation of direct-acting antivirals (DAA) in 2011 resulted in improved SVR and a revolution in the field of anti-HCV therapy. Despite the development of drug resistance, the initial NS3-4A protease inhibitors (PI) telaprevir and boceprevir undoubtedly possessed advantages over dual IFN/RBV treatment [78] . The success of both drugs was subsequently eclipsed by the licensing of simeprevir and sofosbuvir [79] [80] [81] [82] . The field of HCV therapy is rapidly evolving, and as a result, a large number of new antiviral drugs are currently being evaluated in advanced clinical trials [83] . The prevention of HCV infection has been hampered by the slow development of promising vaccine candidates. Phase I of two different vaccine trials have been completed. Testing of a prime-boost regimen with the candidate developed by GlaxoSmithKline after preliminary studies in the chimpanzee model has also been performed [84] . Importantly, conserved epitopes in HCV genotype 1 and 3 were successfully identified, suggesting the potential for cross-genotypic protection. Priming with this vaccine followed by boosting with a modified vaccinia virus is currently being evaluated in a phase I/II trial [85] , with the goal of preventing persistence in HCV-nave IDUs at high risk for infection. HCV transmission is a dynamic process that primarily occurs via parenteral routes and especially by unsafe injections, which have significantly facilitated virus spread [86] . The time of initial spread of HCV into Western countries and the population dynamics of the epidemic can only be indirectly inferred [13] . Current evidence supports a recent spread; however, the lack of samples prior to the Second World War has hindered our ability to reconstruct the HCV epidemic. Because vertical and sexual transmission contribute little to HCV spread, the restriction of HCV transmission through primarily parenteral routes implicates unsafe injections (i.e., inadequate medical treatment, large-scale vaccination programs, blood transfusion, and injecting drug use) as the main vehicle for HCV transmission [13] . Limited information is available about the transmission of HCV prior to the invention of injections. However, cultural traditions, such as tattooing and scarification, were likely to have played an important role in the spread of HCV. HCV intrahost populations frequently exist as an ensemble of genetically distinct but closely related variants [18, 87] . Analysis of HCV intrahost genetic variation is the basis of genetic relatedness and epidemiological studies as well as the identification of drug resistant mutations [88, 89] . While in some instances genetic relatedness might be demonstrated with consensus sequencing of some viral subgenomic regions, HCV outbreak investigation commonly requires a much deeper analysis of the infecting viral population [16, [90] [91] [92] [93] . The rapidness with which HCV intrahost populations diverge significantly affects genetic relatedness studies because molecular epidemiological links can be lost between related cases in a relatively short period of time ( Figure 3 ) [16, [94] [95] [96] . Therefore, the use of more and longer subgenomic regions, such as the NS3, NS5A, and NS5B might help alleviate this issue. The molecular mechanisms that shaped the evolution of HCV are quite diverse. Upon transmission, HCV undergoes a series of events that affect the architecture of the viral population ( Figure 4) . The \"steps\" taken by HCV throughout the infection process are highly complex and depend on fluctuations in the complexity and size of the intrahost population. HCV transmission events are characterized by genetic bottlenecks that inflict a strong selective pressure during acute infection [56, [97] [98] [99] . Indeed, the HCV genetic bottlenecks are so intense following transmission that very few variants are able to establish infection in the new host [100, 101] . Subsequently, genetic drift commonly takes place as the main force driving the molecular evolution of HCV during the early stages of the acute phase as a result of the strong founder effect [102] [103] [104] . During the later stages of HCV infection the extent of genetic drift is significantly reduced [102] [103] [104] [105] , allowing a large viral population size to come to prominence in chronic cases. To a lesser degree, genetic recombination also affects overall HCV genetic heterogeneity. HCV genetic recombination is rare [106] , mainly due to the exclusion of superinfections [107] [108] [109] . However, despite limited HCV recombination, naturally occurring inter-genotype, intra-genotype, and intra-strain HCV recombinants have been reported [110] [111] [112] [113] . Thus, HCV recombination might have important implications for clinical and epidemiological studies [114] . Staging of HCV infections has also been proposed [115] . This hypothesis suggests that during chronic infection, HCV attempts to reach a settlement stage characterized by a viral population undergoing strong negative selection. This \"journey\" involves complex processes including temporal variations attained by incremental changes between communities, random mutations and fluctuations in the frequency of coexisting viral subpopulations; in conjunction these factors constantly reshape the architecture of the viral population [89, 95, 115, 116] . This staging is likely to affect HCV transmission because different viral variants that most likely possess different degrees of transmissibility are available at different time points during the course of the infection [21] . Despite the complexity of the means used by HCV to evolve in an adverse microenvironment, genetic relatedness can be successfully established in many instances if transmission has occurred relatively recently. In addition to time of infection, establishment of relatedness depends on the degree of divergence and other external factors such as a history of antiviral therapy, which can significantly alter the natural course of HCV evolution. HCV transmission networks are difficult to be recognized for numerous reasons [117, 118] . The long incubation periods and the characteristic asymptomatic nature of acute HCV infections make identification of cases a rather challenging task [117] . HCV transmissions are difficult to link to their corresponding source of infection because intrahost viral populations are often genetically related but seldom identical [95, 119] . Importantly, branching in phylogenetic analyses, derived from genetic relatedness studies, do not always correspond to transmission events, particularly in those cases where not all individuals belonging to the transmission network are sampled ( Figure 5 ). Thus, local epidemic sequences can cluster together in the absence of direct transmission [119, 120] . HCV evolution is also affected by preexisting liver conditions, infection with other viruses and also pregnancy as a direct consequence of the alterations of the immune response in the mother [17] . During pregnancy, increased viral loads and a reduction in CD8+ T cell cytotoxicity are commonly observed, favoring the emergence of more fit strains [121] as a result of the impairment of the antiviral response in the mother. Therefore, children infected perinatally might potentially bear infections with the more fit viruses, thereby facilitating persistence [122] . The era of human genetics is expected to play an important role in the clinical management of infectious diseases [123] . HCV infection is not the exception, and several studies have shown that host genetic makeup affects the outcome of infection and therapy [21] . HCV is spontaneously cleared by the host in up to 15% of acutely infected individuals [124] [125] [126] . It has been reported that an early functional inactivation (peripheral tolerance) of the HCV-specific CD4+ T cell response may play a major role in establishing viral persistence [127] [128] [129] [130] , and such dysfunctional T cells can be functionally reactivated [131, 132] . Several studies have suggested that differences in host immune responses determine viral clearance. Indeed, the intensity of the immune response can be dictated, at least in part, by host genetic factors [133] . However, the role of host factors in HCV clearance is not well understood. Different studies have shown correlations between HCV-related disease and the genetic background of the host [75, 134] . For example, certain HLA alleles have been associated with specific outcomes of HCV infection [135, 136] . As a result of the completion of the Human Genome Project and the development of novel methodologies capable of rapidly assessing the host genetic background, a number of disease-associated genes have recently been reported. Consequently, mutations in these genes can be screened and used to assist in the clinical decision making process [137] . This information can also be incorporated into heightened surveillance for a number of infectious diseases, facilitating the incorporation of pharmacogenetics into global surveillance [138] . Recently, several single nucleotide polymorphisms (SNP) in immune-related genes have been associated with HCV disease outcomes. Large-scale genetic analyses have suggested the participation of different SNPs in the spontaneous clearance of HCV, including TNFSF18 (tumor necrosis factor superfamily, member 18), TANK (TRAF family member associated NF-B activator), HAVCR1 (Hepatitis A virus cellular receptor 1) and IL-18BP (IL-18 binding protein) [139] . Additionally, SNPs occurring in the gene encoding tumor necrosis factor alpha (TNF-) have also been proposed to participate in HCV clearance, particularly in subjects of African origin [140, 141] . TNF- is an important cytokine produced primarily by macrophages that participates in the induction of apoptosis, gene regulation and cellular proliferation [142] . Interestingly, high levels of TNF- have been detected in the serum and liver of chronically infected patients [143] . SNPs in transforming growth factor (TGF)-1 and IFN- have also been suggested to participate in HCV clearance [144, 145] . TGF-1 is a suppressor of natural killer (NK) cells that inhibits the production of IFN- and IL-12, thereby blocking the proliferation and cytotoxicity of NK cells [146] . Moreover, dysregulation of TGF-1 has been shown to be involved in the progression of liver cirrhosis and HCC [147] . IFN- is a multifunctional cytokine produced by effector T and NK cells that participates in the development of T helper 1 (Th1) cells and is critical for host defense against a variety of intracellular pathogens, including HCV. IFN- inhibits HCV replication in vitro, and intrahepatic levels are associated with viral clearance in animal models [148, 149] . SNPs in immune-related genes encoding for interleukin-10 (IL-10) have been reported to play role in HCV-related infection [141] ; however, the role of IL-10 in HCV infection is debatable. IL-10 inhibits IFN- production, resulting in an imbalance in the T helper response that in turn leads to viral persistence [150] , while low levels of IL-10 seem to be associated with resolution of HCV infection [151] . Conversely, IL-10 has been proposed to be antifibrogenetic in chronic liver injury [152] . SNPs in the osteopontin (OPN) gene (1748 and 443) have been associated with chronic HCV infection [153] . Response rates were higher in patients with the G/G or G/A alleles at nt 1748 in comparison to patients bearing the A/A alleles. Likewise, the response rate was higher in patients with the T/T alleles at nt 443 than in those with C/C or C/T alleles [153] . OPN is a highly phosphorylated sialoprotein and an important component of the extracellular matrix [154] , which is secreted by lymphocytes, leucocytes, and macrophages. OPN interacts with cellular adhesion molecules and plays a role in different aspects of the cellular immune response, but the exact role of OPN in HVC infection is still unclear. Recent studies have demonstrated an association between a SNP in the exon 7 splice acceptor site of the oligoadenylate synthetase 1 (OAS1) gene and SVR in HCV patients. Patients exhibiting an AA genotype showed poor SVR rates and progressed to more severe disease [155] . OAS1 is an important protein with antiviral activity [156] that is activated by double-stranded RNA. Upon activation, OAS1 inhibits viral RNA and protein synthesis. Thus, nucleotide changes affecting its function are expected to further impair virus infection control. Importantly, a SNP in the mannan-binding lectin (MBL) gene (also known as the mannose binding protein) has been related to HCV infection. HCV patients tend to exhibit YA/YO in comparison to controls and reduced levels of MLB in plasma [157] . Additionally, the frequency of these genotypes is reduced in patients with advanced fibrosis compared to patients with moderate fibrosis. MLB is a pattern recognition receptor (PRR) that has an important function in the innate immune response [158] . Nevertheless, its role in HCV control is not well known. Another host factor associated with HCV infection is the occurrence of SNPs in the promoter region of the IL-28B gene (also known as interferon- 3). The type III IFN family includes IFN- 1, 2, 3 and 4 (IL-29, -28A, -28B, IFN- 4, respectively) [159, 160] . Based on their molecular structures type III IFNs belong to the interleukin-10 (IL-10) superfamily, but functionally they are closely related to type I IFNs (IFN, IFN), which play a major role in antiviral immunity [161] . IFN-s are produced by dendritic cells, neuronal cells, alveolar epithelial cells, and hepatocytes [162] , in response to viral infections through its activation via Toll-like receptors (TLRs) [163] . IFN-s inhibit viral replication and modulates immune-related functions, such as the maturation and differentiation of immune cells [164] [165] [166] . The SNPs in IL-28B have been recognized as strong predictors for both spontaneous and antiviralinduced clearance of HCV. Several studies have shown that patients infected with HCV genotype 1 bearing the C/C, A/A and T/T alleles in rs12979860, rs12980275, and rs8099917, respectively, are more likely to achieve SVR [75] [76] [77] . Recently, the discovery of a new gene within the type III IFN family (IFN-4) situated upstream of IFNL3, and its association with a poor response to IFN- therapy and impaired HCV clearance suggests a more complex role for the interferon response in HCV infection [160] . However, the arrival of second and third generation treatment regimens based on combinations of different DAA with and without IFN has down-played the relevance of these SNPs in the prediction of SVR. DNA sequencing is essential for the study of transmission, phylogenetics, and outbreaks [167] . Upon transmission, a small subset of the HCV population from the source is transferred to a new host (recipient) and then subjected to immune pressures based on the new host\\'s genetic background. Therefore, the viral populations in the two individuals are related but genetically distinct [88, 89, 95, 96] . Additionally, the characterization of viral variants is also useful for the study of virus tropism in host compartments [89] . The selective forces driving the molecular evolution of HCV are complex in nature and difficult to assess. Most molecular approaches, such as population sequencing, required for molecular characterization of HCV intrahost populations, are arduous, lengthy and costly [168] . However, new technologies based on NGS platforms allow a finer resolution and are more affordable [59, 67, 168, 169] , facilitating the molecular characterization of HCV intrahost viral populations [64, 168, 170] . The analysis of the composition of HCV intrahost populations requires nucleotide sequencing of as many variants as possible in the patient. The assessment of the complexity of the HCV intrahost population is challenging due to the difficulty in detecting low-frequency or minor variants. Establishment of genetic relatedness is affected by the variability of the genomic region analyzed, the time since transmission and the length of the subgenomic region. Moreover, sequence errors introduced by the reverse transcriptase and DNA polymerase during amplification of the template can occur. HCV population complexity can also be skewed by template re-sampling during PCR in low titer clinical samples, resulting in apparently false reduced complexity. Additionally, major variants may mask minor variants due to their respective frequencies during re-sampling. Different approaches have been developed to characterize the HCV intrahost population with different inherent limitations. Alternatives approaches developed for HCV intrahost characterization are based on the identification of nucleotide sequences or assessment of the diversity and complexity without the need to sequence the viral genome [171, 172] . Complexity, cost and reliability are some of the factors that play a role in the choice of methods. Historically, some methods were not powerful enough or presented too many shortcomings to overcome. Consequently, these methods have been disused and replaced by more powerful methods capable of analyzing the composition of the HCV intrahost population to an unprecedented level. Non-sequencing-based methods are disadvantageous because the structure of the viral population is not obtained and the performance relies on the DNA properties of the variants being analyzed. In contrast, sequence-based strategies provide a high-resolution snapshot of the circulating viral variants. Several challenges are faced during the implementation of these techniques, including the stochastic nature of PCR-based methodologies that can significantly distort the composition of the HCV intrahost population. Tracking of HCV infection depends on sequence information originated from different subgenomic regions. The 5\\'-UTR region has been widely used for detection owing to its degree of conservation across genotypes, while the NS5B region is the most common target for HCV genotyping [173, 174] . However, these two regions are well conserved and, therefore, do not contain sufficient sequence information to establish genetic relatedness between clinical isolates. Instead, genetic relatedness studies primarily rely on information obtained from HVR1 [16, 89, 93, 95] . Rapid divergence in this region represents a challenge for molecular epidemiological studies and can result in the loss of genetic links between related isolates [16] . Sequencing of multiple and longer subgenomic regions has been proposed as an alternative to overcome the limitations imposed by the rapid molecular evolution of HCV [16] . NS5A has been used to establish relatedness among HCV cases [115] ; the use of this gene can restore links between isolates owing to its lower nucleotide substitution rate. Despite the usefulness of different subgenomic regions for the characterization of clinical isolates, whole genome sequencing should be the ultimate goal for HCV molecular epidemiology. Prior to the implementation of more sophisticated molecular methods for the characterization of individual variants, SSCP was a relatively popular alternative [175] [176] [177] . This approach is based on the denaturation of single-stranded amplicons that are resolved in non-denaturing conditions to differentiate molecules based on their secondary structure [178] . The popularity of this method was originally based on the assumption that different variants may adopt different secondary structures; however, several studies showed the implicit limitations of SSCP for the identification of closely genetically related variants [179] . Sensitivity is another major limitation of SSCP, because only variants present in >3% of the viral population can successfully be identified [180] , and therefore, minor variants are unlikely to be detected. Additionally, the length of the regions studied imposes important limitations, because long fragments may not resolve as clearly as shorter fragments [181] . Other factors affecting the performance of SSCP include DNA concentration and GC content. Despite being an inexpensive and relatively straightforward approach, SSCP lacks the required resolution to accurately assess the composition of the HCV intrahost population. Thus, it is not surprising that its use has diminished with the arrival of more powerful technologies better suited for such demanding task. In many ways, heteroduplex gel shift assays resemble SSCP [182, 183] , and therefore, the benefits and disadvantages between the two methods are rather similar. Overall, the entire intrahost population can be sampled; however, no specific information about specific substitutions in the population can be obtained. This methodology is based on the amplification and hybridization of the viral RNA to a labeled probe. Next, the heteroduplex products are resolved by electrophoresis. Thus, different variants are expected to exhibit different migration patterns due to mismatches with the probe, resulting in differential band profiles depending on the composition of the intrahost population. A slight variation in the conventional heteroduplex assay relies on the cloning of different viral species into plasmids instead. Genetic differences can be calculated using a heuristic approach known as the heteroduplex mobility ratio. However, the accuracy provided by this method is limited and usually requires variants to differ by ~1.5% [182] . Sequencing individual variants is a cumbersome and costly task that provides important information about the composition of the intrahost population. The basis of this approach includes amplification of the region of interest, followed by cloning of the amplicons into a plasmid vector and transformation of bacteria. Then, the transformed bacteria are plated. The bacteria are assumed to carry only one plasmid, and the presence of only one viral variant sequence is confirmed by colony PCR and sequencing. Thus, DNA from an individual bacterial colony represents the sequence of a single variant. Unfortunately, sampling is a major issue for this particular approach because even in large studies that analyze hundreds of clones, only a small fraction of the viral population and genome is sampled [173] . Moreover, the exact number of clones necessary to achieve a given level of confidence depends on the frequency of each incorrect nucleotide at any given nucleotide position in the sequence [184] . It has been suggested that between 20 and 100 clones per sample are required for frequency and entropy analyses. Sequencing 99 clones has been proposed to identify 95% of all variants present at a frequency of at least 3% in the population [185] . However, others have suggested that sequencing 20 clones is sufficient to cover 95% of the major variants (frequency >10% in the population). Regardless of the estimate, under these circumstances, most minor variants go undetected, significantly hampering our capacity to untangle the mechanisms involved in HCV evolution. Recent advances in single genome sequencing have improved our understanding of HCV transmission [101] . In summary, regardless of the number of clones analyzed, the depth with which population sequencing can assess the complexity of the HCV intrahost population is extremely limited. EPLD-PCR is a viable alternative to single-molecule PCR amplification and has largely been used to identify individual viral variants of different viruses including human immunodeficiency virus (HIV), hepatitis B virus (HBV) and HCV [186] [187] [188] . EPLD-PCR relies on the isolation of individual coexisting sequence variants in clinical samples using a limiting-dilution approach followed by sequencing. Overall, EPLD-PCR is less prone to distortion compared to bacterial cloning-based methods because the amplification is based on single DNA molecules. Moreover, real-time-based EPLD-PCR has been shown to be superior to conventional population sequencing methods for the detection of HCV viral variants. However, implementation of EPLD-PCR is cumbersome and time consuming, resulting in a rather expensive methodology. MS was developed as an alternative to the numerous DNA sequencing technologies [171] . Originally, the accuracy, sensitivity and high-throughput methodology of MS set the basis for the implementation of molecular methods aiming to characterize single nucleotide changes in DNA molecules [189] . MS analysis relies on the measurement of the molecular masses of different analytes, which should be ionized and separated before detection. Unfortunately, ionization may lead to partial degradation of the analytes, thereby affecting the resolution of this method. To minimize the effect of ionization on the analytes, methods such as matrix-assisted laser desorption/ionization and electrospray ionization have been developed [190, 191] . MS has been used for resequencing, microbial typing, and single nucleotide polymorphism discovery [172, 192, 193] . Analysis of the HCV intrahost population by MS has been reported [172, 192, 194] . MS represents a high-performance methodology that analyzes base-specific cleaved RNA originating from PCR fragments [171, 172, 195] . This is a cost-effective and reproducible approach capable of high throughput. The resolution of this method easily accomplishes the identification of nucleotide polymorphisms comprising >10% of the amplicon population. The data originating from this type of analysis represents a composite pattern of the variants present in the sample [36] . Importantly, the identification of novel single-nucleotide variants depends on the proper representation of such variants in the reference database required for base calling [196] ; therefore, a constant update of the reference sequences is required. Despite their implicit advantages, MS-based approaches have not been largely used for assessment of HCV intrahost populations based on the fact that de novo sequencing is challenging and also because of the arrival of more powerful and inexpensive methods [173] . The advent of (NGS) has revolutionized the diagnostics of infectious diseases [197] . The development of massively parallel ultra-deep pyrosequencing allows for a high-resolution snapshot of the intrahost viral population. Additionally, newer approaches have overcome the issue associated with short reads, resulting in longer sequences that facilitate the reconstruction of the viral population [93, 173, [198] [199] [200] . Ultra-deep sequencing is based on a limiting dilution approach and allows rapid sequencing of a large number of variants by eliminating the need to separate molecules and clone into bacterial vectors. Ultra-deep sequencing has been used to identify minor variants in different settings [59, 201] . Different platforms are available, including 454 Life Science (Roche), Illumina, Ion Torrent and Pacific Biosciences. These platforms were initially developed as an alternative to the prohibitively expensive Sanger method. Comprehensive characterization of viral populations is easily accomplished with NGS. The advantages of NGS technologies in virology are numerous. Metagenomics is a growing field in virology that has allowed the characterization of viral populations from different types of samples [202] [203] [204] , especially the detection and characterization of viruses associated with disease outcomes [205] . However, overcoming different challenges is required for the implementation of NGS approaches in the study of viral diseases. NGS amplicon sequencing allows sufficient coverage in order to detect variants occurring with a frequency of <0.1%. The low abundance of nucleic acids is a major problem for the molecular characterization of viruses from clinical samples. The low ratio of viral RNA/host RNA commonly observed in clinical samples is one the main obstacles to using NGS methods. Pre-amplification and probe capture are some of the alternatives available to overcome the relatively low frequency of viral RNA commonly present in clinical specimens. Additionally, viral enrichment and concentration have been used as alternative methods to improve sequencing quality and depth. Nevertheless, whole genome (WG) viral sequencing remains challenging. The analytical sensitivity of WG using NGS approaches is not as easy to evaluate and largely depends on the depth of sequencing [206, 207] . However, increasing the depth of sequencing for an optimized sample preparation can decrease the level of detection. Moreover, the analytical sensitivity depends on the length of the genome. Longer lengths translate into a higher number of potentially available reads, as seen for some studies of viruses. This should also be the case for bacterial and fungal genomes, which could be viewed as an advantage for the detection of such microbes because their concentrations in blood can be very low even in samples from infected patients. Several studies have evaluated the diagnostic sensitivity of this technique. Analysis of the human virome in febrile and afebrile children allowed the identification of a wide range of viruses that correlated with the outcome of disease, as well as virus subtyping [208] . Another advantage of the technique is its capacity to identify co-infections, which is important for the development of adaptive therapeutics [197] . Thus, NGS will likely become a routine test for diagnostics of infectious diseases [209] . For this initiative to succeed, improvements in sample preparation, availability of sequencers in central laboratories and validated pipelines for read sorting and taxonomic assignation must be attained. The study of viral genetics in the era of next generation sequencing promises to help to unveil the most intimate details of the virus-host interface [168] . A variety of computational challenges are commonly encountered in the field of virology owing to the high diversity of viruses, their compact genome organization and their rapid rate of evolution [210] . For HCV, the increasing number of viral sequences available in public and private databases has promoted the development of novel ways to analyze considerably large data sets [211, 212] . Relatively recent technological innovations have ignited an explosion in virus genome sequencing that is likely to help us understand the intimate details of HCV biology and its impact on public health. Nonetheless, any benefits derived from the generation of massive sequence data are hindered by the implementation of resources capable of identifying the sequences, as well as assembling, annotating, curating, maintaining and storing extremely large databases [117, 211, 212] . Importantly, virology has recently drawn the interest of the bioinformatics community, which in turn has led to the development of a variety of tools. This rapid expansion of the HCV sequence universe has forced a recalibration of the data model to better provide extant sequence representation and enhanced reference sequence products to serve the needs of the various viral communities [211] [212] [213] . However, despite the growth in viral bioinformatics, a number of questions remain unanswered, including but not restricted to identification, genome annotation, phylogeny, evolution, and genetic diversity. Viral evolution has many implications for clinical virology. The emergence of HCV resistance mutations is one of the most important challenges for successful antiviral therapy [214] . The molecular mechanisms selecting resistance mutations are complex; therefore, new bioinformatics approaches to characterize HCV evolution both at an intra-and interhost level are required. The integration of bioinformatics methods might lead to predictions of viral evolution in HCV chronic patients based on sequences derived from their intrahost viral populations. Hence, the ultimate goal in HCV genetics is the prediction of the course of HCV evolution that in turn could lead to the customized management of the patient and hepatitis C treatment. Viral phylogenetics have also benefited from the exponential growth of bioinformatics. Phylogenetics is an important area in virology, particularly in HCV molecular epidemiology [21] . However, several characteristics of HCV are challenging for phylogenetics. Significant differences in evolution rates (high over the short term and much lower over the long term), gene transfer, evolutionary virus-host relationship, and the lack of physical \"fossil records\" of viruses (ancient viruses) remain difficult obstacles to overcome. Additionally, phylogenetic trees derived from HCV sequence analyses cannot faithfully represent complex evolutionary relationships relevant to HCV such as horizontal gene transfer, recombination or evolutionary virus-host relationships. Novel phylogenetic approaches have been developed in recent years to better represent such relationships [215, 216] . However, the field of phylogenetics warrants further research to address several aspects related to the reconstruction of HCV phylogeny. Establishing the time of infection is one of the most important and challenging tasks in any epidemiological investigation. However, infection metrics among cases with multiple risk factors and possible exposures are difficult to pinpoint. Therefore, determination of time of infection using molecular data represents an attractive line of research. Estimating the time of infection based purely on genetic data has been reported [92, 217, 218] . However, molecular clock estimates vary significantly, and their reliability depends on a number of factors including sampling, temporal and anatomical distribution of sampling, genome region sequenced, super-or re-infection, and the evolutionary models and algorithms used [119] . As a consequence, the implementation of such approaches for genetic relatedness studies should be undertaken cautiously [119] . A database is a collection of data that is organized so that its contents can easily be accessed, managed, and updated. Many databases for infectious diseases compile data obtained from clinical and/or public health laboratories, aiding the monitoring of infectious disease trends and surveillance [219, 220] . However, even with our growing ability to acquire sequence data, its potential to impact disease surveillance is only fully accessed when data are translated into public health actions. Initiatives such as the Viral Bioinformatics Resource Center (VBRC) aim to close the gaps between massive viral sequencing and bioinformatics are a welcome addition to the set of on-line bioinformatics tools available to the scientific community. These tools provide comprehensive web-based genomics resources that are useful for basic and applied virology research. The VBRC consists of relational databases and web applications for data storage, annotation, analysis, and information exchange. Curation of the data results in the ability to search for gene functions relating to biological genotypes and phenotypes with an emphasis on pathogenesis and provides a variety of analytical and visualization tools [213] . The HCV Database Project was initially funded by the Division of Microbiology and Infectious Diseases of the National Institute of Allergies and Infectious Diseases (NIAID). The HCV database is a resource for the scientific community working on HCV genetics, evolution, variability, and vaccine and drug design [211, 212] . The HCV sequences deposited in GenBank composed the backbone of the database, including information such as country, sampling year, isolate names, genotype and subtype, and host species in addition to relevant annotation information associated with the corresponding publications. Annotation fields in the database include genotype, subtype, start and stop coordinates relative to the reference strain HCV-H, sampling country, sampling city, sampling date and sampling tissue. Patient information is also documented in the database, including health status, age, gender, ALT level, treatment and result, co-infection with HIV and hepatitis B, infection date, infection country, infection city, infection route, infection outcome, HLA type, and epidemiological relationship with other patients. The HCV database contains HCV-associated, hand-annotated genetic data and provides access to the central database via web-accessible search interfaces in addition to a number of analysis tools. The Los Alamos HCV database emerged as a result of the success of the HIV Database Group and uses a similar approach [212] . The infrastructure developed for the dynamic alignment of a large numbers of sequences in the HIV database was quickly implemented for the HCV database [221] . Tools designed for the manipulation of viral sequences (gene extraction, coordinates to reference strains, sequence alignment, and nucleotide or protein motifs scanning) were easily adapted for the HCV database. The information is accessed via an interface allowing for advanced searches with sorting and graphical overview capabilities. Importantly, sequence data can be retrieved as a DNA alignment. Pre-made, manually optimized alignments are also provided in a variety of flavors. Synonymous/non-synonymous substitution analysis for all genes and proteins is available. The Geography tool can plot genotype frequencies based on their geographical origin. Other functions include glycosylation site identification, Modeltest, Treemaker, BLAST, PCOORD, Gene Cutter, Consensus, PeptGen, Motiscan, Primalign, Epilign, Seq-convert, OmniRead, SeqPublish and Sequence locator [211, 212] . Unfortunately, this valuable resource is no longer funded by the NIH; therefore, the herculean effort to properly maintain it has been significantly delayed by the lack of financial support. The Virus Pathogen Database and Analysis Resource (ViPR) supports virology researchers studying select agents and other significant public health pathogens belonging to 14 virus families including HCV [222] . Cross-referencing data and integrated computational tools into the online ViPR resource allow complex analyses. ViPR captures data from external and internal sources and makes them accessible through custom searches. ViPR is primarily focused on viruses of human interest; however, families isolated from other host species are also available for comparison. ViPR features a suite of data analysis and visualization tools to perform custom correlative analyses. ViPR uses the NCBI RefSeq strains to extend the manually optimized annotations to the rest of the taxon. Thus, the ViPR resource provides the scientific community with friendly tools capable of performing complex analytical workflows [222] . Importantly, the requirement for the establishment of a database-type of molecular surveillance network that is feasible for international surveillance networks is the standardization of the input data. Several initiatives involving international partnerships and pilot studies have been initiated for an array of pathogens including HCV [219, 223, 224] . Data sharing is critical for an interdisciplinary approach in order to tackle a problem [223] . The introduction of NGS in routine diagnostics is likely to further boost HCV integrative surveillance. By applying analytical tools to genomic data for HCV, public health scientists can track specific mutations that confer the ability to resist drugs or link them to transmission networks. However, the information provided by raw genomic sequences of pathogens must be integrated with knowledge about the host biology as well as societal and environmental factors in order to understand the etiology of epidemics and to anticipate their trajectories [223] . Therefore, the development of a diagnostic pipeline critically relies on database exhaustiveness that can match the rapid growth of databases prompted by NGS. A typical blast analysis of millions of sequences after de novo assembly into larger contigs against the whole NCBI database using relaxed criteria is time-and resource-consuming for diagnostics. In contrast, stringent mapping of non-assembled reads in a comprehensive database in conjunction with long sequence reads will likely improve the overall process. Thus, an HCV integrative molecular surveillance tool should ideally be web-based and include both viral and host factors as well as epidemiological data ( Figure 6 ) [220] . In addition to international public and private sequence databases, the ideal HCV integrative database should be enriched with data obtained from ongoing molecular surveillance projects and outbreak investigations, supplemented if at all possible with sequence information from cases identified in hepatitis clinics and blood banks [117] . As a result, comprehensive analyses including human genetics, phylogenetics and data mining could be easily attained locally by researchers in the field with minimal effort at an affordable cost for local state public health laboratories ( Figure 6 ). Data mining, or \"Knowledge Discovery in Databases\", is an interdisciplinary subfield of computer science referring to the computational process of discovering patterns in large datasets via artificial intelligence, machine learning, statistics, and database systems [225] . The ultimate goal of data mining is the analysis of tremendous volumes of data to discover hidden patterns and relationships in highly complex datasets, rendering valuable information. This is attained by a series of data pre-processing, model and inference considerations, interestingness metrics, complexity considerations, post-processing of discovered structures, and visualization processes [225] . The application of machine learning methods and advanced statistical modeling to laboratory data and the subsequent discovery of patterns is of relevance for the identification of factors associated with disease outcomes [226] . For HCV infection, decision-tree analysis (a core component of data mining analysis) was used to build predictive models for therapeutic outcomes to antiviral therapy in chronic HCV patients and the severity of disease [227] [228] [229] [230] [231] . Decision-tree analysis relies on a tree-shaped structure representing decision \"calls\" with classifying power, where each internal node denotes an attribute, each branch represents an outcome of the test, and each terminal node a class [227] . Decision-tree analysis facilitates the classification of patients into subgroups that can identify the possibility of an outcome of HCV therapy and, thereby, improve diagnostics [232] [233] [234] [235] . Other approaches, such as linear progression and Bayesian network analyses, have also been shown to produce robust predictions for the progression to fibrosis [229] . Interestingly, HCV epistasis is strongly associated with host factors [230] , suggesting that intrahost viral evolution is convergent and that important traits can be modeled, resulting in predictions with clinical relevance. HCV molecular evolution in many ways affects virus spread and disease transmission. The sophistication achieved by HCV in shaping its molecular evolutionary patterns is intricate and has significantly hindered our ability to fully understand the mechanisms exploited by the virus to ensure transmission. The remarkable HCV mutation rate represents a challenging task for molecular epidemiology. In this new era of advanced sequencing technologies, the implementation of enhanced molecular surveillance is of the utmost importance in order to accurately monitor the circulation of viral strains. Comprehensive molecular studies are required to uncover the key participating elements responsible for virulence. Supplementing molecular data with epidemiological information and host factors significantly improves the accuracy of HCV molecular surveillance. Therefore, the fact that comprehensive epidemiological investigations must be carried out cannot be understated. Improved assessment of HCV intrahost genetic variability should clarify the pathway towards advanced integrative molecular surveillance and ultimately help to unveil the mechanisms driving viral transmission. For advanced molecular surveillance to be truly effective, implementation of preventive and control measures along with therapeutic interventions should be performed. The past 25 years since the discovery of HCV have been quite a journey, and the future of HCV control looks more promising than ever before.',\n",
       "   '_search_score': 0.6960899999999999},\n",
       "  {'_id': 'a1213a37009cf0d561bdfe620cbeabf8de2778d1',\n",
       "   'insert_date_': '2020-11-04T18:48:29.826934',\n",
       "   'abstract': 'The SARS coronavirus (SARS-CoV) spike is the largest known viral spike molecule, and shares a similar function with all class 1 viral fusion proteins. Previous structural studies of membrane fusion proteins have largely used crystallography of static molecular fragments, in isolation of their transmembrane domains. In this study we have produced purified, irradiated SARS-CoV virions that retain their morphology, and are fusogenic in cell culture. We used cryo-electron microscopy and image processing to investigate conformational changes that occur in the entire spike of intact virions when they bind to the viral receptor, angiotensin-converting enzyme 2 (ACE2). We have shown that ACE2 binding results in structural changes that appear to be the initial step in viral membrane fusion, and precisely localized the receptor-binding and fusion core domains within the entire spike. Furthermore, our results show that receptor binding and subsequent membrane fusion are distinct steps, and that each spike can bind up to three ACE2 molecules. The SARS-CoV spike provides an ideal model system to study receptor binding and membrane fusion in the native state, employing cryo-electron microscopy and single-particle image analysis.',\n",
       "   'title': 'Conformational Reorganization of the SARS Coronavirus Spike Following Receptor Binding: Implications for Membrane Fusion',\n",
       "   'body': \"Viral membrane fusion proteins are responsible both for binding to cellular receptors, and the subsequent fusion of viral and cellular membranes. The paradigm for class I fusion proteins consists of two heptad repeat regions, and a hydrophobic fusion peptide [1] . This motif is present in SARS-CoV [2] and other coronaviruses [3] , as well as the hemagglutinin (HA) of influenza [4] , gp21 of human T-cell leukemia virus type 1 [5] , gp41 of HIV [6] , GP2 of Ebola virus [7, 8] , and the fusion protein of paramyxovirus [9] [10] [11] [12] . Class I viral fusion proteins can also be divided into two sub-types; those whose fusion mechanism is low pH-dependent such as influenza HA, and those that are pH-independent like the retroviral fusion proteins. In retroviruses, receptor binding itself can trigger fusion, with temperature and redox conditions also influencing the fusion mechanism [13, 14] . The SARS spike appears to be insensitive to redox conditions [15] . For SARS-CoV, it is proposed that the virus is internalized in the cell by endocytosis, and is then exposed to a low pH environment, and it is postulated that proteolytic cleavage between the S1 and S2 domains initiates the membrane fusion process [16] . Although the factors which trigger fusion (endocytosis, pH sensitivity, single receptor vs. primary and co-receptor binding, redox change) differ amongst diverse virus families, all viral fusion proteins are thought to share the same basic fusion mechanism [1, 4, [17] [18] [19] [20] . A notable feature of the SARS spike is its large mass (,500 kD per trimer) and striking, club-shaped appearance, from the end-on, this appears like a three-bladed propeller with a radius of 90  [21] . Despite the structural differences, the SARS spike performs the same fundamental task in viral entry to the host cell as other class I viral fusion proteins, such as the influenza HA (,220 kD per trimer). The SARS spike can be subdivided into four structural domains (from N to C terminus); the two large external domains S1 and S2 are largely responsible for receptor binding and membrane fusion, respectively. In many class I viral fusion proteins the analogous peptides are generated by proteolysis of the spike precursor during the maturation process in the host cell, resulting in two peptides with the fusion peptide on the N-terminus of S2. In SARS-CoV the S1/S2 assignment is given based on sequence homology to other viral fusion proteins, however there is no peptide cleavage during maturation. The final two small domains are comprised of a transmembrane domain, and a carboxyterminal cytoplasmic domain, which principally anchor the spike to the viral envelope. The cell-surface molecule ACE2 is the receptor for the SARS spike protein [22] and is a relatively large macromolecule with a diameter of 70  . In comparison, the receptor for influenza HA, sialic acid, is much smaller with a 10  diameter. The precise mechanisms by which class I viral fusion proteins gain access to the host cell remain unknown. The hypothetical entry process includes several steps that take place in sequence: receptor binding, fusion core re-arrangement, fusion peptide insertion in host cell membrane, refolding of heptad repeats, membrane fusion, and finally viral nucleocapsid transfer [23] . The structures of ACE2 bound to a fragment of the SARS spike containing the receptor-binding domain and the pre-and postfusion configurations of the fusion core heptad repeats of the spike have been solved to atomic resolution [2, 3, [24] [25] [26] . In addition, the atomic resolution structures of two neutralizing antibodies bound to the SARS spike receptor-binding domain have been solved [27, 28] showing that blocking of the receptor binding domain, preventing attachment of virions to cell-surface ACE2, is the likely mechanism of virus neutralization by these antibodies. The aim of the present study was to delineate any possible structural changes in the SARS spike that accompanied receptor-binding, and to precisely localize the receptor binding domains (which are only 14% of the total mass of the spike) within the overall structure, employing cryo-EM and image processing of intact virions bound to soluble ACE2. We demonstrate the structural dynamics which accompany spike-receptor binding, which may be involved in triggering membrane fusion. To study the structure of the spike-ACE2 complex, we infected Vero E6 cells with SARS-CoV and purified supernatant virus by iodoxanol density gradient centrifugation. Previously, we have shown that specimens could be c-irradiated with a sufficient dose (2Mrad) for viral inactivation, while still preserving protein antigenicity. Inactivation was verified by passage in cell culture and testing by quantitative PCR [29] . Purified c-irradiated SARS-CoV preparations had a fusogenic activity when added to Vero E6 cells at high multiplicity (,500-3000 virions per cell), causing the formation of syncytia in the absence of replication or cytopathic effects ( Figure 1A ). These syncytia had identical morphology to those observed in cytopathic studies of SARS-CoV in tissues and in tissue cultured cells; syncytia have also been observed upon expression of coronavirus S protein, or following addition of cells expressing the S-protein to cells with surface-expressed ACE2. [22, [30] [31] [32] [33] [34] . The same inactivated c-irradiated virus preparations were incubated with a human ACE2-human Fc chimeric protein, for analysis of the complex by cryo-EM, 3D image processing, and immuno-electron microscopy (immuno-EM) ( Figure 1 , Movie S1). The ACE2-Fc construct was selected since it is soluble and dimeric, and it was anticipated that the additional mass was would be beneficial in locating the receptor attached to the spike. Immuno-EM confirmed that recombinant ACE2 protein bound to virions, and that binding affinity was not affected by c-irradiation ( Figure 1B) . We did not observe any structural changes associated with prior irradiation. Image averaging further suppressed any possible random radiation induced structural changes. Cryo-EM coupled with 3D single particle image analysis was employed to investigate the binding of ACE2 to the spike of SARS-CoV. Although SARS-CoV is approximately spherical when observed by cryo-EM, both its size and shape vary slightly, and thus it is not amenable to single-particle averaging techniques. However, individual spikes on the surface envelope of the virus do provide a repetitive structure that is ideal for single particle techniques. Spikes on the surface of virus particles are readily imaged by cryo-EM in the frozen-hydrated state ( Figure 1C ). Three-dimensional image processing was carried out on 11,153 selected spike images taken at various defocus levels, using routine single-particle image processing [35, 36] . The resolution of the final 3D structure was evaluated by a Fourier shell correlation of 0.5 to be 18.5  (Figure 2 ). The structure of the unbound spike was compared with that of the spike-ACE2 complex ( Figures 1E, 1F , 3, Movie S2). In both cases the spike portion of the 3D structure is fairly similar, indicating that ACE2 binding does not result in a fundamental structural unfolding of the spike. However, the overall height of the spike was reduced from 160  to 150  following ACE2 binding. When viewed from the end-on perspective and the spike undergoes a rotation of ,5u following binding (Movie S2), and the mass at the center of the axis of symmetry on the distal end of the spike redistributes itself from one small central blob to three blobs or nubs (Figure 3 The precise location of ACE2 binding on the distal end of the spike is centered at 70  from the central axis of the spike, with a 30  gap between the axis of symmetry and ACE2. One ACE2 molecule was bound to each of the three propeller-like blades of the spike, making a structure 220  high ( Figure 1F ). The density of these ACE2 molecules is high in the 3D density map, indicating a high occupancy of binding sites. In addition, the structure shows that binding of one ACE2 to the spike would not sterically hinder binding of additional ACE2 molecules on the other two propellerblades of each trimer. The cryo-EM 3D structures of the spike and the spike-ACE2 complex were combined with the atomic resolution structures of the SARS spike receptor-binding domain-ACE2 complex [24] and the heptad repeat pre-and post-fusion cores [2, 25] to interpret the cryo-EM structure. The receptor-binding domain-ACE2 data were docked with a correlation score of 0.965 using the SITUS software package ( Figure 4 ) [37] . As expected the receptor-binding domain docked to the distal end of the spike with ACE2 filling the extra mass on the spike (shown by the color violet in Figure 4 ). The empty upper region of the mass is likely composed of the Fc component of the chimeric protein. Although this chimeric molecule is dimeric, only one leg of the ACE-2 is able to bind to each of the three receptor binding domains of the trimer. We anticipate the mass distal to the hinge region to be flexible, and therefore components were blurred out in the averaging process, leaving only a portion of the additional mass in the 3-D map. The C-terminus of the docked ACE2 is in a location consistent with this interpretation. Previously we modeled the putative location of the receptor-binding domain and ACE2 in the cryo-EM structure of the unbound spike using solely the receptor-binding domain atomic resolution data [21] . In the present study, we have shown that our previous prediction of the location of the receptor-binding domain was only partially correct. The actual location of the receptor binding domain and ACE2 are in fact shifted 29  towards the 3-fold axis, corresponding to about 40% of the diameter of ACE2. The results from the present study will aid in the precise location of future atomic resolution structure fragments of the SARS spike, as well as in the structural mapping of neutralizing antibody-binding epitopes (which are in progress). The coordinates of the three dimensional reconstruction of the SARS spike (EMD-1423) and the SARS spike-ACE2 complex (EMD-1425) have been deposited in the macromolecular database (http://www.ebi.ac.uk/ msd). Our cryo-EM results demonstrate that there is a structural transition of the spike that occurs upon receptor binding (Figure 3 , Movie S2). The overall height of the spike is reduced by 10  due to a shift in the mass in the S1 domain (Figure 3 D,I) and the three S1 blades of the spike twist ,5u along the axis of symmetry. On the distal end of the spike, a small blob feature is replaced by three small nubs which appear radially. (Figure 3 F,J) . We propose that this structural re-arrangement represents an initial ''priming'' of the spike for membrane fusion, which brings the cellular and viral membranes 10  closer to each other to facilitate fusion. The coronavirus S protein is known to mediate both virion-cell membrane fusion, and intercellular membrane fusion [22, [30] [31] [32] [33] and in the present study we have shown that purified inactivated virions are also fusion-competent. We show that the addition of SARS-CoV virions to cultured cells that express ACE-2 can induce fusion, presumably from the fusion activity of spikes on virions bridging neighboring cells. Similar observations on the induction of ''fusion-from-without'' were made when cells surfaceexpressing the spike protein were mixed with cells expressing ACE-2 [35] . The trigger that initiates membrane fusion varies amongst class I fusion proteins, with some activated by receptor binding, others by pH, and others by redox conditions. In most proposed models of membrane fusion it is postulated that the S1 domain or analogous receptor binding domains dissociate from the spike during the membrane fusion process. This dynamic process was demonstrated for influenza HA by Kemble et al. [38] in their investigation where they engineered intermonomer disulfide bonds between the HA S1 subunits. The result of this was that fusion activity was impaired; however it could be restored under reducing conditions. It is likely that the SARS spike shares a similar mechanism, and the structural changes that we have detected represent the initial step in this process. By analogy with other class I viral fusion proteins, we anticipate that the fusion core of the SARS spike must undergo similar structural re-arrangements during fusion. The receptor-binding domain is localized in a position on the distal end of the molecule, closer to the 3-fold axis than anticipated, yet still in a position that would not impede these structural re-arrangements. Putative mechanisms by which class I viral fusion proteins achieve membrane fusion have been proposed [1, 4, [17] [18] [19] , but complete structural evidence for the role of intermediate structures in these mechanisms has yet to be obtained. The structural biology of this process has been best characterized for the influenza hemagglutinin, and paramyxovirus fusion (F) protein, for which the prefusion and membrane fusion pH structures have been determined by X-ray crystallography [4, 11, 12, 39, 40] . All of the subsequent models for class I viral fusion proteins are based on the structural data of these two fusion proteins. A drawback in all of these models is that they are based on recombinant ectodomains that are not proven to exist as a component in the complete molecule, and they lack both membrane-interacting residues, and lipids [4] . The cryo-EM structures presented in this investigation are of intact SARS spike bound to native virion lipid envelopes. The cryo-EM data are very instructive when atomic resolution fragments are docked within the overall molecule, especially as the entire SARS spike has proven to be a difficult subject for X-ray crystallography, and atomic resolution data exist for only a few fragments of the SARS spike. Our cryo-EM results show that it is possible for the spike to attach to three ACE2 receptors at once; this may serve to hold on to the host membrane like a tripod so as to accurately orientate the fusion core. In addition the 30  gap between the axis of symmetry and ACE2 provides sufficient space for fusion core rearrangement. Damico et al., [14] demonstrated that the kinetics of binding of the Rous sarcoma virus envelope protein to its receptor suggested that binding of each spike molecule to multiple receptor monomers occurred. It may therefore be possible that other structurally homologous viral envelope proteins can also bind multiple receptors, and this may be a general adaptation that provides the correct temporal and spatial arrangement to bring about membrane fusion. The observation that the SARS spike could bind three soluble ACE2 receptors provides three possible binding states with one, two or three membrane-bound receptors attached to the spike. With one or two receptors bound, the spike and virus have a wide range of movement possible. Whereas, in the case of three bound receptors the spike and its fusion core will be arranged perpendicular to the cell surface with minimal movement possible. It is interesting to note that binding three receptors is the minimum number of binding events required to achieve this perpendicular orientation in three dimensional space. This observation matches up with the conserved trimeric structures of class I fusion proteins which are common amongst enveloped viruses, thus indicating a possible conserved structural-functional relationship may exist. Another line of reasoning is that oligomerisation favors a better ''chance'' of successful receptor binding and subsequent insertion of the fusion peptide into the host membrane. According to this model, oligomeric multiples higher then three would be favored, whereas the ''tripod'' model of receptor binding favors the conserved trimeric structures of class I viral fusion proteins. In Figure 3 we have we have presented the ACE2-SARS structures as they were solved in this cryo-EM investigation. For other fusion proteins like influenza HA1 and HIV GP120 it has been modeled that the re-arrangements upon membrane fusion are dramatic involving a shedding of the above mentioned domains. In this investigation we have detected structural movement of S1 upon ACE2 binding. This could represent the initial phase of this dramatic process that is followed by fusion core re-arrangement, fusion peptide insertion in the host cell membrane, refolding of heptad repeats, and finally membrane fusion. Analyzing the structure of the spike-receptor complex is essential to understanding how viruses can adapt to utilize receptors from different species and how they may evolve to gain specificity for new receptor types. RNA viruses have a high rate of mutation and recombination [41] . In SARS coronaviruses, the spike is able to retain specific binding affinity for the ACE2 of more than one host species, and rapid evolution to gain specificity for novel ACE2 species has been demonstrated [42, 43] . The structure and large size of the spike of coronaviruses appears to be an adaptation related to utilization of large host cell-surface molecules such as ACE2 as specific receptors. Amongst the Coronavirus family, specific cell surface receptors for the spike protein are all in the range of 60-110 kD [44] . These large host receptor molecules are in turn functionally constrained and thus conserved across species barriers. In utilizing binding to a large receptor molecule, it may be important that the spike S1 domain also acts as a ''spacer arm'' holding the receptor far enough away from the three-fold axis of symmetry of the spike S2 domain to permit fusion core re-arrangement and subsequent membrane fusion. Such a property necessitates having a large spike molecule. Moreover, multiple receptor binding may have functional significance, enhancing the binding and entry of viruses. Crosslinking of adjacent host receptor molecules could increase the affinity of the virus for its target cell, as well as improving the kinetics of fusion. The SARS spike-ACE2 complex is an ideal model system for the investigation of class I viral fusion protein dynamics, utilizing cryo-EM to investigate fusion protein binding and structural changes in the native state, within virion membranes that are fusogenic. The presence of a common cell fusion mechanism shared by diverse virus families holds the prospect of developing broad-spectrum anti-virals that target these conserved mechanisms, as well as the potential use of these fusogenic proteins in drug delivery systems. SARS-CoV (Tor3 strain) was inoculated onto Vero E6 cells grown in Dulbecco's modified eagle medium supplemented with 10% foetal bovine serum, 100 U/ml penicillin, 100 mg/ml streptomycin, and 0.22 mg/ml l-glutamine at 37uC with 5% CO 2 . Cell supernatant was harvested (300 ml) with a SARS-CoV TCID 50 titre of 5610 7 /ml and virus was pelleted using a Beckman SW32 rotor at 28000 rpm for 90 minutes. The virus pellet was resuspended in PBS buffer, layered onto of a 12-30% iodoxanol gradient (Optiprep) and centrifuged in a SW40 rotor at 38000 rpm for 2.5 hours. Fractions (0.7 ml) were collected from the gradient with an Auto Densi-Flow gradient collector (Labcono) and dialyzed against PBS. SARS-CoV-enriched fractions were checked by SDS-Page and Western blotting, and rendered noninfectious by irradiation in a gamma cell on dry ice with a 2Mrad exposure for 90 minutes. Irradiated specimens were tested for infectivity by inoculation onto Vero E6 cells, and examined for cytopathogenic effects for 10 days, followed by blind passage of the cells and testing for the growth of SARS-CoV by PCR [29] . To test for spike fusogenic activity Vero E6 cells were cultured in 96 well plates (Corning). Once the cells were confluent the medium was removed from each well and 20 ml of gamma irradiated SARS-CoV containing 10 10 pfu/ml diluted 1:1 and 1:9 in culture media. The virus dilutions were incubated with the cells for 1 hour at 37uC. After the incubation the plates were washed three times with fresh culture medium to remove unbound virus. The vero E6 cells were observed by light microscopy 2 and 4 days post treatment for the presence of syncytia. Virus growth was not detected at either dilution of irradiated virus, and no cytopathic effect was observed. The presence of nucleic acid in syncytia was investigated by propidium iodide staining and observation by fluorescence microscopy. The human ACE2-Fc (human IgG1) recombinant protein (Adipogen) had an initial concentration of 1 mg/ml in PBS and was diluted 1:10 in PBS for incubation with SARS-CoV, for the remainder of this document this is referred to as ACE2. 2 ml of SARS-CoV were injected into 4 ml of PBS on a formvar-carbon coated 400-mesh nickel grid, and incubated for 1 minute. All incubations were carried out at 20uC. Grids were then washed in PBS six times for 1 minute, followed by a 10-minute block in PBS-G-BSA (PBS pH7.2, 0.2% glycine, 2% BSA). Grids were then washed in PBS-G (PBS pH7.2, 0.2% glycine) six times for 1 minute, followed by a 1-hour incubation on a 20 ml drop of ACE2. Grids were then washed in PBS-G (PBS pH7.2, 0.2% glycine) six times for 1 minute, followed by a 1-hour incubation with rabbit anti-human ACE2 polyclonal IgG (AdipoGen; diluted 1:200 in PBS). Grids were then washed in PBS-G (PBS pH7.2, 0.2% glycine) six times for 1 minute, followed by a 30 minute incubation with goat anti-rabbit IgG (conjugated to 10 nm gold, Sigma; diluted 1:10 in PBS). Grids were then washed in PBS-G (PBS pH7.2, 0.2% glycine) three times for 1 minute. Grids were fixed for 2 minutes (1% paraformaldehyde, 2% glutaraldehyde in PBS), washed in deionised water and negatively stained with 2% methylamine tungstate (Nanoprobes), and observed by transmission electron microscopy in a Tecnai 20 G2 transmission electron microscope (FEI) operated at 200 kV, digital images were collected using an AMT Advantage XR-12 digital camera. SARS-CoV was incubated with ACE2 for one hour, and 4 ml samples were applied to glow-discharged holey carbon films supported on 400-mesh copper grids. After blotting immediately for 2-5 seconds with filter paper, grids were plunged into liquid ethane cooled by liquid nitrogen, using a custom built gravityoperated freezing device. Specimens were transferred to a Tecnai 20 G2 transmission electron microscope (FEI) operated at 200 kV, equipped with a Gatan 626.DH low-temperature specimen holder. Observations were made at temperatures of ,2185uC and images recorded at 29,0006 magnification on Kodak SO-163 electron image film at a dose of 10-20 electrons/ 2 with an exposure of 1-2 seconds. Film was developed in Kodak D19 for 12 minutes at room temperature, rinsed for 2 minutes in water, and fixed for 10 minutes with Kodak fixer. The exact magnification in the microscope that SARS-CoV was imaged at was determined to be 29,9686 using a calibration grid (Pelco International). Images of SARS-CoV were digitized on a Nikon super coolscan 9000 ED scanner at a pixel size of 2.125 . Processing was carried out using the EMAN and SPIDER/WEB image processing program packages [35, 45] on SGI Fuel and Tezro workstations running IRIX 6.5, or a Dell PE6850 with 4way 64-bit Xeon Dual Core 3.6 GHz CPUs and 32GB Ram running Linux (Fedora Core 5). Images were corrected for contrast transfer function (ctf) using the ''ctfit'' function in the EMAN software package, which estimates defocus and corrects for ctf by phase-flipping. The data set composed of images centred on spikes-ACE2 (n = 11,153; recorded at 2.9-11.9  defocus) were processed using SPIDER by the projection matching approach [36] which incorporated three-fold symmetry into the search function and reconstruction procedure. After each refinement cycle the new reference volume was either theresholded or segmented using the ''floodfill'' function in the SITUS software package [37] to select the spike-ACE2 structure and minimize the influence of neighbouring spikes in the reference structure that was used in the subsequent iteration of the projection match alignment. The initial reference model used in this procedure was the structure of the SARS spike that we previously solved by cryo-electron microscopy [21] . The population of spike images was composed of both side view projections as well as end-on projections, with the latter being easily identified in the images with greater defocus values. These high-defocus/end-on images were essential to eliminate the ''missing cone'' from the reconstruction. The resolution of the cryo-EM reconstruction was estimated by Fourier shell correlation to be 18.5 using the 0.5 FSC criteria. The structures of the pre-fusion HR2 domain and the ACE2-receptor-binding domain were docked into the cryo-EM reconstruction using the ''floodfill'' and ''colores'' functions in the SITUS software package. The resultant docking generated one distinct location for the ACE2-receptor-binding domain, which placed ACE2 in the additional mass that was not present in the reconstruction of the spike alone. The docking of the entire 2FXP.pdb [2] structure was previously conducted [21] . Three-dimensional cryo-EM reconstructions, and the atomic resolution structures 2AJF.pdb [24] , 2FXP.pdb [2] , were visualized using UCSF Chimera (Computer Graphics Laboratory, University of California, San Francisco, supported by NIH P41 RR-01081) [46] . The spike component of the spike-viral envelope reconstruction was segmented from the entire reconstruction using the ''floodfill'' algorithm in the SITUS software package. Movie S1 of the rotating SARS spike-ACE2 complex was generated on a Silicon Graphics workstation using UCSF Chimera, and the ''mediarecorder'' and ''mediaconvert'' software provided with the Silicon Graphics IRIX 6.5 operating system. Movie S2 showing the morphing of the spike between the SARS spike and SARS spike-ACE2 complex was generated using the UCSF Chimera software package with the ''morph map'' morphing tool plug-in. Both reconstructions were masked with a cylindrical mask, which when applied masked the peripheral boundaries of the SARS-CoV envelope in both reconstructions. Movie S1 Three-dimensional reconstruction of the of the SARS spike-ACE2 complex. This movie supplements Figs. 1, 4 showing both the reconstruction of the spike-ACE2 complex as well as the location of the docked atomic structures. The movie starts showing the spike-ACE2 complex viewed from an end-on perspective. The reconstruction then tilts 90o, followed by a 120o rotation along the three-fold axis of symmetry of the spike. In the second section of the movie the cryo-EM reconstruction becomes transparent revealing the location of the docked atomic structures, followed by a final rotation of 90o back to the initial end-on perspective. The following color scheme was used: cryo-EM surface; ACE2, violet; spike, green; stalk, blue; envelope, beige; nucleocapsid, red. Atomic Structures: PDB ID code, 2AJF, (SARS spike receptorbinding domain, red; ACE2, white; ACE2 C-terminus, blue); and PDB ID code, 2FXP, HR2 fusion core, yellow. Found at: doi:10.1371/journal.pone.0001082.s001 (7.24 MB MOV) Movie S2 Three-dimensional morphing of the SARS spike, and ACE2 bound structures. This movie supplements Figs. 1, 3 showing the putative structural re-arrangement of the spike as it binds to ACE2. The movie starts showing the SARS spike and then gradually morphs to the SARS spike-ACE2 complex, and back to the SARS spike. The morphing reconstruction is shown from the side and end-on perspectives, and the mass attributed to the nucleocapsid is presented without morphing. As one watches the move play back and forth (this can be done by selecting ''loop'' in a QuickTime Player) one can see the spike twist by ,5o when viewed from the end-on perspective, and the spike becomes squatter by ,10 &#x00C5; when viewed from the side perspective. The following color scheme was used: cryo-EM surface; ACE2, violet; spike, green; stalk, blue; envelope, beige; nucleocapsid, red. \",\n",
       "   '_search_score': 0.6960299999999999},\n",
       "  {'_id': 'a6203f5bc0f1ac39f4c722a6d9aa6918858f5de1',\n",
       "   'insert_date_': '2020-11-04T18:54:06.070979',\n",
       "   'abstract': 'The outbreak of a new coronavirus (SARS-CoV-2) disease has become pandemic. To be more effectively controlling the disease, it is critical to set up an optimal quarantine period so that about 95% of the cases developing symptoms will be retained for isolation. At the moment, the WHO-established quarantine period is 14 days based on previous reports which had studied small sizes of hospitalized cases (10 and ~100, respectively), however, over 80% of adult-and 95% of child-cases were not necessary to stay in hospitals, and therefore, had not been hospitalized. Therefore, we are questioning if the current-inferred median incubation time is representative for the whole Covid-19 population, and if the current quarantine period is optimal. We compiled and analyzed the patient-level information of 2015 laboratory-confirmed Covid-19 cases including 99 children in 28 Chinese provinces. This cohort represents a wide-range spectrum of Covid-19 disease with both hospitalized and non-hospitalized cases. The full range of incubation periods of the Covid-19 cases ranged from 0 to 33 days among 2015 cases. There were 6 (0.13%) symptom-free cases including 4 females with a median age of 25.5 years and 2 males with a median age of 36 years. The median incubation period of both male and female adults was similar (7-day) but significantly shorter than that (9-day) of child cases (P=0.02). This cohort contained 4 transmission generations, and incubation periods of the cases between generations were not significantly different, suggesting that the virus has not been rapidly adapted to human beings. Interestingly, incubation periods of 233 cases (11.6%) were longer than the WHO-established quarantine period (14 days). Data modeling suggested that if adults take an extra 4-day or 7-day of isolation (i.e., a quarantine period of 18 or 21 days), 96.2% or 98.3%, respectively, of the people who are developing symptoms will be more effectively quarantined. Patients transmitted via lunch/dinner parties (i.e., gastrointestinal tract infection . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)',\n",
       "   'title': 'Is a 14-day quarantine period optimal for effectively controlling coronavirus disease 2019 (COVID-19)?',\n",
       "   'body': \"The outbreak of a new coronavirus (SARS-CoV-2) disease (Covid-19) has become a worldwide pandemic. Understanding the characteristics of the incubation period is crucial to design public health efforts for effectively isolating the cases and contacts to control outbreaks of Covid-19. SARS-CoV-2 can be transmitted by either respiratory droplets, contaminated surfaces, and objects or an oral-fecal transmission route to lead gastrointestinal tract infection 1 Because outbreaks are being occurred in many countries, it is critical to analyze real-time data of the Covid-19 cases to gain insights to inform public-health intervention strategies for managing and preventing the disease. One of the key issues for controlling Covid-19 is to set up a suitable quarantine period which could capture about 95% of the cases developing symptoms so that the practice of isolations will be more effective and save public health resources. Thus far, the incubation time studies have been focused on hospitalized patients who were highly selected based on medical conditions, and further, sample sizes for inferring incubation times were small, for example, 10 and ~100, respectively 3, 4 . However, the majority (i.e., about 80% of adults and 95% of children) of the Covid-19 cases were not necessary to be hospitalized, and therefore, had not been hospitalized due to their mild conditions. Here, we asked if the median incubation time inferred from the hospitalized patients are representative for the whole population of Covid-19, and if the current quarantine period determined based on hospitalized patients is optimal. To answer these questions, we manually curated the patient-level data from the case reports on official websites of China's provincial public health agencies. This cohort represents a general population of Covid-19 cases without a selection based on medical conditions. The cohort contained 2015 cases including 99 children who had symptom onsets between January 1, 2020, and February 25, 2020. All the curated cases contained exposure, onset, and diagnosis dates. We analyzed the incubation time for the Covid-19 population containing both hospitalized and nonhospitalized cases. . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review) The copyright holder for this preprint this version posted March 18, 2020. we read the case reports in Chinese to curate age, sex, demographic information (i.e., name of the provinces and cities), dates of exposures, symptom onset and diagnosis (i.e., confirmed the SARS-CoV-2 infection through laboratory-tests), when available. The patient data were curated by XJ, WXC, and YCC who are native Chinese speakers and translated the summary of each patient into English. To make sure the data accuracy, the curations have been cleaned and checked 4 times by four other researchers, YWN, XL, LL, and EW. We mainly focused on the cases which were allowed to inferring incubation time. Among 4567 cases, after removing the cases which did not have exposure dates, or whose exposure dates were not sure, we obtained 2015 cases whose incubation times could be inferred. Because the information of Covid-19 cases in Hubei province including Wuhan, the city at the center of the epidemic, was not publicly available, this cohort included only non-Hubei cases. Study design and data We curated the patient-level data as described above and characterized patient baseline information such as incubation periods. All the data in this study have been deidentified and publicly available through local Chinese health agencies, therefore, patient consent and ethics approval was not required. The incubation period of a Covid-19 case is the time between infectious exposure and symptom onset. Over 2,000 cases had sufficient information for inferring incubation periods. Most of which had clear records for the dates of exposures and symptom onsets. For imported cases (e.g., travelers from Wuhan city), the exposure dates were defined as the dates of leaving the epicentral cities such as Wuhan. For the cases, who had been quarantined and monitored for the disease, the diagnosed dates were set as the dates for their symptom onsets. 2 tests and t-tests were used for statistical analysis using R, a free software environment for statistical computing. We considered p values of less than 0.05 to be significant. . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint this version posted March 18, 2020.  We collected patient-level data for 2015 Covid-19 cases reported between January 1 and February 25, 2020 (Table 1, Supplementary Table 1) . 887 (44.0%) cases were female adults with a median age of 46 years (IQR 41.5-50.5), 1029 (51.0%) were male adults and the median age was 45 years (IQR 40.5-50.5), and 99 (4.9 %) cases were children with the median age was 9 years (IQR 7-11) with a boy-to-girl ratio of 1.3:1. 774 (38.4%) were imported cases, most reported a travel history to Wuhan (133, 17.2%) or were residents of Wuhan (641, 82.8%), while 240 (11.9%) of the imported case were either the residents or returning travelers from non-Wuhan regions. The incubation period is essential in the control of infectious diseases. To halt a pandemic, a suitable quarantine period should be set by investigating incubation periods. Thus, understanding the characteristics of the incubation period is crucial to design public health efforts. In this cohort, the full range of incubation periods of the Covid-19 cases ranged from 0 to 33 days among 2015 cases. Both male and female adults had their median incubation periods of 7-day, which is 1.8-day longer than those in other reports 3,4 (see discussion). However, a similar result was released in the recent news from the Chinese Medical Association that the median incubation periods were 5-7 days 5 . The median incubation period (9-day) of children were significantly longer than those of adults (t-test P = 0.02). For adults, the incubation periods for the majority (66.5%) of the cases were 1-9 days: most of them (57.6% and 52.0% for males and females, respectively) had 1-5 days incubation. Importantly, we found that the incubation periods of 11.5% (n=233, 123 and 105 were male and female, respectively) of the cases were longer than 14 days, the current WHO-set 14-day quarantine period. These results agree with a recent study which also reported 12.7% (n=13) of the cases whose incubation periods were longer than 14 days 4 . These data suggested that the official 14-day quarantine period only captured 88.5% of the population developing Covid-19. However, among the 233 cases with longer incubation periods, most (n=198, 85.0%) had 15-21 days of incubation periods, suggesting that implementation of the 21-day quarantine period for everybody will capture 98.3% of the cases. However, we knew that fewer children were infected, . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint this version posted March 18, 2020. . https://doi.org/10.1101/2020.03.15.20036533 doi: medRxiv preprint we plotted the age-distribution of the cases having a longer quarantine period (Fig 1) . Based on Fig 1, we concluded that if adults take an extra 7 days for quarantine, we will capture 98.3% of the cases. Data modeling also showed that a 17-day or 18-day quarantine period will capture 94.8% and 96.2%, respectively, of the cases. The incubation period of 2.0% (n=19 and 20 for male and female adults, respectively) of the total cases was 0-day, while some cases were transmitted by having contacts with Covid-19 patients who had not developed symptom onset yet, suggesting that SARS-CoV-2 might be contagious (i.e., is shedding virus) during incubation periods, and these patients could be more susceptible to this virus. There were 6 (0.13%) symptom-free cases including 4 females with a median age of 25.5 years (IQR 24.0-27.0) and 2 males with a median age of 36 years (IQR 24.0-48.0). These symptom-free cases were much younger than the majority of the patients. The fecal-oral transmission has been found in Covide-19 cases 1 . Food sharing in lunch/dinner parties could facilitate fecal-oral transmission. Interestingly, the incubation time (9-day IQR 7.75-10.25) of the cases transmitted by lunch/dinner party contacts was significantly longer than others (P=0.03), suggesting that oral transmission via sharing food had longer incubation periods than transmitting via respiratory droplets. There were 4 generations of transmitted cases based on contact tracing information. The Wuhanimported cases were defined as the 1-generation cases, while the 2-generation cases included both non-Wuhan imported cases, and new cases which were transmitted directly by a 1generation case or an unknown cause. The 3-generation cases were the ones transmitted directly by a 2-generation case. The incubation time was not significantly different between the patients of transmission generations, suggesting that the virus is not rapidly adapted to humans. The incubation period is essential in making intervention strategies to control infectious diseases. Thus, understanding of incubation time of a wide-range spectrum of Covid-19 for establishing an optimal quarantine period is crucial to designing public health interventions. Two recent studies have reported incubation periods for Covid-19 cases 3, 4 , but only for the hospitalized patients, . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint this version posted March 18, 2020. . https://doi.org/10.1101/2020.03.15.20036533 doi: medRxiv preprint who were highly selected and represented only 20% of adult-and 5% of child-cases of the general Covid-19 population. Further, the sample size (about 100 and 10 cases, respectively) for inferring incubation periods were small in these studies. Here, we manually curated a Covid-19 cohort containing baseline and incubation information of nearly 2,000 patients including 99 children. This cohort included both hospitalized and non-hospitalized cases. By analyzing this cohort, we characterized the incubation times of Covid-19 adults and children as well, and examined the effectiveness of the 14-day quarantine period. The median incubation period for adults in this cohort was 7-day, which was longer than 5-5.2 days reported previously 3, 4 . Recently, the Chinese Medical Association also found that the coronavirus median incubation period was 5-7 days 5 . Kids had an incubation period of 9-day and significantly longer than adults. Based on previous studies, WHO has set a 14-day quarantine period for Covid-19. However, we found that that the incubation periods of 11.5% (n=233) of the cases were longer than 14 days, meaning that the current quarantine policy only stops 88% of the cases for virus spreading. This finding is similar to another report showing that 13 (12.7%) cases had their incubation time longer than 14 days 4 . In general, a quarantine period that retains about 95% of the cases developing symptoms will be more effective 6 . Thus, we further analyzed the subset (n=233) and found that 98.7% and 97.9%, respectively, of the men and women cases who are developing the disease, will be quarantined, if everybody or adults alone, respectively, take an extra 7-day of isolation (i.e., a quarantine period of 21 days). If taking a quarantine period of 17-day or 18-day, 94.8% and 96.2%, respectively, of the cases developing symptoms will be successfully quarantined. As the outbreak is fast-moving in the world, based on this analysis we recommended that an extension of adults' quarantine period to 17 or 21 days could be more effective. Of note, the outbreak occurred during the Chinese New Year holiday, further the holiday period had been extended to another 10 days 7 . Moreover, until now (as of March 15, 2020) schools and universities have been still shut down. Thus, due to the implementation of the aggressive social distancing measures many people in China have been isolated for more than 60 days. . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint this version posted March 18, 2020. . https://doi.org /10.1101 /10. /2020 The full range of incubation periods of the Covid-19 cases ranged from zero to 33 days in this study, including ~30 cases whose incubation periods ranged 22-33 days, and 6 symptom-free cases, suggesting that a wide-range heterogeneous response to the SARS-CoV-2 virus in the population, and the challenges for preventing virus-spreading. Furthermore, some cases start to transmit this virus before the symptom onset. This is very different from the transmission patterns of SARS (Severe Acute Respiratory Syndrome), for which transmission rarely occurred until after the 4-5 days after symptom onset 8 . When a virus has infectiousness before symptom onset, control of outbreaks using contact tracing and isolation is more challenging 9 . Interestingly, the cases transmitted by lunch/dinner party contacts had a 2-day longer incubation period than the cases transmitted via respiratory droplets. The Chinese style dish-and foodsharing during lunch/dinner parties can easily contaminant the shared dishes by virus-carriers and thus, could facilitate oral transmission for leading gastrointestinal tract infection, which has been reported as a transmission means for Covide-19 1 . It has been reported that 10% of patients presented with diarrhea and nausea 1-2 days before the development of fever and respiratory symptoms 10, 11 . These observations explained why the cases transmitted by lunch/dinner party contacts had significantly longer incubation periods. Therefore, the early signs of the cases transmitted by lunch/dinner parties could be diarrhea and nausea which often develop before fever and respiratory symptoms in these patients. Several limitations of this study are the following: (1) the cohort did not include patient-level information from the patients in Hubei province including Wuhan city, because they were exhaust-dealing with the outbreak and could not make the information in their official websites. (2) all the cases are Chinese patients. Acknowledgments: EW and XJ designed the project. XJ, WXC, and YCC manually curated data and translated the summary of each patient into English, while YWN, XL, LL, and EW checked the curations and cleaned the data. XJ and YWN conducted data analysis. EW and XJ wrote the manuscript. This work is supported by the AISH Translational Chair Program in Cancer Genomics. . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review) The copyright holder for this preprint this version posted March 18, 2020. 1. The distribution of the cases having a longer incubation period. The figure shows the ages of the cases who had the incubation days (15-day, 16-day and so on) which were longer than 14 days. Red, blue and green represent men, women, and children, respectively. . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review) The copyright holder for this preprint this version posted March 18, 2020. . https://doi.org/10.1101/2020.03.15.20036533 doi: medRxiv preprint\",\n",
       "   '_search_score': 0.6938412},\n",
       "  {'_id': '00f72ff0dc58a60ef31950d0f28c33dfaf1bf2eb',\n",
       "   'insert_date_': '2020-11-04T18:46:14.829261',\n",
       "   'abstract': 'Background: The novel coronavirus, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), infected over 3300 healthcare workers in early 2020 in China. Little information is known about nosocomial infections of healthcare workers in the initial period. We analysed data from healthcare workers with nosocomial infections in Wuhan Union Hospital (Wuhan, China) and their family members. Methods: We collected and analysed data on exposure history, illness timelines and epidemiological characteristics from 25 healthcare workers with laboratory-confirmed coronavirus disease 2019 (COVID-19) and two healthcare workers in whom COVID-19 was highly suspected, as well as 10 of their family members with COVID-19, between 5 January and 12 February 2020. The demographics and clinical features of the 35 laboratory-confirmed cases were investigated and viral RNA of 12 cases was sequenced and analysed. Results: Nine clusters were found among the patients. All patients showed mild to moderate clinical manifestation and recovered without deterioration. The mean period of incubation was 4.5 days, the meanSD clinical onset serial interval (COSI) was 5.23.2 days, and the median virus shedding time was 18.5 days. Complete genomic sequences of 12 different coronavirus strains demonstrated that the viral structure, with small irrelevant mutations, was stable in the transmission chains and showed remarkable traits of infectious traceability. Conclusions: SARS-CoV-2 can be rapidly transmitted from person to person, regardless of whether they have symptoms, in both hospital settings and social activities, based on the short period of incubation and COSI. The public health service should take practical measures to curb the spread, including isolation of cases, tracing close contacts, and containment of severe epidemic areas. Besides this, healthcare workers should be alert during the epidemic and self-quarantine if self-suspected of infection. This article has supplementary material available from',\n",
       "   'title': 'Nosocomial outbreak of COVID-19 pneumonia in Wuhan, China',\n",
       "   'body': \"17 years after the 2003 epidemic of severe acute respiratory syndrome (SARS) [1, 2] , a novel coronavirus, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), was isolated from bronchoalveolar lavage of several patients with unknown origin pneumonia in Wuhan City, China [3] . SARS-CoV-2 has caused coronavirus disease 2019 across the country and abroad. A total of 83 157 COVID-19 cases with 4.02% mortality has been reported in China up to 7 April 2020 [4] . The confirmed cases have been identified in >180 countries and regions around the world, and >1 300 000 people have been infected globally outside China, including 78 932 deaths [5] . The hunt for patient zero is critical, but epidemiological investigations are often complex and unclear [6] . A COVID-19 patient usually presents with fever, a nonproductive cough, myalgia and malaise. Symptoms, including a productive cough and headache, are less common. Dyspnoea is observed in more than one-quarter of patients. Cases with persistent lymphopenia usually develop fatal comorbidities, including acute respiratory distress syndrome, acute cardiac injury, secondary infection and multiorgan dysfunction [7] [8] [9] . SARS-CoV-2 is a positive-sense and single-stranded RNA virus of zoonotic origin belonging to Betacoronavirus lineage B [3] . It has successfully crossed the animal-to-human barrier and proved to be capable of epidemic spread and may even have endemic persistence in the human population [10] . The calculated R0 of SARS-CoV-2 is as high as 2.68 [11] , making prevention measures and intervention tactics very challenging. Unfortunately, current general hospital settings in Wuhan and many other cities are acting as epidemic hot spots to facilitate transmission and exacerbate spread [12] . So far, >3300 healthcare workers (HCWs) in China have been diagnosed as infected, mainly through nosocomial transmission [13] . HCWs are among the first to suffer during the epidemic. The transmission dynamics of COVID-19 in hospitals in the initial period, especially among HCWs, are important for deeper understanding of the epidemiology of the disease. Here we investigated possible transmission links by integrating epidemiological data and whole-genome sequencing of 25 infected HCWs and two HCWs in whom infection was highly suspected, as well as their family members who were successively infected with SARS-CoV-2 in the initial period in Wuhan Union Hospital. We also collected and analysed the clinical data of the 35 laboratory-confirmed cases among these individuals. A total of 35 laboratory-confirmed cases and two highly suspected cases were enrolled between 5 January and 12 February 2020, including 27 HCWs and 10 relatives out of seven families. All participants were interviewed using a standard questionnaire that elevated communicable diseases. Items such as HCWs' contact history with the confirmed COVID-19 patients, including where, when and how they had been exposed, were assessed. We mapped all cases with the precise time of symptom onset. Cluster outbreaks, potential exposures and possible patterns of transmission were estimated. We collected clinical data from the 35 laboratory-confirmed cases, including symptoms and signs, laboratory examinations, chest imaging, comorbidities and complications, and clinical treatment and outcomes, which were arranged using a standardised data collection form, referencing the case record form shared by the International Severe Acute Respiratory and emerging Infection Consortium (ISARIC) [14] and the household transmission investigation protocol for SARS-CoV-2 infection from the World Health Organization (WHO) [15] . The SARS-CoV-2 laboratory test assays were based on the previous WHO recommendation [16] . RNA was extracted from oropharyngeal swabs of patients suspected of having SARS-CoV-2 infection using the respiratory sample RNA isolation kit. Then real-time reverse transcription PCR (RT-PCR) assay of SARS-CoV-2 RNA was conducted to amplify and test two target genes, including open reading frame 1ab (ORF1ab) and nucleocapsid protein (N). Details of the manufacturer's protocols for RNA extraction and real-time RT-PCR assay, and the diagnostic criteria, are illustrated in the supplementary material. Whole-genome sequencing and comparative genome analysis Nasopharyngeal and/or oropharyngeal swabs were obtained from all COVID-19 cases for SARS-CoV-2 nucleic acid assay, and 22 were processed using whole-genome sequencing. Full genomes were sequenced using the BioelectronSeq 4000 (CapitalBio Corporation, Beijing, China) and assembled using the de novo genome assembler, SPAdes, version 3.10.1 [17] . The complete genome was annotated by Prokka, version 1.14.5 [18] . The mutations were analysed with the use of C-Sibelia, version 3.07 [19] , based on the existing Wuhan-Hu-1 (NC_045512.2) genome, and were annotated by SnpEff, version 4.3 [20] . We integrated information from 60 published genomic sequences of SARS-CoV-2. Full-length genomes were combined with published SARS-CoV-2 genomes and other coronaviruses and aligned using the FFT-NS-2 model by MAFFT, version 7.455 [21] . Maximum-likelihood phylogenies were inferred under a generalised-time-reversal (GTR)+ gamma substitution model and bootstrapped 1000 times to assess confidence using RAxML, version 8.2.12 [22] . The mutations of assembled amino acid sequences of Spike protein were compared with Wuhan-Hu-1 (NC_045512.2), bat-SL-CoVZC45 (MG772933.1) and SARS coronavirus isolate Tor2/FP1-10851 ( JX163927.1) using Clustal Omega [23] . For participants with detailed onset information, a log-normal distribution was used to fit the incubation period, and the probability distribution of the incubation period was estimated. For participants with detailed medical visit information, the Weibull distribution was used to fit and determine onset-to-first-medical-visit and onset-to-admission distributions based on the dates of the onset of the illness, first medical visit and hospital admission. The clinical onset serial interval (COSI) was calculated, for which the probability distribution was estimated by the gamma distribution. For clinical data, categorical variables were expressed as number ( proportion). Continuous variables were expressed as medians and compared by Mann-Whitney U-tests. p-values <0.05 were considered significant. Statistical analyses were conducted using SPSS software, version 23.0 (IBM Corp., Armonk, NY, USA), unless otherwise indicated. Ethical approval was waived by the institutional review board of the hospital, since we collected and analysed all data from the patients according to the policy for public health outbreak investigation of emerging infectious diseases issued by the National Health Commission of the People's Republic of China. Nosocomial outbreak of COVID-19 pneumonia and its epidemiological transmission patterns One male patient (aged 69 years) in the Department of Neurosurgery of Wuhan Union Hospital developed a fever of 38C on 6 January 2020 and was finally diagnosed with COVID-19 on 16 January 2020. A female patient (aged 55 years) had a fever on 11 January 2020, and COVID-19 was confirmed on 18 January 2020. During this period, the HCWs in the Department of Neurosurgery either had direct contact with the two index patients without sufficiently efficient personal protective equipment, or they had indirect contact through their co-workers in the department gatherings on 12 and 13 January 2020. 12 of the HCWs in the Department of Neurosurgery were finally diagnosed with laboratory-confirmed COVID-19 between 16 and 24 January 2020. Besides these, two HCWs (labelled as cases O and Z) were diagnosed as probable cases with similar clinical manifestations but negative viral nucleotide tests. Beyond the Department of Neurosurgery, there were 13 HCWs of other departments who were laboratory-confirmed COVID-19 cases. These departments also had suspects enrolled at the same time. After the 25 HCWs' onset of illness, 10 out of 43 family members were confirmed as COVID-19 cases via nucleotide tests. The remaining 33 family members of the HCWs were not secondarily infected, due to the strict self-quarantine strategies taken by the HCWs immediately after their onset of illness, including wearing a facial mask when they came home, living alone in a separated room, never eating together with their families, etc. In summary, from 5 to 29 January 2020, 25 HCWs and 10 family members out of seven families were diagnosed with COVID-19 in a single hospital. The detailed epidemiological and contact history, as well as cluster information, are shown in the supplementary material. The time distribution of symptom onset and speculative transmission pattern are shown in figure 1a and b, respectively.  According to the information from 14 laboratory-confirmed cases who had specific dates of exposure and symptom onset, we estimated that the mean incubation period was 4.5 days (95% CI 3.0-6.4 days). The 95th percentile of the probability distribution for the incubation period was 11.4 days (95% CI 4.0-12.0 days) (figure 2a). Through the nine transmission chains (figure 1b), we estimated that the meanSD of the COSI distribution was 5.23.2 days (95% CI 3.8-6.8 days) (figure 2b). Among the data obtained from the 35 confirmed cases (including the HCWs and their family members) and two suspected cases, the interval from the date of onset to the first medical visit was estimated to have a mean of 3.0 days (95% CI 2.2-3.9 days) (figure 2c). Of the 27 hospitalised cases (23 HCWs and four family members), the interval from the date of onset to hospital admission was estimated to have a mean of 6.6 days (95% CI 5.3-8.2 days) (figure 2d). There were no significant differences in these four indicators between HCWs and their family members. The clinical outcomes of the patients are shown in On admission, infiltrations in chest computed tomography (CT) images were detected in most cases. In total, 29 (82.9%) showed ground-glass opacities, four (11.4%) showed consolidations, and one (2.9%) presented mixed ground-glass opacities and consolidation. Only one patient (2.9%) had normal chest imaging (supplementary table S2 ). Figure 3 shows two series of typical CT images of moderate pneumonia (HCWs A and C) and supplementary figure S5 shows a series of CT images from a suspected case (HCW O). We obtained 12 whole-genome sequences (GenBank accession numbers MT079843-MT079854) after implementing de novo sequencing from 22 swab specimens from 22 COVID-19 patients. These 12 whole-genome sequences were from patients B, C, E, F, G, H, J, M, Q, R, V and e, as labelled in figure 1b , and their corresponding sample numbers are summarised in supplementary table S3. The remaining 10 samples were removed due to insufficient coverage of the extracted virus genome. Compared with Wuhan-Hu-1, except for V, the remaining 11 viral genomes all had two nucleotide mutations. We found that these two mutation sites both exist in 16 (26.2%) published virus genomes (supplementary tables S4-S6). V showed two missense mutations completely different from the other 11 cases; V was a nurse in the laboratory department, responsible for delivering clinical specimens, so we assume that the difference in virus sequence may be because he was exposed to many other patients' clinical specimens due to his job. The sequences from R and her husband e are the same (figure 4a). A total of six missense mutations were found in the 12 genomes of SARS-CoV-2 strains, but none of them were located in the genes for structural proteins in the new coronavirus, including S, E, M or N proteins (figure 4a). Five of the mutations were located on the nonstructural proteins, which are hydrolysed from the virus-encoded polyproteins 1a/1ab1. The one remaining missense mutation ( position 28 144, T>C) was located on the cofactor gene ORF8. The phylogenetic tree of full-length genomes showed that SARS-CoV-2 strains form a monophyletic clade with a bootstrap support of 100% (supplementary figure S3) . The most closely related sequence to this clade is bat-SL-CoV. Sequences from six HCWs (C, H  At the very early stage of the outbreak of COVID-19 in Wuhan City and Hubei Province in China, we lacked efficient viral identification capacity to diagnose probable COVID-19 cases. There were no effective prevention measures in general hospital settings to isolate and manage the suspected COVID-19 cases, or valid containment measures to block the transmission pathways [12] . HCWs are the highest risk occupational population in outbreaks of such acute respiratory infectious diseases, such as SARS and COVID-19, which are transmitted by contact or respiratory droplets, aerosols or fomites [24] . Here, we have described several clusters of COVID-19 cases in a hospital, showing person-to-person transmission of SARS-CoV-2, and have assessed the epidemiological features of COVID-19. We have reported 25 laboratory-confirmed cases and two highly suspected cases among HCWs, who were infected successively in Wuhan Union Hospital. Of these, 14 were co-workers in the Department of Neurosurgery. They formed two clusters, with two index patients as the sources, which further emphasised the importance of adequate protection for HCWs during their daily work. Moreover, seven family clusters associated with these HCWs have occurred, mainly due to the close contact between the affected HCWs and their family members without any isolation measures. Fortunately, the majority of HCWs, who isolated themselves from their family members at the onset of illness, effectively protected their family members from getting infected. Through mapping the time of onset and possible routes of transmission (figure 1b), we can conclude that the general hospitals are just like an epidemic hub, gathering a lot of patients who are the sources of infection, which would efficiently facilitate and exacerbate the transmission and spreading of virus via public transport or alternative ways. Through phylogenetic analysis, we found clustered cases that were closer in the phylogenetic tree (figure 4b). Mutation analysis showed that the sequence from each sample had different mutation sites, most of which were synonymous mutations. Although some missense mutations existed, none of them were located in the genes for structural proteins of SARS-CoV-2 (figure 4a). Additionally, the amino acid sequences of the structural proteins had no changes (supplementary figures S1 and S2). Most of the initial symptoms in our cases were myalgia, malaise and fever. Some patients had no fever but only mild symptoms such as nasal congestion, which may have led to patients being overlooked, causing a wider spread of COVID-19. In our cases, the intervals of onset of illness to first medical visit and onset of illness to hospital admission were estimated to have a mean of 3.0 days and 6.6 days, respectively, which were shorter than in other studies [25] . By 12 February, 21 out of 27 hospitalised cases (77.8%) had recovered and were discharged; none of them were admitted to the intensive care unit or had died. From the chest CT images from HCWs A and C, very mild or moderate infiltrations were presented, and these became alleviated and resolved significantly in a short time ( figure 3) . Thus, early diagnosis, isolation and timely treatments can speed up the recovery of patients and decrease the deteriorative tendency. The median time for virus shedding to become negative in our cases was a long period of 18.5 days (range 12-25 days). It is worth mentioning that nurse F suffered severe diarrhoea during hospitalisation. Even after the nucleic acid test of nasopharyngeal swabs turned negative and lung CT images got better, the viral nucleic acid test of the stool maintained a sustainable positive result, 1 month after onset. We must thus pay close attention to the status of virus clearance of patients with COVID-19. Such a long virus shedding period perhaps highlights the production of efficiently neutralising antibodies in a comparatively postponed manner or even an unsatisfactory titre in the patient's plasma. Therefore, it might be necessary to extend the period of hospitalisation for infectious control purposes, and we should adopt a prudent strategy for treating critically severe patients with convalescent plasma from COVID-19 patients. Our current study has some limitations. First, only 37 patients were enrolled, which might have led to potential bias of the incubation period and COSI in our study. More accurate calculation of incubation period and COSI would be possible with a larger sample size. Secondly, the investigation of the kinetics of virus shedding and the relevant viral loading in both respiratory and intestinal tracts was not possible, since the novel causative pathogen has only just been identified. Thirdly, the potential for superficial exposure in infection transmission was not investigated, which may need further studies. Fourthly, the kinetics of viral antibodies, especially neutralised antibodies, were not monitored due to laboratory limitations. Finally, viral genome analysis and traceability from each patient have not been performed and should be conducted in future studies. In summary, similar to the 2003 SARS outbreak in Guangzhou, the current epidemic of SARS-CoV-2 resulting in COVID-19 in Wuhan is mainly due to efficient person-to-person transmission or super-spreading events in hospital settings and social activities, based on the short period of incubation and COSI. Practical strategies and measures, such as isolation of patients, tracing and quarantine of close contacts and containment of severe epidemic areas, are crucial to block the spread. Earlier detection and diagnosis of patients with COVID-19 will result in better prognoses.\",\n",
       "   '_search_score': 0.6832560999999999},\n",
       "  {'_id': '00f244245f02c27974ee0dd3bc21fe47abcc03cb',\n",
       "   'insert_date_': '2020-11-04T18:46:14.825469',\n",
       "   'abstract': 'The shedding, tissue distribution and quasispecies composition of feline coronaviruses were studied in naturally and experimentally infected cats. The infection remained subclinical, but the majority of the animals shed the virus via faeces throughout the experiment. Sequences corresponding to the viral nucleocapsid region were amplified by reverse-transcription polymerase chain reaction from the cortex, dura mater, pancreas, lungs, third eyelid, and the heart muscle in four cases. Interestingly, the ORF7b viral region -a supposed virulence factor -was detected in fewer organs, raising the possibility that this region can be affected by deletions during virus replication in vivo. It is demonstrated that the composition of the viral quasispecies differs between organs, and that genomic regions with different functions undergo distinct processes of selection, which should be considered during the evolution of feline coronaviruses.',\n",
       "   'title': 'Preliminary studies on feline coronavirus distribution in naturally and experimentally infected cats',\n",
       "   'body': \"FELINE CORONAVIRUSES (FCoV) are members of the family Coronaviridae. Their genome consists of a singlestranded positive-sense RNA (de Groot et al 1987 , Luytjes 1995 . Regarding their pathogenicity, FCoV comprise two biotypes: feline infectious peritonitis (FIP) viruses (FIPV) and feline enteric coronaviruses (FECV; Pedersen 1987) . Based on in-vitro neutralisation using monoclonal antibodies, the existence of at least two serotypes of FCoV (type I and II) has been shown. Each serotype contains viruses of both the FIPV and FECV biotype (Hohdatsu et al 1991a,b) . Type I and II viruses differ in their in vitro growth characteristics (Pedersen 1987) and importantly, in the composition of their spike protein (Herrewegh et al 1998) . In the field, the prevalence of FCoV type I seems to be higher, since about 70 per cent of FIP cases are due to infection with type I viruses (Hohdatsu et al 1992) . FIP is a highly fatal disease in Felidae (Foley et al 1997a) . The pathogenesis of the disease is still not fully understood. The diagnosis of the disease is rather complicated, and therefore it represents one of the major pet animal health problems that still remains to be solved. Thus, serological data on their own are not always informative (Olsen 1993) in diagnosis. Vaccines may not provide protection since they may even induce accelerated pathogenesis and early death, associated with antibody-dependent enhancement of infection (Vennema et al 1990 , Harvey et al 1996 . Nevertheless, a commercially available vaccine has shown promising results (Fehr et al 1997) . FECV is not considered a direct cause of morbidity in cats. It may produce mild enteritis but most experimental and natural infections remain subclinical. However, it presents a considerable health risk to cats because it has been revealed that it is the parent of FIPV (Foley et al 1997b , Vennema et al 1998 . Recent findings suggest that genetic changes frequently occur in feline coronaviruses (Herrewegh et al 1995 , Vennema et al 1998 . For instance, it has been demonstrated that type II feline coronaviruses have arisen from an RNA recombination event between canine coronavirus and a type I feline coronavirus (Herrewegh et al 1998) . Further observations indicate that deletions in different parts of the FECV genome may be responsible for the formation of FIP viruses (Vennema et al 1998) . However, the mechanism of these events is not totally understood. It is known that type I FECV can establish persistent infection in cats when the virus replicates in the enterocytes of the crypts of Lieberkhn in the large intestine . During virus replication, the specific changes in the viral genome may lead to the development of the highly pathogenic FIP viruses and consequently, manifestation of the disease. Possibly a malfunction of the immune system contributes to this process (Poland et al 1996) . The laboratory diagnosis and control of FIP is seriously hampered by the failure of most FCoV to replicate in cell cultures . Considering this fact, the aim of this study was to develop and apply efficient approaches for the detection and differentiation of FCoV subpopulations. Using these techniques, we investigated the persistence, distribution and quasispecies composition of FCoV in naturally and experimentally infected cats. In order to mimic natural conditions, 12 randomly selected mixed-breed domestic cats of both genders were used. The animals originated from a vendor and were clinically healthy, and between 4-6 months of age. The cats had not been immunised with any vaccine. The faeces of cats Nos 6, 8, 9 and 10 were positive for FCoV using a reverse-transcription polymerase chain reaction (RT-PCR). Therefore these animals were regarded as carriers, while the remaining eight cats were repeatedly negative for FCoV before starting the experiment. Six FCoV free animals (Nos 1, 2, 3, 4, 5 and 7) were experimentally infected with FCoV, and two cats (Nos 11 and 12) were used as negative controls. The animals were kept in the same room but in separate cages and had no physical contact with each other. They were fed with commercially available cat food, milk and fresh water. Faecal samples were collected from cats Nos 6, 8, 9 and 10 at the beginning of the experiment. The fresh samples were pooled, homogenised in PBS (1:10, w/v), centrifuged at 3100  g for 20 minutes and the supernatant fluid was given orally to six FCoV free cats (Nos 1, 2, 3, 4, 5 and 7) at a dose of 1 ml cat -1 . The negative controls and the FCoV carriers were mock-infected with PBS. The cats were monitored daily for clinical signs. Rectal swab samples were collected 7 days before starting the experiment, on the day of experimental inoculation, and on post-infection days (PID): 10, 20, 30, 40, 50, 60, 70 and 80. The swabs were processed as described by Foley et al (1997b) . Briefly, the swabs were vortexed in 1.5 ml doubledistilled water and then the suspensions were centrifuged at 5000  g for 5 minutes. On PID 80 the cats were killed and gross and histopathological examinations were carried out. The following tissue samples were collected for RT-PCR: brain cortex, medulla oblongata, dura mater, liver, kidneys, spleen, lymph nodes, lungs, small and large intestines, pancreas, third eyelid and heart muscle. From the organ samples, 10 per cent (w/v) suspensions were prepared in double-distilled water and then centrifuged at 7000  g for 10 minutes. The supernatants were used immediately for RNA extraction or stored at -70C. The supernatants of the faeces or tissue homogenates were used for RNA preparation by the procedures of Boom et al (1990) and Cheung et al (1994) as follows: 100 l suspension was added to 20 l size-fractionated silica in 900 l lysis buffer (120 g guanidine thiocyanate dissolved in 100 ml 0.1 M Tris-HCl, [pH 6.4], and 22 ml 0.2 M EDTA, [pH 8.0], and 2.6 g Triton X-100). The samples were vortexed and left for 10 minutes at room temperature. The supernatant was removed after a quick spin (12000  g for 1 minute), and the silica-RNA pellet was washed twice with 450 l washing buffer (120 g guanidine thiocyanate dissolved in 100 ml 0.1 M Tris-HCl, [pH 6.4]) and twice with 1 ml of 70% (v/v) ethanol. After a final wash with 1 ml acetone, the pellet was dried at 56C for 10 minutes. The RNA was eluted by resuspending the pellet in 40 l diethyl pyrocarbonate-treated (DEPC) water at 56C for 10 minutes. After pelleting the silica particles (12000  g for 3 minutes), the supernatant fluid containing the RNA was used for reverse transcription. The cDNA synthesis was performed in 28 l reaction mixtures. A preliminary mixture containing 5 l RNA, 5 l DEPC-treated water, and 1 l random hexamers (0.02 U; Pharmacia Biotech, Uppsala, Sweden) was incubated at 65C for 5 minutes for RNA denaturation. The tube was placed on ice, and 17 l RT-mix containing 2.5 l of each deoxynucleotide triphosphate (2 mM; Pharmacia), 5 l of 5  reaction buffer (0.25 M Tris-HCl, [pH 8.3], 0.375 M KCl, 15 mM MgCl 2 ), 1 l RNAguard (24 U; Pharmacia), and 1 l Moloney murine leukaemia virus reverse transcriptase (200 U; Gibco BRL, Bethesda, MD, USA) was added. The reaction mixture was incubated at 37C for 90 minutes, followed by incubation at 98C for 5 minutes to inactivate the enzyme. Primers were selected from the highly conserved nucleocapsid and ORF7b regions of the FIP virus strain 79-1146 genome (Vennema et al 1991) using the OLIGO 4.0 program (National Biosciences Inc., Plymouth, MN, USA). Sequences of the primers are shown in Table 1 . It should be noted that the N-PCR has been applied for more than 6 years in our routine diagnostic laboratory for the diagnosis of FIP and several hundreds of samples from clinical cases were found to be positive (unpublished data). The two PCR assays were carried out in 50 l volumes. The reaction mixtures contained 5 l 10  GeneAmp PCR buffer II (Perkin-Elmer Cetus, Norwalk, CT, USA), 15-pmol OFIP1A and OFIP2A primers for the first nucleocapsid PCR, 15 pmol P399 and P997 primers for the first ORF7b PCR, 2.5 l of each deoxynucleotide triphosphate (10 mM; Pharmacia), 2.5 mM MgCl 2 , AmpliTaq DNA polymerase (1 U; Perkin-Elmer Cetus), and 5 l cDNA. Two droplets of mineral oil (Sigma Chemical, St. Louis, MO, USA) were added to prevent evaporation. The second (nested) PCR assays were performed on 2 l amounts of the first PCR products with 15-15 pmol of primer OFIP4 and OFIP3 for the nucleocapsid and 15-15 pmol of primer P503 and P970 for the ORF7b. The following temperature profile was used: 94C for 45 seconds, 45C for 60 seconds (50C for 60 seconds for the first five cycles), and 72C for 2 minutes. For the second round of amplification, 96C for 45 seconds, 52C for 1.5 minutes, and 72C for 3 minutes was applied. This cycle profile was repeated 30 times followed by a final extension step at 72C for 7 minutes. The amplified DNA products were detected by standard 2 per cent agarose gel-electrophoresis. To avoid false positive and negative results, the precautions of our routine diagnostic PCR laboratory were used throughout the experiments (Belk and Ballagi-Pordny 1993) . The sensitivity of the two PCR assays was evaluated and compared by testing 10-fold dilutions of the tissue culture supernatants of reference virus strains (DF-2, NW-1, 79-1683, Black, UCD-2, and UCD-3) and clinical samples, such as rectal swab and heart muscle homogenate originating from cat No 9. The concentration of the nucleic acid solutions was determined by a spectrophotometer (GeneQuant RNA/DNA Calculator, Pharmacia, Uppsala, Sweden). In serial 10-fold dilution tests, the two PCR assays showed the same sensitivity. As calculated, 3.2 pg ml -1 total RNA was the minimal target where visible PCR product was obtained (not shown). In order to generate single stranded PCR products for the SSCP analysis, asymmetric PCR was performed on 2 l amounts of the completed first PCR mixtures by using the nested primers at a ratio of 1:10 (5:50 pmol). The same cycle profile was applied as for the second round of amplification. Five microlitre amounts of the single stranded PCR products were added to 15 l SSCP loading buffer (95 per cent formamide, 0.05 per cent xylene cyanol, 0.05 per cent bromophenol blue), kept at 95C for 5 minutes, put on ice for 5 minutes and finally loaded onto 10 per cent polyacrylamide gels (Bio-Rad, Hercules, CA, USA). Electrophoresis was performed in a Mini Protean Cell (Bio-Rad) at 140 V constant power for 4 hours at room temperature. Subsequently, the gels were submerged into a 0.5 g ml -1 ethidium-bromide solution for 5 minutes and visualised under UV light. After experimental infection, the clinical status of the cats was not severely affected. Only a mild diarrhoea was observed among the experimentally infected animals at PID 3-6. The appetite, behaviour and body temperature remained normal, while the body weight of the cats increased by 15 to 25 per cent up to the end of the experiment. By testing the swab samples, the N-PCR and the ORF7b-PCR assays gave consistent results. In cat Nos 5, 9 and 10, FCoV was detected at each occasion of testing. In other animals, the virus was detected at various occasions, as shown in By testing the large intestines in the N-PCR assay, virus was detected in eight animals. Cat Nos 6 and 7 were negative. In cat Nos 1, 5, 9 and 10, the virus was also demonstrated in the following organs; cat No 1: cortex, dura mater, lungs, pancreas, third eyelid and heart muscle; cat No 5: third eyelid and heart muscle; and cat Nos 9 and 10: heart muscle. The ORF7b region was detected in the large intestines of the same eight animals as in the N-PCR assay. It was also demonstrated in the pancreas of cat No 1 and in the heart muscle of cat Nos 5 and 9. In the SSCP assays, the quasispecies nature of the detected viruses was revealed as illustrated in Fig 1. The bands in the same lane represent viral subpopulations with slight nucleotide sequence differences (Lzaro and Estivill 1992) . Generally the N region showed higher variability than the ORF7b region. The consistency of the patterns was confirmed by repeated PCR-SSCP analyses (data not shown). The changes of the viral quasispecies in the large intestine of cat No 1 (Fig 1a,c) could be characterised as follows. At 10 days after infection, there appeared to be one major and one minor variant of virus subpopulation. At PID 20 there was an additional minor variant. The latter remained until termination of the experiment at PID 80. The first minor variant appeared to be replaced by a third minor variant at PID 30 which also remained present to the end. The major variant was always present throughout the experiment and was never replaced. The third minor variant sequence in the N gene that appeared at PID 30 might correlate with a variant in the 7b gene sequence (Fig 1c) . In cat No 9 (Fig 1b,d) the major and minor N variants present at day 0 were most likely the same as the variants present throughout the experiment from day -7 to PID 70, with the appearance and disappearance of additional minor subpopulations. Changes of the 7b region showed similar characteristics but to a more moderate extent. The N-SSCP patterns of the organs were more homogeneous than those of the swab samples, as demonstrated in Fig 2. Apparently each organ consistently contained three predominant variants. The ORF7b-SSCP patterns of the organs had substantially the same pattern as the swabs (not shown). Recent studies suggest that FECV is the parent of FIPV, and the alterations in the genome of the 'harmless' parent lead to the development of the fatal disease of infectious peritonitis (Vennema et al 1992 , Herrewegh et al 1995 , Poland et al 1996 , Vennema et al 1998 . Considering that FECV is extremely widespread in the cat populations (Foley et al 1997b) , detailed studies are required to investigate the biology of the two virus biotypes, with particular regard to the genetic stability of FECV and the genomic regions FIG 2: SSCP patterns of the products amplified by the nucleocapsid region specific pcr from the organ samples of cat No. 1. The lanes represent the following organs: 1. cortex; 2. dura mater; 3. lung; 4. large intestine; 5. pancreas; 6. heart muscle; 7. third eyelid. responsible for the development of viruses of high pathogenicity. These investigations will rely on the ability to detect and characterise the different viruses at various stages of the infection. In the present study we investigated the invasion, the tissue distribution, and the variability of FCoV. Our aim was to establish and compare two PCR assays with respect to their applicability for monitoring purposes. By applying these assays to amplify two conserved regions of the FCoV genome, we found that eight of 10 cats shed the virus in the faeces throughout the 80-day course of the experiment. This finding supports the observations of Herrewegh et al (1997) who reported that the FCoV infection is maintained by chronically infected carriers. The short-lasting and mild diarrhoea that was observed among the experimentally infected animals at the early stage of infection could be the clinical symptom of FCEV infection, or it may have been the consequence of the massive exposure of the animals to the FCoV, as suggested by Foley et al (1997a) . The two cats (Nos 6 and 7), which became and remained PCR-negative at the relatively early stage of the experiment, might have had strong and effective immune response against FCoV. A crucial step in the pathogenesis of FIP is when certain virus variants leave the intestines and reach different organs (Pedersen et al 1981) . It is not known how long this process takes or which conditions are required for this event. The present experiment showed that despite the relatively short observation period of 80 days, in one cat FCoV apparently invaded different organs. Considering the suggestions of Pedersen et al (1981) , this may be an important step in the development of FIP. It is interesting that the N and ORF7b-PCR assays gave identical results when the swab and large intestine samples were tested. However, in the case of additional organs, the N-PCR scored considerably more positive samples. An obvious explanation would be the different sensitivity of the two systems. This is not likely, however, considering that the two assays yielded identical results not only on the swab and large intestine samples, but also when serial dilutions of FCoV and clinical specimens were tested. We therefore hypothesise that the viruses detected in the cats could presumably have their ORF7b region affected by deletion(s). Such a phenomenon was reported for avirulent, tissue culture adapted feline coronaviruses and it was concluded that the loss of virulence was associated with the deletions in the ORF7b region (de Groot and Horzinek 1995). According to Vennema et al (1998) , as well as being a virulence factor the presence of this region is an advantage for the virus replication. Therefore, this phenomenon should be investigated further, both in vitro and in vivo, and studies should also involve the ORF3c region which is considered to exert a regulatory effect on the ORF7b (Vennema et al 1998) . The quasispecies nature of the members of the Coronaviridae family has been well documented, mainly for murine coronaviruses (Makino et al 1986 , Baric et al 1990 , Holland et al 1992 , Lai 1996 . The present study revealed that FCoV also comprised virus subpopulations. However, there were differences in the prevalence of viral subpopulations between body compartments, i.e., not all variants present in the swab samples were present in the organ samples, and some variants were more prevalent in the organ samples than in the large intestine. Additionally, during in vivo replication, the SSCP patterns of the N region showed higher variability than the ORF7b region. These findings suggest that presumably only selected virus variants became able to leave the large intestine and invade the blood circulation and different organs. It appears, however, that the selection pressure is not on the N gene, since the predominant subpopulations (the master sequences) could be demonstrated in each N-PCR positive organ specimen. Nevertheless, presumably a new selection process takes place in every infected organ that may result in a virus quasispecies that is different in composition from that which originally replicated in the large intestine. This process, in conjunction with the relatively impaired reaction of the respective immune response of particular cats could eventually lead to the development of FIP. The present study demonstrated that FCoV is able to establish chronic enteric infection in cats. The data indicate that the viruses undergo continuous but subtle changes during the chronic replication, and that the extent of variation and the composition of the viral quasispecies is not the same in the different organs. Additionally, even neighbouring genomic regions showed distinct variability: the N region varied more than the ORF7b region. The possible role of ORF7b in virus replication is indicated by the highly conservative SSCP pattern of this region exhibited throughout the experiment. The exact information on ORF7b and its product, the gp7b, should reveal the role of the region in the development of FIP. In conclusion, this study provided information concerning alternations in the genome of FCoV during in vivo replication. It was shown that the composition of the viral quasispecies differs between organs, and that genomic regions with different functions undergo distinct processes of selection, which should be considered during the evolution of feline coronaviruses. Future investigations will aim to determine the significance of the changes of particular viral genomic regions. Beyond this, the immune responses against FCoV infections and the probable impairment of these functions, that eventually participate in the pathogenesis of FIP, will be elaborated.\",\n",
       "   '_search_score': 0.6714923000000002},\n",
       "  {'_id': 'a9853a6bc65595e9211549f413674a37ad424a6c',\n",
       "   'insert_date_': '2020-11-04T19:00:46.149569',\n",
       "   'abstract': 'We identified a Norwalk-like calicivirus (CV) whose genome likely was derived from naturally occurring recombination. This strain (Arg320) was detected by the EIA developed against recombinant Mexico virus (rMxV) capsids, but the viral RNA polymerase sequence was closer to Lordsdale virus, in a separate genetic cluster of Norwalk-like viruses. A 3.3 kb cDNA from the RNA polymerase region to the 3 end of the genome of Arg320 was cloned and sequenced. The sequence demonstrated that the capsid region of Arg320 shared 95% amino acid identity with MxV, but 68% identity with Lordsdale virus, while the RNA polymerase region shared 95% identity with Lordsdale virus, but 87% identity with MxV. Pair-wise sequence comparisons identified a potential recombination site at the polymerase/capsid junction. This is the first example of a naturally occurring recombinant in the CV family. Further studies to search for and characterize other strains may be necessary for understanding the genetic diversity of the family.',\n",
       "   'title': 'Characterization of a novel human calicivirus that may be a naturally occurring recombinant',\n",
       "   'body': 'Caliciviruses (CVs) are single-stranded, positive-sense RNA viruses that are closely related to picornaviruses. Four CV genera have been described, including \"Norwalk-like viruses\", \"Sapporo-like viruses\", Vesivirus, and Lagovirus [1] . CVs are genetically diverse but share a common genomic organization. All CVs have at least three genes, from 5 to 3 being a nonstructural polyprotein, a capsid gene, and a small protein of unknown function. In Sapporo-like viruses and Lagoviruses, the capsid gene is fused to the non-structural gene and most Sapporolike viruses have an additional small ORF overlapping the capsid gene [10, 11, 19] . CVs share amino acid motifs with picornaviruses but have a different genomic organization. The capsid gene of CVs is switched in its 5 -3 position in the genome compared with that of picornaviruses [10, 11, 19] , suggesting the two families might derive from one another by recombination. RNA recombination has been shown to occur in animal hosts by recovery of recombinant polioviruses from vaccinees [14, 15] , however, the frequency of such recombination in nature remains unknown. RNA recombination has been suggested among CVs [5] , but direct evidence remains lacking. In this study, we describe a human CV (HuCV) whose genome likely was derived from recombination. A continuous genomic fragment spanning the RNA polymerase region to the 3 end of the genome was sequenced and the sequence demonstrated that the RNA polymerase and capsid genes of the strain shared closest sequence identities with distinct HuCV genetic clusters. Sequence alignment also suggested a potential recombination site. Stool specimens from children hospitalized with diarrhea in Mendoza, Argentina, were collected in 1995 (3) . A subset of 150 stool specimens that were negative for rotaviruses was selected for testing of HuCVs by recombinant EIAs and RT-PCR. Stool specimens were kept at 20  C before testing. EIAs specific for detection of Norwalk virus (NV) and Mexico virus (MxV) based on hyperimmune antisera against baculovirus-expressed recombinant NV (rNV) and MxV (rMxV) capsid proteins were used for detection of viral antigens in stool specimens [8, 9] . A positive result was defined by a P/N ratio  2.0 between the OD values in the wells coated with pre-immunized vs post-immunized hyperimmune antisera in each assay. Viral RNA was extracted from stool specimens using the Trizol method (GibcoBRL, Grand Island, NY). RT-PCR was performed to detect two regions of HuCV genomic RNA. The first RT-PCR used primers NV36/51 (8) to amplify the RNA-dependent RNA polymerase. The second RT-PCR used primer NV36 and oligo-dT to amplify a 3.3 kb cDNA fragment from the RNA polymerase region to the 3 end of the genome. Both RT-PCR used similar conditions except the RT-PCR for a long product (>3 kb) used a long (3-5 min) extension time [10] . Viral cDNA fragments with expected sizes determined by agarose gel electrophoresis were cloned into pGEM-T (Promega, Madison, WI). For sequencing the cDNA (400 bp) amplified by primer pair NV36/51, the reverse and forward primers flanking the cDNA in the plasmid vector were used. Multiple steps were used to determine the sequence of the entire 3.3 kb cDNA. The end sequences of the cDNA were determined using the same forward and reverse primers. Internal sequences were determined using subclones generated by restriction enzyme digestion and re-ligation when convenient restriction enzyme sites were available. The remaining sequences of the cDNAs were determined by genomic walking with primers based upon newly obtained sequences adjacent to those regions. All sequences were confirmed by sequencing from both strands and at least twice. The sequences of the cloned viral cDNA were analyzed using PC/GENE software, version 6.90 (IntelliGenetics, Mountain View, CA). Multiple sequence alignment and a dendrogram predicting genetic relationships were generated using the CLUSTAL program, based upon the method of Higgins et al. [7] . Pair-wise sequence comparison between different strains was performed manually on the alignment and numbers of sequence identities between strains were calculated in a moving frame of 25 nt. Sequences used in the analysis included Lordsdale virus (X86557) and the MxV (U22498) from the GenBank. The RNA polymerase sequences of additional strains were derived from either the GenBank or our database. The GenBank sequence number for strain Arg320 is AF190818 for the RNA polymerase region and AF190817 for the 3.3 kb cDNA at the 3 end of the genome. In the collection of diarrhea stools from children in Mendoza, seven (5%) stool specimens from seven hospitalized children were positive for HuCVs by the rMxV EIA and by RT-PCR with RNA polymerase region primers. None of the seven specimens was positive by the rNV EIA. Sequence analysis of the RT-PCR products showed that the seven strains belonged to two genetic clusters of Norwalk-like viruses, containing the prototype Snow Mountain virus (SMV, 2). Two of the seven strains belonged to the MxV cluster and the other five belonged to the Lordsdale cluster. The two MxV-like strains revealed 95% nucleotide and amino acid identities with the prototype MxV. The five \"Lordsdale like\" strains showed 85% nt identity with the prototype Lordsdale virus, but only 75% nt identity with MxV. In our previous studies of HuCV variation, if a strain was positive by the rMxV EIA, it usually shared a high sequence identity (>95%) in the RNA polymerase region with the prototype MxV. We hypothesized the five \"Lordsdale-like\" strains might be recombinants. One Mendoza stool (Arg320) containing a \"Lordsdale-like\" virus was selected to clone a cDNA from the RNA polymerase region to the 3 end of the genome. A 3.3 kb cDNA was amplified using a one-step RT-PCR with primer NV36 and oligo-dT. The cDNA then was cloned and sequenced. To ensure the sequence was not derived from contamination by another virus, the two ends of the cDNA were re-sequenced on purified plasmid DNAs from a single bacterial colony. Sequences identical with previously determined sequences at the two ends were obtained from the re-sequencing. The resulting sequence contained 3 312 bp plus a poly-A tail and revealed typical HuCV genomic features. The 5 end of the cDNA encoded the C-terminus of ORF1, including the RNA polymerase gene. This RNA polymerase sequence matched the sequence obtained earlier. The second ORF started at nt 853 and ended at nt 2 496, encoding a protein of 548 aa which is predicted to be the viral capsid protein. The third ORF started at nt 2 499 and ended at nt 3 259, encoding a protein of 252 aa with unknown function. A noncoding region of 55 nt was found between the third ORF and the poly-A tail. There was no sequence match in the GenBank or in our library of unpublished CV sequences. Sequence alignment of the 3.3 kb Arg320 cDNA with corresponding regions of MxV and Lordsdale virus showed that Arg320 shared 86% and 71% nt identities with MxV and Lordsdale virus, respectively, however, the relative sequence identities differed between the RNA polymerase and capsid sequences. The capsid region of Arg320 shared high (95%) aa identity with MxV, confirming the rMxV EIA result, but low (68%) identity with Lordsdale virus. The RNA polymerase region of Arg320 shared high (95%) aa identity with Lordsdale virus, but lower (87%) aa identity with MxV. Multiple sequence alignment of the capsid sequences placed Arg320 in the MxV cluster, but multiple sequence alignment of the RNA polymerase genes placed it in the Lordsdale cluster (Fig. 1) .  To localize the potential recombination site and to understand a possible recombination mechanism, we performed pair-wise sequence comparisons between Arg320, MxV, and Lordsdale virus. The plotted relative nt identities showed that Arg320 shared a constantly low level of sequence identity in the RNA polymerase region and a constantly high level of nt identity in the capsid region with MxV, and a constantly high level of sequence identity in the polymerase region but a constantly low level of nt identity in the capsid region with Lordsdale virus. A single cross of the two curves was observed at the ORF1/capsid junction (Fig. 2) . The identity levels found in the capsid genes extended to ORF3 and the 3 noncoding region. When MxV and Lordsdale virus were compared, no significant change of sequence identity between the RNA polymerase and capsid regions was observed and the curves did not cross with the two curves described above (Fig. 2) . Although MxV and Lordsdale virus shared 70% nt identity over the 3.3 kb cDNA, a small region with significantly higher nt identity was observed around the ORF1/capsid junction (Figs. 2 and 3 ). This highly conserved region starts at nt 750, 103 nt upstream from the capsid, and ends at nt 875, including 23 nt of the capsid gene. Of 126 nt in the region, 116 (92%) were identical between the two Comparison of sequence identity and genomic organization of Arg320, MxV, and Lordsdale virus. Insertion/deletions were identified based upon amino acid sequence alignment of the capsid and ORF3, respectively. The two insertion/deletion sites in capsid gene shared between Arg320 and MxV were found at amino acids 295 and 401 for the two capsid genes. The single insertion/deletion site in ORF3 shared by Arg320 and MxV was found at amino acid 42 of the two viruses strains. The predicted subgenomic RNAs of the two strains start in this region (nt 850). The first 28 nt at the 5 end of the viral subgenomic RNA were identical between MxV and Lordsdale virus (Fig. 3 ). Additional features of Arg320\\'s genome revealed different relationships with MxV and Lordsdale virus. For example, the capsid protein of Arg320 (548 aa) was larger than that of Lordsdale virus (539 aa), but was the same as that of MxV (Fig. 4) . Two identical deletion/insertion regions were identified in the capsid gene of Arg320 and MxV, but not for Lordsdale virus. ORF3 of Arg320 also had the same size (252 aa) as that of MxV, which is smaller than that of Lordsdale virus (276 aa). One deletion/insertion was identified in ORF3 of MxV and Arg320, but not for Lordsdale virus. This study describes a HuCV strain that likely was derived from naturally occurring recombination. The viral RNA polymerase gene shares closest sequence with one HuCV strain but its capsid gene shares closest sequence identity with another HuCV strain. Spontaneous point mutation would not result in such sequence changes between two regions next to each other in a genome. The cDNA used to obtain the sequence was generated from one RT-PCR reaction covering the entire region. Therefore, the cDNA was from a single virus, not from multiple viruses due to contamination or dual infection of the child. The sequences of both ends of the cDNA were confirmed by simultaneously sequencing of purified cDNA from a single bacterial colony, excluding the possibility of contamination during the sequencing. Finally, the two potential parental strains are in the range of genetic diversity among strains currently co-circulating [4, 20, 24, 25] , therefore, the recombination event could have occurred recently, but not necessarily during the infection of this child. Naturally occurring recombination of HuCVs was suggested when the capsid and RNA polymerase regions of SMV were characterized [6] . The capsid nucleotide sequences of SMV and Melksham virus (MeV) were almost identical (94%) but the RNA polymerase sequence were significantly different (79%). When MeV was compared with MxV and a Japanese strain, Oth-25, there seemed no significant difference of sequence identities between the capsid and RNA polymerase regions. Therefore SMV likely is a recombinant virus. However, the capsid and RNA polymerase region amplicons of SMV were generated separately and that fact does not exclude the possibility of different source of strains, which was not the case for Arg320 described in this study. The change of relative sequence identity occurred at the ORF1/capsid junction, indicating the recombination site occurred in this region. This site also is suggested to be the break and rejoin site between caliciviruses and picornaviruses, although direct evidence of recombination between the two families remains lacking. The high sequence conservation in the ORF1/capsid junction among the two strains representing the putative parent strains and the putative recombinant suggests the recombination might be driven by homologous RNA interaction. The \"copy choice\" model of RNA recombination has been preferred for singlestranded RNA viruses, including picornaviruses and coronaviruses [2, 13, 17, 18, 21, 23] . Co-infection of different HuCV strains in one patient has been observed in our surveillance of HuCV-associated outbreaks of acute gastroenteritis in Virginia [12] and mixed infection of multiple strains in one outbreak has been reported [5] . Therefore, high sequence conservation would increase the interaction between RNA molecules from different strains and recombination could occur as a result of such homologous interaction. In the copy-choice model, recombination could occur during RNA replication with the involvement of viral RNA polymerase, which may switch templates of two different strains at the highly conserved region of the gemomes of the two strains. CVs contain subgenomic RNA that covers the entire 3 end of the genome from the capsid gene to the 3 end. Therefore, the CV subgenomic RNA could act as an independent unit participating in the recombination event. It remains unclear how CV subgenomic RNA is involved in virus replication, but it is clear that both CV genomic and subgenomic RNAs share highly conserved 5 end sequence and both RNAs are assembled into virions. The high conserved 5 end of the subgenomic RNAs between MxV and Lordsdale virus also is included in the highly conserved ORF1/capsid region. Therefore, interaction between genomic and subgenomic RNAs could occur by the same mechanisms as that of genomic-genomic interaction, which would significantly increase the chance of recombination events that otherwise result only from genomic-genomic RNA interaction. Furthermore, because both genomic and subgenomic RNA are assembled into virions, co-infection of different strains may result in reassortment of different genomic RNAs from different strains, as found in the influenza virus [26] , which could result in recombination in future generations, although direct evidence of reassortment of CV RNAs remains lacking. Although recombination was predicted to occur at the ORF1/capsid junction, the precise recombination site was difficult to pinpoint because of the high conservation in this region between the two strains representing putative parents. Furthermore, because the putative recombination event may be predicted to have occurred in this part, the sequences at both sides of the recombination site of the progeny strain may have changed since the recombination event due to naturally occurring point mutations. The parent strains also would have changed due to the same mechanism. Thus, we may never observe 100% sequence identity of corresponding regions that would identify the parent strains. Because the potential recombination event described in this paper may have occurred recently, the sequences adjoining the potential recombination site maintained a high level of identity, which allowed us to identify the recombinant. It would not be difficult to imagine many CV strains that currently are circulating could have derived from recombination events in the remote past. The existence of a recombinant within the CV family also provides understanding of the relationship between picornaviruses and caliciviruses which belong to a superfamily [22] that probably was divided into two families by recombination. This study extends our knowledge of the genetic diversity of CVs. If the RNA recombination described in this study is a common phenomenon, we would see more diversity of the family, which would make genotyping more difficult. For example, many reports on CV genotyping have been based upon sequences of the RNA polymerase region, due to its relatively high sequence conservation and ease of designing PCR primers. In comparison, only a few capsid genes have been characterized. The viral capsid protein is responsible for antigenicity and possibly protective immunity. Genotying of CVs based upon the RNA polymerase sequences clearly is not the best choice if ORF1/capsid gene recombination is common. In addition, it remains unclear whether additional recombination sites exist. Continued searches for Arg320-like and other recombinant strains are necessary. The significance of this study has been emphasized by the occurrence of another four strains in Mendoza that are apparently similar (sharing >98% amino acid sequence identities in the RNA polymerase region with Arg320) and by a recent identification of an Arg320-like strain in an outbreak of gastroenteritis in Virginia [12] . These findings indicate that such recombinants are genetically stable, are still circulating, cause illness, and possibly have a wide distribution. The fact that strains similar to both putative parents of the recombinant are currently co-circulating not only allowed us to identify the recombinant, but also provide indirect evidence that natural recombination is still occurring. In another words, if the parental strains had less survival advantage than the recombinant and the recombination event occurred long ago, we would not be able to recognize the recombinant because representatives of the parental genetic clusters would no longer exist. Therefore, molecular recombination could markedly increase the genetic diversity of currently circulating strains in the CV family.',\n",
       "   '_search_score': 0.6653039999999999}],\n",
       " 'count': 759}"
      ]
     },
     "execution_count": 32,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "resultsb"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-05T08:35:11.670565Z",
     "start_time": "2020-11-05T08:35:11.003574Z"
    }
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'abstract': 'text',\n",
       " 'abstract_vector_': 'vector',\n",
       " 'body': 'text',\n",
       " 'insert_date_': 'date',\n",
       " 'title': 'text',\n",
       " 'title_vector_': 'vector'}"
      ]
     },
     "execution_count": 28,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "vi_client.collection_schema(\"corpus-dask\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-11-05T08:35:12.480568Z",
     "start_time": "2020-11-05T08:35:12.475567Z"
    }
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[('a3116a1ea2e157c8855023d968cc0f0bd6c9846f', 0.9573084999999999),\n",
       " ('a05179c57d5e1cdddc839ff3c0b70a614efbb6cb', 0.8702154000000002),\n",
       " ('a18236e9c75536c022502b9a9d0efdc6b0a3f0c2', 0.8499500000000002),\n",
       " ('a45430e796de9329f33ca9a4ea936b3c569c5296', 0.8114971999999998),\n",
       " ('a09090caa2498f6ff886f0fd7b10a7e45818751c', 0.7965499999999999),\n",
       " ('00623268d0ce12f11d6b3acb0cc012821f592b93', 0.7556745999999999),\n",
       " ('008cf2148ee4383534191b3736096878cad82c67', 0.7367054999999998),\n",
       " ('a912f654028e787673b235c9ba2eb2d161e372aa', 0.7334266),\n",
       " ('a17525ba0365d376469cb2e8faeb0bad6d6ed682', 0.7328115),\n",
       " ('007075cfdca03250c97761eb0e9c8e1bc8a94d6c', 0.7250315999999999),\n",
       " ('0022361a636d679fc3493cc0b20bb6127bf63a6a', 0.7245579000000002),\n",
       " ('a1949f80e726d045fe1cccb24a0e911cc0b912b9', 0.7034755000000001),\n",
       " ('a03517f26664be79239bcdf3dbb0966913206a86', 0.7024539000000001),\n",
       " ('a911adbaf5828c2f84558d48e2e72e00810069aa', 0.6969185000000002),\n",
       " ('a007977dad90a07b3beb9f689e3be8b3f7d2a7f6', 0.6960899999999999),\n",
       " ('a1213a37009cf0d561bdfe620cbeabf8de2778d1', 0.6960299999999999),\n",
       " ('a6203f5bc0f1ac39f4c722a6d9aa6918858f5de1', 0.6938412),\n",
       " ('00f72ff0dc58a60ef31950d0f28c33dfaf1bf2eb', 0.6832560999999999),\n",
       " ('00f244245f02c27974ee0dd3bc21fe47abcc03cb', 0.6714923000000002),\n",
       " ('a9853a6bc65595e9211549f413674a37ad424a6c', 0.6653039999999999)]"
      ]
     },
     "execution_count": 29,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "[(result[\"_id\"],result[\"_search_score\"]) for result in resultsb[\"results\"]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.6"
  },
  "latex_envs": {
   "LaTeX_envs_menu_present": true,
   "autoclose": true,
   "autocomplete": true,
   "bibliofile": "biblio.bib",
   "cite_by": "apalike",
   "current_citInitial": 1,
   "eqLabelWithNumbers": true,
   "eqNumInitial": 1,
   "hotkeys": {
    "equation": "Ctrl-E",
    "itemize": "Ctrl-I"
   },
   "labels_anchors": false,
   "latex_user_defs": false,
   "report_style_numbering": false,
   "user_envs_cfg": false
  },
  "varInspector": {
   "cols": {
    "lenName": 16,
    "lenType": 16,
    "lenVar": 40
   },
   "kernels_config": {
    "python": {
     "delete_cmd_postfix": "",
     "delete_cmd_prefix": "del ",
     "library": "var_list.py",
     "varRefreshCmd": "print(var_dic_list())"
    },
    "r": {
     "delete_cmd_postfix": ") ",
     "delete_cmd_prefix": "rm(",
     "library": "var_list.r",
     "varRefreshCmd": "cat(var_dic_list()) "
    }
   },
   "types_to_exclude": [
    "module",
    "function",
    "builtin_function_or_method",
    "instance",
    "_Feature"
   ],
   "window_display": false
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
